<!DOCTYPE html>
<html lang="en">

  <head>
    <title>osmos::feed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="osmos::feed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <!-- %before-head-end.html% -->
  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-28 datetime="2022-09-28T13:56:44.000Z">2022-09-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-28 datetime="2022-09-28T13:56:44.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119210"
                        open
                      >
                        <summary class="article-expander__title">Biologists Use Genetic Circuits to Program Plant Roots</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/biologists-use-genetic-circuits-to-program-plant-roots-20220928/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Programming-Plant-Roots_1200_Social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using inserted genetic circuitry, synthetic biologists controlled the growth of plant roots for the first time.            
The post Biologists Use Genetic Circuits to Program Plant Roots first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.27.509702v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Cell type-independent profiling of interactions between intracellular pathogens and the human phosphoproteome [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.27.509702v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Interactions between proteins from intracellular pathogens and host proteins in an infected cell are often mediated by post-translational modifications encoded in the host proteome. Identifying protein modifications, such as phosphorylation, that dictate these interactions remains a defining challenge in unraveling the molecular mechanisms of pathogenesis. We have developed a platform in engineered bacteria that displays over 110,000 phosphorylated human proteins coupled to a fluorescent reporter system capable of identifying the host-pathogen interactome of phosphoproteins (H-PIP). This resource broadly enables cell-type independent interrogation and discovery of proteins from intracellular pathogens capable of binding phosphorylated human proteins. As an example of the H-PIP platform, we generated a unique, high-resolution SARS-CoV-2 interaction network which expanded our knowledge of viral protein function and identified understudied areas of host pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.26.509630v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Identification of Rho-dependent termination site in vivo using synthetic sRNA [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.26.509630v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Rho promotes Rho-dependent termination (RDT) at the Rho-dependent terminator, producing a variable-length region at the 3-end of mRNA without secondary structure. Determining the exact RDT site in vivo is challenging because the 3&#x27;-end of mRNA is rapidly removed by 3&#x27;- to 5&#x27;-exoribonuclease digestion after RDT. Here, we applied synthetic sRNA (sysRNA) to pinpoint RDT sites in vivo by exploiting its complementary base-pairing ability to target mRNA. Through the combined assays of rapid amplification of cDNA 3-ends, primer extension, and capillary electrophoresis, we could precisely locate and quantify mRNA 3-ends. We found that complementary double-stranded RNA (dsRNA) formed between sysRNA and mRNA was efficiently cleaved by RNase III in the middle of the dsRNA region. The formation of dsRNA seems to protect the cleaved RNA 3-ends from rapid degradation by 3- to 5- exonuclease, thereby stabilizing the mRNA 3-end. We further verified that the signal intensity at the 3-end was positively correlated with amounts of mRNA. By constructing a series of sysRNAs with target sites in close proximity, and comparing the difference in signal intensity at the 3-end of wild-type and Rho-impaired strains, we finally identified a region of increased mRNA expression within 21 bp range, which was determined as RDT site. Our results demonstrated the ability to use sysRNA as a novel tool to precisely localize RDTs in vivo and expanded the range of sysRNA applications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.16.508262v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">An integrase toolbox to record gene-expression during plant development [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.16.508262v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>There are many open questions about the mechanisms that coordinate the dynamic, multicellular behaviors required for organogenesis. Synthetic circuits that can record in vivo signaling networks have been critical in elucidating animal development. Here, we report on the transfer of this technology to plants using orthogonal serine integrases to mediate site-specific and irreversible DNA recombination visualized by switching between fluorescent reporters. When combined with promoters expressed during lateral root initiation, integrases amplified reporter signal and permanently marked all descendants. In addition, we have developed a suite of methods to tune the threshold for integrase switching, including: RNA/protein degradation tags, a nuclear localization signal, and a split-intein system. These tools improved the robustness of integrase-mediated switching with different promoters and the stability of switching behavior over multiple generations. This integrase toolbox can be used to build history-dependent circuits to decode the order of expression during organogenesis in many contexts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19565-y"
                        open
                      >
                        <summary class="article-expander__title">De novo metatranscriptomic exploration of gene function in the millipede holobiont</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19565-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19565-y/MediaObjects/41598_2022_19565_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - De novo metatranscriptomic exploration of gene function in the millipede holobiont</span>
                                &ensp;<span class="article-reading-time">(&hairsp;40
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33522-3"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Structure of the PAPP-ABP5 complex reveals mechanism of substrate recognition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33522-3">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Author Correction: Structure of the PAPP-ABP5 complex reveals mechanism of substrate recognition</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41397-022-00287-3"
                        open
                      >
                        <summary class="article-expander__title">Publisher Correction: Cyrius: accurate CYP2D6 genotyping using whole-genome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41397-022-00287-3">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>The Pharmacogenomics Journal - Publisher Correction: Cyrius: accurate CYP2D6 genotyping using whole-genome sequencing data</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41397-022-00289-1"
                        open
                      >
                        <summary class="article-expander__title">Drug genetic associations with COVID-19 manifestations: a data mining and network biology approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41397-022-00289-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41397-022-00289-1/MediaObjects/41397_2022_289_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Pharmacogenomics Journal - Drug genetic associations with COVID-19 manifestations: a data mining and network biology approach</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20671-0"
                        open
                      >
                        <summary class="article-expander__title">Unravelling the role of transient redox partner complexes in P450 electron transfer mechanics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20671-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20671-0/MediaObjects/41598_2022_20671_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Unravelling the role of transient redox partner complexes in P450 electron transfer mechanics</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20106-w"
                        open
                      >
                        <summary class="article-expander__title">Pseudotime analysis reveals novel regulatory factors for multigenic onset and monogenic transition of odorant receptor expression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20106-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20106-w/MediaObjects/41598_2022_20106_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Pseudotime analysis reveals novel regulatory factors for multigenic onset and monogenic transition of odorant receptor expression</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20229-0"
                        open
                      >
                        <summary class="article-expander__title">Differential transcriptomic responses to heat stress in surface and subterranean diving beetles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20229-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20229-0/MediaObjects/41598_2022_20229_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Differential transcriptomic responses to heat stress in surface and subterranean diving beetles</span>
                                &ensp;<span class="article-reading-time">(&hairsp;34
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41593-022-01166-7"
                        open
                      >
                        <summary class="article-expander__title">The three-dimensional landscape of cortical chromatin accessibility in Alzheimer’s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41593-022-01166-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-022-01166-7/MediaObjects/41593_2022_1166_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The authors generated the largest epigenome atlas of postmortem brains with Alzheimer’s disease. They reported regulatory genomic signatures associated with AD, including variability in open chromatin regions, transcription factor networks and cis-regulatory domains.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20634-5"
                        open
                      >
                        <summary class="article-expander__title">Multiscale modeling of collective cell migration elucidates the mechanism underlying tumor–stromal interactions in different spatiotemporal scales</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20634-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20634-5/MediaObjects/41598_2022_20634_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Multiscale modeling of collective cell migration elucidates the mechanism underlying tumor–stromal interactions in different spatiotemporal scales</span>
                                &ensp;<span class="article-reading-time">(&hairsp;40
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03954-0"
                        open
                      >
                        <summary class="article-expander__title">Predicting fertility from sperm motility landscapes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03954-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03954-0/MediaObjects/42003_2022_3954_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dimension reduction methods on porcine sperm motility landscapes reveal heterogeneity and hierarchy in sperm movement behavior and show high-speed and straight-lined motion as predictive features of fertility.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20632-7"
                        open
                      >
                        <summary class="article-expander__title">The degradation of performance of a state-of-the-art skin image classifier when applied to patient-driven internet search</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20632-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20632-7/MediaObjects/41598_2022_20632_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - The degradation of performance of a state-of-the-art skin image classifier when applied to patient-driven internet search</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z">2022-09-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927"
                        open
                      >
                        <summary class="article-expander__title">Neurodegenerative disease can progress in newly identified patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/neuron-cell.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine-learning method finds patterns of health decline in ALS, informing future clinical trial designs and mechanism discovery. The technique also extends to Alzheimer’s and Parkinson’s.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119076"
                        open
                      >
                        <summary class="article-expander__title">How Big Is Infinity?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/how-big-is-infinity-20220927/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Infinity_academy_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Of all the endless questions children and mathematicians have asked about infinity, one of the most fascinating has to do with its size.            
The post How Big Is Infinity? first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0095009?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Bifurcation structure and chaos in dynamics of nanomagnet coupled to Josephson junction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0095009?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Irregular easy axis reorientation features are observed in numerical simulations of the nanomagnet coupled to the Josephson junction. We study magnetization bifurcations and chaos that appear in this system due to the interplay of superconductivity and magnetism. The bifurcation structure of magnetization under the variation of Josephson to magnetic energy ratio as a control parameter demonstrates several precessional motions that are related to chaotic behavior and orbits with different periodicities in the ferromagnetic resonance region. The effect of an external periodic signal on the bifurcation structure is also investigated. The results demonstrate high-frequency modes of a periodic motion and a chaotic response near resonance. Far from the ferromagnetic resonance, we observe a quasiperiodic behavior. The obtained results explain the irregular reorientation of the easy axis and the transitions between different types of motion.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2021.12.08.471839v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Assessing single-component gene drive systems in the mosquito Aedes aegypti via single generation crosses and modeling [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2021.12.08.471839v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The yellow fever mosquito Aedes aegypti is a major vector of arthropod-borne viruses, including dengue, chikungunya, and Zika. A novel approach to mitigate arboviral infections is to generate mosquitoes refractory to infection by overexpressing antiviral effector molecules. Such an approach requires a mechanism to spread these antiviral effectors through a population, for example, by using CRISPR/Cas9-based gene drive (GD) systems. Critical to the design of a single-locus autonomous GD is that the selected genomic locus be amenable to both GD and appropriate expression of the antiviral effector. In our study, we used reverse engineering to target two intergenic genomic loci, which had previously shown to be highly permissive for antiviral effector gene expression, and we further investigated the use of three promoters (nanos, {beta}2-tubulin, or zpg) for Cas9 expression. We then quantified the accrual of insertions or deletions (indels) after single generation crossings, measured maternal effects, and assessed fitness costs associated with the various transgenic lines to model the rate of GD fixation. Overall, MGDrivE modeling suggested that when an autonomous GD is placed into an intergenic locus, the GD system will eventually be blocked by the accrual of GD blocking resistance alleles and ultimately be lost in the population. Moreover, while genomic locus and promoter selection were critically important for the initial establishment of the autonomous GD, it was the fitness of the GD line that most strongly influenced the persistence of the GD in the simulated population. As such, we propose that when autonomous CRISPR/Cas9 based GD systems are anchored in an intergenic locus, they temporarily result in a strong population replacement effect, but as GD-blocking indels accrue, the GD becomes exhausted due to the fixation of CRISPR resistance alleles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.25.509419v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Top-down design of protein nanomaterials with reinforcement learning [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.25.509419v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The multisubunit protein assemblies that play critical roles in biology are the result of evolutionary selection for function of the entire assembly, and hence the subunits in structures such as icosahedral viral capsids often fit together with remarkable shape complementarity1,2. In contrast, the large multisubunit assemblies that have been created by de novo protein design, notably the icosahedral nanocages used in a new generation of potent vaccines3-7, have been built by first designing symmetric oligomers with cyclic symmetry and then assembling these into nanocages while keeping the internal structure fixed8-14, which results in more porous structures with less extensive shape matching between the components. Such hierarchical &quot;bottom-up&quot; design approaches have the advantage that one…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20615-8"
                        open
                      >
                        <summary class="article-expander__title">Identification and verification of CCNB1 as a potential prognostic biomarker by comprehensive analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20615-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20615-8/MediaObjects/41598_2022_20615_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Identification and verification of CCNB1 as a potential prognostic biomarker by comprehensive analysis</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20275-8"
                        open
                      >
                        <summary class="article-expander__title">An open-source computational tool for measuring bacterial biofilm morphology and growth kinetics upon one-sided exposure to an antimicrobial source</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20275-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20275-8/MediaObjects/41598_2022_20275_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - An open-source computational tool for measuring bacterial biofilm morphology and growth kinetics upon one-sided exposure to an antimicrobial source</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33291-z"
                        open
                      >
                        <summary class="article-expander__title">Gene expression based inference of cancer drug sensitivity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33291-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33291-z/MediaObjects/41467_2022_33291_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Predicting treatment response in cancer remains a highly complex task. Here, the authors develop Precily, a deep neural network framework to predict treatment response in cancer by considering gene expression, pathway activity estimates and drug features, and test this method in multiple datasets and preclinical models.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-18284-8"
                        open
                      >
                        <summary class="article-expander__title">Predicting and preventing COVID-19 outbreaks in indoor environments: an agent-based modeling study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-18284-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-18284-8/MediaObjects/41598_2022_18284_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Predicting and preventing COVID-19 outbreaks in indoor environments: an agent-based modeling study</span>
                                &ensp;<span class="article-reading-time">(&hairsp;32
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s43587-022-00294-w"
                        open
                      >
                        <summary class="article-expander__title">Healthy aging and the human gut microbiome: why we cannot just turn back the clock</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s43587-022-00294-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs43587-022-00294-w/MediaObjects/43587_2022_294_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The aging research field has largely focused on reversing aging-related changes in the body. However, emerging evidence about the gut microbiome indicates that it may not be optimal to just turn back the clock. Here, we advocate for a more tailored and function-focused approach promoting health across the lifespan.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20278-5"
                        open
                      >
                        <summary class="article-expander__title">Targeting and ultrabroad insight into molecular basis of Resistance-nodulation-cell division efflux pumps</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20278-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20278-5/MediaObjects/41598_2022_20278_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Targeting and ultrabroad insight into molecular basis of Resistance-nodulation-cell division efflux pumps</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41597-022-01692-w"
                        open
                      >
                        <summary class="article-expander__title">Pan-tumor CAnine cuTaneous Cancer Histology (CATCH) dataset</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41597-022-01692-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41597-022-01692-w/MediaObjects/41597_2022_1692_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurement(s) canine cutaneous tissue Technology Type(s) bright-field microscopy • H&amp;E slide staining • whole slide scanning Sample Characteristic - Organism Canis</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20041-w"
                        open
                      >
                        <summary class="article-expander__title">Signals of local bioclimate-driven ecomorphological changes in wild birds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20041-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20041-w/MediaObjects/41598_2022_20041_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Signals of local bioclimate-driven ecomorphological changes in wild birds</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19666-8"
                        open
                      >
                        <summary class="article-expander__title">Genomic epidemiology of Mycobacterium abscessus in a Canadian cystic fibrosis centre</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19666-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19666-8/MediaObjects/41598_2022_19666_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Genomic epidemiology of Mycobacterium abscessus in a Canadian cystic fibrosis centre</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03999-1"
                        open
                      >
                        <summary class="article-expander__title">Phospholipid synthesis inside phospholipid membrane vesicles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03999-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03999-1/MediaObjects/42003_2022_3999_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A giant vesicle system combines fatty acid synthesis pathways, CoA recycling, and cell-free acyltransferase gene expression to perform phospholipid synthesis inside.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-26 datetime="2022-09-26T14:41:16.000Z">2022-09-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-26 datetime="2022-09-26T14:41:16.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119061"
                        open
                      >
                        <summary class="article-expander__title">Physicists Rewrite a Quantum Rule That Clashes With Our Universe</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/physicists-rewrite-a-quantum-rule-that-clashes-with-our-universe-20220926/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Unitarity_1200_Social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The past and the future are tightly linked in conventional quantum mechanics. Perhaps too tightly. A tweak to the theory could let quantum possibilities increase as space expands.            
The post Physicists Rewrite a Quantum Rule That Clashes With Our Universe first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00235"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Reprogramming the Cleavage Specificity of Botulinum Neurotoxin Serotype B1</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00235">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00235</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00156"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Quantification of Genome Editing and Transcriptional Control Capabilities Reveals Hierarchies among Diverse CRISPR/Cas Systems in Human Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00156</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0087627?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Abrupt transition of the efficient vaccination strategy in a population with heterogeneous fatality rates</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0087627?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
An insufficient supply of an effective SARS-CoV-2 vaccine in most countries demands an effective vaccination strategy to minimize the damage caused by the disease. Currently, many countries vaccinate their population in descending order of age (i.e., descending order of fatality rate) to minimize the deaths caused by the disease; however, the effectiveness of this strategy needs to be quantitatively assessed. We employ the susceptible–infected–recovered–dead model to investigate various vaccination strategies. We constructed a metapopulation model with heterogeneous contact and fatality rates and investigated the effectiveness of vaccination strategies to reduce epidemic mortality. We found that the fatality-based strategy, which is currently employed in many countries, is more effective when the contagion rate is high and vaccine supply is low, but the contact-based method outperforms the fatality-based strategy when there is a sufficiently high supply of the vaccine. We identified a discontinuous transition of the optimal vaccination strategy and path-dependency analogous to hysteresis. This transition and path-dependency imply that combining the fatality-based and contact-based strategies is ineffective in reducing the number of deaths. Furthermore, we demonstrate that such phenomena occur in real-world epidemic diseases, such as tuberculosis and COVID-19. We also show that the conclusions of this research are valid even when the complex epidemic stages, efficacy of the vaccine, and reinfection are considered.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0121186?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Erratum: “Variational inference of the drift function for stochastic differential equations driven by Lévy processes” [Chaos 32, 061103 (2022)]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0121186?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102904?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Crises and chaotic scattering in hydrodynamic pilot-wave experiments</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102904?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Theoretical foundations of chaos have been predominantly laid out for finite-dimensional dynamical systems, such as the three-body problem in classical mechanics and the Lorenz model in dissipative systems. In contrast, many real-world chaotic phenomena, e.g., weather, arise in systems with many (formally infinite) degrees of freedom, which limits direct quantitative analysis of such systems using chaos theory. In the present work, we demonstrate that the hydrodynamic pilot-wave systems offer a bridge between low- and high-dimensional chaotic phenomena by allowing for a systematic study of how the former connects to the latter. Specifically, we present experimental results, which show the formation of low-dimensional chaotic attractors upon destabilization of regular dynamics and a final transition to high-dimensional chaos via the merging of distinct chaotic regions through a crisis bifurcation. Moreover, we show that the post-crisis dynamics of the system can be rationalized as consecutive scatterings from the nonattracting chaotic sets with lifetimes following exponential distributions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107866?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Cluster synchronization induced by manifold deformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107866?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Pinning control of cluster synchronization in a globally connected network of chaotic oscillators is studied. It is found in simulations that when the pinning strength exceeds a critical value, the oscillators are synchronized into two different clusters, one formed by the pinned oscillators and the other one formed by the unpinned oscillators. The numerical results are analyzed by the generalized method of master stability function (MSF), in which it is shown that whereas the method is able to predict the synchronization behaviors of the pinned oscillators, it fails to predict the synchronization behaviors of the unpinned oscillators. By checking the trajectories of the oscillators in the phase space, it is found that the failure is attributed to the deformed synchronization manifold of the unpinned oscillators, which is clearly deviated from that of isolated oscillator under strong pinnings. A similar phenomenon is also observed in the pinning control of cluster synchronization in a complex network of symmetric structures and in the self-organized cluster synchronization of networked neural oscillators. The findings are important complements to the generalized MSF method and provide an alternative approach to the manipulation of synchronization behaviors in complex network systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100698?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Noise-induced switching in an oscillator with pulse delayed feedback: A discrete stochastic modeling approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100698?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study the dynamics of an oscillatory system with pulse delayed feedback and noise of two types: (i) phase noise acting on the oscillator and (ii) stochastic fluctuations of the feedback delay. Using an event-based approach, we reduce the system dynamics to a stochastic discrete map. For weak noise, we find that the oscillator fluctuates around a deterministic state, and we derive an autoregressive model describing the system dynamics. For stronger noise, the oscillator demonstrates noise-induced switching between various deterministic states; our theory provides a good estimate of the switching statistics in the linear limit. We show that the robustness of the system toward this switching is strikingly different depending on the type of noise. We compare the analytical results for linear coupling to numerical simulations of nonlinear coupling and find that the linear model also provides a qualitative explanation for the differences in robustness to both types of noise. Moreover, phase noise drives the system toward higher frequencies, while stochastic delays do not, and we relate this effect to our theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0110865?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Imaging in reflecting spheres</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0110865?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study the formation of images in a reflective sphere in three configurations using caustics on the field of light rays. The optical wavefront emerging from a source point reaching a subject following passage through the optical system is, in general, a Gaussian surface with partial focus along the two principal directions of the Gaussian surface; i.e., there are two images of the source point, each with partial focus. As the source point moves, the images move on two surfaces, referred to as viewable surfaces. In our systems, one viewable surface consists of points with radial focus and the other consists of points with azimuthal focus. The problems we study are (1) imaging of a parallel beam of light, (2) imaging of the infinite viewed from a location outside the sphere, and (3) imaging of a planar object viewed through the point of its intersection with the radial line normal to the plane. We verify the existence of two images experimentally and show that the distance between them agrees with the computations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100315?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Two competing simplicial irreversible epidemics on simplicial complex</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100315?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Higher-order interactions have significant implications for the dynamics of competing epidemic spreads. In this paper, a competing spread model for two simplicial irreversible epidemics (i.e., susceptible–infected–removed epidemics) on higher-order networks is proposed. The simplicial complexes are based on synthetic (including homogeneous and heterogeneous) and real-world networks. The spread process of two epidemics is theoretically analyzed by extending the microscopic Markov chain approach. When the two epidemics have the same 2-simplex infection rate and the 1-simplex infection rate of epidemic [math] ([math]) is fixed at zero, an increase in the 1-simplex infection rate of epidemic [math] ([math]) causes a transition from continuous growth to sharp growth in the spread of epidemic [math] with [math]. When [math], the growth of epidemic [math] is always continuous. With the increase of [math], the outbreak threshold of epidemic [math] is delayed. When the difference in 1-simplex infection rates between the two epidemics reaches approximately three times, the stronger side obviously dominates. Otherwise, the coexistence of the two epidemics is always observed. When the 1-simplex infection rates are symmetrical, the increase in competition will accelerate the spread process and expand the spread area of both epidemics; when the 1-simplex infection rates are asymmetrical, the spread area of one epidemic increases with an increase in the 1-simplex infection rate from this epidemic while the other decreases. Finally, the influence of 2-simplex infection rates on the competing spread is discussed. An increase in 2-simplex infection rates leads to sharp growth in one of the epidemics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098707?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Learning spatiotemporal chaos using next-generation reservoir computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098707?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Forecasting the behavior of high-dimensional dynamical systems using machine learning requires efficient methods to learn the underlying physical model. We demonstrate spatiotemporal chaos prediction using a machine learning architecture that, when combined with a next-generation reservoir computer, displays state-of-the-art performance with a computational time [math]–[math] times faster for training process and training data set [math] times smaller than other machine learning algorithms. We also take advantage of the translational symmetry of the model to further reduce the computational cost and training data, each by a factor of [math]10.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0095737?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamics study of integrable turbulence with fourth-order nonlinear Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0095737?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In this paper, we focus on the fourth-order nonlinear Schrödinger equation, which can describe the optical system and the Heisenberg spin system. We consider a continuous wave perturbed by the one-dimensional random rough surface as the initial condition. First, we numerically resolve the eigenvalues under different control parameters utilizing the Fourier collocation method. Then, we simulate the evolution of this equation under the above initial conditions via the symmetrical split-step Fourier method. Moreover, we investigate the “steady” chaotic state by evolving a large number of initial conditions for the same control parameters. We find that the control parameters of the initial condition affect the number and intensity of rogue waves (RWs) in integrable turbulence. In particular, we locate the inflection point where the control parameter affects the velocities of solitons and the inconsistency within the parameter of the contribution to the generation of RWs. We further verify that the collision between breathers, solitons, and breathers and solitons can generate RWs. These results will enable us to understand the turbulent state and the formation mechanism of RWs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0088880?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Synchronization in a multilevel network using the Hamilton–Jacobi–Bellman (HJB) technique</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0088880?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
This paper presents the optimal control and synchronization problem of a multilevel network of Rössler chaotic oscillators. Using the Hamilton–Jacobi–Bellman technique, the optimal control law with a three-state variable feedback is designed such that the trajectories of all the Rössler oscillators in the network are optimally synchronized at each level. Furthermore, we provide numerical simulations to demonstrate the effectiveness of the proposed approach for the cases of one and three networks. A perfect correlation between the MATLAB and PSpice results was obtained, thus allowing the experimental validation of our designed controller and shows the effectiveness of the theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.23.509237v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Tunable dynamics in a multi-strain transcriptional pulse generator [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.23.509237v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A major challenge in synthetic biology is the manipulation of engineered gene circuits toward a specified behavior. This challenge becomes more difficult as synthetic systems become more complex by incorporating additional genes or strains. Here we demonstrate that circuit dynamics can be tuned in synthetic consortia through the manipulation of strain fractions within the community. To do this, we constructed a microbial consortium comprised of three strains of engineered Escherichia coli that, when co-cultured, use homoserine lactone (HSL) mediated intercellular signaling to create a multi-strain incoherent type-1 feedforward loop (I1-FFL). Like naturally occurring I1-FFL motifs in gene networks, this engineered microbial consortium acts as a pulse generator of gene expression. We demonstrated that the amplitude of the pulse can be easily tuned by adjusting the relative population fractions of the strains. We created a mathematical model for the temporal dynamics of the microbial consortium and, using this model, identified population fractions that produced desired pulse characteristics. Our work demonstrates that intercellular gene circuits can be effectively tuned simply by adjusting the starting fractions of each strain type.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.24.509315v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">An approach towards identifying the site of interaction between the Streptomyces phage OC31 protein Gp3 and the coiled-coil domain of its serine integrase [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.24.509315v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Streptomyces phage OC31 encodes a serine integrase which recombines the host (attB) and phage (attP) attachment sites to form new attL and attR sites. For attL and attR to be recombined by the integrase, the presence of its cognate recombination directionality factor (RDF) is required. It is hypothesized that the RDF binds the coiled-coil domain of the integrase to accomplish this switch in recombination directionality. Yet, nothing is known about the site of interaction between the RDF and integrase. Here, we tried to identify the region on the coiled-coil domain of OC31 integrase to which its cognate RDF (gp3) might bind. Mutant integrases fused to their cognate RDF (gp3) were created and selected for inactivity in attL x attR recombination. It was not possible to characterize the mutants and identify the Int-RDF binding region, due to unforeseen errors that occurred during the construction of the mutant library, but we demonstrated that simple experimental approaches could be used to identify this region. Despite this, an integrase mutant (P398L D595N) fused to gp3 was characterized. This mutant was catalyzing a more unidirectional attL x attR recombination reaction with reduced attP x attB recombination compared to the wild-type integrase. This mutant was selected since it could be used within the field of molecular biology to construct inversion switches which are a key element by which cells can be computerized.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.24.509247v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Continuous directed evolution of a feedback-resistant Arabidopsis arogenate dehydratase in plantized E. coli [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.24.509247v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Continuous directed evolution (CDE) is a powerful tool for enzyme engineering due to the depth and scale of evolutionary search that it enables. If suitably controlled and calibrated, CDE could be widely applied in plant breeding and biotechnology to improve plant enzymes ex planta. We tested this concept by evolving Arabidopsis arogenate dehydratase (AtADT2) for resistance to feedback inhibition. We used an Escherichia coli platform with a phenylalanine biosynthesis pathway reconfigured ( plantized) to mimic the plant pathway, a T7RNA polymerase-base deaminase hypermutation system (eMutaT7), and 4-fluorophenylalanine as selective agent. Selection schemes were pre-validated using a known feedback-resistant AtADT2 variant. We obtained variants that had 4-fluorophenylalanine resistance at least matching the known variant and that carried mutations in the ACT domain responsible for feedback inhibition. We conclude that ex planta CDE of plant enzymes in a microbial platform is a viable way to tailor characteristics that involve interaction with small molecules.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2021.07.26.453865v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Longitudinal single-cell chemical imaging of engineered strains reveals heterogeneity in fatty acid production [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2021.07.26.453865v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Understanding metabolic heterogeneity is critical for optimizing microbial production of valuable chemicals, but requires tools that can quantify metabolites at the single-cell level over time. Here, we develop longitudinal hyperspectral stimulated Raman scattering (SRS) chemical imaging to directly visualize free fatty acids in engineered Escherichia coli over many cell cycles. We also develop compositional analysis to determine the chain length and unsaturation of the fatty acids in living cells. Our method reveals substantial heterogeneity in fatty acid production among and within colonies that emerges over the course of many generations. Interestingly, the strains display distinct types of production heterogeneity in an enzyme-dependent manner. By pairing time-lapse and SRS imaging, we examine the relationship between growth and production at the single-cell level. Single-cell quantification does not show a significant growth-production tradeoff in a strain that exhibits high production heterogeneity. Our results demonstrate that cell-to-cell production heterogeneity is pervasive and provide a means to link single-cell and population-level production.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19608-4"
                        open
                      >
                        <summary class="article-expander__title">ProteinGLUE multi-task benchmark suite for self-supervised protein modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19608-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19608-4/MediaObjects/41598_2022_19608_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - ProteinGLUE multi-task benchmark suite for self-supervised protein modeling</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s43588-022-00319-9"
                        open
                      >
                        <summary class="article-expander__title">Getting the most out of noisy surveillance data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s43588-022-00319-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs43588-022-00319-9/MediaObjects/43588_2022_319_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A recent study proposes a metric to quantify how much information different types of epidemiological surveillance data, such as case counts and death counts, convey about the real-time transmission of an epidemic.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42256-022-00532-1"
                        open
                      >
                        <summary class="article-expander__title">Transformer-based protein generation with regularized latent space optimization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42256-022-00532-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42256-022-00532-1/MediaObjects/42256_2022_532_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The space of possible proteins is vast, and optimizing proteins for specific target properties computationally is an ongoing challenge, even with large amounts of data. Castro and colleagues combine a transformer-based model with regularized prediction heads to form a smooth and pseudoconvex latent space that allows for easier navigation and more efficient optimization of proteins.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;40
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20089-8"
                        open
                      >
                        <summary class="article-expander__title">Actomyosin contractility as a mechanical checkpoint for cell state transitions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20089-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20089-8/MediaObjects/41598_2022_20089_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Actomyosin contractility as a mechanical checkpoint for cell state transitions</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s43588-022-00321-1"
                        open
                      >
                        <summary class="article-expander__title">Computation empowers CRISPR discovery and technology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s43588-022-00321-1">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>CRISPR has revolutionized biomedical and bioengineering research as a programmable genome engineering technology. Computational tools have been integral throughout the discovery of CRISPR biology and the development of CRISPR-based technologies that hold promise to solve many challenges ahead.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s44161-022-00136-3"
                        open
                      >
                        <summary class="article-expander__title">Interactions between aging and sex differences for cardiometabolic risk factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s44161-022-00136-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs44161-022-00136-3/MediaObjects/44161_2022_136_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We explored age-dependent patterns in sex differences for a wide range of cardiometabolic disease risk factors and observed widespread and divergent patterns for different risk-factor groups. These findings highlight the importance of taking both age and sex into account when assessing a person’s risk of developing cardiometabolic disease.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41564-022-01234-5"
                        open
                      >
                        <summary class="article-expander__title">Computational discovery of dynobactin antibiotics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41564-022-01234-5">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>A computational analysis of biosynthetic gene clusters with unique structural features unveils new natural product scaffolds, leading to the discovery of an antibiotic targeting BamA with activity against Gram-negative pathogens.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42256-022-00534-z"
                        open
                      >
                        <summary class="article-expander__title">scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42256-022-00534-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42256-022-00534-z/MediaObjects/42256_2022_534_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cell type annotation is a core task for single cell RNA-sequencing, but current bioinformatic tools struggle with some of the underlying challenges, including high dimensionality, data sparsity, batch effects and a lack of labels. In a self-supervised approach, a transformer model called scBERT is pretrained on millions of unlabelled public single cell RNA-seq data and then fine-tuned with a small number of labelled samples for cell annotation tasks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20040-x"
                        open
                      >
                        <summary class="article-expander__title">Identification of the effects of COVID-19 on patients with pulmonary fibrosis and lung cancer: a bioinformatics analysis and literature review</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20040-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20040-x/MediaObjects/41598_2022_20040_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Identification of the effects of COVID-19 on patients with pulmonary fibrosis and lung cancer: a bioinformatics analysis and literature review</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32945-2"
                        open
                      >
                        <summary class="article-expander__title">Modeling tissue-specific breakpoint proximity of structural variations from whole-genomes to identify cancer drivers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32945-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32945-2/MediaObjects/41467_2022_32945_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Identifying structural variants (SVs) under positive selection in cancer is challenging. Here, the authors develop CSVDriver, a method that computes SV breakpoint proximity and the contribution of elements such as topologically associating domains, and identifies loci that show signs of positive selection and contain known and putative drivers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01136-x"
                        open
                      >
                        <summary class="article-expander__title">Chemically inducible split protein regulators for mammalian cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01136-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01136-x/MediaObjects/41589_2022_1136_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>An INSPIRE platform for therapeutic regulators and sensors, based on split human domains binding physiological or clinically approved compounds, has been developed and demonstrated on several human proteins in a relevant ligand concentration range.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33221-z"
                        open
                      >
                        <summary class="article-expander__title">The material properties of a bacterial-derived biomolecular condensate tune biological function in natural and synthetic systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33221-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33221-z/MediaObjects/41467_2022_33221_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>“Intracellular phase separation is emerging as a universal principle for organizing biochemical reactions in time and space. Here the authors show that PopZ condensate dynamics support cell division and using PopZ modular architecture, the tunable PopTag platform was developed to enable designer condensates.”</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-25 datetime="2022-09-25T00:00:00.000Z">2022-09-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-25 datetime="2022-09-25T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.22.508994v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Designing a Minimal Artificial Glycolytic Pathway [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.22.508994v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The canonical glycolysis generates two molecules of acetyl-coenzyme A (acetyl-CoA) from one glucose through eleven cascade biochemical reactions. Here, we designed and constructed a Minimal Artificial Glycolytic (MAG) pathway consisting of only three types of biochemical reactions, with phosphoketolase as the core, combined with phosphatase and isomerase as auxiliary enzymes. It could theoretically achieve a 100% carbon yield to acetyl-CoA from any monosaccharide by integrating one-carbon condensation reaction. We tested the MAG pathway in vitro and in vivo, demonstrating the catabolism of typical C1-C6 carbohydrates to acetyl-CoA with yields from 82% to 95%. This novel glycolytic pathway provides a promising route for biomanufacturing with stoichiometric productivity from multiple carbon sources in the future.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-24 datetime="2022-09-24T00:00:00.000Z">2022-09-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-24 datetime="2022-09-24T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33288-8"
                        open
                      >
                        <summary class="article-expander__title">Synthetic neuromorphic computing in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33288-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33288-8/MediaObjects/41467_2022_33288_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Computational properties of neuronal networks have been applied to computing systems using simplified models comprising repeated connected nodes. Here the authors create layered assemblies of genetically encoded devices that perform non-binary logic computation and signal processing using combinatorial promoters and feedback regulation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03987-5"
                        open
                      >
                        <summary class="article-expander__title">Targeting non-coding RNA family members with artificial endonuclease XNAzymes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03987-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03987-5/MediaObjects/42003_2022_3987_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Engineered RNA endonuclease XNAzymes can specifically target individual disease-associated non-coding RNAs (ncRNAs) in vitro with potential as platform technology for precision detection or knockdown of individual microRNAs or longer ncRNAs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-23 datetime="2022-09-23T04:00:00.000Z">2022-09-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-23 datetime="2022-09-23T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00216"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Novel Split Intein-Mediated Enzymatic Channeling Accelerates the Multimeric Bioconversion Pathway of Ginsenoside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00216">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00216</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00291"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineered Spore-Forming Bacillus as a Microbial Vessel for Long-Term DNA Data Storage</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00291">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00291</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33181-4"
                        open
                      >
                        <summary class="article-expander__title">Activation of stably silenced genes by recruitment of a synthetic de-methylating module</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33181-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33181-4/MediaObjects/41467_2022_33181_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stably silenced genes with methylated CpG at the promoter are refractory to current CRISPR activation systems. Here the authors create a more robust activation system, TETact that recruits DNA-demethylating TET1 with transcriptional activators.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-22 datetime="2022-09-22T15:50:00.000Z">2022-09-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-22 datetime="2022-09-22T15:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/cell-rover-exploring-augmenting-inner-world-cell-0922"
                        open
                      >
                        <summary class="article-expander__title">Cell Rover: Exploring and augmenting the inner world of the cell</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/cell-rover-exploring-augmenting-inner-world-cell-0922">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/cell-antenna.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MIT researchers demonstrate an intracellular antenna that&#x27;s compatible with 3D biological systems and can operate wirelessly inside a living cell.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119021"
                        open
                      >
                        <summary class="article-expander__title">The New Math of Wrinkling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/the-new-math-of-wrinkling-patterns-20220922/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/How-to-Wrinkle_1200_Social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A comprehensive mathematical framework treats wrinkling patterns as elegant solutions to geometric problems.            
The post The New Math of Wrinkling first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00111"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A p‑Coumaroyl-CoA Biosensor for Dynamic Regulation of Naringenin Biosynthesis in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00111">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00111</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093842?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamical systems defined on simplicial complexes: Symmetries, conjugacies, and invariant subspaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093842?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We consider the general model for dynamical systems defined on a simplicial complex. We describe the conjugacy classes of these systems and show how symmetries in a given simplicial complex manifest in the dynamics defined thereon, especially with regard to invariant subspaces in the dynamics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107583?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Structural position vectors and symmetries in complex networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107583?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Symmetries, due to their fundamental importance to dynamical processes on networks, have attracted a great deal of current research. Finding all symmetric nodes in large complex networks typically relies on automorphism groups from algebraic-group theory, which are solvable in quasipolynomial time. We articulate a conceptually appealing and computationally extremely efficient approach to finding and characterizing all symmetric nodes by introducing a structural position vector (SPV) for each node in networks. We establish the mathematical result that symmetric nodes must have the same SPV value and demonstrate, using six representative complex networks from the real world, that all symmetric nodes in these networks can be found in linear time. Furthermore, the SPVs not only characterize the similarity of nodes but also quantify the nodal influences in propagation dynamics. A caveat is that the proved mathematical result relating the SPV values to nodal symmetries is not sufficient; i.e., nodes having the same SPV values may not be symmetric, which arises in regular networks or networks with a dominant regular component. We point out with an analysis that this caveat is, in fact, shared by the known existing approaches to finding symmetric nodes in the literature. We further argue, with the aid of a mathematical analysis, that our SPV method is generally effective for finding the symmetric nodes in real-world networks that typically do not have a dominant regular component. Our SPV-based framework, therefore, provides a physically intuitive and computationally efficient way to uncover, understand, and exploit symmetric structures in complex networks arising from real-world applications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0108672?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Matrix coupling and generalized frustration in Kuramoto oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0108672?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The Kuramoto model describes the synchronization of coupled oscillators that have different natural frequencies. Among the many generalizations of the original model, Kuramoto and Sakaguchi (KS) proposed a frustrated version that resulted in dynamic behavior of the order parameter, even when the average natural frequency of the oscillators is zero. Here, we consider a generalization of the frustrated KS model that exhibits new transitions to synchronization. The model is identical in form to the original Kuramoto model but written in terms of unit vectors and with the coupling constant replaced by a coupling matrix. The matrix breaks the rotational symmetry and forces the order parameter to point in the direction of the eigenvector with the highest eigenvalue, when the eigenvalues are real. For complex eigenvalues, the module of order parameter oscillates while it rotates around the unit circle, creating active states. We derive the complete phase diagram for the Lorentzian distribution of frequencies using the Ott–Antonsen ansatz. We also show that changing the average value of the natural frequencies leads to further phase transitions where the module of the order parameter goes from oscillatory to static.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107504?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The qualitative and quantitative relationships between pattern formation and average degree in networked reaction-diffusion systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107504?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The Turing pattern is an important dynamic behavior characteristic of activator–inhibitor systems. Differentiating from traditional assumption of activator–inhibitor interactions in a spatially continuous domain, a Turing pattern in networked reaction-diffusion systems has received much attention during the past few decades. In spite of its great progress, it still fails to evaluate the precise influences of network topology on pattern formation. To this end, we try to promote the research on this important and interesting issue from the point of view of average degree—a critical topological feature of networks. We first qualitatively analyze the influence of average degree on pattern formation. Then, a quantitative relationship between pattern formation and average degree, the exponential decay of pattern formation, is proposed via nonlinear regression. The finding holds true for several activator–inhibitor systems including biology model, ecology model, and chemistry model. The significance of this study lies that the exponential decay not only quantitatively depicts the influence of average degree on pattern formation, but also provides the possibility for predicting and controlling pattern formation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-21 datetime="2022-09-21T14:06:52.000Z">2022-09-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-21 datetime="2022-09-21T14:06:52.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;118980"
                        open
                      >
                        <summary class="article-expander__title">High-Temperature Superconductivity Understood at Last</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/high-temperature-superconductivity-understood-at-last-20220921/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Superconductivity-Mystery-Reolved_1200_Social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A new atomic-scale experiment all but settles the origin of the strong form of superconductivity seen in cuprate crystals, confirming a 35-year-old theory.            
The post High-Temperature Superconductivity Understood at Last first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00365"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Light-Dependent Control of Bacterial Expression at the mRNA Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00365">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00365</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00638"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Precise Genomic Riboregulator Control of Metabolic Flux in Microbial Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00638">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00638</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33191-2"
                        open
                      >
                        <summary class="article-expander__title">A de novo matrix for macroscopic living materials from bacteria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33191-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33191-2/MediaObjects/41467_2022_33191_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Engineered living materials (ELMs) embed living cells in a biopolymer matrix to create novel materials with tailored functions. In this work, the authors engineered bacteria to grow novel macroscopic materials that can be reshaped, functionalized, and used to filter contaminated water while also showing that the stiffness of these materials can be tuned through genetic changes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-20 datetime="2022-09-20T07:00:00.000Z">2022-09-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-20 datetime="2022-09-20T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0507?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">virDTL: Viral Recombination Analysis Through Phylogenetic Reconciliation and Its Application to Sarbecoviruses and SARS-CoV-2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0507?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00159"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A Versatile in Vivo DNA Assembly Toolbox for Fungal Strain Engineering</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00159">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00159</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096689?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Synchronization of Belousov–Zhabotinsky oscillators with electrochemical coupling in a spontaneous process</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096689?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
A passive electrochemical coupling approach is proposed to induce spontaneous synchronization between chemical oscillators. The coupling exploits the potential difference between a catalyst redox couple in the Belousov–Zhabotinsky (BZ) reaction, without external feedback, to induce surface reactions that impact the kinetics of the bulk system. The effect of coupling in BZ oscillators under batch condition is characterized using phase synchronization measures. Although the frequency of the oscillators decreases nonlinearly over time, by a factor of 2 or more within 100 cycles, the coupling is strong enough to maintain synchronization. In such a highly drifting system, the Gibbs–Shannon entropy of the cyclic phase difference distribution can be used to quantify the coupling effect. We extend the Oregonator BZ model to account for the drifting natural frequencies in batch condition and for electrochemical coupling, and numerical simulations of the effect of acid concentration on synchronization patterns are in agreement with the experiments. Because of the passive nature of coupling, the proposed coupling scheme can open avenues for designing pattern recognition and neuromorphic computation systems using chemical reactions in a spontaneous process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101933?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Transport and nonequilibrium phase transitions in polygonal urn models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101933?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study the deterministic dynamics of [math] point particles moving at a constant speed in a 2D table made of two polygonal urns connected by an active rectangular channel, which applies a feedback control on the particles, inverting the horizontal component of their velocities when their number in the channel exceeds a fixed threshold. Such a bounce-back mechanism is non-dissipative: it preserves volumes in phase space. An additional passive channel closes the billiard table forming a circuit in which a stationary current may flow. Under specific constraints on the geometry and on the initial conditions, the large [math] limit allows nonequilibrium phase transitions between homogeneous and inhomogeneous phases. The role of ergodicity in making a probabilistic theory applicable is discussed for both rational and irrational urns. The theoretical predictions are compared with the numerical simulation results. Connections with the dynamics of feedback-controlled biological systems are highlighted.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0091342?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Group percolation in interdependent networks with reinforcement network layer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0091342?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In many real-world interdependent network systems, nodes often work together to form groups, which can enhance robustness to resist risks. However, previous group percolation models are always of a first-order phase transition, regardless of the group size distribution. This motivates us to investigate a generalized model for group percolation in interdependent networks with a reinforcement network layer to eliminate collapse. Some backup devices that are equipped for a density [math] of reinforced nodes constitute the reinforcement network layer. For each group, we assume that at least one node of the group can function in one network and a node in another network depends on the group to function. We find that increasing the density [math] of reinforcement nodes and the size [math] of the dependency group can significantly enhance the robustness of interdependent networks. Importantly, we find the existence of a hybrid phase transition behavior and propose a method for calculating the shift point of percolation types. The most interesting finding is the exact universal solution to the minimal density [math] of reinforced nodes (or the minimum group size [math]) to prevent abrupt collapse for Erdős–Rényi, scale-free, and regular random interdependent networks. Furthermore, we present the validity of the analytic solutions for a triple point [math] (or [math]), the corresponding phase transition point [math], and second-order phase transition points [math] in interdependent networks. These findings might yield a broad perspective for designing more resilient interdependent infrastructure networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0103427?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Application of the Slater criteria to localize invariant tori in Hamiltonian mappings</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0103427?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We investigate the localization of invariant spanning curves for a family of two-dimensional area-preserving mappings described by the dynamical variables [math] and [math] by using Slater’s criterion. The Slater theorem says there are three different return times for an irrational translation over a circle in a given interval. The returning time, which measures the number of iterations a map needs to return to a given periodic or quasi periodic region, has three responses along an invariant spanning curve. They are related to a continued fraction expansion used in the translation and obey the Fibonacci sequence. The rotation numbers for such curves are related to a noble number, leading to a devil’s staircase structure. The behavior of the rotation number as a function of invariant spanning curves located by Slater’s criterion resulted in an expression of a power law in which the absolute value of the exponent is equal to the control parameter [math] that controls the speed of the divergence of [math] in the limit the action [math] is sufficiently small.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33207-x"
                        open
                      >
                        <summary class="article-expander__title">Screening microbially produced Δ9-tetrahydrocannabinol using a yeast biosensor workflow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33207-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33207-x/MediaObjects/41467_2022_33207_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Microbial production of cannabinoids promises a cheaper and more sustainable route to these important therapeutic molecules, but strain improvement and screening is challenging. Here, the authors develop a yeast-based Δ9-tetrahydrocannabinol (THC) biosensor for screening microbial mutant libraries.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-19 datetime="2022-09-19T12:42:09.000Z">2022-09-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-19 datetime="2022-09-19T12:42:09.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096983?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Resilience of supply-chain systems under perturbations: A network approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096983?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Supply-chain systems (SCSs) are an indispensable part of our daily infrastructures. Note that a small perturbation in a SCS can be amplified, eliciting cascading failures. It is of significant value to ensure a high resilience of SCSs. However, due to the complexity of SCSs, it is quite challenging to study their resilience under conditions of perturbations. In view of this, this paper presents a complex network perspective toward the resilience of SCSs. To achieve this goal, a complex SCS is modeled as a multilayer supply-chain network (SCN) with physical organizations being modeled as nodes and interactions among them as edges. A modeled SCN contains three types of nodes, i.e., suppliers, manufacturers, and retailers. An algorithm is proposed to construct a multilayer SCN. For each layer of a multilayer SCN, two kinds of networks, i.e., networks with Poisson degree distributions and networks with power-law degree distributions, are considered. For a given multilayer SCN, a ripple-effect network model is proposed to analyze its resilience under perturbations. Regarding the perturbations, two scenarios, i.e., random node failures and target node failures, are adopted in this research. In order to validate the effectiveness of the proposed network perspective, simulations on computer-generated SCNs are carried out. Interestingly, it is found that the resilience of SCNs under both random and target perturbations presents a discontinuous phase-change phenomenon, which indicates that SCNs are quite fragile under perturbations. It is further noticed that SCNs with power-law degree distributions are relatively more robust than SCNs with Poisson degree distributions. Although SCNs are found to be fragile, it has been discovered that denser interactions between different system organizations can enhance the network’s resilience.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097417?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamic scaling and stochastic fractal in nucleation and growth processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097417?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
A class of nucleation and growth models of a stable phase is investigated for various different growth velocities. It is shown that for growth velocities [math] and [math], where [math] and [math] are the mean domain size of the metastable phase (M-phase) and the mean nucleation time, respectively, the M-phase decays following a power law. Furthermore, snapshots at different time [math] that are taken to collect data for the distribution function [math] of the domain size [math] of the M-phase are found to obey dynamic scaling. Using the idea of data-collapse, we show that each snapshot is a self-similar fractal. However, for [math], such as in the classical Kolmogorov–Johnson–Mehl–Avrami model, and for [math], the decays of the M-phase are exponential and they are not accompanied by dynamic scaling. We find a perfect agreement between numerical simulation and analytical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0121435?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Erratum: “Saddle-center and periodic orbit: Dynamics near symmetric heteroclinic connection” [Chaos 31, 023113 (2021)]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0121435?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00289"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Transcriptional Reactivation of Lignin Biosynthesis for the Heterologous Production of Etoposide Aglycone in Nicotiana benthamiana</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00289">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00289</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33255-3"
                        open
                      >
                        <summary class="article-expander__title">Biosynthesis of plant hemostatic dencichine in Escherichia coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33255-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33255-3/MediaObjects/41467_2022_33255_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Biosynthetic pathway of dencichine, a plant derived nature product that has found various pharmacological applications, is still elusive. Here, the authors design artificial pathways through retro-biosynthesis approaches and achieve its efficient production in E. coli by systematic metabolic and enzymatic engineering.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-16 datetime="2022-09-16T12:44:04.000Z">2022-09-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-16 datetime="2022-09-16T12:44:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0090443?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">A global synchronization theorem for oscillators on a random graph</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0090443?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Consider [math] identical Kuramoto oscillators on a random graph. Specifically, consider Erdős–Rényi random graphs in which any two oscillators are bidirectionally coupled with unit strength, independently and at random, with probability [math]. We say that a network is globally synchronizing if the oscillators converge to the all-in-phase synchronous state for almost all initial conditions. Is there a critical threshold for [math] above which global synchrony is extremely likely but below which it is extremely rare? It is suspected that a critical threshold exists and is close to the so-called connectivity threshold, namely, [math] for [math]. Ling, Xu, and Bandeira made the first progress toward proving a result in this direction: they showed that if [math], then Erdős–Rényi networks of Kuramoto oscillators are globally synchronizing with high probability as [math]. Here, we improve that result by showing that [math] suffices. Our estimates are explicit: for example, we can say that there is more than a [math] chance that a random network with [math] and [math] is globally synchronizing.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0088293?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Riemann–Hilbert approach to two-component modified short-pulse system and its nonlocal reductions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0088293?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In this paper, a Riemann–Hilbert approach to a two-component modified short-pulse (mSP) system on the line with zero boundary conditions is developed. A parametric representation of the solution to the related Cauchy problem is obtained. Four nonlocal integrable reductions, namely, the real reverse space-time nonlocal focusing and defocusing mSP equations and the complex reverse space-time nonlocal focusing and defocusing mSP equations, are studied in detail. For each case, soliton solutions are presented, and, unlike their local counterparts, the nonlocal equations exhibit certain novel properties induced by the impact of nonlocality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0111507?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Patched patterns and emergence of chaotic interfaces in arrays of nonlocally coupled excitable systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0111507?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We disclose a new class of patterns, called patched patterns, in arrays of non-locally coupled excitable units with attractive and repulsive interactions. The self-organization process involves the formation of two types of patches, majority and minority ones, characterized by uniform average spiking frequencies. Patched patterns may be temporally periodic, quasiperiodic, or chaotic, whereby chaotic patterns may further develop interfaces comprised of units with average frequencies in between those of majority and minority patches. Using chaos and bifurcation theory, we demonstrate that chaos typically emerges via a torus breakup and identify the secondary bifurcation that gives rise to chaotic interfaces. It is shown that the maximal Lyapunov exponent of chaotic patched patterns does not decay, but rather converges to a finite value with system size. Patched patterns with a smaller wavenumber may exhibit diffusive motion of chaotic interfaces, similar to that of the incoherent part of chimeras.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097175?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Target recovery of the economic system based on the target reinforcement path method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097175?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
An effective and stable operation of an economic system leads to a prosperous society and sustainable world development. Unfortunately, the system faces inevitable perturbations of extreme events and is frequently damaged. To maintain the system&#x27;s stability, recovering its damaged functionality is essential and is complementary to strengthening its resilience and forecasting extreme events. This paper proposes a target recovery method based on network and economic equilibrium theories to defend the economic system against perturbations characterized as localized attacks. This novel method stimulates a set of economic sectors that mutually reinforce damaged economic sectors and is intuitively named the target reinforcement path (TRP) method. Developing a nonlinear dynamic model that simulates the economic system&#x27;s operation after being perturbed by a localized attack and recovering based on a target recovery method, we compute the relaxation time for this process to quantify the method&#x27;s efficiency. Furthermore, we adopt a rank aggregation method to comprehensively measure the method&#x27;s efficiency by studying the target recovery of three country-level economic systems (China, India, and Japan) for 73 different regional attack scenarios. Through a comparative analysis of the TRP method and three other classic methods, the TRP method is shown to be more effective and less costly. Applicatively, the proposed method exhibits the potential to recover other vital complex systems with spontaneous recovery ability, such as immune, neurological, and ecological systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105340?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Instability of oscillations in the Rosenzweig–MacArthur model of one consumer and two resources</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105340?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The system of two resources [math] and [math] and one consumer [math] is investigated within the Rosenzweig–MacArthur model with a Holling type II functional response. The rates of consumption of particular resources are normalized as to keep their sum constant. Dynamic switching is introduced as to increase the variable [math] in a process of finite speed. The space of parameters where both resources coexist is explored numerically. The results indicate that oscillations of [math] and mutually synchronized [math], which appear equal for the rates of consumption, are destabilized when these rates are modified. Then, the system is driven to one of fixed points or to a limit cycle with a much smaller amplitude. As a consequence of symmetry between the resources, the consumer cannot change the preferred resource once it is chosen.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0083137?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Identifying the spatiotemporal organization of high-traffic events in a mobile communication system using event synchronization and complex networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0083137?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In mobile communication systems, congestion is related to high-traffic events (HTEs) that occur in the coverage areas of base stations. Understanding, recognizing, and predicting these HTEs and researching their occurrence rules provides theoretical and decision-making support for preventing system congestion. Communication sectors are regarded as nodes, and if HTEs occur synchronously among sectors, then the corresponding nodes are connected. The total number of synchronous HTEs determines the edge weights. The mobile-communication spatiotemporal data are mapped to a weighted network, with the occurrence locations of HTEs as the basic elements. Network analysis provides a structure for representing the interaction of HTEs. By analyzing the topological features of the event synchronization network, the associations among the occurrence times of HTEs can be mined. We find that the event synchronization network is a small-world network, the cumulative strength distribution is exponential, and the edge weight obeys a power law. Moreover, the node clustering coefficient is negatively correlated with the node degree. A congestion coefficient based on several topological parameters is proposed, and the system congestion is visualized. The congestion coefficient contains information about the synchronous occurrence of HTEs between a sector and its neighbors and information about the synchronous occurrence of HTEs among its neighbors. For the mobile communication system considered in this study, the congestion coefficient of a large number of sectors is small and the risk of system congestion is low.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0292?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Novel Network Method Major Minor Variation Clustering Enables Identification of Poliovirus Clusters with High-Resolution Linkages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0292?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33111-4"
                        open
                      >
                        <summary class="article-expander__title">Unleashing the potential of noncanonical amino acid biosynthesis to create cells with precision tyrosine sulfation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33111-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33111-4/MediaObjects/41467_2022_33111_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Incorporation of noncanonical amino acids into proteins holds great promise for altering structure and function of these proteins. Here the authors generate metabolically modified prokaryotic and eukaryotic cells that can biosynthesize sTyr and incorporate it into proteins in a site-specific manner.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-15 datetime="2022-09-15T04:00:00.000Z">2022-09-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-15 datetime="2022-09-15T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00247"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Functional Comparison between Endogenous and Synthetic Notch Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00247">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00247</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-14 datetime="2022-09-14T02:05:31.000Z">2022-09-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-14 datetime="2022-09-14T02:05:31.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093841?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Nucleation of stem cell domains in a bistable activator–inhibitor model of the shoot apical meristem</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093841?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Shoot apical meristems (SAMs) give rise to all above-ground tissues of a plant. Expansion of meristematic tissue is derived from the growth and division of stem cells that reside in a central zone of the SAM. This reservoir of stem cells is maintained by expression of a transcription factor WUSCHEL that is responsible for the development of stem cells in the central zone. WUSCHEL expression is self-activating and downregulated by a signaling pathway initiated by CLAVATA proteins, which are upregulated by WUSCHEL. This classic activator–inhibitor network can generate localized patterns of WUSCHEL activity by a Turing instability provided certain constraints on reaction rates and diffusion constants of WUSCHEL and CLAVATA are satisfied, and most existing mathematical models of nucleation and confinement of stem cells in the SAM rely on Turing&#x27;s mechanism. However, Turing patterns have certain properties that are inconsistent with observed patterns of stem cell differentiation in the SAM. As an alternative mechanism, we propose a model for stem cell confinement based on a bistable-switch in WUSCHEL–CLAVATA interactions. We study the bistable-switch mechanism for pattern formation in a spatially continuous domain and in a discrete cellularized tissue in the presence of a non-uniform field of a rapidly diffusing hormone. By comparing domain formation by Turing and bistable-switch mechanisms in these contexts, we show that bistable switching provides a superior account of nucleation and confinement of the stem cell domain under reasonable assumptions on reaction rates and diffusion constants.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/d41586-022-02231-8"
                        open
                      >
                        <summary class="article-expander__title">Life brought to artificial cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/d41586-022-02231-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.nature.com/lw1024/magazine-assets/d41586-022-02231-8/d41586-022-02231-8_23476970.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A synthetic system that reproduces several attributes of living cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33058-6"
                        open
                      >
                        <summary class="article-expander__title">Layered feedback control overcomes performance trade-off in synthetic biomolecular networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33058-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33058-6/MediaObjects/41467_2022_33058_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Layered feedback is an optimization strategy in feedback control designs widely used in engineering. Here, combining simulation and experimentation, the authors apply layered control - a powerful optimization strategy in engineering - to synthetic biomolecular networks in living bacteria to show layered control overcomes performance trade-offs in biology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41586-022-05223-w"
                        open
                      >
                        <summary class="article-expander__title">Living material assembly of bacteriogenic protocells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41586-022-05223-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-022-05223-w/MediaObjects/41586_2022_5223_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A bacteriogenic strategy for constructing membrane-bounded, molecularly crowded, and compositionally, structurally and morphologically complex synthetic cells provides opportunities for the fabrication of new synthetic cell modules and augmented living/synthetic cell constructs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-13 datetime="2022-09-13T15:20:00.000Z">2022-09-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-13 datetime="2022-09-13T15:20:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/smart-innovation-center-awarded-nrf-grant-0913"
                        open
                      >
                        <summary class="article-expander__title">SMART Innovation Center awarded five-year NRF grant for new deep tech ventures</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/smart-innovation-center-awarded-nrf-grant-0913">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/IC%20stock%20image%202.jpeg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>By providing researchers with financial and strategic support from the early stages, the Innovation Center hopes to bring new and disruptive technologies to market.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0658?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">BDEtools: A MATLAB Package for Boolean Delay Equation Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0658?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">RESISTOR: A New OSPREY Module to Predict Resistance Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00265"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering of the Translesion DNA Synthesis Pathway Enables Controllable C‑to‑G and C‑to‑A Base Editing in Corynebacterium glutamicum</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00265">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00265</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101913?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Fractional Brownian motion with random Hurst exponent: Accelerating diffusion and persistence transitions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101913?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Fractional Brownian motion, a Gaussian non-Markovian self-similar process with stationary long-correlated increments, has been identified to give rise to the anomalous diffusion behavior in a great variety of physical systems. The correlation and diffusion properties of this random motion are fully characterized by its index of self-similarity or the Hurst exponent. However, recent single-particle tracking experiments in biological cells revealed highly complicated anomalous diffusion phenomena that cannot be attributed to a class of self-similar random processes. Inspired by these observations, we here study the process that preserves the properties of the fractional Brownian motion at a single trajectory level; however, the Hurst index randomly changes from trajectory to trajectory. We provide a general mathematical framework for analytical, numerical, and statistical analysis of the fractional Brownian motion with the random Hurst exponent. The explicit formulas for probability density function, mean-squared displacement, and autocovariance function of the increments are presented for three generic distributions of the Hurst exponent, namely, two-point, uniform, and beta distributions. The important features of the process studied here are accelerating diffusion and persistence transition, which we demonstrate analytically and numerically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0087838?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Effect of seasonally forced noisy environment on aquatic prey–predator model with water level fluctuations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0087838?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In this paper, the impact of environmental noise on prey–predator interactions with seasonal fluctuations in water levels has been studied. Both intensity of noise and water level variations together play a significant role in the dynamics of an aquatic prey–predator system. Analytically, we have shown the existence of a positive solution and its uniqueness, the ultimate bound of the system’s solutions, and the global attractivity of the solution. Moreover, parametric conditions for which model species either persist (strongly or weakly) in the system or go into extinction are derived and their biological significance is also discussed. The sufficient condition of stochastic permanence is also analyzed. To justify the analytical results, we have performed numerical simulations of both deterministic and stochastic systems and observed some significant dynamics in the considered system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0087258?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Linear vs nonlinear transport during chaotic advection in fluid flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0087258?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The goal of this study is explicit demarcation of the region of validity of a linear canonical representation for chaotic advection of Lagrangian fluid parcels in “chaotic seas” in two-dimensional (2D) and three-dimensional (3D) time-periodic fluid flows governed by Hamiltonian mechanics. The concept of lobe dynamics admits exact and unique geometric demarcation of this region and, inherently, distinction of the portions of chaotic seas with essentially linear vs nonlinear Lagrangian transport. This, furthermore, admits explicit establishment of a topological equivalence between the (embedded) Hamiltonian structure of the Lagrangian dynamics in 2D (3D) flows and their canonical form. The linear transport region in physical space encompasses four adjacent subregions that each corresponds to one of the four quadrants in canonical space and may exchange material with their environment in two essentially nonlinear ways. First, exchange between quadrants within the linear transport region and, second, exchange with the exterior of this region. Both forms of exchange can be linked to specific subsets of material elements defined by interacting lobes and combined give rise to circulation through the quadrants of the linear transport region that systematically exchanges the material with the exterior.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0103247?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Effects of species vigilance on coexistence in evolutionary dynamics of spatial rock–paper–scissors game</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0103247?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Recognizing surrounding situations, such as enemy attacks, which can be realized by predator–prey relationships, is one of the common behaviors of the population in ecosystems. In this paper, we explore the relationship between such species’ behavior and biodiversity in the spatial rock–paper–scissors game by employing the ecological concept “vigilance.” In order to describe the vigilance process, we adopt a multiplex structure where two distinct layers describe virtual and physical interactions. By investigating the process of evolution in species, we also found that species with different vigilance go together. In addition, by utilizing the dynamic time warping method, we found that species with the same vigilance have consistent behavior, but species with different vigilance have diverse behavior. Our findings may lead to broader interpretations of mechanisms promoting biodiversity via vigilance in species ecosystems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-12 datetime="2022-09-12T07:00:00.000Z">2022-09-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-12 datetime="2022-09-12T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Differentiable Learning of Sequence-Specific Minimizer Schemes with DeepMinimizer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101257?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Transient chaotic behavior of fuzzy controlled polishing processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101257?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
This paper investigates the dynamics of a fuzzy controlled polishing machine where the effect of temporal sampling is also taken into account. Chaotic and transient chaotic behaviors are experienced for certain control parameter combinations. In the case of transient chaotic motion, closed-form algebraic expressions are determined for the expected value of the kickout number and for the corresponding standard deviation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100166?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Constructing differential equations using only a scalar time-series about continuous time chaotic dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100166?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We propose a simple method of constructing a system of differential equations of chaotic behavior based on the regression only from scalar observable time-series data. The estimated system enables us to reconstruct invariant sets and statistical properties as well as to infer short time-series. Our successful modeling relies on the introduction of a set of Gaussian radial basis functions to capture local structures. The proposed method is used to construct a system of ordinary differential equations whose orbit reconstructs a time-series of a variable of the well-known Lorenz system as a simple but typical example. A system for a macroscopic fluid variable is also constructed.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00314"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Intestinal Engineered Probiotics as Living Therapeutics: Chassis Selection, Colonization Enhancement, Gene Circuit Design, and Biocontainment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00314">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00314</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00195"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Steady-State Operation of a Cell-Free Genetic Band-Detection Circuit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00195">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00195</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33046-w"
                        open
                      >
                        <summary class="article-expander__title">Robust data storage in DNA by de Bruijn graph-based de novo strand assembly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33046-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33046-w/MediaObjects/41467_2022_33046_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA data storage is a rapidly developing technology with great potential due to its high density, long-term durability, and low maintenance cost. Here the authors present a strand assembly algorithm (DBGPS) using de Bruijn graph and greedy path search.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33033-1"
                        open
                      >
                        <summary class="article-expander__title">Kinetic compartmentalization by unnatural reaction for itaconate production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33033-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33033-1/MediaObjects/41467_2022_33033_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Unlike eukaryotic system, bacterial hosts lack membranous system, which is one of the limitations for efficient metabolic engineering. Here, the authors report a kinetic compartmentalization strategy to increase substrate availability from competitive reactions for the efficient production of itaconate in E. coli.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-10 datetime="2022-09-10T04:00:00.000Z">2022-09-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-10 datetime="2022-09-10T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00363"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Reprogramming T7RNA Polymerase in Escherichia coli Nissle 1917 under Specific Lac Operon for Efficient p‑Coumaric Acid Production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00363">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00363</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-09 datetime="2022-09-09T04:00:00.000Z">2022-09-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-09 datetime="2022-09-09T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00629"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Pseudotyping Bacteriophage P2 Tail Fibers to Extend the Host Range for Biomedical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00629">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00629</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-08 datetime="2022-09-08T08:05:17.000Z">2022-09-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-08 datetime="2022-09-08T08:05:17.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0094943?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Non-parametric tests for serial dependence in time series based on asymptotic implementations of ordinal-pattern statistics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0094943?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Ordinal patterns can be used to construct non-parametric hypothesis tests that aim to discover (possibly non-linear) serial dependence in a real-valued time series. We derive the asymptotic distribution of the vector of sample frequencies of ordinal patterns and that of various corresponding tests statistics such that the targeted tests for serial dependence are easily implemented based on asymptotic approximations. Simulations are used to check the finite-sample performance of these tests as well as their power properties with respect to various alternative scenarios. The application and interpretation of the tests in practice are illustrated by an environmental data example.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098163?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">On the origin of chaotic attractors with two zero Lyapunov exponents in a system of five biharmonically coupled phase oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098163?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study chaotic dynamics in a system of four differential equations describing the interaction of five identical phase oscillators coupled via biharmonic function. We show that this system exhibits strange spiral attractors (Shilnikov attractors) with two zero (indistinguishable from zero in numerics) Lyapunov exponents in a wide region of the parameter space. We explain this phenomenon by means of bifurcation analysis of a three-dimensional Poincaré map for the system under consideration. We show that chaotic dynamics develop here near a codimension three bifurcation, when a periodic orbit (fixed point of the Poincaré map) has the triplet of multipliers [math]. As it is known, the flow normal form for such bifurcation is the well-known three-dimensional Arneodó–Coullet–Spiegel–Tresser (ACST) system, which exhibits spiral attractors. According to this, we conclude that the additional zero Lyapunov exponent for orbits in the observed attractors appears due to the fact that the corresponding three-dimensional Poincaré map is very close to the time-shift map of the ACST-system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102421?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Detecting bifurcations in dynamical systems with CROCKER plots</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102421?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Existing tools for bifurcation detection from signals of dynamical systems typically are either limited to a special class of systems or they require carefully chosen input parameters and a significant expertise to interpret the results. Therefore, we describe an alternative method based on persistent homology—a tool from topological data analysis—that utilizes Betti numbers and CROCKER plots. Betti numbers are topological invariants of topological spaces, while the CROCKER plot is a coarsened but easy to visualize data representation of a one-parameter varying family of persistence barcodes. The specific bifurcations we investigate are transitions from periodic to chaotic behavior or vice versa in a one-parameter collection of differential equations. We validate our methods using numerical experiments on ten dynamical systems and contrast the results with existing tools that use the maximum Lyapunov exponent. We further prove the relationship between the Wasserstein distance to the empty diagram and the norm of the Betti vector, which shows that an even more simplified version of the information has the potential to provide insight into the bifurcation parameter. The results show that our approach reveals more information about the shape of the periodic attractor than standard tools, and it has more favorable computational time in comparison with the Rösenstein algorithm for computing the maximum Lyapunov exponent.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098454?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Cross-subject emotion recognition using visibility graph and genetic algorithm-based convolution neural network</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098454?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
An efficient emotion recognition model is an important research branch in electroencephalogram (EEG)-based brain–computer interfaces. However, the input of the emotion recognition model is often a whole set of EEG channels obtained by electrodes placed on subjects. The unnecessary information produced by redundant channels affects the recognition rate and depletes computing resources, thereby hindering the practical applications of emotion recognition. In this work, we aim to optimize the input of EEG channels using a visibility graph (VG) and genetic algorithm-based convolutional neural network (GA-CNN). First, we design an experiment to evoke three types of emotion states using movies and collect the multi-channel EEG signals of each subject under different emotion states. Then, we construct VGs for each EEG channel and derive nonlinear features representing each EEG channel. We employ the genetic algorithm (GA) to find the optimal subset of EEG channels for emotion recognition and use the recognition results of the CNN as fitness values. The experimental results show that the recognition performance of the proposed method using a subset of EEG channels is superior to that of the CNN using all channels for each subject. Last, based on the subset of EEG channels searched by the GA-CNN, we perform cross-subject emotion recognition tasks employing leave-one-subject-out cross-validation. These results demonstrate the effectiveness of the proposed method in recognizing emotion states using fewer EEG channels and further enrich the methods of EEG classification using nonlinear features.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0091738?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Symmetry breaking of solitons in the PT-symmetric nonlinear Schrödinger equation with the cubic–quintic competing saturable nonlinearity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0091738?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The symmetry breaking of solitons in the nonlinear Schrödinger equation with cubic–quintic competing nonlinearity and parity-time symmetric potential is studied. At first, a new asymmetric branch separates from the fundamental symmetric soliton at the first power critical point, and then, the asymmetric branch passes through the branch of the fundamental symmetric soliton and finally merges into the branch of the fundamental symmetric soliton at the second power critical point, while the power of the soliton increases. This leads to the symmetry breaking and double-loop bifurcation of fundamental symmetric solitons. From the power-propagation constant curves of solitons, symmetric fundamental and tripole solitons, asymmetric solitons can also exist. The stability of symmetric fundamental solitons, asymmetric solitons, and symmetric tripole solitons is discussed by the linear stability analysis and direct simulation. Results indicate that symmetric fundamental solitons and symmetric tripole solitons tend to be stable with the increase in the soliton power. Asymmetric solitons are unstable in both high and low power regions. Moreover, with the increase in saturable nonlinearity, the stability region of fundamental symmetric solitons and symmetric tripole solitons becomes wider.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096099?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dipole and quadrupole nonparaxial solitary waves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096099?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The cubic nonlinear Helmholtz equation with third and fourth order dispersion and non-Kerr nonlinearity, such as the self steepening and the self frequency shift, is considered. This model describes nonparaxial ultrashort pulse propagation in an optical medium in the presence of spatial dispersion originating from the failure of slowly varying envelope approximation. We show that this system admits periodic (elliptic) solitary waves with a dipole structure within a period and also a transition from a dipole to quadrupole structure within a period depending on the value of the modulus parameter of a Jacobi elliptic function. The parametric conditions to be satisfied for the existence of these solutions are given. The effect of the nonparaxial parameter on physical quantities, such as amplitude, pulse width, and speed of the solitary waves, is examined. It is found that by adjusting the nonparaxial parameter, the speed of solitary waves can be decelerated. The stability and robustness of the solitary waves are discussed numerically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100772?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">When costly migration helps to improve cooperation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100772?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Motion is a typical reaction among animals and humans trying to reach better conditions in a changing world. This aspect has been studied intensively in social dilemmas where competing players’ individual and collective interests are in conflict. Starting from the traditional public goods game model, where players are locally fixed and unconditional cooperators or defectors are present, we introduce two additional strategies through which agents can change their positions of dependence on the local cooperation level. More importantly, these so-called sophisticated players should bear an extra cost to maintain their permanent capacity to evaluate their neighborhood and react accordingly. Hence, four strategies compete, and the most successful one can be imitated by its neighbors. Crucially, the introduction of costly movement has a highly biased consequence on the competing main strategies. In the majority of parameter space, it is harmful to defectors and provides a significantly higher cooperation level when the population is rare. At an intermediate population density, which would be otherwise optimal for a system of immobile players, the presence of mobile actors could be detrimental if the interaction pattern changes slightly, thereby blocking the optimal percolation of information flow. In this parameter space, sophisticated cooperators can also show the co-called Moor effect by first avoiding the harmful vicinity of defectors; they subsequently transform into an immobile cooperator state. Hence, paradoxically, the additional cost of movement could be advantageous to reach a higher general income, especially for a rare population when subgroups would be isolated otherwise.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0088465?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Symmetry breaking yields chimeras in two small populations of Kuramoto-type oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0088465?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://aip.scitation.org/action/showOpenGraphArticleImage?doi&#x3D;10.1063/5.0088465&amp;id&#x3D;images/medium/5.0088465.figures.online.highlight_f1.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Despite their simplicity, networks of coupled phase oscillators can give rise to intriguing collective dynamical phenomena. However, the symmetries of globally and identically coupled identical units do not allow solutions where distinct oscillators are frequency-unlocked—a necessary condition for the emergence of chimeras. Thus, forced symmetry breaking is necessary to observe chimera-type solutions. Here, we consider the bifurcations that arise when full permutational symmetry is broken for the network to consist of coupled populations. We consider the smallest possible network composed of four phase oscillators and elucidate the phase space structure, (partial) integrability for some parameter values, and how the bifurcations away from full symmetry lead to frequency-unlocked weak chimera solutions. Since such solutions wind around a torus they must arise in a global bifurcation scenario. Moreover, periodic weak chimeras undergo a period-doubling cascade leading to chaos. The resulting chaotic dynamics with distinct frequencies do not rely on amplitude variation and arise in the smallest networks that support chaos.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;56
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01058-0"
                        open
                      >
                        <summary class="article-expander__title">Publisher Correction: Standardized excitable elements for scalable engineering of far-from-equilibrium chemical networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01058-0">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Chemistry - Publisher Correction: Standardized excitable elements for scalable engineering of far-from-equilibrium chemical networks</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-06 datetime="2022-09-06T10:00:00.000Z">2022-09-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-06 datetime="2022-09-06T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/alphafold-potential-protein-drug-0906"
                        open
                      >
                        <summary class="article-expander__title">Analyzing the potential of AlphaFold in drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/alphafold-potential-protein-drug-0906">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/MIT-Better-Predictions-01-press.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Study finds computer models that predict molecular interactions need improvement before they can help identify drug mechanisms of action.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00561"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Development of Base Editors for Simultaneously Editing Multiple Loci in Lactococcus lactis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00561">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00561</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01610-3"
                        open
                      >
                        <summary class="article-expander__title">Tracking the order of transcriptional events in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01610-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01610-3/MediaObjects/41592_2022_1610_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Researchers describe Retro-Cascorder, a tool that employs prokaryotic retrons and CRISPR integrases to write transcriptional events into the genome of Escherichia coli.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32744-9"
                        open
                      >
                        <summary class="article-expander__title">Generation of an Escherichia coli strain growing on methanol via the ribulose monophosphate cycle</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32744-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32744-9/MediaObjects/41467_2022_32744_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using one carbon compounds as feedstock is a promising approach in abating climate change. Here, the authors report the conversion of E. coli into a synthetic methylotroph that assimilates methanol via the ribulose monophosphate cycle and a set of distinctive mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32496-6"
                        open
                      >
                        <summary class="article-expander__title">MVsim is a toolset for quantifying and designing multivalent interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32496-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32496-6/MediaObjects/41467_2022_32496_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Arising through multiple binding elements, multivalency can specify the avidity, duration, cooperativity, and selectivity of biomolecular interactions, but quantitative prediction and design of these properties has remained challenging. Here the authors enable facile analysis and engineering of multivalent binding by developing MVsim, a simulator that incorporates biochemical and biophysical parameters of interacting molecules and is accessible through a graphical user interface.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32941-6"
                        open
                      >
                        <summary class="article-expander__title">Signal processing and generation of bioactive nitric oxide in a model prototissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32941-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32941-6/MediaObjects/41467_2022_32941_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A challenge for synthetic biology is the design and construction of prototissue. Here, the authors spatially segregate layers of enzyme-decorated coacervate protocells as a model prototissue capable of chemical signal processing and modulating outputs of nitric oxide to inhibit blood clot formation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-05 datetime="2022-09-05T07:00:00.000Z">2022-09-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-05 datetime="2022-09-05T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0447?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Machine-Learning Prospects for Detecting Selection Signatures Using Population Genomics Data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0447?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 943-960, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0399?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">An Integer Linear Programming Approach for Scaffolding Based on Exemplar Breakpoint Distance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0399?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 961-973, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0417?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Optimal Design of Synthetic DNA Sequences Without Unwanted Binding Sites</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0417?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 974-986, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0269?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">RDscan: A New Method for Improving Germline and Somatic Variant Calling Based on Read Depth Distribution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0269?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 987-1000, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0536?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">kmer2vec: A Novel Method for Comparing DNA Sequences by word2vec Embedding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0536?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1001-1021, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0549?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Sharing Rewards Undermines Coordinated Hunting</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0549?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1022-1030, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0349?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Cross-Organ Transcriptomic Comparison Reveals Universal Factors During Maturation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0349?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1031-1044, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0464?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Harnessing Fuzzy Rule Based System for Screening Major Histocompatibility Complex Class I Peptide Epitopes from the Whole Proteome: An Implementation on the Proteome of Leishmania donovani</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0464?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1045-1058, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01021-z"
                        open
                      >
                        <summary class="article-expander__title">A modular XNAzyme cleaves long, structured RNAs under physiological conditions and enables allele-specific gene silencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01021-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01021-z/MediaObjects/41557_2022_1021_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Oligonucleotide catalysts such as ribozymes and DNAzymes can cleave RNA efficiently and specifically but are typically dependent on high concentrations of divalent cations, limiting their biological applications. A modular XNAzyme catalyst composed of 2′-deoxy-2′-fluoro-β-d-arabino nucleic acid (FANA) has now been developed that can cleave long (&gt;5 kb), highly structured mRNAs under physiological conditions and enables allele-specific catalytic RNA knockdown inside cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-03 datetime="2022-09-03T00:00:00.000Z">2022-09-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-03 datetime="2022-09-03T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32921-w"
                        open
                      >
                        <summary class="article-expander__title">Expanding the terpene biosynthetic code with non-canonical 16 carbon atom building blocks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32921-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32921-w/MediaObjects/41467_2022_32921_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Establishing methods to access the chemical space that lies beyond canonical terpenoid biosynthesis will increase the applications of isoprenoids. Here, the authors reconstruct the modular structure of terpene biosynthesis on 16-carbon backbones by engineered yeast and synthesize 28 different unique terpenes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;53
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-02 datetime="2022-09-02T09:53:01.000Z">2022-09-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-02 datetime="2022-09-02T09:53:01.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097075?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Action potential and chaos near the edge of chaos in memristive circuits</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097075?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Memristor-based neuromorphic systems have a neuro-bionic function, which is critical for possibly overcoming Moore&#x27;s law limitation and the von Neumann bottleneck problem. To explore neural behaviors and complexity mechanisms in memristive circuits, this paper proposes an N-type locally active memristor, based on which a third-order memristive circuit is constructed. Theoretical analysis shows that the memristive circuit can exhibit not only various action potentials but also self-sustained oscillation and chaos. Based on Chua&#x27;s theory of local activity, this paper finds that the neural behaviors and chaos emerge near the edge of chaos through subcritical Hopf bifurcation, in which the small unstable limit cycle is depicted by the dividing line between the attraction basin of the large stable limit cycle and the attraction basin of the stable equilibrium point. Furthermore, an analog circuit is designed to imitate the action potentials and chaos, and the simulation results are in agreement with the theoretical analysis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096532?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Effects of infection fatality ratio and social contact matrices on vaccine prioritization strategies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096532?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Effective strategies of vaccine prioritization are essential to mitigate the impacts of severe infectious diseases. We investigate the role of infection fatality ratio (IFR) and social contact matrices on vaccination prioritization using a compartmental epidemic model fueled by real-world data of different diseases and countries. Our study confirms that massive and early vaccination is extremely effective to reduce the disease fatality if the contagion is mitigated, but the effectiveness is increasingly reduced as vaccination beginning delays in an uncontrolled epidemiological scenario. The optimal and least effective prioritization strategies depend non-linearly on epidemiological variables. Regions of the epidemiological parameter space, in which prioritizing the most vulnerable population is more effective than the most contagious individuals, depend strongly on the IFR age profile being, for example, substantially broader for COVID-19 in comparison with seasonal influenza. Demographics and social contact matrices deform the phase diagrams but do not alter their qualitative shapes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03832-9"
                        open
                      >
                        <summary class="article-expander__title">A visualization reporter system for characterizing antibiotic biosynthetic gene clusters expression with high-sensitivity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03832-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03832-9/MediaObjects/42003_2022_3832_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A quorum sensing based visualization reporter system is presented for the characterization of promoters in Gram-positive bacteria, utilizing violacein production, especially for use in the identification of secondary metabolites.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32829-5"
                        open
                      >
                        <summary class="article-expander__title">Automated model-predictive design of synthetic promoters to control transcriptional profiles in bacteria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32829-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32829-5/MediaObjects/41467_2022_32829_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcription rates are regulated by the interactions between RNA polymerase, sigma factor, and promoter DNA sequences in bacteria. Here the authors combine massively parallel experiments &amp; machine learning to develop a predictive biophysical model of transcription, validated across 22132 bacterial promoters, and apply it to the design and debugging of genetic circuits.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32889-7"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal control of signal-driven enzymatic reaction in artificial cell-like polymersomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32889-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32889-7/MediaObjects/41467_2022_32889_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Researchers have been trying to mimick the cellular spatiotemporal control in normal cells with different approaches. Here, the authors present semi-permeable polymersomes comprising of amphiphilic triblock copolymers to achieve external signal-driven complex coacervation and biophysical reconstitution of cytoskeleton.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;41
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-01 datetime="2022-09-01T07:00:00.000Z">2022-09-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-01 datetime="2022-09-01T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0212?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Quartet Based Gene Tree Imputation Using Deep Learning Improves Phylogenomic Analyses Despite Missing Data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0212?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0265?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Inconsistency of Triplet-Based and Quartet-Based Species Tree Estimation under Intralocus Recombination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0265?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01114-3"
                        open
                      >
                        <summary class="article-expander__title">Horizontal gene transfer enables programmable gene stability in synthetic microbiota</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01114-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01114-3/MediaObjects/41589_2022_1114_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Dynamic redundancy by horizontal gene transfer stabilizes gene abundances amidst compositional fluctuations in microbial communities, which suggests a means to program gene stability of complex microbiota.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/xlxs4/rss-reader/actions/runs/3147929183">
        <time id="build-timestamp" datetime="2022-09-29T01:01:53.069Z">2022-09-29T01:01:53.069Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>