<!DOCTYPE html>
<html lang="en">

  <head>
    <title>A curated list of immersive ramblings</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="A curated list of immersive ramblings" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <!-- %before-head-end.html% -->
  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-23 datetime="2022-12-23T19:25:23.163Z">2022-12-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-23 datetime="2022-12-23T19:25:23.163Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ScienceDirect Publication: Journal of Differential Equations</button>
                  <a class="source-heading__link" href="https://www.sciencedirect.com/journal/journal-of-differential-equations">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622007082"
                        open
                      >
                        <summary class="article-expander__title">Sharp decay rates of degenerate hyperbolic-parabolic coupled system: Rectangular domain vs one-dimensional domain</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622007082?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 March 2023
Source: Journal of Differential Equations, Volume 349
Author(s): Zhong-Jie Han, Han-Qi Song, Kai Yu</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622007069"
                        open
                      >
                        <summary class="article-expander__title">Existence and regularity for a general class of quasilinear elliptic problems involving the Hardy potential</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622007069?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 March 2023
Source: Journal of Differential Equations, Volume 349
Author(s): Giusy Chirillo, Luigi Montoro, Luigi Muglia, Berardino Sciunzi</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622007136"
                        open
                      >
                        <summary class="article-expander__title">Bifurcations and pattern formation in a predator-prey model with memory-based diffusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622007136?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 March 2023
Source: Journal of Differential Equations, Volume 350
Author(s): Meng Liu, Hongbin Wang, Weihua Jiang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622007124"
                        open
                      >
                        <summary class="article-expander__title">Global bifurcation structure and some properties of steady periodic water waves with vorticity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622007124?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 March 2023
Source: Journal of Differential Equations, Volume 349
Author(s): Guowei Dai, Yong Zhang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622007100"
                        open
                      >
                        <summary class="article-expander__title">Well-posedness for a class of compressible non-Newtonian fluids equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622007100?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 March 2023
Source: Journal of Differential Equations, Volume 349
Author(s): Bilal Al Taki</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622007094"
                        open
                      >
                        <summary class="article-expander__title">Existence and global second-order regularity for anisotropic parabolic equations with variable growth</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622007094?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 March 2023
Source: Journal of Differential Equations, Volume 349
Author(s): Rakesh Arora, Sergey Shmarev</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/mit-dlc-holiday-traditions-2022-1223"
                        open
                      >
                        <summary class="article-expander__title">MIT’s departments, labs, and centers celebrate the holidays</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/mit-dlc-holiday-traditions-2022-1223">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/MIT_EAPS-01-PRESS.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Across the Institute, MIT’s communities took part in light-hearted traditions new and old.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.21.521368v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Oral delivery of GLP-1R agonist by an engineered probiotic yeast strain has anti-obesity effects in mice [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.21.521368v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Obesity is rapidly increasing within the global population and is one of the leading causes of chronic diseases, including type 2 diabetes (T2D), non-alcoholic fatty liver disease, and cardiovascular diseases. Glucagon-like peptide-1 receptor (GLP-1R) agonists have emerged as promising therapeutic agents for treating T2D and obesity. However, the route of administration of the GLP-1R agonists is currently by injection or high oral dosages of the therapeutic combined with absorption enhancers. Oral delivery of GLP-1R agonists remains the preferred administration route due to convenience and high patient compliance. Thus, strategies to improve the oral delivery of this therapeutic are needed. In this study, we engineered the probiotic yeast Saccharomyces boulardii strain to produce Exendin-4, a GLP-1R agonist, in the gastrointestinal tract to reduce the adverse effects of diet-induced obesity in male C57BL/6 mice. The biological efficiency of the secreted Exendin-4 from S. boulardii was characterised ex vivo on isolated pancreatic islets, demonstrating induced insulin secretion. Furthermore, in vivo characterisation of the engineered strain identified a synergistic effect of cold exposure and Sb-Exe4 by successfully inhibiting appetite and promoting body weight loss under cold exposure (8{degrees}C). In addition, the combination of cold and Sb-Exe4 improved the glucose and lipid homeostasis in the mice by increasing the circulating glucagon level and reducing the inflammatory marker TNF-. Our results demonstrate that S. boulardii can be genetically modified to secrete and deliver active therapeutic GLP-1R agonists in the gastrointestinal tract improving the metabolism of the host.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.20.521182v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Harnessing photosynthetic bacterium for light-powered biocatalysis [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.20.521182v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The traditional whole-cell biocatalysis typically utilizes the heterotrophic microbes as the biocatalyst, which requires carbohydrates to power the cofactor (ATP, NAD(P)H) regeneration. In this study, we sought to harness purple non-sulfur photosynthetic bacterium (PNSB) as the biocatalyst to achieve light-driven cofactor regeneration for cascade biocatalysis. We substantially improved the performance of PNSB-based biocatalysis by using a highly active and conditional expression system, blocking the side-reactions, controlling the feeding strategy, and attenuating the light shading effect. We found that 50 mM ferulic acid could be completely converted to vanillyl alcohol in the recombinant strain, reaching 7.7 g/L vanillyl alcohol. In addition, &gt;99.9% conversion of p-coumaric acid to p-hydroxybenzoic alcohol (6.21 g/L) was similarly achieved under light-anaerobic conditions. Moreover, we examined the isoprenol utilization pathway (IUP) for pinene synthesis and 13.81 mM pinene (1.88 g/L) with 92.1% conversion rate from isoprenol was obtained. Taken together, these results suggested that PNSB could be a promising host for light-powered biotransformation, which offers an efficient approach for synthesizing value-added chemicals in a green and sustainable manner.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.20.521091v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">High-throughput functional characterization of combinations of transcriptional activators and repressors [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.20.521091v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite growing knowledge of the functions of individual human transcriptional effector domains, much less is understood about how multiple effector domains within the same protein combine to regulate gene expression. Here, we measure transcriptional activity for 8,400 effector domain combinations by recruiting them to reporter genes in human cells. In our assay, weak and moderate activation domains synergize to drive strong gene expression, while combining strong activators often results in weaker activation. In contrast, repressors combine linearly and produce full gene silencing, and repressor domains often overpower activation domains. We use this information to build a synthetic transcription factor whose function can be tuned between repression and activation independent of recruitment to target genes by using a small molecule drug. Altogether, we outline the basic principles of how effector domains combine to regulate gene expression and demonstrate their value in building precise and flexible synthetic biology tools.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01754-2"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: scBasset: sequence-based modeling of single-cell ATAC-seq using convolutional neural networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01754-2">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Methods - Author Correction: scBasset: sequence-based modeling of single-cell ATAC-seq using convolutional neural networks</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26141-x"
                        open
                      >
                        <summary class="article-expander__title">Assessing and removing the effect of unwanted technical variations in microbiome data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26141-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26141-x/MediaObjects/41598_2022_26141_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Assessing and removing the effect of unwanted technical variations in microbiome data</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41429-022-00590-y"
                        open
                      >
                        <summary class="article-expander__title">Correction: The occurrence of antibiotic resistance genes in the microbiota of yak, beef and dairy cattle characterized by a metagenomic approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41429-022-00590-y">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>The Journal of Antibiotics - Correction: The occurrence of antibiotic resistance genes in the microbiota of yak, beef and dairy cattle characterized by a metagenomic approach</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41746-022-00724-4"
                        open
                      >
                        <summary class="article-expander__title">Reply to: Immortal time bias in the analysis of drug prescription trajectories</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41746-022-00724-4">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>npj Digital Medicine - Reply to: Immortal time bias in the analysis of drug prescription trajectories</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-25549-9"
                        open
                      >
                        <summary class="article-expander__title">A network approach for low dimensional signatures from high throughput data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-25549-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-25549-9/MediaObjects/41598_2022_25549_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A network approach for low dimensional signatures from high throughput data</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41588-022-01225-6"
                        open
                      >
                        <summary class="article-expander__title">Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41588-022-01225-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41588-022-01225-6/MediaObjects/41588_2022_1225_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MetaSTAAR enables functionally informed rare variant association analysis in biobank-scale cohorts using an efficient, sparse matrix approach for summary statistic storage.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;41
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35595-6"
                        open
                      >
                        <summary class="article-expander__title">Thousands of human non-AUG extended proteoforms lack evidence of evolutionary selection among mammals</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35595-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35595-6/MediaObjects/41467_2022_35595_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Analysis of a large number of Ribo-seq datasets and genomic alignments led to detection of novel non-AUG proteoforms. Unexpectedly the number of non-AUG proteoforms identified with Ribo-seq greatly exceeds those with strong phylogenetic support.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41551-022-00952-9"
                        open
                      >
                        <summary class="article-expander__title">A deep-learning model for transforming the style of tissue images from cryosectioned to formalin-fixed and paraffin-embedded</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41551-022-00952-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41551-022-00952-9/MediaObjects/41551_2022_952_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A deep-learning model that transforms cryosectioned whole-slide tissue images into the style of whole-slide formalin-fixed and paraffin-embedded tissue improves the rates of accurate tumour subtyping.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26520-4"
                        open
                      >
                        <summary class="article-expander__title">Deep learning-based 3D OCT imaging for detection of lamina cribrosa defects in eyes with high myopia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26520-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26520-4/MediaObjects/41598_2022_26520_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Deep learning-based 3D OCT imaging for detection of lamina cribrosa defects in eyes with high myopia</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26704-y"
                        open
                      >
                        <summary class="article-expander__title">Correlation of internet search enquiries, incidence of ophthalmological diseases and corneal, cataract and refractive surgical procedures</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26704-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26704-y/MediaObjects/41598_2022_26704_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Correlation of internet search enquiries, incidence of ophthalmological diseases and corneal, cataract and refractive surgical procedures</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26439-w"
                        open
                      >
                        <summary class="article-expander__title">Artificial intelligence convolutional neural networks map giant kelp forests from satellite imagery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26439-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26439-w/MediaObjects/41598_2022_26439_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Artificial intelligence convolutional neural networks map giant kelp forests from satellite imagery</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26446-x"
                        open
                      >
                        <summary class="article-expander__title">In silico activity and ADMET profiling of phytochemicals from Ethiopian indigenous aloes using pharmacophore models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26446-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26446-x/MediaObjects/41598_2022_26446_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - In silico activity and ADMET profiling of phytochemicals from Ethiopian indigenous aloes using pharmacophore models</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04352-2"
                        open
                      >
                        <summary class="article-expander__title">A molecular barcode and web-based data analysis tool to identify imported Plasmodium vivax malaria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04352-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04352-2/MediaObjects/42003_2022_4352_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>An online, amendable classifier for detecting malaria parasite country of origin from genomic data is developed using a machine learning approach.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22561-x"
                        open
                      >
                        <summary class="article-expander__title">A novel feature for monitoring the enzymatic harvesting process of adherent cell cultures based on lens-free imaging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22561-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22561-x/MediaObjects/41598_2022_22561_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A novel feature for monitoring the enzymatic harvesting process of adherent cell cultures based on lens-free imaging</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26299-4"
                        open
                      >
                        <summary class="article-expander__title">A metagenomic insight into the microbiomes of geothermal springs in the Subantarctic Kerguelen Islands</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26299-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26299-4/MediaObjects/41598_2022_26299_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A metagenomic insight into the microbiomes of geothermal springs in the Subantarctic Kerguelen Islands</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-22 datetime="2022-12-22T17:00:13.000Z">2022-12-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-22 datetime="2022-12-22T17:00:13.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;121394"
                        open
                      >
                        <summary class="article-expander__title">The Year in Comments</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/thoughtful-and-funny-feedback-from-quantas-audience-in-2022-20221222/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/12/YiR-COMMENTS-byMyriamWares-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>While the internet is awash in the noise of unmoderated comments, Quanta has attempted to provide a quiet, information-rich spot where readers, listeners and viewers can contemplate the complexity of nature.              
The post The Year in Comments first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;121400"
                        open
                      >
                        <summary class="article-expander__title">The Year in Math</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/the-biggest-math-breakthroughs-in-2022-20221222/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/12/YiR-MATH-byMyriamWares-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Four Fields Medals were awarded for major breakthroughs in geometry, combinatorics, statistical physics and number theory, even as mathematicians continued to wrestle with how computers are changing the discipline.            
The post The Year in Math first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;23
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;121396"
                        open
                      >
                        <summary class="article-expander__title">The Year in Physics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/the-biggest-discoveries-in-physics-in-2022-20221222/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/12/YiR-PHYSICS-byMyriamWares-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In a year filled with sweet new observations in astronomy and tantalizing breakthroughs in condensed matter physics, the brand-new space telescope takes the cake.            
The post The Year in Physics first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/mri-light-detection-1222"
                        open
                      >
                        <summary class="article-expander__title">New sensor uses MRI to detect light deep in the brain</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/mri-light-detection-1222">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/MIT-MRI-Light-01.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using this approach, researchers can map how light spreads in opaque environments.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00506"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Logic Circuits Based on 2A Peptide Sequences in the Yeast Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00506">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00506</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00392"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A Low-Cost, Thermostable, Cell-Free Protein Synthesis Platform for On-Demand Production of Conjugate Vaccines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00392">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00392</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.22.521680v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Systematic multi-trait AAV capsid engineering for efficient gene delivery [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.22.521680v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Broadening gene therapy applications requires manufacturable vectors that efficiently transduce target cells in humans and preclinical models. Conventional selections of adeno-associated virus (AAV) capsid libraries are inefficient at searching the vast sequence space for the small fraction of vectors possessing multiple traits essential for clinical translation. Here, we present Fit4Function, a generalizable machine learning (ML) approach for systematically engineering multi-trait AAV capsids. By leveraging a capsid library that evenly samples the manufacturable sequence space, reproducible screening data are generated to train accurate sequence-to-function models. Combining six models, we designed a multi-trait (liver-targeted, manufacturable) capsid library and validated 89% of library variants on all six predetermined criteria. Furthermore, the models, trained only on mouse in vivo and human in vitro Fit4Function data, accurately predicted AAV capsid variant biodistribution in macaque. Top candidates exhibited high production yields, efficient murine liver transduction, up to 1000-fold greater human hepatocyte transduction, and increased enrichment, relative to AAV9, in a screen for liver transduction in macaques. The Fit4Function strategy ultimately makes it possible to predict cross-species traits of peptide-modified AAV capsids and is a critical step toward assembling an ML atlas that predicts AAV capsid performance across dozens of traits.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.21.521521v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Language models generalize beyond natural proteins [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.21.521521v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Learning the design patterns of proteins from sequences across evolution may have promise toward generative protein design. However it is unknown whether language models, trained on sequences of natural proteins, will be capable of more than memorization of existing protein families. Here we show that language models generalize beyond natural proteins to generate de novo proteins. We focus on two protein design tasks: fixed backbone design where the structure is specified, and unconstrained generation where the structure is sampled from the model. Remarkably although the models are trained only on sequences, we find that they are capable of designing structure. A total of 228 generated proteins are evaluated experimentally with high overall success rates (152/228 or 67%) in producing a soluble and monomeric species by size exclusion chromatography. Out of 152 experimentally successful designs, 35 have no significant sequence match to known natural proteins. Of the remaining 117, sequence identity to the nearest sequence match is at median 27%, below 20% for 6 designs, and as low as 18% for 3 designs. For fixed backbone design, the language model generates successful designs for each of eight experimentally evaluated artificially created fixed backbone targets. For unconstrained generation, sampled proteins cover diverse topologies and secondary structure compositions, and have high experimental success rate (71/129 or 55%). The designs reflect deep patterns linking sequence and structure, including motifs that occur in related natural structures, and motifs that are not observed in similar structural contexts in known protein families. The results show that language models, though only trained on sequences, learn a deep grammar that enables the design of protein structure, extending beyond natural proteins.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.21.521526v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">A high-level programming language for generative protein design [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.21.521526v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Combining a basic set of building blocks into more complex forms is a universal design principle. Most protein designs have proceeded from a manual bottom-up approach using parts created by nature, but top-down design of proteins is fundamentally hard due to biological complexity. We demonstrate how the modularity and programmability long sought for protein design can be realized through generative artificial intelligence. Advanced protein language models demonstrate emergent learning of atomic resolution structure and protein design principles. We leverage these developments to enable the programmable design of de novo protein sequences and structures of high complexity. First, we describe a high-level programming language based on modular building blocks that allows a designer to easily compose a set of desired properties. We then develop an energy-based generative model, built on atomic resolution structure prediction with a language model, that realizes all-atom structure designs that have the programmed properties. Designing a diverse set of specifications, including constraints on atomic coordinates, secondary structure, symmetry, and multimerization, demonstrates the generality and controllability of the approach. Enumerating constraints at increasing levels of hierarchical complexity shows that the approach can access a combinatorially large design space.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.20.521207v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Photovoltaic enzymes by design and evolution [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.20.521207v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The global energy crisis challenges us to develop more efficient strategies for the sustainable production of energy. Given the excellent efficiency of the natural photosynthetic apparatus, biohybrid photovoltaic devices present an attractive solution for solar energy conversion. However, their composition, stability, and complexity can limit their inclusion into photovoltaic devices. Here, we combined computational design and directed evolution to overcome these limitations and create tailor-made photoenzymes. Photo-biocatalysts were designed by introducing photosensitizer binding sites into heme-containing helical bundle proteins. The designed binding sites were specific for the target photosensitizer and readily transplanted into other helical bundles. The best design was highly evolvable and reached nanomolar ligand affinity after mutagenesis and screening. The evolved enzyme generated 2.6 times higher photocurrents than the photosensitizer alone, primarily driven by increased photostability. Evolvability is a unique advantage of our protein-based approach over abiological photovoltaic and will be critical to developing efficient biohybrid systems.



O_FIG O_LINKSMALLFIG WIDTH&#x3D;200 HEIGHT&#x3D;114 SRC&#x3D;&quot;FIGDIR/small/521207v1_ufig1.gif&quot; ALT&#x3D;&quot;Figure 1&quot;&gt;
View larger version (31K):
org.highwire.dtl.DTLVardef@12655org.highwire.dtl.DTLVardef@17bc6aaorg.highwire.dtl.DTLVardef@12fd205org.highwire.dtl.DTLVardef@855b63_HPS_FORMAT_FIGEXP  M_FIG C_FIG</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35520-x"
                        open
                      >
                        <summary class="article-expander__title">Reply to: A balanced measure shows superior performance of pseudobulk methods in single-cell RNA-sequencing analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35520-x">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Reply to: A balanced measure shows superior performance of pseudobulk methods in single-cell RNA-sequencing analysis</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35519-4"
                        open
                      >
                        <summary class="article-expander__title">A balanced measure shows superior performance of pseudobulk methods in single-cell RNA-sequencing analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35519-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35519-4/MediaObjects/41467_2022_35519_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nature Communications - A balanced measure shows superior performance of pseudobulk methods in single-cell RNA-sequencing analysis</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35564-z"
                        open
                      >
                        <summary class="article-expander__title">Benchmarking tools for detecting longitudinal differential expression in proteomics data allows establishing a robust reproducibility optimization regression approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35564-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35564-z/MediaObjects/41467_2022_35564_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Longitudinal proteomics holds great promise for biomarker discovery, but the data interpretation has remained a challenge. Here, the authors evaluate several tools to detect longitudinal differential expression in proteomics data and introduce RolDE, a robust reproducibility optimization approach.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;64
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41746-022-00720-8"
                        open
                      >
                        <summary class="article-expander__title">Confounders mediate AI prediction of demographics in medical imaging</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41746-022-00720-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41746-022-00720-8/MediaObjects/41746_2022_720_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>npj Digital Medicine - Confounders mediate AI prediction of demographics in medical imaging</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41746-022-00725-3"
                        open
                      >
                        <summary class="article-expander__title">In-ear infrasonic hemodynography with a digital health device for cardiovascular monitoring using the human audiome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41746-022-00725-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41746-022-00725-3/MediaObjects/41746_2022_725_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>npj Digital Medicine - In-ear infrasonic hemodynography with a digital health device for cardiovascular monitoring using the human audiome</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41564-022-01276-9"
                        open
                      >
                        <summary class="article-expander__title">SARS-CoV-2 Africa dashboard for real-time COVID-19 information</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41564-022-01276-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41564-022-01276-9/MediaObjects/41564_2022_1276_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The SARS-CoV-2 Africa dashboard is an interactive tool that enables visualization of SARS-CoV-2 genomic information in African countries. The customizable app allows users to visualize the number of sequences deposited in each country, and the variants circulating over time. Our dashboard enables near real-time exploration of public data that can inform policymakers, healthcare professionals and the public about the ongoing pandemic.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01708-8"
                        open
                      >
                        <summary class="article-expander__title">Multiplexed transcriptome discovery of RNA-binding protein binding sites by antibody-barcode eCLIP</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01708-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01708-8/MediaObjects/41592_2022_1708_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Antibody-barcode eCLIP (ABC) uses proximity ligation to couple DNA-barcoded antibodies to RNA-binding protein (RBP)-protected RNA fragments for multiplexed eCLIP. ABC can be used to interrogate several RBPs in a single tube with results on par with eCLIP.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26000-9"
                        open
                      >
                        <summary class="article-expander__title">MicroRNAs within the Basal-like signature of Quadruple Negative Breast Cancer impact overall survival in African Americans</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26000-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26000-9/MediaObjects/41598_2022_26000_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - MicroRNAs within the Basal-like signature of Quadruple Negative Breast Cancer impact overall survival in African Americans</span>
                                &ensp;<span class="article-reading-time">(&hairsp;32
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41594-022-00892-7"
                        open
                      >
                        <summary class="article-expander__title">Diverse silent chromatin states modulate genome compartmentalization and loop extrusion barriers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41594-022-00892-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41594-022-00892-7/MediaObjects/41594_2022_892_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The authors find that silent chromatin is more diverse than just facultative and constitutive heterochromatin. These inactive types have distinct three-dimensional interaction characteristics that are transposable if the underlying chromatin state is altered.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;57
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35621-7"
                        open
                      >
                        <summary class="article-expander__title">Graph-based pangenomics maximizes genotyping density and reveals structural impacts on fungal resistance in melon</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35621-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35621-7/MediaObjects/41467_2022_35621_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The power of pangenomic graphs to improve genetic mapping is still unclear. Here, the authors demonstrate its value in identification of genetic variants associated with disease resistance traits in melon using PanPipes, a pangenome construction and low-coverage genotype-by-sequencing pipeline.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04343-3"
                        open
                      >
                        <summary class="article-expander__title">A statistical framework for high-content phenotypic profiling using cellular feature distributions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04343-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04343-3/MediaObjects/42003_2022_4343_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A high-content screening assay based on a comprehensive set of cytological features, together with a robust statistical analysis workflow for profiling broad-based cellular phenotypic responses to small molecules or genetic perturbations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01674-1"
                        open
                      >
                        <summary class="article-expander__title">SVDSS: structural variation discovery in hard-to-call genomic regions using sample-specific strings from accurate long reads</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01674-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01674-1/MediaObjects/41592_2022_1674_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SVDSS combines multiple strategies to improve detection of structural variation using low-error long-read sequencing data.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35596-5"
                        open
                      >
                        <summary class="article-expander__title">A method to build extended sequence context models of point mutations and indels</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35596-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35596-5/MediaObjects/41467_2022_35596_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The mutation rate at any specific position in the human genome depends on sequence context. Here, the authors develop a method for predicting mutation rates of point mutations and indels based on sequence context; the results can be used to find genes where de novo mutations cause disease and genes under strong selective constraint.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;32
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35529-2"
                        open
                      >
                        <summary class="article-expander__title">Intracellular phase separation of globular proteins facilitated by short cationic peptides</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35529-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35529-2/MediaObjects/41467_2022_35529_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Phase separation provides intracellular organisation via membraneless entities called biomolecular condensates. Here, the authors show that short, cationic peptide tags can drive biomolecular condensation of engineered proteins in E. coli through associative interactions with RNA.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01706-w"
                        open
                      >
                        <summary class="article-expander__title">Virus-assisted directed evolution of enhanced suppressor tRNAs in mammalian cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01706-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01706-w/MediaObjects/41592_2022_1706_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Virus-assisted directed evolution of tRNAs (VADER) enables selection of improved suppressor tRNAs in mammalian cells. Improved pyrrolysyl- and tyrosyl-tRNA variants enhance unnatural amino acid incorporation in mammalian systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01082-0"
                        open
                      >
                        <summary class="article-expander__title">Genetically programmed cell-based synthesis of non-natural peptide and depsipeptide macrocycles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01082-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01082-0/MediaObjects/41557_2022_1082_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Macrocyclic peptides can be genetically encoded and synthesized in cells; however, the programmable diversity is limited. Now, macrocycles containing two non-canonical amino acids have been genetically encoded and synthesized in codon-reassigned Syn61Δ3 cells. Incorporating diverse hydroxy acids in Syn61Δ3 cells enables the synthesis of non-natural depsipeptides containing either one or two ester bonds.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-21 datetime="2022-12-21T14:55:57.000Z">2022-12-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-21 datetime="2022-12-21T14:55:57.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;121398"
                        open
                      >
                        <summary class="article-expander__title">The Year in Computer Science</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/the-biggest-discoveries-in-computer-science-in-2022-20221221/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/12/YiR-CS-byMyriamWares-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Computer scientists this year learned how to transmit perfect secrets, why transformers seem so good at everything, and how to improve on decades-old algorithms (with a little help from AI).            
The post The Year in Computer Science first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;121402"
                        open
                      >
                        <summary class="article-expander__title">The Year in Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/major-biology-discoveries-from-2022-20221221/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/12/YiR-BIOLOGY-byMyriamWares-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Momentum for new ideas in Alzheimer’s research joined advances in neuroscience, developmental biology and origin-of-life studies to make 2022 a memorable year of biological insights.            
The post The Year in Biology first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00649"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Encoding, Decoding, and Rendering Information in DNA Nanoswitch Libraries</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00649">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00649</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0123426?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Conjoined Lorenz twins—a new pseudohyperbolic attractor in three-dimensional maps and flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0123426?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
We describe new types of Lorenz-like attractors for three-dimensional flows and maps with symmetries. We give an example of a three-dimensional system of differential equations, which is centrally symmetric and mirror symmetric. We show that the system has a Lorenz-like attractor, which contains three saddle equilibrium states and consists of two mirror-symmetric components that are adjacent at the symmetry plane. We also found a discrete-time analog of this “conjoined-twins” attractor in a cubic three-dimensional Hénon map with a central symmetry. We show numerically that both attractors are pseudohyperbolic, which guarantees that each orbit of the attractor has a positive maximal Lyapunov exponent, and this property is preserved under small perturbations. We also describe bifurcation scenarios for the emergence of the attractors in one-parameter families of three-dimensional flows and maps possessing the symmetries.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.03.518932v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Point-of-care peptide hormone production enabled by cell-free protein synthesis [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.03.518932v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In resource-limited settings, it can be difficult to safely deliver sensitive biologic medicines to patients due to cold chain and infrastructure constraints. Point-of-care drug manufacturing could circumvent these challenges since medicines could be produced locally and used on-demand. Towards this vision, we combine cell-free protein synthesis (CFPS) and a 2-in-1 affinity purification and enzymatic cleavage scheme to develop a platform for point-of-care drug manufacturing. As a model, we use this platform to synthesize a panel of peptide hormones, an important class of medications that can be used to treat a wide variety of diseases including diabetes, osteoporosis, and growth disorders. With this approach, temperature-stable lyophilized CFPS reaction components can be rehydrated with DNA encoding a SUMOylated peptide hormone of interest when needed. Strep-Tactin(R) affinity purification and on-bead SUMO protease cleavage yields peptide hormones in their native form that are recognized by ELISA antibodies and that can bind their respective receptors. With further development to ensure proper biologic activity and patient safety, we envision this platform could be used to manufacture valuable peptide hormone drugs at the point-of-care in resource-limited settings.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.07.20.500902v3?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Evolutionary-scale prediction of atomic level protein structure with a language model [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.07.20.500902v3?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Artificial intelligence has the potential to open insight into the structure of proteins at the scale of evolution. It has only recently been possible to extend protein structure prediction to two hundred million cataloged proteins. Characterizing the structures of the exponentially growing billions of protein sequences revealed by large scale gene sequencing experiments would necessitate a breakthrough in the speed of folding. Here we show that direct inference of structure from primary sequence using a large language model enables an order of magnitude speed-up in high resolution structure prediction. Leveraging the insight that language models learn evolutionary patterns across millions of sequences, we train models up to 15B parameters, the largest language model of proteins to date. As the language models are scaled they learn information that enables prediction of the three-dimensional structure of a protein at the resolution of individual atoms. This results in prediction that is up to 60x faster than state-of-the-art while maintaining resolution and accuracy. Building on this, we present the ESM Metagenomic Atlas. This is the first large-scale structural characterization of metagenomic proteins, with more than 617 million structures. The atlas reveals more than 225 million high confidence predictions, including millions whose structures are novel in comparison with experimentally determined structures, giving an unprecedented view into the vast breadth and diversity of the structures of some of the least understood proteins on earth.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-26389-3"
                        open
                      >
                        <summary class="article-expander__title">Construction of a machine learning-based artificial neural network for discriminating PANoptosis related subgroups to predict prognosis in low-grade gliomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-26389-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-26389-3/MediaObjects/41598_2022_26389_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Construction of a machine learning-based artificial neural network for discriminating PANoptosis related subgroups to predict prognosis in low-grade gliomas</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41551-022-00996-x"
                        open
                      >
                        <summary class="article-expander__title">Predicting pathogenicity from non-coding mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41551-022-00996-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41551-022-00996-x/MediaObjects/41551_2022_996_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine-learning model identifies cell-type-specific predictors of the pathogenic effects of promoter and enhancer mutations in the human genome.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41576-022-00570-w"
                        open
                      >
                        <summary class="article-expander__title">New cuts for CRISPR effectors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41576-022-00570-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41576-022-00570-w/MediaObjects/41576_2022_570_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Two new studies in Science characterize a CRISPR-associated nuclease–protease system that can be leveraged as a programmable protease-based RNA sensor.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-20 datetime="2022-12-20T08:00:00.000Z">2022-12-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-20 datetime="2022-12-20T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1441122?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">QM/MM Methods for Crystalline Defects. Part 3: Machine-Learned MM Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1441122?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 4, Page 1490-1518, December 2022. 
 Abstract. We develop and analyze a framework for consistent QM/MM (quantum/classic) hybrid models of crystalline defects, which admits general atomistic interactions including traditional off-the-shell interatomic potentials as well as state of art “machine-learned interatomic potentials.&quot; We (i) establish an a priori error estimate for the QM/MM approximations in terms of matching conditions between the MM and QM models, and (ii) demonstrate how to use these matching conditions to construct practical machine learned MM potentials specifically for QM/MM simulations.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/15M1048549?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">One-Way Functions and (Im)perfect Obfuscation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/15M1048549?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://epubs.siam.org/na101/home/literatum/publisher/siam/journals/covergifs/smjcat/cover.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 6, Page 1769-1795, December 2022. 
 Abstract. A program obfuscator takes a program and outputs a “scrambled” version of it, where the goal is that the obfuscated program will not reveal much about its structure beyond what is apparent from executing it. There are several ways of formalizing this goal. Specifically, in indistinguishability obfuscation, first defined by Barak et al. [Advances in Cryptology - CRYPTO, 2001, Lect. Notes Comput. Sci. 2139, Springer, Berlin, Heidelberg, pp. 1–18], the requirement is that the results of obfuscating any two functionally equivalent programs (circuits) will be computationally indistinguishable. In 2013, a fascinating candidate construction for indistinguishability obfuscation was proposed by Garg et al. [Pr…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/17M1155375?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Balanced Allocation: Patience Is Not a Virtue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/17M1155375?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 6, Page 1743-1768, December 2022. 
 Abstract. Load balancing is a well-studied problem, with balls-in-bins being the primary framework. The greedy algorithm [math] of Azar et al. [SIAM J. Comput., 29 (1999), pp. 180–200] places each ball by probing [math] random bins and placing the ball in the least loaded of them. With high probability, the maximum load under [math] is exponentially lower than the result when balls are placed uniformly randomly. Vöcking [J. ACM, 50 (2003), pp. 568–589] showed that a slightly asymmetric variant, [math], provides a further significant improvement. However, this improvement comes at the additional computational cost of imposing structure on the bins. Here, we present a fully decentralized and easy-to-implement algorithm called [math] that combines the simplicity of [math] and the improved balance of [math]. The key idea in [math] is to probe until a different bin size from the first observation is located and then place the ball. Although the number of probes could be quite large for some of the balls, we show that [math] requires only at most [math] probes on average per ball (in both the standard and the heavily loaded settings). Thus the number of probes is no greater than that of either [math] or [math]. More importantly, we show that [math] closely matches the improved maximum load ensured by [math] in both the standard and heavily loaded settings. We further provide a tight lower bound on the maximum load up to [math] terms. We additionally give experimental data that [math] is indeed as good as [math], if not better, in practice.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/19M1262504?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Subexponential Parameterized Algorithms for Planar and Apex-Minor-Free Graphs via Low Treewidth Pattern Covering</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/19M1262504?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 6, Page 1866-1930, December 2022. 
 Abstract. We prove the following theorem. Given a planar graph [math] and an integer [math], it is possible in polynomial time to randomly sample a subset [math] of vertices of [math] with the following properties: [math] induces a subgraph of [math] of treewidth [math], and for every connected subgraph [math] of [math] on at most [math] vertices, the probability that [math] covers the whole vertex set of [math] is at least [math], where [math] is the number of vertices of [math]. Together with standard dynamic programming techniques for graphs of bounded treewidth, this result gives a versatile technique for obtaining (randomized) subexponential-time parameterized algorithms for problems on planar graphs, usually with running time bound [math]. The technique can be applied to problems expressible as searching for a small, connected pattern with a prescribed property in a large host graph; examples of such problems include Directed [math]-Path, Weighted [math]-Path, Vertex Cover Local Search, and Subgraph Isomorphism, among others. Up to this point, it was open whether these problems could be solved in subexponential parameterized time on planar graphs, because they are not amenable to the classic technique of bidimensionality. Furthermore, all our results hold in fact on any class of graphs that exclude a fixed apex graph as a minor, in particular on graphs embeddable in any fixed surface.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1391018?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Quantum Speedup for Graph Sparsification, Cut Approximation, and Laplacian Solving</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1391018?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 6, Page 1703-1742, December 2022. 
 Abstract. Graph sparsification underlies a large number of algorithms, ranging from approximation algorithms for cut problems to solvers for linear systems in the graph Laplacian. In its strongest form, “spectral sparsification” reduces the number of edges to near-linear in the number of nodes, while approximately preserving the cut and spectral structure of the graph. In this work we demonstrate a polynomial quantum speedup for spectral sparsification and many of its applications. In particular, we give a quantum algorithm that, given a weighted graph with [math] nodes and [math] edges, outputs a classical description of an [math]-spectral sparsifier in sublinear time [math]. This contrasts with the optimal classical complexity [math]. We also prove that our quantum algorithm is optimal up to polylog-factors. The algorithm builds on a string of existing results on sparsification, graph spanners, quantum algorithms for shortest paths, and efficient constructions for [math]-wise independent random strings. Our algorithm implies a quantum speedup for solving Laplacian systems and for approximating a range of cut problems such as min cut and sparsest cut.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1471742?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Multi-Item Nontruthful Auctions Achieve Good Revenue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1471742?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 6, Page 1796-1838, December 2022. 
 Abstract. We present a general framework for designing approximately revenue-optimal mechanisms for multi-item additive auctions, which applies to both truthful and nontruthful auctions. Given a (not necessarily truthful) single-item auction format [math] satisfying certain technical conditions, we run simultaneous item auctions augmented with a personalized entry fee for each bidder that must be paid before the auction can be accessed. These entry fees depend only on the prior distribution of bidder types and in particular are independent of realized bids. We bound the revenue of the resulting two-part tariff mechanism using a novel geometric technique that enables revenue guarantees for many common nontruthful auctions that previously had none. Our approach adapts and extends the duality framework of Cai, Devanur, and Weinberg [SIAM J. Comput., 50 (2021), pp. STOC16-160–STOC16-200] beyond truthful auctions. Our framework can be used with many common auction formats, such as simultaneous first-price, simultaneous second-price, and simultaneous all-pay auctions. Our results for first-price and all-pay are the first revenue guarantees of nontruthful mechanisms in multidimensional environments, addressing an open question in the literature [T. Roughgarden, V. Syrgkanis, and E. Tardos, J. Artificial Intelligence Res., 59 (2017), pp. 59–101]. If all-pay auctions are used, we prove that the resulting mechanism is also credible in the sense that the auctioneer cannot benefit by deviating from the stated mechanism after observing agent bids. This is the first static credible mechanism for multi-item additive auctions that achieves a constant factor of the optimal revenue. If second-price auctions are used, we obtain a truthful [math]-approximate mechanism with fixed entry fees that are amenable to tuning via online learning techniques.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00381"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Circular Guide RNA for Improved Stability and CRISPR-Cas9 Editing Efficiency in Vitro and in Bacteria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00381">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00381</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0122351?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Copula-based analysis of the generalized friendship paradox in clustered networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0122351?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
A heterogeneous structure of social networks induces various intriguing phenomena. One of them is the friendship paradox, which states that on average, your friends have more friends than you do. Its generalization, called the generalized friendship paradox (GFP), states that on average, your friends have higher attributes than yours. Despite successful demonstrations of the GFP by empirical analyses and numerical simulations, analytical, rigorous understanding of the GFP has been largely unexplored. Recently, an analytical solution for the probability that the GFP holds for an individual in a network with correlated attributes was obtained using the copula method but by assuming a locally tree structure of the underlying network [Jo et al., Phys. Rev. E 104, 054301 (2021)]. Considering the abundant triangles in most social networks, we employ a vine copula method to incorporate the attribute correlation structure between neighbors of a focal individual in addition to the correlation between the focal individual and its neighbors. Our analytical approach helps us rigorously understand the GFP in more general networks, such as clustered networks and other related interesting phenomena in social networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0123886?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Multiplexing-based control of stochastic resonance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0123886?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
We show that multiplexing (Here, the term “multiplexing” means a special network topology where a one-layer network is connected to another one-layer networks through coupling between replica nodes. In the present paper, this term does not refer to the signal processing issues and telecommunications.) allows us to control noise-induced dynamics of multilayer networks in the regime of stochastic resonance. We illustrate this effect on an example of two- and multi-layer networks of bistable overdamped oscillators. In particular, we demonstrate that multiplexing suppresses the effect of stochastic resonance if the periodic forcing is present in only one layer. In contrast, multiplexing allows us to enhance the stochastic resonance if the periodic forcing and noise are present in all the interacting layers. In such a case, the impact of multiplexing has a resonant character: the most pronounced effect of stochastic resonance is achieved for an appropriate intermediate value of coupling strength between the layers. Moreover, multiplexing-induced enhancement of the stochastic resonance can become more pronounced for the increasing number of coupled layers. To visualize the revealed phenomena, we use the evolution of the dependence of the signal-to-noise ratio on the noise intensity for varying strength of coupling between the layers.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0104704?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Langevin equation for the collective motions of a binary astrophysical system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0104704?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
According to Thirring’s condition, the thermodynamic equilibrium is not possible between two systems with negative heat capacities. Since the existence of negative heat capacity is a rule rather an exception in astrophysics, we would like to analyze the possibility of the thermodynamic equilibrium for a paradigmatic astrophysical situation: the binary system. The possible explanation arises after decomposing the dynamical variables into internal and collective degrees of freedom, which enables us to reinterpret binary systems as two systems [math] and [math] that interact between them through a third low dimensional system [math]. Our central result is the derivation of a Langevin equation to describe the dynamics of the collective degrees of freedom. Our preliminary analysis evidences that the proposed framework is able not only to describe the system stability, but also unstable processes, such as escape or collapse of the binary system. These processes crucially depend on the angular momentum, the quadrupole–orbit coupling among internal and collective degrees of freedom, as well as internal temperatures of each system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.12.16.520786v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Reductive Enzyme Cascades for Valorization of PET Deconstruction Products [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.12.16.520786v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>To better incentivize the collection of plastic wastes, new chemical transformations must be developed that add value to plastic deconstruction products. Polyethylene terephthalate (PET) is a common plastic whose deconstruction through chemical or biological means has received much attention. However, a limited number of alternative products have been formed from PET deconstruction, and only a small share could serve as building blocks for alternative materials or therapeutics. Here, we demonstrate the production of useful mono-amine and diamine building blocks from known PET deconstruction products. We achieve this by designing one-pot biocatalytic transformations that are informed by the substrate specificity of an {omega}-transaminase and diverse carboxylic acid reductases (CAR) towards…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="285"
                        open
                      >
                        <summary class="article-expander__title">Journal of Mathematical Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/285">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01851-y"
                        open
                      >
                        <summary class="article-expander__title">Two-phase model of compressive stress induced on a surrounding hyperelastic medium by an expanding tumour</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01851-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In vitro experiments in which tumour cells are seeded in a gelatinous medium, or hydrogel, show how mechanical interactions between tumour cells and the tissue in which they are embedded, together with local levels of an externally-supplied, diffusible nutrient (e.g., oxygen), affect the tumour’s growth dynamics. In this article, we present a mathematical model that describes these in vitro experiments. We use the model to understand how tumour growth generates mechanical deformations in the hydrogel and how these deformations in turn influence the tumour’s growth. The hydrogel is viewed as a nonlinear hyperelastic material and the tumour is modelled as a two-phase mixture, comprising a viscous tumour cell phase and an isotropic, inviscid interstitial fluid phase. Using a combination of numerical and analytical techniques, we show how the tumour’s growth dynamics change as the mechanical properties of the hydrogel vary. When the hydrogel is soft, nutrient availability dominates the dynamics: the tumour evolves to a large equilibrium configuration where the proliferation rate of nutrient-rich cells on the tumour boundary balances the death rate of nutrient-starved cells in the central, necrotic core. As the hydrogel stiffness increases, mechanical resistance to growth increases and the tumour’s equilibrium size decreases. Indeed, for small tumours embedded in stiff hydrogels, the inhibitory force experienced by the tumour cells may be so large that the tumour is eliminated. Analysis of the model identifies parameter regimes in which the presence of the hydrogel drives tumour elimination.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10915"
                        open
                      >
                        <summary class="article-expander__title">Journal of Scientific Computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10915">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02074-5"
                        open
                      >
                        <summary class="article-expander__title">Adaptive FEM for Helmholtz Equation with Large Wavenumber</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02074-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A posteriori upper and lower bounds are derived for the linear finite element method (FEM) for the Helmholtz equation with large wavenumber. It is proved rigorously that the standard residual type error estimator seriously underestimates the true error of the FE solution for the mesh size h in the preasymptotic regime, which is first observed by Babuška et al. (Int J Numer Methods Eng 40:3443–3462, 1997) for a one dimensional problem. By establishing an equivalence relationship between the error estimators for the FE solution and the corresponding elliptic projection of the exact solution, an adaptive algorithm is proposed and its convergence and quasi-optimality are proved under the condition that 
                      
                      \(k^3h_0^{1+\alpha }\)
                      
                     is sufficiently small, where k is the wavenumber, 
                      
                      \(h_0\)
                      
                     is the initial mesh size, and 
                      
                      \(\frac{1}{2}&lt;\alpha \le 1\)
                      
                     is a regularity constant depending on the maximum reentrant angle of the domain. Numerical tests are given to verify the theoretical findings and to show that the adaptive continuous interior penalty finite element method (CIP-FEM) with appropriately selected penalty parameters can greatly reduce the pollution error and hence the residual type error estimator for this CIP-FEM is reliable and efficient even in the preasymptotic regime.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-19 datetime="2022-12-19T05:00:00.000Z">2022-12-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-19 datetime="2022-12-19T05:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00425"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Optimization of Pinocembrin Biosynthesis in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00425">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00425</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00338"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Coupling Cell Communication and Optogenetics: Implementation of a Light-Inducible Intercellular System in Yeast</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00338">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00338</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00511"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Continuous Directed Evolution of a Feedback-Resistant Arabidopsis Arogenate Dehydratase in Plantized Escherichia coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00511">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00511</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00472"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering of Specific Single-Module Nonribosomal Peptide Synthetases of the RXP Type for the Production of Defined Peptides</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00472">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00472</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0125266?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Periodic motions in a class of discontinuous delayed systems with a parabolic boundary</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0125266?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
Various periodic motions are investigated for a mass-spring-damper oscillator described by second-order discontinuous delayed systems with a parabolic boundary. The [math]-function related to a delayed variable is first developed to judge the crossing, sliding, and grazing motions at the boundary points. Then, basic and local mappings, together with their motion equations, are classified by current and delayed states of a local flow. It is shown that periodic motions can be predicted analytically in terms of the return mapping and the initial value function provided posteriorly. Finally, numerical simulations are given to illustrate the existence of slowly oscillating periodic solutions without or with sliding portions as well as a crossing periodic solution not slowly oscillating. These results not only enrich and extend switchability theory of discontinuous delayed systems, but also reveal some complicated periodic behaviors which cannot be found in continuous delayed systems and discontinuous systems without time delay.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0136898?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">In memoriam—Polina S. Landa (15 February 1931–21 October 2022)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0136898?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097187?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">A combinatorial view of stochastic processes: White noise</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097187?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
White noise is a fundamental and fairly well understood stochastic process that conforms to the conceptual basis for many other processes, as well as for the modeling of time series. Here, we push a fresh perspective toward white noise that, grounded on combinatorial considerations, contributes to giving new interesting insights both for modeling and theoretical purposes. To this aim, we incorporate the ordinal pattern analysis approach, which allows us to abstract a time series as a sequence of patterns and their associated permutations, and introduce a simple functional over permutations that partitions them into classes encoding their level of asymmetry. We compute the exact probability mass function (p.m.f.) of this functional over the symmetric group of degree [math], thus providing the description for the case of an infinite white noise realization. This p.m.f. can be conveniently approximated by a continuous probability density from an exponential family, the Gaussian, hence providing natural sufficient statistics that render a convenient and simple statistical analysis through ordinal patterns. Such analysis is exemplified on experimental data for the spatial increments from tracks of gold nanoparticles in 3D diffusion.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01844-x"
                        open
                      >
                        <summary class="article-expander__title">Persistence in a large network of sparsely interacting neurons</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01844-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This article presents a biological neural network model driven by inhomogeneous Poisson processes accounting for the intrinsic randomness of synapses. The main novelty is the introduction of sparse interactions: each firing neuron triggers an instantaneous increase in electric potential to a fixed number of randomly chosen neurons. We prove that, as the number of neurons approaches infinity, the finite network converges to a nonlinear mean-field process characterised by a jump-type stochastic differential equation. We show that this process displays a phase transition: the activity of a typical neuron in the infinite network either rapidly dies out, or persists forever, depending on the global parameters describing the intensity of interconnection. This provides a way to understand the emergence of persistent activity triggered by weak input signals in large neural networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01841-0"
                        open
                      >
                        <summary class="article-expander__title">Sensitivity of the electrical response of a node of Ranvier model to alterations of the myelin sheath geometry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01841-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Nodes of Ranvier play critical roles in the generation and transmission of action potentials. Alterations in node properties during pathology and/or development are known to affect the speed and quality of electrical transmission. From a modelling standpoint, nodes of Ranvier are often described by systems of ordinary differential equations neglecting or greatly simplifying their geometric structure. These approaches fail to accurately describe how fine scale alteration in the node geometry or in myelin thickness in the paranode region will impact action potential generation and transmission. Here, we rely on a finite element approximation to describe the three dimensional geometry of a node of Ranvier. With this, we are able to investigate how sensitive is the electrical response to alterations in the myelin sheath and paranode geometry. We could in particular investigate irregular loss of myelin, which might be more physiologically relevant than the uniform loss often described through simpler modelling approaches.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-17 datetime="2022-12-17T00:00:00.000Z">2022-12-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-17 datetime="2022-12-17T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01852-x"
                        open
                      >
                        <summary class="article-expander__title">Long-range seed dispersal enables almost stationary patterns in a model for dryland vegetation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01852-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Spatiotemporal patterns of vegetation are a ubiquitous feature of semi-arid ecosystems. On sloped terrain, vegetation patterns occur as stripes perpendicular to the contours. Field studies report contrasting long-term dynamics between different observation sites; some observe slow uphill migration of vegetation bands while some report stationary patterns. In this paper, we show that long-range seed dispersal provides a mechanism that enables the occurrence of both migrating and stationary patterns. We utilise a nonlocal PDE model in which seed dispersal is accounted for by a convolution term. The model represents vegetation patterns as periodic travelling waves and numerical continuation shows that both migrating and almost stationary patterns are stable if seed dispersal distances are sufficiently large. We use a perturbation theory approach to obtain analytical confirmation of the existence of almost stationary patterned solutions and provide a biological interpretation of the phenomenon.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-16 datetime="2022-12-16T08:00:00.000Z">2022-12-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-16 datetime="2022-12-16T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1450719?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Epidemics from the Eye of the Pathogen</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1450719?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 2036-2056, December 2022. 
 Abstract. While a common trend in disease modeling is to develop models of increasing complexity, it was recently pointed out that outbreaks appear remarkably simple when viewed in the incidence vs. cumulative cases (ICC) plane. This article details the theory behind this phenomenon by analyzing the stochastic Susceptible, Infected, Recovered (SIR) model in the cumulative cases domain. We prove that the Markov chain associated with this model reduces, in the ICC plane, to a pure birth chain for the cumulative number of cases, whose limit leads to an independent increments Gaussian process that fluctuates about a deterministic ICC curve. We calculate the associated variance and quantify the additional variability due to estimating incidence over a finite period of time. We also illustrate the universality brought forth by the ICC concept on real-world data for Influenza A and for the COVID-19 outbreak in Arizona.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1454055?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Near-Collision Dynamics in a Noisy Car-Following Model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1454055?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 2080-2110, December 2022. 
 Abstract. We consider a small stochastic perturbation of an optimal velocity car-following model. We give a detailed analysis of behavior near a collision singularity. We show that collision is impossible in a simplified model without noise and then show that collision is asymptotically unlikely over large time intervals in the presence of small noise, with the large time interval scaling like the square of the reciprocal of the strength of the noise. Our calculations depend on careful boundary-layer analyses.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1471547?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Minimal Object Characterizations Using Harmonic Generalized Polarizability Tensors and Symmetry Groups</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1471547?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 2057-2079, December 2022. 
 Abstract. We introduce a new type of object characterization, which is capable of accurately describing small isolated inclusions for potential field inverse problems such as in electrostatics, magnetostatics, and related low frequency Maxwell problems. Relevant applications include characterizing ferrous unexploded ordnance from magnetostatic field measurements in magnetometry, describing small conducting inclusions for medical imaging using electrical impedance tomography, performing geological ground surveys using electrical resistivity imaging, characterizing objects by electrosensing fish to navigate and identify food, and describing the effective properties of dilute composites. Our object characterization builds on the generalized polarizability tensor (GPT) object characterization concept and provides an alternative to the contracted GPT. We call the new characterizations harmonic GPTs (HGPTs) as their coefficients correspond to products of harmonic polynomials. Then, we show that the number of independent coefficients of HGPTs needed to characterize objects can be significantly reduced by considering the symmetry group of the object and propose a systematic approach for determining the subspace of symmetric harmonic polynomials that is fixed by the group and its dimension. This enables us to determine the independent HGPT coefficients for different symmetry groups.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M147668X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">A Continuous Integral Model for White Blood Cell Production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M147668X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 2111-2130, December 2022. 
 Abstract. Production of blood cells is a complex and tightly regulated process. Its quantitative understanding is highly relevant for the treatment of various diseases, including blood cancers. White blood cells (WBCs) originate from the so-called hematopoietic stem cell (HSC) population. HSCs give rise to immature WBCs that divide and further differentiate into mature cells. It is still a matter of debate whether cell differentiation resembles a discrete or a continuous process. We propose a novel continuous integral model (CIM) to describe WBC production. In agreement with recent experimental results, the maturity level of cells is described by a one-dimensional continuous variable. The process of maturation is modeled by an integral term that accounts for the influx of cells of maturity level [math] from the differentiation of cells with lower maturity levels. In the model, cell proliferation and differentiation are controlled by a feedback signal which is a function of the mature cell population. We investigate the relationships between the proposed CIM, the multicompartment ordinary differential equations model introduced in [A. Marciniak-Czochra et al., Stem Cells Dev., 18 (2009), pp. 377–385] which treats differentiation as a discrete process, and the partial differential equations model of transport type studied by Doumic et al. [SIAM J. Appl. Math., 71 (2011), pp. 1918–1940] which describes differentiation as a continuous process. We show that both models can be derived from the CIM and we perform numerical simulations to compare them qualitatively.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M135515X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Relaxed Locally Correctable Codes with Nearly-Linear Block Length and Constant Query Complexity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M135515X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 6, Page 1839-1865, December 2022. 
 Abstract. Locally correctable codes (LCCs) are error correcting codes [math] which admit local algorithms that correct any individual symbol of a corrupted codeword via a minuscule number of queries. For systematic codes, this notion is stronger than that of locally decodable codes (LDCs), where the goal is to only recover individual symbols of the message. One of the central problems in algorithmic coding theory is to construct [math]-query LCCs and LDCs with minimal block length. Alas, state-of-the-art of such codes requires super-polynomial block length to admit [math]-query algorithms for local correction and decoding, despite much attention during the last two decades. The study of relaxed LCCs and LDCs, which allow the correction algorithm to abort (but not err) on a small fraction of the locations, provides a way to circumvent this barrier. This relaxation turned out to allow constant-query correcting and decoding algorithms for codes with polynomial block length. Focusing on local correction, Gur, Ramnarayan, and Rothblum [Proceedings of the 9th Innovations in Theoretical Computer Science Conference, ITCS’18, 2018, pp. 1–27] showed that there exist [math]-query relaxed LCCs that achieve nearly-quartic block length [math], for an arbitrarily small constant [math]. We construct an [math]-query relaxed LCC with nearly-linear block length [math], for an arbitrarily small constant [math]. This significantly narrows the gap between the lower bound which states that there are no [math]-query relaxed LCCs with block length [math]. In particular, our construction matches the parameters achieved by Ben-Sasson et al. [SIAM J. Comput., 36 (2006), pp. 889–974], who constructed relaxed LDCs with the same parameters. This resolves an open problem raised by Gur, Ramnarayan, and Rothblum [Proceedings of the 9th Innovations in Theoretical Computer Science Conference, ITCS’18, 2018, pp. 1–27].</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0126782?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Consequences of fear effect and prey refuge on the Turing patterns in a delayed predator–prey system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0126782?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
This study presents a qualitative analysis of a modified Leslie–Gower prey–predator model with fear effect and prey refuge in the presence of diffusion and time delay. For the non-delayed temporal system, we examined the dissipativeness and persistence of the solutions. The existence of equilibria and stability analysis is performed to comprehend the complex behavior of the proposed model. Bifurcation of codimension-1, such as Hopf-bifurcation and saddle-node, is investigated. In addition, it is observed that increasing the strength of fear may induce periodic oscillations, and a higher value of fear may lead to the extinction of prey species. The system shows a bistability attribute involving two stable equilibria. The impact of providing spatial refuge to the prey population is also examined. We noticed that prey refuge benefits both species up to a specific threshold value beyond which it turns detrimental to predator species. For the non-spatial delayed system, the direction and stability of Hopf-bifurcation are investigated with the help of the center manifold theorem and normal form theory. We noticed that increasing the delay parameter may destabilize the system by producing periodic oscillations. For the spatiotemporal system, we derived the analytical conditions for Turing instability. We investigated the pattern dynamics driven by self-diffusion. The biological significance of various Turing patterns, such as cold spots, stripes, hot spots, and organic labyrinth, is examined. We analyzed the criterion for Hopf-bifurcation for the delayed spatiotemporal system. The impact of fear response delay on spatial patterns is investigated. Numerical simulations are illustrated to corroborate the analytical findings.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0131433?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Stochastic averaging for a type of fractional differential equations with multiplicative fractional Brownian motion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0131433?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
This paper focuses on the averaging principle of Caputo fractional stochastic differential equations (SDEs) with multiplicative fractional Brownian motion (fBm), where Hurst parameter [math] and the integral of fBm as a generalized Riemann–Stieltjes integral. Under suitable assumptions, the averaging principle on Hölder continuous space is established by giving the estimate of Hölder norm. Specifically, we show that the solution of the original fractional SDEs converges to the solution of the proposed averaged fractional SDEs in the mean square sense and gives an example to illustrate our result.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0117473?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Synchronization of a higher-order network of Rulkov maps</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0117473?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
In neuronal network analysis on, for example, synchronization, it has been observed that the influence of interactions between pairwise nodes is essential. This paper further reveals that there exist higher-order interactions among multi-node simplicial complexes. Using a neuronal network of Rulkov maps, the impact of such higher-order interactions on network synchronization is simulated and analyzed. The results show that multi-node interactions can considerably enhance the Rulkov network synchronization, better than pairwise interactions, for involving more and more neurons in the network.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0114542?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Learning chaotic systems from noisy data via multi-step optimization and adaptive training</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0114542?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
A data-driven sparse identification method is developed to discover the underlying governing equations from noisy measurement data through the minimization of Multi-Step-Accumulation (MSA) in error. The method focuses on the multi-step model, while conventional sparse regression methods, such as the Sparse Identification of Nonlinear Dynamics method (SINDy), are one-step models. We adopt sparse representation and assume that the underlying equations involve only a small number of functions among possible candidates in a library. The new development in MSA is to use a multi-step model, i.e., predictions from an approximate evolution scheme based on initial points. Accordingly, the loss function comprises the total error at all time steps between the measured series and predicted series with the same initial point. This enables MSA to capture the dynamics directly from the noisy measurements, resisting the corruption of noise. By use of several numerical examples, we demonstrate the robustness and accuracy of the proposed MSA method, including a two-dimensional chaotic map, the logistic map, a two-dimensional damped oscillator, the Lorenz system, and a reduced order model of a self-sustaining process in turbulent shear flows. We also perform further studies under challenging conditions, such as noisy measurements, missing data, and large time step sizes. Furthermore, in order to resolve the difficulty of the nonlinear optimization, we suggest an adaptive training strategy, namely, by gradually increasing the length of time series for training. Higher prediction accuracy is achieved in an illustrative example of the chaotic map by the adaptive strategy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41929-022-00901-4"
                        open
                      >
                        <summary class="article-expander__title">Mimicking intercellular communication</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41929-022-00901-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41929-022-00901-4/MediaObjects/41929_2022_901_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nature Catalysis - Mimicking intercellular communication</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02072-7"
                        open
                      >
                        <summary class="article-expander__title">Solving Blind Ptychography Effectively Via Linearized Alternating Direction Method of Multipliers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02072-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The problem of blind ptychography is to determine the specimen object and the scanning probe simultaneously from diffraction data. By formulating the problem as a nonconvex optimization, we propose linearized alternating direction method of multipliers (LADMM) to effectively solve the blind ptychography problem. Different from the iterative ptychographic engines (including ePIE, rPIE and other variants), all of the diffraction patterns are simultaneously exploited in our optimization based approach to address the ill-posedness of the problem. Compared to the existing ADMM-type algorithm in the literature (Chang et al. in SIAM J Imaging Sci 12(1):153–185, 2019), ours is based on a new splitting form for the bilinear term, which enables closed-form solutions for each subproblem. Convergence to stationary point of the considered optimization is provided under certain assumptions. It is observed that the performance of ADMM-type method highly depends on the penalty parameter. Thus LADMM with adaptive penalty parameter is developed to prioritize the reconstruction performance and have empirical convergence guarantee. With numerical comparison to other competitive algorithms, our proposed method outperforms the current state-of-the-art ones on simulated and experimental data, especially for the only limited available diffraction data case, which validates the effectiveness of the proposed algorithm.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02066-5"
                        open
                      >
                        <summary class="article-expander__title">A Dimension Splitting Method for Time Dependent PDEs on Irregular Domains</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02066-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We develop a simple and efficient dimension splitting method for solving time dependent partial differential equations (PDEs) on multiple space dimensional irregular domains using a one dimensional kernel-free boundary integral (KFBI) method. The proposed method extends the alternating direction implicit methods and locally one dimensional methods to more general cases involving complex geometry. The KFBI method is a potential theory based Cartesian grid method, which works as an improvement of conventional boundary integral methods. In the KFBI method, boundary or volume integrals are evaluated by solving equivalent interface problems without using any analytic expression of Green’s functions. The one dimensional interface problems after dimension splitting are solved by finite difference method. The resulting linear systems are tri-diagonal and efficiently solved by the Thomas algorithm. The one dimensional kernel-free boundary integral method is rigorously proved to have second-order convergence rate in the maximum norm. Multiple numerical examples, including different types of PDEs and a free boundary problem, are presented to demonstrate the advantages of the proposed method. Numerical results show that the proposed method is efficient and achieves overall second order accuracy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02062-9"
                        open
                      >
                        <summary class="article-expander__title">First-Order Positivity-Preserving Entropy Stable Scheme for the 3-D Compressible Navier–Stokes Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02062-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we extend the positivity-preserving, entropy stable first-order scheme developed for the one-dimensional compressible Navier–Stokes equations in Upperman et al. (J Comput Phys 466, 2022) to three spatial dimensions. The new first-order scheme is provably entropy stable, design-order accurate for smooth solutions, and guarantees the pointwise positivity of thermodynamic variables for 3-D compressible viscous flows. Similar to the 1-D counterpart, the proposed scheme for the 3-D Navier–Stokes equations is discretized on Legendre-Gauss-Lobatto grids used for high-order spectral collocation methods. The positivity of density is achieved by adding an artificial dissipation in the form of the first-order Brenner–Navier–Stokes diffusion operator. Another distinctive feature of the proposed scheme is that the Navier–Stokes viscous terms are discretized by high-order spectral collocation summation-by-parts operators. To eliminate time step stiffness caused by the high-order approximation of the viscous terms and the temperature positivity constraint, the velocity and temperature limiters developed for the 1-D compressible Navier–Stokes equations in Upperman et al. (J. Comput. Phys., 466, 2022) are generalized to three spatial dimensions. These limiters bound the magnitude of velocity and temperature gradients and preserve the entropy stability and positivity properties of the baseline scheme. Numerical results are presented to demonstrate design-order accuracy and positivity-preserving properties of the new first-order scheme for 2-D and 3-D inviscid and viscous flows with strong shocks and contact discontinuities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-15 datetime="2022-12-15T05:00:00.000Z">2022-12-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-15 datetime="2022-12-15T05:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00455"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Investigating the Specificity of the Dehydration and Cyclization Reactions in Engineered Lanthipeptides by Synechococcal SyncM</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00455">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00455</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00507"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering Pheromone-Mediated Quorum Sensing with Enhanced Response Output Increases Fucosyllactose Production in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00507">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00507</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01194-1"
                        open
                      >
                        <summary class="article-expander__title">De novo engineering of a bacterial lifestyle program</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01194-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01194-1/MediaObjects/41589_2022_1194_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>This article presents a synthetic genetic program for orthogonal, tunable and programmable control of bacterial lifestyle and associated phase-specific gene expression, offering a versatile platform for microbial engineering in complex contexts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01209-x"
                        open
                      >
                        <summary class="article-expander__title">A change of phase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01209-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01209-x/MediaObjects/41589_2022_1209_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Most engineered bacteria are designed to grow and function in a free-swimming state. A new method enables engineered bacteria to reversibly transition into a biofilm state.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34902-5"
                        open
                      >
                        <summary class="article-expander__title">Accuracy and data efficiency in deep learning models of protein expression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34902-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34902-5/MediaObjects/41467_2022_34902_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Synthetic biology often involves engineering microbial strains to express high-value proteins. Here the authors build deep learning predictors of protein expression from sequence that deliver accurate models with fewer data than previously assumed, helping to lower costs of model-driven strain design.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;40
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-14 datetime="2022-12-14T16:00:00.000Z">2022-12-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-14 datetime="2022-12-14T16:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/five-mit-named-2023-quad-fellows-1214"
                        open
                      >
                        <summary class="article-expander__title">Five from MIT named 2023 Quad Fellows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/five-mit-named-2023-quad-fellows-1214">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/2023-Quad-Fellows.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The new fellowship from the governments of Australia, India, Japan, and the United States, administered by Schmidt Futures, supports graduate education in STEM fields.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1480732?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Circadian Regulation of Electrolyte and Water Transport in the Rat Kidney</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1480732?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Ahead of Print. 
 Abstract. Circadian oscillations have been reported in major renal functions such as renal blood flow, glomerular filtration rate, and urinary excretion. These oscillations can be explained, in part, by the rhythmic variations in renal transporter expressions. However, given that a multitude of renal transporters are regulated by the circadian clock, and that the functions of some of these transporters are coupled, the underlying mechanisms that link variations in transporter expression to the observed changes in kidney function have yet to be fully elucidated. To better understand the impact of the circadian clock on renal solute and water transport, we have developed a computational model of the epithelial transport of the rat kidney that represents the observed rhythmic variations in glomerular filtration rate (GFR) and in the activities of Na+/H+ exchanger 3, sodium-gLucose cotransporter 1, epithelial Na+ channels, pendrin, and renal outer-medullary K+ channels. The model predicts the rhythmic oscillations in GFR and key renal transporter activities give rise to a significant shift in transport loads among different nephron segments. Together, these oscillations yield 3-4-fold daily fluctuations in urine output and urinary electrolyte excretion rates.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00434"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering the Biosynthesis of Late-Stage Vinblastine Precursors Precondylocarpine Acetate, Catharanthine, Tabersonine in Nicotiana benthamiana</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00434">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00434</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00539"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] CaExTun: Mitigating Cas9-Related Toxicity in Streptomyces through Species-Specific Expression Tuning with Randomized Constitutive Promoters</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00539">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00539</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35500-1"
                        open
                      >
                        <summary class="article-expander__title">Biosynthesis of mushroom-derived type II ganoderic acids by engineered yeast</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35500-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35500-1/MediaObjects/41467_2022_35500_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The biosynthetic pathway of type II ganoderic acids (GAs) in Ganoderma lucidum, a traditional medicinal mushroom, is unknown. Here, the authors assemble the genome of type II GAs accumulating accession, identify CYPs involving in type II GAs biosynthesis, and achieve their production in engineered baker’s yeast.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;52
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-13 datetime="2022-12-13T20:30:00.000Z">2022-12-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-13 datetime="2022-12-13T20:30:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/reframing-first-generation-academic-experience-1213"
                        open
                      >
                        <summary class="article-expander__title">Reframing the first-generation academic experience</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/reframing-first-generation-academic-experience-1213">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/2022-FirstGenSummit.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>First-gen MIT graduate students are claiming their identity, forming community, and holding space for one another.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.29076.ip?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Special Issue: RECOMB 2022, Part 2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.29076.ip?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1269-1269, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0261?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Improving RNA Assembly via Safety and Completeness in Flow Decompositions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0261?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1270-1287, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Differentiable Learning of Sequence-Specific Minimizer Schemes with DeepMinimizer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1288-1304, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0336?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">NetMix2: A Principled Network Propagation Algorithm for Identifying Altered Subnetworks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0336?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1305-1323, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0378?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Multimodal Genotype and Phenotype Data Integration to Improve Partial Data-Based Longitudinal Prediction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0378?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1324-1345, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">RESISTOR: A New OSPREY Module to Predict Resistance Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1346-1352, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0280?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Kernel Mixed Model for Transcriptome Association Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0280?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1353-1356, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0262?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Accurate Binning of Metagenomic Contigs Using Composition, Coverage, and Assembly Graphs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0262?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1357-1376, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0411?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">On the Hardness of Sequence Alignment on De Bruijn Graphs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0411?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1377-1396, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0413?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Group Testing Matrix Design for PCR Screening with Real-Valued Measurements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0413?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 12, Page 1397-1411, December 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00416"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Reconstitution of Multi-Protein Complexes through Ribozyme-Assisted Polycistronic Co-Expression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00416">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00416</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0126682?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Taming cardiac arrhythmias: Terminating spiral wave chaos by adaptive deceleration pacing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0126682?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
Sequences of weak electrical pulses are considered a promising alternative for terminating ventricular and atrial fibrillations while avoiding strong defibrillation shocks with adverse side effects. In this study, using numerical simulations of four different 2D excitable media, we show that pulse trains with increasing temporal intervals between successive pulses (deceleration pacing) provide high success rates at low energies. Furthermore, we propose a simple and robust approach to calculate inter-pulse spacing directly from the frequency spectrum of the dynamics (for instance, computed based on the electrocardiogram), which can be practically used in experiments and clinical applications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0129066?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Recovering obstacles from their traveling times</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0129066?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
Noakes and Stoyanov [Mathematics 9, 2434 (2021)] introduced a method of recovering strictly convex planar obstacles from their set of traveling times. We provide an extension of this construction for obstacles on Riemannian surfaces under some general curvature conditions. It is required that no smooth geodesic intersects more than two obstacles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0106514?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Forced signal and predictability in a prototype climate model: Implications for fingerprinting based detection in the presence of multidecadal natural variability</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0106514?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
Advanced numerical models used for climate prediction are known to exhibit biases in their simulated climate response to variable concentrations of the atmospheric greenhouse gases and aerosols that force a non-uniform, in space and time, secular global warming. We argue here that these biases can be particularly pronounced due to misrepresentation, in these models, of the multidecadal internal climate variability characterized by large-scale, hemispheric-to-global patterns. This point is illustrated through the development and analysis of a prototype climate model comprised of two damped linear oscillators, which mimic interannual and multidecadal internal climate dynamics and are set into motion via a combination of stochastic driving, representing weather noise, and deterministic external forcing inducing a secular climate change. The model time series are paired with pre-specified patterns in the physical space and form, conceptually, a spatially extended time series of the zonal-mean near-surface temperature, which is further contaminated by a spatiotemporal noise simulating the rest of climate variability. The choices of patterns and model parameters were informed by observations and climate-model simulations of the 20th century near-surface air temperature. Our main finding is that the intensity and spatial patterns of the internal multidecadal variability associated with the slow-oscillator model component greatly affect (i) the ability of modern pattern-recognition/fingerprinting methods to isolate the forced response of the climate system in the 20th century ensemble simulations and (ii) climate-system predictability, especially decadal predictability, as well as the estimates of this predictability using climate models in which the internal multidecadal variability is underestimated or otherwise misrepresented.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01850-z"
                        open
                      >
                        <summary class="article-expander__title">Role of senescent tumor cells in building a cytokine shield in the tumor microenvironment: mathematical modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01850-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Cellular senescence can induce dual effects (promotion or inhibition) on cancer progression. While immune cells naturally respond and migrate toward various chemotactic sources from the tumor mass, various factors including senescent tumor cells (STCs) in the tumor microenvironment may affect this chemotactic movement. In this work, we investigate the mutual interactions between the tumor cells and the immune cells that either inhibit or facilitate tumor growth by developing a mathematical model that consists of taxis–reaction–diffusion equations and receptor kinetics for the key players in the interaction network. We apply a mathematical model to a transwell Boyden chamber invasion assay used in the experiments to illustrate that STCs can play a pivotal role in negating immune attack through tight regulation of intra- and extra-cellular signaling molecules. In particular, we show that senescent tumor cells in cell cycle arrest can block intratumoral infiltration of CD8+ T cells by secreting a high level of CXCL12, which leads to significant reduction its receptors, CXCR4, on T cells, and thus impaired chemotaxis. The predictions of nonlinear responses to CXCL12 were in good agreement with experimental data. We tested several hypotheses on immune-tumor interactions under various biochemical conditions in the tumor microenvironment and developed new concepts for anti-tumor strategies targeting senescence induced immune impairment.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-12 datetime="2022-12-12T16:00:00.000Z">2022-12-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-12 datetime="2022-12-12T16:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/diaphragm-implantable-ventilator-1212"
                        open
                      >
                        <summary class="article-expander__title">MIT engineers design a soft, implantable ventilator</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/diaphragm-implantable-ventilator-1212">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/MIT-Implantable-Ventilator-01-PRESS.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The new design works with the diaphragm to improve breathing.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/two-mit-named-2023-marshall-scholars-1212"
                        open
                      >
                        <summary class="article-expander__title">Two from MIT named 2023 Marshall Scholars</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/two-mit-named-2023-marshall-scholars-1212">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/2022-Marshall-scholars.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Rachel Chae and Sihao Huang ’22 will pursue graduate studies in the United Kingdom.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/mucus-cholera-infection-1212"
                        open
                      >
                        <summary class="article-expander__title">Molecules found in mucus could prevent cholera infection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/mucus-cholera-infection-1212">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202212/MIT-Cholera-Mucins-01-press.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Harnessing these protective molecules may offer a new way to treat the disease, which spreads through contaminated water.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0125535?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Instability mechanisms of repelling peak solutions in a multi-variable activator–inhibitor system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0125535?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 12, December 2022. 
We study the linear stability properties of spatially localized single- and multi-peak states generated in a subcritical Turing bifurcation in the Meinhardt model of branching. In one spatial dimension, these states are organized in a foliated snaking structure owing to peak–peak repulsion but are shown to be all linearly unstable, with the number of unstable modes increasing with the number of peaks present. Despite this, in two spatial dimensions, direct numerical simulations reveal the presence of stable single- and multi-spot states whose properties depend on the repulsion from nearby spots as well as the shape of the domain and the boundary conditions imposed thereon. Front propagation is shown to trigger the growth of new spots while destabilizing others. The results indicate that multi-variable models may support new types of behavior that are absent from typical two-variable models.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00496"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] CRISPR-Based Diagnostics: Challenges and Potential Solutions toward Point-of-Care Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00496">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00496</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00407"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] DisGUVery: A Versatile Open-Source Software for High-Throughput Image Analysis of Giant Unilamellar Vesicles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00407">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00407</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41586-022-05622-z"
                        open
                      >
                        <summary class="article-expander__title">Programming Multicellular Assembly with Synthetic Cell Adhesion Molecules</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41586-022-05622-z">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature - Programming Multicellular Assembly with Synthetic Cell Adhesion Molecules</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-11 datetime="2022-12-11T00:00:00.000Z">2022-12-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-11 datetime="2022-12-11T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02069-2"
                        open
                      >
                        <summary class="article-expander__title">Fast and Accurate Artificial Compressibility Ensemble Algorithms for Computing Parameterized Stokes–Darcy Flow Ensembles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02069-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Accurate simulations of the Stokes–Darcy system face many difficulties including the coupling of flows in two different subdomains via interface conditions, the incompressibility constraint for the free flow and uncertainties in model parameters. In this report, we propose and study efficient, decoupled, artificial compressibility (AC) ensemble schemes based on a recently developed SAV approach for fast computation of Stokes–Darcy flow ensembles. The proposed algorithms (1) do not require any time step condition and (2) decouple the computation of the velocity and pressure in the free flow region, and (3) result in a common coefficient matrix for all realizations after spatial discretization for which efficient iterative linear solvers such as block CG or block GMRES can be used to greatly reduce the computational cost. We prove the long time stability under two parameter conditions, without any timestep constraints. In particular, for one single simulation, they are unconditionally stable schemes. Several numerical tests are presented to demonstrate the efficiency of the algorithms and illustrate their applications in realistic flow problems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02064-7"
                        open
                      >
                        <summary class="article-expander__title">Binary Quantized Network Training With Sharpness-Aware Minimization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02064-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The quantized neural network is a common way to improve inference and memory efficiency for deep learning methods. However, it is challenging to solve this optimization problem with good generalization due to its high nonlinearity and nonconvexity. This paper proposes an algorithm to train one-bit quantization neural networks based on sharpness-aware minimization Foret et al. (Sharpness-aware minimization for efficiently improving generalization, 2021) with two types of gradient approximation. The idea is to improve the generalization by finding a local minimum of a flat landscape of both continuous and quantized neural network loss. The convergence theory is partially established under a one-bit quantization setting. Experiments on the CIFAR-10 Krizhevsky (Learning multiple layers of features from tiny images, 2009) and SVHN Netzer et al. (Reading digits in natural images with unsupervised feature learning, 2011) datasets show improvement in generalization with the proposed algorithm compared to other state-of-the-art quantized training methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-10 datetime="2022-12-10T05:00:00.000Z">2022-12-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-10 datetime="2022-12-10T05:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00609"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Low Resource Integrated Platform for Production and Analysis of Capped mRNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00609">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00609</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-25895-8"
                        open
                      >
                        <summary class="article-expander__title">Direct encapsulation of biomolecules in semi-permeable microcapsules produced with double-emulsions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-25895-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-25895-8/MediaObjects/41598_2022_25895_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Direct encapsulation of biomolecules in semi-permeable microcapsules produced with double-emulsions</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-12-09 datetime="2022-12-09T08:00:00.000Z">2022-12-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-12-09 datetime="2022-12-09T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;sjaday&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1464300?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;sjaday&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Melnikov Theory for Two-Dimensional Manifolds in Three-Dimensional Flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1464300?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;sjaday&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Dynamical Systems, Volume 21, Issue 4, Page 2642-2696, December 2022. 
 Abstract. We present a geometric Melnikov method to analyze a two-dimensional stable or unstable manifold associated with a saddle point in three-dimensional nonvolume preserving autonomous flows. The time-varying perturbed location of such a manifold is obtained under very general, nonvolume preserving and with arbitrary time-dependence, perturbations. We demonstrate the explicit computability of the leading-order spatio-temporal location of the manifold using our formulas. In unperturbed situations with a two-dimensional heteroclinic manifold, we adapt our theory to quantify the splitting into a stable and unstable manifold, and thereby obtain an instantaneous flux quantification in terms of a Melnikov function. The time-varying instantaneous flux theory does not require any intersections between perturbed manifolds, nor rely on descriptions of lobe dynamics. Our theory has specific application to transport in fluid mechanics, where the flow is in three dimensions and flow separators in forward/backward time are two-dimensional stable/unstable manifolds. We demonstrate our theory using both the classical and swirling versions of Hill’s spherical vortex.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1475776?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;sjaday&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">A Projected Subgradient Method for the Computation of Adapted Metrics for Dynamical Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1475776?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;sjaday&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Dynamical Systems, Volume 21, Issue 4, Page 2610-2641, December 2022. 
 Abstract. In this paper, we extend a recently established subgradient method for the computation of Riemannian metrics that optimizes certain singular value functions associated with dynamical systems. This extension is threefold. First, we introduce a projected subgradient method which results in Riemannian metrics whose parameters are confined to a compact convex set and we can thus prove that a minimizer exists; second, we allow inexact subgradients and study the effect of the errors on the computed metrics; and third, we analyze the subgradient algorithm for three different choices of step sizes: constant, exogenous, and Polyak. The new methods are illustrated by application to dimension and entropy estimation of the Hénon map.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1440219?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Mean Field Analysis of Hypergraph Contagion Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1440219?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 1987-2007, December 2022. 
 Abstract. We typically interact in groups, not just in pairs. For this reason, it has recently been proposed that the spread of information, opinion, or disease should be modeled over a hypergraph rather than a standard graph. The use of hyperedges naturally allows for a nonlinear rate of transmission, in terms of both the group size and the number of infected group members, as is the case, for example, when social distancing is encouraged. We consider a general class of individual-level, stochastic, susceptible-infected-susceptible models on a hypergraph, and focus on a mean field approximation proposed in [G. F. de Arruda, G. Petri, and Y. Moreno, Phys. Rev. Res., 2 (2020), 023032]. We derive spectral conditions under which the mean field model predicts local or global stability of the infection-free state. We also compare these results with (a) a new condition that we derive for decay to zero in mean for the exact process, (b) conditions for a different mean field approximation in [D. J. Higham and H.-L. de Kergorlay, Proc. A, 477 (2021), 20210232], and (c) numerical simulations of the microscale model.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1440876?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Stationarity and Inference in Multistate Promoter Models of Stochastic Gene Expression via Stick-Breaking Measures</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1440876?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 1953-1986, December 2022. 
 Abstract. In a general stochastic multistate promoter model of dynamic messenger ribonucleic acid (mRNA)/protein interactions, we identify the stationary joint distribution of the promoter state, mRNA, and protein levels through an explicit “stick-breaking&quot; construction perhaps of interest in itself. This derivation is a constructive advance over previous work where the stationary distribution is solved only in restricted cases. Moreover, the stick-breaking construction allows us to sample directly from the stationary distribution, permitting inference procedures and model selection. In this context, we discuss numerical Bayesian experiments to illustrate the results.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1446162?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Large Deviations for Additive Functionals of Reflected Jump-Diffusions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1446162?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 2008-2035, December 2022. 
 Abstract. We consider a jump-diffusion process on a bounded domain with reflection at the boundary and establish long-term results for a general additive process of its path. This includes the long-term behavior of its occupation time in the interior and on the boundaries. We derive a characterization of the large deviation rate function which quantifies the rate of exponential decay of probabilities of rare events for paths of the process. The characterization relies on a solution of a partial integro-differential equation with boundary constraints. We develop a practical implementation of our results in terms of a numerical solution for the PIDE. We illustrate the method on a few standard examples (Brownian motion, birth-death processes) and on a particular jump-diffusion model arising from applications to biochemical reactions.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/xlxs4/rss-reader/actions/runs/3767809121">
        <time id="build-timestamp" datetime="2022-12-23T19:25:39.294Z">2022-12-23T19:25:39.294Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>