<!DOCTYPE html>
<html lang="en">

  <head>
    <title>osmos::feed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="osmos::feed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <!-- %before-head-end.html% -->
  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-26 datetime="2022-11-26T00:36:15.651Z">2022-11-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-26 datetime="2022-11-26T00:36:15.651Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ScienceDirect Publication: Journal of Differential Equations</button>
                  <a class="source-heading__link" href="https://www.sciencedirect.com/journal/journal-of-differential-equations">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622006659"
                        open
                      >
                        <summary class="article-expander__title">Lack of the strict dissipativity and modification for the dissipative Bresse system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622006659?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 February 2023
Source: Journal of Differential Equations, Volume 347
Author(s): Tomoyuki Suzuki, Yoshihiro Ueda</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200657X"
                        open
                      >
                        <summary class="article-expander__title">Well-posedness and exponential stability for a Klein-Gordon system with locally distributed viscoelastic dampings in a past-history framework</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200657X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 February 2023
Source: Journal of Differential Equations, Volume 346
Author(s): J.C.O. Faria, A.Y. Souza Franco</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622006660"
                        open
                      >
                        <summary class="article-expander__title">General study on limit cycle bifurcation near a double homoclinic loop</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622006660?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 February 2023
Source: Journal of Differential Equations, Volume 347
Author(s): Maoan Han, Junmin Yang, Jibin Li</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622006623"
                        open
                      >
                        <summary class="article-expander__title">Hopf bifurcation for general network-organized reaction-diffusion systems and its application in a multi-patch predator-prey system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622006623?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 February 2023
Source: Journal of Differential Equations, Volume 346
Author(s): Wei Gou, Zhen Jin, Hao Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622006647"
                        open
                      >
                        <summary class="article-expander__title">On entropy and pressure via preimage structure for random dynamical systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622006647?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 February 2023
Source: Journal of Differential Equations, Volume 346
Author(s): Xinsheng Wang, Weisheng Wu, Yujun Zhu</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200660X"
                        open
                      >
                        <summary class="article-expander__title">Transitions in stochastic non-equilibrium systems: Efficient reduction and analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200660X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 February 2023
Source: Journal of Differential Equations, Volume 346
Author(s): MickaÃ«l D. Chekroun, Honghu Liu, James C. McWilliams, Shouhong Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-25 datetime="2022-11-25T05:00:00.000Z">2022-11-25</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-25 datetime="2022-11-25T05:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00385"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Tuning of Gene Expression in Clostridium phytofermentans Using Synthetic Promoters and CRISPRi</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00385">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00385</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.24.517869v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Synthetic in vivo compartmentalisation improves metabolic flux and modulates the product profile of promiscuous enzymes [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.24.517869v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Enzyme spatial organisation and compartmentalisation are naturally evolved mechanisms for facilitating multi-step biocatalysis. We explored the synthetic in vivo co-encapsulation of two different cargo proteins in yeast using a self-assembling virus-like particle. Co-encapsulation was verified using single particle techniques for both end-to-end fusion of the cargo proteins with the encapsulation anchor at one end, and coexpression of each cargo protein with their individual anchors. The co-encapsulation of a bifunctional geranyl diphosphate/farnesyl diphosphate synthase and a bifunctional linalool/nerolidol synthase delivered nerolidol titres up to 30 times that of an unorganised free enzyme control, a remarkable improvement from a single engineering step. Interestingly, striking differences in the ratio of products (linalool and nerolidol) were observed with each spatial organisation approach. This work presents the largest reported titre fold increases from in vivo enzyme compartmentalisation and suggests that enzyme spatial organisation could be used to modulate the product profile of promiscuous enzymes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.23.517630v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Enzymatic assembly of small synthetic genes with repetitive elements [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.23.517630v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although gene synthesis efficiency has improved through the years, current methods offered by DNA synthesis vendors are limited when it comes to repetitive sequences. Here, we describe a method for the enzymatic assembly of repetitive small synthetic genes. The method involves an initial step where the gene of interest is split in silico into small synthons of up to 80 base pairs flanked by Golden Gate-compatible four-base pair overhangs. Synthons are enzymatically synthesized by oligo extension and then assembled into the synthetic gene by Golden Gate Assembly. We construct eight different synthetic genes ranging from 133 to 456 base pairs encoding RNA structures with repetitive elements that are challenging to synthesize. We report assembly fidelities of up to 87.5 % that decrease for increasing number of synthons. This method is envisioned as an important addition to the molecular cloning toolbox and especially useful for construction of challenging and repetitive genes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33381-y"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Biological heterogeneity in idiopathic pulmonary arterial hypertension identified through unsupervised transcriptomic profiling of whole blood</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33381-y">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Author Correction: Biological heterogeneity in idiopathic pulmonary arterial hypertension identified through unsupervised transcriptomic profiling of whole blood</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34954-7"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Mothers with higher twinning propensity had lower fertility in pre-industrial Europe</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34954-7">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Author Correction: Mothers with higher twinning propensity had lower fertility in pre-industrial Europe</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41584-022-00876-3"
                        open
                      >
                        <summary class="article-expander__title">Understanding synovial cell diversity in post-traumatic OA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41584-022-00876-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41584-022-00876-3/MediaObjects/41584_2022_876_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Research related to the role of the synovium and its cell constituents during the pathogenies of osteoarthritis (OA) has taken a back seat to that of the cartilage and chondrocytes. The influence of synoviocytes in OA is increasingly recognized, but are synoviocytes equal in their contributions to disease progression?</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-23213-w"
                        open
                      >
                        <summary class="article-expander__title">Predicting the potential global distribution of an invasive alien pest Trioza erytreae (Del Guercio) (Hemiptera: Triozidae)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-23213-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-23213-w/MediaObjects/41598_2022_23213_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Predicting the potential global distribution of an invasive alien pest Trioza erytreae (Del Guercio) (Hemiptera: Triozidae)</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41431-022-01233-4"
                        open
                      >
                        <summary class="article-expander__title">Expanding the genetics and phenotypic spectrum of Lysine-specific demethylase 5C (KDM5C): a report of 13 novel variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41431-022-01233-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41431-022-01233-4/MediaObjects/41431_2022_1233_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>European Journal of Human Genetics - Expanding the genetics and phenotypic spectrum of Lysine-specific demethylase 5C (KDM5C): a report of 13 novel variants</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-24581-z"
                        open
                      >
                        <summary class="article-expander__title">Determining epigenetic memory in kidney proximal tubule cell derived induced pluripotent stem cells using a quadruple transgenic reprogrammable mouse</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-24581-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-24581-z/MediaObjects/41598_2022_24581_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Determining epigenetic memory in kidney proximal tubule cell derived induced pluripotent stem cells using a quadruple transgenic reprogrammable mouse</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34877-3"
                        open
                      >
                        <summary class="article-expander__title">A single-cell analysis reveals tumor heterogeneity and immune environment of acral melanoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34877-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34877-3/MediaObjects/41467_2022_34877_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Studying the cell composition of acral melanoma at the single-cell level could provide some clues about its poor response to immunotherapy. Here, the authors analyse acral and cutaneous melanoma patient samples using single-cell RNA-sequencing, and reveal a severe immunosuppressive state in acral melanomas</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-24900-4"
                        open
                      >
                        <summary class="article-expander__title">Lung_PAYNet: a pyramidal attention based deep learning network for lung nodule segmentation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-24900-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-24900-4/MediaObjects/41598_2022_24900_Fig1_HTML.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Lung_PAYNet: a pyramidal attention based deep learning network for lung nodule segmentation</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-24 datetime="2022-11-24T00:00:00.000Z">2022-11-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-24 datetime="2022-11-24T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.24.517818v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Engineering stringent genetic biocontainment of yeast with a protein stability switch [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.24.517818v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Synthetic biology holds immense promise to tackle key problems we are facing, for instance in resource use, environmental health, and human health care. However, comprehensive safety measures are needed to deploy genetically engineered microorganisms in open-environment applications. Here, we describe a genetic biocontainment system based on conditional stability of essential proteins. We used a yeast-adapted destabilizing domain degron, which can be stabilized by estradiol addition (ERdd). Leveraging the yeast GFP collection and lab automation platforms, we ERdd-tagged 775 essential genes and screened for strains with estradiol dependent growth. Three genes, SPC110, DIS3 and RRP46, were found to be particularly suitable. Respective strains showed no growth defect in the presence of estradiol and strong growth inhibition in its absence. Of these, SPC110-ERdd offered the most stringent containment, with an escape frequency of 7.0x10-8, and full growth restoration at 100 nM estradiol. By systematically analysing the containment escapees, we identified the non- essential C-terminal region of SPC110 as target for escape mutations. Its removal decreased the escape frequency with a single ERdd tag further to 4.3x10-9. Combining SPC110-ERdd with a second ERdd tag on either DIS3 or RRP46 resulted in escape frequencies below the detection limit of the used assay (&lt;2x10-10). Being based on conditional protein stability, this approach is mechanistically orthogonal to previously reported intrinsic biocontainment systems. It thus can be readily combined with other systems, for instance ones based on transcriptional or translational control of essential gene expression, to achieve multiplexed, extremely stringent control over the survival of engineered organisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.21.517450v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">A tRNA Modification-based strategy for Identifying amiNo acid Overproducers (AMINO) [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.21.517450v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Amino acids have a multi-billion-dollar market with rising demand, prompting the development of high-performance microbial factories. However, a general screening strategy applicable to all proteinogenic and non-proteinogenic amino acids is still lacking. Modification of the critical structure of tRNA could decrease the aminoacylation level of tRNA catalyzed by aminoacyl-tRNA synthetases. Involved in a two-substrate sequential reaction, amino acids with increased concentration could elevate the reduced aminoacylation rate caused by specific tRNA modification. Here, we developed a selection system for overproducers of specific amino acids using corresponding engineered tRNAs and reporter genes. As a proof-of-concept, overproducers of five amino acids such as L-tryptophan were screened out by growth-based and/or fluorescence-activated cell sorting (FACS)-based screening from random mutation libraries of Escherichia coli and Corynebacterium glutamicum, respectively. This study provided a universal strategy that could be applied to screen overproducers of proteinogenic and non-proteinogenic amino acids in amber-stop-codon-recoded or non-recoded hosts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.22.516435v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Repurposing the mammalian RNA-binding protein Musashi-1 as an allosteric translation repressor in bacteria [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.22.516435v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The RNA recognition motif (RRM) is the most common RNA-binding protein domain identified in nature. However, RRM-containing proteins are only prevalent in eukaryotic phyla, in which they play central regulatory roles. Here, we engineered an orthogonal post-transcriptional control system of gene expression in the bacterium Escherichia coli with the mammalian RNA-binding protein Musashi-1, which is a stem cell marker with neurodevelopmental role that contains two canonical RRMs. In the circuit, Musashi-1 is regulated transcriptionally and works as an allosteric translation repressor thanks to a specific interaction with the leader coding region of a messenger RNA and its structural plasticity to respond to fatty acids. We fully characterized the genetic system at the population and single-cell levels showing a significant fold change in reporter expression, and the underlying molecular mechanism by assessing the in vitro binding kinetics and in vivo functionality of a series of RNA mutants. Moreover, the dynamic response of the system was well recapitulated by a bottom-up mathematical model. This work illustrates how RRM-based regulation can be adapted to simple organisms, thereby paving the way for engineering more complex circuits in prokaryotes by combining transcription and translation control with proteins.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.22.517491v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">A termination-aware vector design improves heterologous gene expression in Pseudomonas putida [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.22.517491v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcriptional terminators are key players in the flow of genetic information, but are often overlooked in circuit design. In this work, we used the Standard European Vector Architecture (SEVA) as a scaffold to investigate the effects of different terminators in the output of a reporter construct expressed in two bacterial species and found that replacing the conventional T1 and T0 transcriptional terminators of the SEVA vector format with a set of broad-host metagenomic terminators resulted in a significant improvement in the signal of a fluorescent device in Pseudomonas putida KT2440. Interestingly, this new vector design displayed the same performance as pSEVA231 in Escherichia coli DH10B. Our results support the notion that transcriptional terminators may affect the interconnected mesh of biological processes taking place in gene expression, leading to a host-dependent output that should be heeded for optimal circuit expression in different microbial hosts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41397-022-00297-1"
                        open
                      >
                        <summary class="article-expander__title">Bioinformatics driven discovery of small molecule compounds that modulate the FOXM1 and PPARA pathway activities in breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41397-022-00297-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41397-022-00297-1/MediaObjects/41397_2022_297_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The Pharmacogenomics Journal - Bioinformatics driven discovery of small molecule compounds that modulate the FOXM1 and PPARA pathway activities in breast cancer</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01685-y"
                        open
                      >
                        <summary class="article-expander__title">AlphaFill: enriching AlphaFold models with ligands and cofactors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01685-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01685-y/MediaObjects/41592_2022_1685_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The AlphaFill algorithm transplants missing small molecules and ions from experimentally determined structures to predicted protein models in the AlphaFold protein structure database. All AlphaFill entries are available for visual inspection and download through the AlphaFill website.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04245-4"
                        open
                      >
                        <summary class="article-expander__title">DrugnomeAI is an ensemble machine-learning framework for predicting druggability of candidate drug targets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04245-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04245-4/MediaObjects/42003_2022_4245_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DrugnomeAI predicts the druggability likelihood for every protein-coding gene in the human exome by small molecules, monoclonal antibodies, and proteolysis-targeting chimeras (PROTACs).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01600-y"
                        open
                      >
                        <summary class="article-expander__title">Novel recombinases for large DNA insertions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01600-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01600-y/MediaObjects/41587_2022_1600_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bioinformatic mining and engineering of serine recombinases yields improved genome editing tools.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01552-3"
                        open
                      >
                        <summary class="article-expander__title">Single-cell multi-omics allows functional characterization of structural variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01552-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01552-3/MediaObjects/41587_2022_1552_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We developed a computational method based on Strand-seq that combines nucleosome occupancy and structural variation analysis in single cells to identify the functional effects of somatic structural variants in leukemia. The method revealed the molecular consequences of somatic driver mutations and informed specific therapeutic targeting of a chromothriptic leukemic subclone.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s43588-022-00353-7"
                        open
                      >
                        <summary class="article-expander__title">Single-cell-specific drug activities are revealed by a tensor imputation algorithm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s43588-022-00353-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs43588-022-00353-7/MediaObjects/43588_2022_353_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We developed a computational method to reveal the drug-induced single-cell transcriptomic landscape. This algorithm enabled us to impute unknown drug-induced single-cell gene expression profiles using tensor imputation, predict cell type-specific drug efficacy, detect cell-type-specific marker genes, and identify the trajectories of regulated biological pathways while considering intercellular heterogeneity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-24724-2"
                        open
                      >
                        <summary class="article-expander__title">Prevalence of classes 1 and 2 integrons in multidrug-resistant Acinetobacter baumanni isolates recovered from some aquatic environment in South Africa</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-24724-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-24724-2/MediaObjects/41598_2022_24724_Fig1a_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Prevalence of classes 1 and 2 integrons in multidrug-resistant Acinetobacter baumanni isolates recovered from some aquatic environment in South Africa</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-24302-6"
                        open
                      >
                        <summary class="article-expander__title">A continuous-time stochastic Boolean model provides a quantitative description of the budding yeast cell cycle</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-24302-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-24302-6/MediaObjects/41598_2022_24302_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A continuous-time stochastic Boolean model provides a quantitative description of the budding yeast cell cycle</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01551-4"
                        open
                      >
                        <summary class="article-expander__title">Functional analysis of structural variants in single cells using Strand-seq</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01551-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01551-4/MediaObjects/41587_2022_1551_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Linking structural variants to gene expression information identifies aberrant signaling pathways.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;62
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34760-1"
                        open
                      >
                        <summary class="article-expander__title">Evaluating cancer etiology and risk with a mathematical model of tumor evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34760-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34760-1/MediaObjects/41467_2022_34760_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Modelling how endogenous mutations accumulate in tissues is valuable to understand how cancers develop and evolve. Here, the authors establish a mathematical model that can predict the number of endogenous somatic mutations in the lifetime of tissues and approximate the time to cancer development.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-23515-z"
                        open
                      >
                        <summary class="article-expander__title">Proteomics reveals unique identities of human TGF-Î²-induced and thymus-derived CD4+ regulatory T cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-23515-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-23515-z/MediaObjects/41598_2022_23515_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Proteomics reveals unique identities of human TGF-Î²-induced and thymus-derived CD4+ regulatory T cells</span>
                                &ensp;<span class="article-reading-time">(&hairsp;41
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34904-3"
                        open
                      >
                        <summary class="article-expander__title">AlphaPeptDeep: a modular deep learning framework to predict peptide properties for proteomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34904-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34904-3/MediaObjects/41467_2022_34904_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deep learning (DL) has been frequently used in mass spectrometry-based proteomics but there is still a lot of potential. Here, the authors develop a framework that enables building DL models to predict arbitrary peptide properties with only a few lines of code.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s43588-022-00352-8"
                        open
                      >
                        <summary class="article-expander__title">Pathway trajectory analysis with tensor imputation reveals drug-induced single-cell transcriptomic landscape</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s43588-022-00352-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs43588-022-00352-8/MediaObjects/43588_2022_352_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A method is proposed to predict drug-induced single-cell gene-expression profiles with tensor imputation, detect cell type-specific maker genes, and identify the trajectories of regulated pathways considering intercellular heterogeneity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34959-2"
                        open
                      >
                        <summary class="article-expander__title">Genomic disparities between cancers in adolescent and young adults and in older adults</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34959-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34959-2/MediaObjects/41467_2022_34959_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The biological underpinnings underlying the increased mortality and morbidity in adolescents and young adults (AYA) remains poorly understood. Here, the authors investigate the clinical and genomic disparities in AYA and older adults in a cohort of more than 100,000 cancer patients.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-24485-y"
                        open
                      >
                        <summary class="article-expander__title">Contrast phase recognition in liver computer tomography using deep learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-24485-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-24485-y/MediaObjects/41598_2022_24485_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Contrast phase recognition in liver computer tomography using deep learning</span>
                                &ensp;<span class="article-reading-time">(&hairsp;27
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-24440-x"
                        open
                      >
                        <summary class="article-expander__title">Development of an insilico model of eccrine sweat using molecular modelling techniques</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-24440-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-24440-x/MediaObjects/41598_2022_24440_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Development of an insilico model of eccrine sweat using molecular modelling techniques</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41593-022-01189-0"
                        open
                      >
                        <summary class="article-expander__title">A deep learning framework for inference of single-trial neural population dynamics from calcium imaging with subframe temporal resolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41593-022-01189-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-022-01189-0/MediaObjects/41593_2022_1189_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Zhu et al. develop a deep learning method to precisely infer single-trial neural dynamics from calcium imaging with subframe temporal resolution, which shows improvement over the state-of-the-art methods in capturing high-frequency dynamics and predicting behavior.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-35038-2"
                        open
                      >
                        <summary class="article-expander__title">Resource sharing is sufficient for the emergence of division of labour</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-35038-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-35038-2/MediaObjects/41467_2022_35038_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Division of labour, where members of a group specialise on different tasks, is a central feature of many social organisms. Using a theoretical model, the authors demonstrate that division of labour can emerge spontaneously within a group of entirely identical individuals.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41416-022-02059-z"
                        open
                      >
                        <summary class="article-expander__title">Lynch syndrome, molecular mechanisms and variant classification</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41416-022-02059-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41416-022-02059-z/MediaObjects/41416_2022_2059_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>British Journal of Cancer - Lynch syndrome, molecular mechanisms and variant classification</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34980-5"
                        open
                      >
                        <summary class="article-expander__title">Enhanced access to the human phosphoproteome with genetically encoded phosphothreonine</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34980-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34980-5/MediaObjects/41467_2022_34980_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein phosphorylation is a ubiquitous post-translational modification used to regulate cellular processes and proteome architecture by modulating protein-protein interactions. Here the authors optimize genetically encoded phosphothreonine to study the regulation of CHK2 kinase using large-scale DNA arrays that enable phosphoproteome expression techniques to identify sitespecific overlap between CHK2 substrates and 14-3-3 interactions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;48
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01527-4"
                        open
                      >
                        <summary class="article-expander__title">Drag-and-drop genome insertion of large sequences without double-strand DNA cleavage using CRISPR-directed integrases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01527-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01527-4/MediaObjects/41587_2022_1527_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Large sequences are integrated site specifically into the human genome without double-strand DNA cleavage.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="285"
                        open
                      >
                        <summary class="article-expander__title">Journal of Mathematical Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/285">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01836-x"
                        open
                      >
                        <summary class="article-expander__title">The effects of internal forces and membrane heterogeneity on three-dimensional cell shapes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01836-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The shape of cells and the control thereof plays a central role in a variety of cellular processes, including endo- and exocytosis, cell division and cell movement. Intra- and extracellular forces control the shapes, and while the shape changes in some processes such as exocytosis are intracellularly-controlled and localized in the cell, movement requires force transmission to the environment, and the feedback from it can affect the cell shape and mode of movement used. The shape of a cell is determined by its cytoskeleton (CSK), and thus shape changes involved in various processes involve controlled remodeling of the CSK. While much is known about individual components involved in these processes, an integrated understanding of how intra- and extracellular signals are coupled to the control of the mechanical changes involved is not at hand for any of them. As a first step toward understanding the interaction between intracellular forces imposed on the membrane and cell shape, we investigate the role of distributed surrogates for cortical forces in producing the observed three-dimensional shapes. We show how different balances of applied forces lead to such shapes, that there are different routes to the same end state, and that state transitions between axisymmetric shapes need not all be axisymmetric. Examples of the force distributions that lead to protrusions are given, and the shape changes induced by adhesion of a cell to a surface are studied. The results provide a reference framework for developing detailed models of intracellular force distributions observed experimentally, and provide a basis for studying how movement of a cell in a tissue or fluid is influenced by its shape.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-23 datetime="2022-11-23T14:57:13.000Z">2022-11-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-23 datetime="2022-11-23T14:57:13.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;120848"
                        open
                      >
                        <summary class="article-expander__title">AI Reveals New Possibilities in Matrix Multiplication</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/ai-reveals-new-possibilities-in-matrix-multiplication-20221123/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/11/DeepmindMatrixMultiplication-cr.MahmetEminGu%CC%88zel-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Inspired by the results of a game-playing neural network, mathematicians have been making unexpected advances on an age-old math problem.            
The post AI Reveals New Possibilities in Matrix Multiplication first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1425499?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">The Local Equilibrium State of a Crystal Surface Jump Process in the Rough Scaling Regime</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1425499?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 4, Page 1315-1360, December 2022. 
 Abstract. We investigate the local equilibrium (LE) distribution of a crystal surface jump process as it approaches its hydrodynamic (continuum) limit in a nonstandard scaling regime introduced by Marzuola and Weare. The atypical scaling leads to a local equilibrium state whose structure is novel to the best of our knowledge. The distinguishing characteristic of the new, rough LE state is that the ensemble average of single lattice site observables do not vary smoothly across lattice sites. We investigate numerically and analytically how the rough LE state affects the convergence mechanism via three key limits, and we show that by comparison more standard, âsmoothâ LE states satisfy stronger versions of these limits.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00366"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Cell-Free Protein Synthesis of Particulate Methane Monooxygenase into Nanodiscs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00366">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00366</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00542"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Correction to âGB_SynP: A Modular dCas9-Regulated Synthetic Promoter Collection for Fine-Tuned Recombinant Gene Expression in Plantsâ</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00542">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00542</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00464"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Poly-Î³-Glutamic Acid Production by Engineering a DegU Quorum-Sensing Circuit in Bacillus subtilis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00464">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00464</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093804?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Guidelines for data-driven approaches to study transitions in multiscale systems: The case of Lyapunov vectors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093804?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This study investigates the use of covariant Lyapunov vectors and their respective angles for detecting transitions between metastable states in dynamical systems, as recently discussed in several atmospheric sciences applications. In a first step, the needed underlying dynamical models are derived from data using a non-parametric model-based clustering framework. The covariant Lyapunov vectors are then approximated based on these data-driven models. The data-based numerical approach is tested using three well-understood example systems with increasing dynamical complexity, identifying properties that allow for a successful application of the method: in particular, the method is identified to require a clear multiple time scale structure with fast transitions between slow subsystems. The latter slow dynamics should be dynamically characterized by invariant neutral directions of the linear approximation model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102424?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Finite-time recurrence analysis of chaotic trajectories in Hamiltonian systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102424?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
In this work, we show that a finite-time recurrence analysis of different chaotic trajectories in two-dimensional non-linear Hamiltonian systems provides useful prior knowledge of their dynamical behavior. By defining an ensemble of initial conditions, evolving them until a given maximum iteration time, and computing the recurrence rate of each orbit, it is possible to find particular trajectories that widely differ from the average behavior. We show that orbits with high recurrence rates are the ones that experience stickiness, being dynamically trapped in specific regions of the phase space. We analyze three different non-linear maps and present our numerical observations considering particular features in each of them. We propose the described approach as a method to visually illustrate and characterize regions in phase space with distinct dynamical behaviors.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2021.11.08.467706v4?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">ICOR: Improving codon optimization with recurrent neural networks [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2021.11.08.467706v4?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>BackgroundIn protein sequences--as there are 61 sense codons but only 20 standard amino acids-- most amino acids are encoded by more than one codon. Although such synonymous codons do not alter the encoded amino acid sequence, their selection can dramatically affect the expression of the resulting protein. Codon optimization of synthetic DNA sequences is important for heterologous expression. However, existing solutions are primarily based on choosing high-frequency codons only, neglecting the important effects of rare codons. In this paper, we propose a novel recurrent-neural-network based codon optimization tool, ICOR, that aims to learn codon usage bias on a genomic dataset of Escherichia coli. We compile a dataset of over 7,000 non-redundant, high-expression, robust genes which are useâ¦</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04226-7"
                        open
                      >
                        <summary class="article-expander__title">NetREX-CF integrates incomplete transcription factor data with gene expression to reconstruct gene regulatory networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04226-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04226-7/MediaObjects/42003_2022_4226_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>NetREX-CF is a computational method that integrates collaborative filtering and a mathematical model of transcriptional regulation to predict gene regulatory networks from incomplete transcription factor (TF) binding data and context-specific gene expression. NetREX-CF is used to obtain a comprehensive regulatory network and benchmark resource for Drosophila Schneider 2 cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;54
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34879-1"
                        open
                      >
                        <summary class="article-expander__title">Spatially aware dimension reduction for spatial transcriptomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34879-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34879-1/MediaObjects/41467_2022_34879_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Spatial transcriptomics analyses can be affected by noise and spatial correlation across tissue locations. Here, the authors develop SpatialPCA, a spatially-aware dimensionality reduction method that explicitly models spatial correlation structures, and show its application to the analysis of healthy and tumour tissues.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;79
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34109-8"
                        open
                      >
                        <summary class="article-expander__title">Reversion mutations in germline BRCA1/2-mutant tumors reveal a BRCA-mediated phenotype in non-canonical histologies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34109-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34109-8/MediaObjects/41467_2022_34109_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Mutations in BRCA1/2 are associated with a homologous recombination deficiency phenotype in BRCA-associated cancers. Reversion mutations can restore BRCA1/2 function and result in treatment resistance in these cancer-types. Here, the authors show that, in select cases, reversion mutations in BRCA1/2 can indicate prior BRCA-mediated tumorigenesis in non-canonical histologies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01617-3"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Modular, programmable RNA sensing using ADAR editing in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01617-3">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Biotechnology - Author Correction: Modular, programmable RNA sensing using ADAR editing in living cells</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01607-5"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Engineering the amoeba Dictyostelium discoideum for biosynthesis of a cannabinoid precursor and other polyketides</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01607-5">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Biotechnology - Author Correction: Engineering the amoeba Dictyostelium discoideum for biosynthesis of a cannabinoid precursor and other polyketides</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01831-2"
                        open
                      >
                        <summary class="article-expander__title">Two results about the Sackin and Colless indices for phylogenetic trees and their shapes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01831-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The Sackin and Colless indices are two widely-used metrics for measuring the balance of trees and for testing evolutionary models in phylogenetics. This short paper contributes two results about the Sackin and Colless indices of trees. One result is the asymptotic analysis of the expected Sackin and Colless indices of tree shapes (which are full binary rooted unlabelled trees) under the uniform model where tree shapes are sampled with equal probability. Another is a short direct proof of the closed formula for the expected Sackin index of phylogenetic trees (which are full binary rooted trees with leaves being labelled with taxa) under the uniform model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01834-z"
                        open
                      >
                        <summary class="article-expander__title">Plastic energy allocation toward life-history functions in a consumer-resource interaction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01834-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Various environmental alterations resulting from the current global change compromise the persistence of species in their habitual environment. To cope with the obvious risk of extinction, plastic responses provide organisms with rapid acclimatization to new environments. The premise of plastic rescue has been theoretically studied from mathematical models in both deterministic and stochastic environments, focusing on analyzing the persistence and stability of the populations. Here, we evaluate this premise in the framework of a consumer-resource interaction considering the energy investment towards reproduction vs. maintenance as a plastic trait according to positive/negative variation of the available resource. A basic consumer-resource mathematical model is formulated based on the principle of biomass conversion that incorporates the energy allocation toward vital functions of the life-cycle of consumer individuals. Our mathematical approach is based on the impulsive differential equations at fixed moments considering two impulsive effects associated with the instants at which consumers obtain environmental information and when energy allocation strategy change occurs. From a preliminary analysis of the non-plastic temporal dynamics, namely when the energy allocation is constant over time and without experiencing changes concerning the variation of resources, both the persistence and stability of the consumer-resource dynamic are dependent on the energy allocation strategies belonging to a set termed stability range. We found that the plastic energy allocation can promote a stable dynamical pattern in the consumer-resource interaction depending on both the magnitude of the energy allocation change and the time lag between environmental sensibility instants and when the expression of the plastic trait occurs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10915"
                        open
                      >
                        <summary class="article-expander__title">Journal of Scientific Computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10915">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02044-x"
                        open
                      >
                        <summary class="article-expander__title">Solving Non-linear Kolmogorov Equations in Large Dimensions by Using Deep Learning: A Numerical Comparison of Discretization Schemes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02044-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Non-linear partial differential Kolmogorov equations are successfully used to describe a wide range of time dependent phenomena, in natural sciences, engineering or even finance. For example, in physical systems, the AllenâCahn equation describes pattern formation associated to phase transitions. In finance, instead, the BlackâScholes equation describes the evolution of the price of derivative investment instruments. Such modern applications often require to solve these equations in high-dimensional regimes in which classical approaches are ineffective. Recently, an interesting new approach based on deep learning has been introduced byby E, Han and Jentzen [1, 2]. The main idea is to construct a deep network which is trained from the samples of discrete stochastic differential equations underlying Kolmogorovâs equation. The network is able to approximate, numerically at least, the solutions of the Kolmogorov equation with polynomial complexity in whole spatial domains. In this contribution we study variants of the deep networks by using different discretizations schemes of the stochastic differential equation. We compare the performance of the associated networks, on benchmarked examples, and show that, for some discretization schemes, improvements in the accuracy are possible without affecting the observed computational complexity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02049-6"
                        open
                      >
                        <summary class="article-expander__title">Projection Based Semi-Implicit Partitioned Reduced Basis Method for Fluid-Structure Interaction Problems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02049-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this manuscript a POD-Galerkin based Reduced Order Model for unsteady Fluid-Structure Interaction problems is presented. The model is based on a partitioned algorithm, with semi-implicit treatment of the coupling conditions. A ChorinâTemam projection scheme is applied to the incompressible NavierâStokes problem, and a Robin coupling condition is used for the coupling between the fluid and the solid. The coupled problem is based on an Arbitrary Lagrangian Eulerian formulation, and the Proper Orthogonal Decomposition procedure is used for the generation of the reduced basis. We extend existing works on a segregated Reduced Order Model for Fluid-Structure Interaction to unsteady problems that couple an incompressible, Newtonian fluid with a linear elastic solid, in two spatial dimensions. We consider three test cases to assess the overall capabilities of the method: an unsteady, non-parametrized problem, a problem that presents a geometrical parametrization of the solid domain, and finally, a problem where a parametrization of the solidâs shear modulus is taken into account.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;62
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02051-y"
                        open
                      >
                        <summary class="article-expander__title">A Multifidelity Monte Carlo Method for Realistic Computational Budgets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02051-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A method for the multifidelity Monte Carlo (MFMC) estimation of statistical quantities is proposed which is applicable to computational budgets of any size. Based on a sequence of optimization problems each with a globally minimizing closed-form solution, this method extends the usability of a well known MFMC algorithm, recovering it when the computational budget is large enough. Theoretical results verify that the proposed approach is at least as optimal as its namesake and retains the benefits of multifidelity estimation with minimal assumptions on the budget or amount of available data, providing a notable reduction in variance over simple Monte Carlo estimation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02056-7"
                        open
                      >
                        <summary class="article-expander__title">Machine Learning Moment Closure Models for the Radiative Transfer Equation III: Enforcing Hyperbolicity and Physical Characteristic Speeds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02056-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This is the third paper in a series in which we develop machine learning (ML) moment closure models for the radiative transfer equation. In our previous work (Huang et al. in J Comput Phys 453:110941, 2022), we proposed an approach to learn the gradient of the unclosed high order moment, which performs much better than learning the moment itself and the conventional 
                      
                      \(P_N\)
                      
                     closure. However, while the ML moment closure has better accuracy, it is not able to guarantee hyperbolicity and has issues with long time stability. In our second paper (Huang et al., in: Machine learning moment closure models for the radiative transfer equation II: enforcing global hyperbolicity in gradient based closures, 2021. arXiv:2105.14410), we identified a symmetrizer which leads to conditions that enforce that the gradient based ML closure is symmetrizable hyperbolic and stable over long time. The limitation of this approach is that in practice the highest moment can only be related to four, or fewer, lower moments. In this paper, we propose a new method to enforce the hyperbolicity of the ML closure model. Motivated by the observation that the coefficient matrix of the closure system is a lower Hessenberg matrix, we relate its eigenvalues to the roots of an associated polynomial. We design two new neural network architectures based on this relation. The ML closure model resulting from the first neural network is weakly hyperbolic and guarantees the physical characteristic speeds, i.e., the eigenvalues are bounded by the speed of light. The second model is strictly hyperbolic and does not guarantee the boundedness of the eigenvalues. Several benchmark tests including the Gaussian source problem and the two-material problem show the good accuracy, stability and generalizability of our hyperbolic ML closure model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02054-9"
                        open
                      >
                        <summary class="article-expander__title">Rank Properties and Computational Methods for Orthogonal Tensor Decompositions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02054-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The orthogonal decomposition factorizes a tensor into a sum of an orthogonal list of rank-one tensors. The corresponding rank is called orthogonal rank. We present several properties of orthogonal rank, which are different from those of tensor rank in many aspects. For instance, a subtensor may have a larger orthogonal rank than the whole tensor. To fit the orthogonal decomposition, we propose an algorithm based on the augmented Lagrangian method. The gradient of the objective function has a nice structure, inspiring us to use gradient-based optimization methods to solve it. We guarantee the orthogonality by a novel orthogonalization process. Numerical experiments show that the proposed method has a great advantage over the existing methods for strongly orthogonal decompositions in terms of the approximation error.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02045-w"
                        open
                      >
                        <summary class="article-expander__title">Kernel-Based Methods for Solving Time-Dependent Advection-Diffusion Equations on Manifolds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02045-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we extend the class of kernel methods, the so-called diffusion maps (DM) and ghost point diffusion maps (GPDM), to solve the time-dependent advection-diffusion PDE on unknown smooth manifolds without and with boundaries. The core idea is to directly approximate the spatial components of the differential operator on the manifold with a local integral operator and combine it with the standard implicit time difference scheme. When the manifold has a boundary, a simplified version of the GPDM approach is used to overcome the bias of the integral approximation near the boundary. The Monte-Carlo discretization of the integral operator over the point cloud data gives rise to a mesh-free formulation that is natural for randomly distributed points, even when the manifold is embedded in high-dimensional ambient space. Here, we establish the convergence of the proposed solver on appropriate topologies, depending on the distribution of point cloud data and boundary type. We provide numerical results to validate the convergence results on various examples that involve simple geometry and an unknown manifold.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02053-w"
                        open
                      >
                        <summary class="article-expander__title">A Cross-Product Free JacobiâDavidson Type Method for Computing a Partial Generalized Singular Value Decomposition of a Large Matrix Pair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02053-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A cross-product free (CPF) JacobiâDavidson type method is proposed to compute a partial generalized singular value decomposition (GSVD) of a large regular matrix pair 
                      
                      \(\{A,B\}\)
                      
                    , called CPF-JDGSVD. It implicitly solves the mathematically equivalent generalized eigenvalue problem of the cross-product matrix pair 
                      
                      \(\{A^TA,B^TB\}\)
                      
                     using the RayleighâRitz projection method but does not form the cross-product matrices explicitly, and thus avoids the possible accuracy loss of the computed generalized singular values and generalized singular vectors. The method is an inner-outer iteration method, where the expansion of the right searching subspace forms the inner iterations that approximately solve the correction equations involved and the outer iterations extract approximate GSVD components with respect to the subspaces. A convergence result is established for the outer iterations, compact bounds are derived for the condition numbers of the correction equations, and the least solution accuracy requirements on the inner iterations are found, which can maximize the overall efficiency of CPF-JDGSVD as much as possible. Based on them, practical stopping criteria are designed for the inner iterations. A thick-restart CPF-JDGSVD algorithm with deflation and purgation is developed to compute several GSVD components of 
                      
                      \(\{A,B\}\)
                      
                     associated with the generalized singular values closest to a given target 
                      
                      \(\tau \)
                      
                    . Numerical experiments illustrate the efficiency of the algorithm.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-22 datetime="2022-11-22T15:50:49.000Z">2022-11-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-22 datetime="2022-11-22T15:50:49.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;120793"
                        open
                      >
                        <summary class="article-expander__title">A Mathematician Dancing Between Algebra and Geometry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/wei-ho-is-drawn-to-algebra-geometry-and-the-human-side-of-math-20221122/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/11/WeiHo-cr.CarolineGutman-Social-2.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Wei Ho, the first director of the Women and Mathematics program at the Institute for Advanced Study, combines algebra and geometry in her work on an ancient class of curves.            
The post A Mathematician Dancing Between Algebra and Geometry first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00552"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Self-Sufficient In Vitro Multi-Enzyme Cascade for Efficient Synthesis of Danshensu from lâDOPA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00552">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00552</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00374"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Repurposing the Endogenous CRISPR-Cas9 System for High-Efficiency Genome Editing in Lacticaseibacillus paracasei</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00374">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00374</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0126799?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The impact of positive and negative information on SIR-like epidemics in delayed multiplex networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0126799?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
In order to better study the interaction between epidemic propagation and information diffusion, a new coupling model on multiplex networks with time delay is put forward in this paper. One layer represents the information diffusion about epidemics. There is not only information about the positive prevention of infectious diseases but also negative preventive information. Meanwhile, the dissemination of information at this layer will be influenced by the mass media, which can convey positive and reliable preventive measures to help the public avoid exposure to contagion. The other layer represents the transmission of infectious diseases, and the public in this layer no longer only exchange information related to infectious diseases in the virtual social network like the information layer but spread infectious diseases through contact among people. The classical SIR model is used to model for epidemic propagation. Since each infected individual needs to spend enough time to recover, the infected one at one time does not necessarily change to the recovered one at the next time, so time delay is an essential factor to be considered in the model. Based on the microscopic Markov chain approach, this paper obtains an explicit expression for epidemic threshold in the two-layered multiplex networks with time delay, which reveals some main factors affecting epidemic threshold. In particular, the time delay has a noticeable effect on the epidemic threshold to some extent. Finally, the influence of these main factors on the epidemic threshold and their interaction are proved through numerical simulations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0123514?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Motion of dark solitons in a non-uniform flow of BoseâEinstein condensate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0123514?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We study motion of dark solitons in a non-uniform one-dimensional flow of a BoseâEinstein condensate. Our approach is based on Hamiltonian mechanics applied to the particle-like behavior of dark solitons in a slightly non-uniform and slowly changing surrounding. In one-dimensional geometry, the condensateâs wave function undergoes the jump-like behavior across the soliton, and this leads to generation of the counterflow in the background condensate. For a correct description of solitonâs dynamics, the contributions of this counterflow to the momentum and energy of the soliton are taken into account. The resulting Hamilton equations are reduced to the Newton-like equation for the solitonâs path, and this Newton equation is solved in several typical situations. The analytical results are confirmed by numerical calculations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0099859?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Quasi-objective eddy visualization from sparse drifter data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0099859?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We employ a recently developed single-trajectory Lagrangian diagnostic tool, the trajectory rotation average [math], to visualize oceanic vortices (or eddies) from sparse drifter data. We apply the [math] to two drifter data sets that cover various oceanographic scales: the Grand Lagrangian Deployment and the Global Drifter Program. Based on the [math], we develop a general algorithm that extracts approximate eddy boundaries. We find that the [math] outperforms other available single-trajectory-based eddy detection methodologies on sparse drifter data and identifies eddies on scales that are unresolved by satellite-altimetry.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02058-5"
                        open
                      >
                        <summary class="article-expander__title">Eigenvalues of Quaternion Tensors with Applications to Color Video Processing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02058-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This paper studies the eigenvalues of quaternion tensors and their applications. Firstly, we investigate the definitions and properties of the left and right eigenvalues of the quaternion tensors. Then we establish the quaternion tensor Lanczos method to calculate the eigenvalues and eigentensors of a Hermitian quaternion tensor and the principal component analysis for the quaternion tensor. Moreover, we present the k-means method for quaternion tensor and use it to color video clustering. Finally, we conducted experiments to prove the effectiveness of the proposed theory and algorithms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-21 datetime="2022-11-21T21:45:00.000Z">2022-11-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-21 datetime="2022-11-21T21:45:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/scientists-unveil-functional-landscape-essential-genes-1121"
                        open
                      >
                        <summary class="article-expander__title">Scientists unveil the functional landscape of essential genes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/scientists-unveil-functional-landscape-essential-genes-1121">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202211/koch-gamma-screen.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Researchers harness new pooled, image-based screening method to probe the functions of over 5,000 essential genes in human cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;120730"
                        open
                      >
                        <summary class="article-expander__title">The Simple Geometry Behind Brownie Bake Offs and Equal Areas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/simple-geometry-shows-when-two-shapes-are-equal-20221121/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/11/NovemberAcademy-cr.RobertNeubecker-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Proving that two polygons have the same area can be as easy as cutting them up and rearranging the pieces.             
The post The Simple Geometry Behind Brownie Bake Offs and Equal Areas first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0116643?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">A data-driven approach for discovering the most probable transition pathway for a stochastic carbon cycle system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0116643?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Many natural systems exhibit tipping points where changing environmental conditions spark a sudden shift to a new and sometimes quite different state. Global climate change is often associated with the stability of marine carbon stocks. We consider a stochastic carbonate system of the upper ocean to capture such transition phenomena. Based on the OnsagerâMachlup action functional theory, we calculate the most probable transition pathway between the metastable and oscillatory states via a neural shooting method. Furthermore, we explore the effects of external random carbon input rates on the most probable transition pathway, which provides a basis to recognize naturally occurring tipping points. Particularly, we investigate the transition pathwayâs dependence on the transition time and further compute the optimal transition time using a physics-informed neural network, toward the maximum carbonate concentration state in the oscillatory regimes. This work may offer some insights into the effects of noise-affected carbon input rates on transition phenomena in stochastic models.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0089694?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamical stability indicator based on autoregressive moving-average models: Critical transitions and the Atlantic meridional overturning circulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0089694?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
A statistical indicator for dynamic stability, known as the [math] indicator, is used to gauge the stability and, hence, detect approaching tipping points of simulation data from a reduced five-box model of the North Atlantic Meridional Overturning Circulation (AMOC) exposed to a time-dependent hosing function. The hosing function simulates the influx of fresh water due to the melting of the Greenland ice sheet and increased precipitation in the North Atlantic. The [math] indicator is designed to detect changes in the memory properties of the dynamics and is based on fitting auto-regressive moving-average models in a sliding window approach to time series data. An increase in memory properties is interpreted as a sign of dynamical instability. The performance of the indicator is tested on time series subject to different types of tipping, namely, bifurcation-induced, noise-induced, and rate-induced tipping. The numerical analysis shows that the indicator indeed responds to the different types of induced instabilities. Finally, the indicator is applied to two AMOC time series from a full complexity Earth systems model (CESM2). Compared with the doubling [math] scenario, the quadrupling [math] scenario results in stronger dynamical instability of the AMOC during its weakening phase.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100595?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Transition from inhomogeneous limit cycles to oscillation death in nonlinear oscillators with similarity-dependent coupling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100595?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We consider a system of coupled nonlinear oscillators in which the interaction is modulated by a measure of the similarity between the oscillators. Such a coupling is common in treating spatially mobile dynamical systems where the interaction is distance dependent or in resonance-enhanced interactions, for instance. For a system of StuartâLandau oscillators coupled in this manner, we observe a novel route to oscillation death via a Hopf bifurcation. The individual oscillators are confined to inhomogeneous limit cycles initially and are damped to different fixed points after the bifurcation. Analytical and numerical results are presented for this case, while numerical results are presented for coupled RÃ¶ssler and Sprott oscillators.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.18.517014v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Accurate Computational Design of 3D Protein Crystals [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.18.517014v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein crystallization plays a central role in structural biology1, with broad impact2 in pharmaceutical formulation3, drug delivery4, biosensing5, and biocatalysis6,7. Despite this importance, the process of protein crystallization remains poorly understood and highly empirical8-10, with largely unpredictable crystal contacts, lattice packing arrangements, and space group preferences, and the programming of protein crystallization through precisely engineered sidechain-sidechain interactions across multiple protein-protein interfaces is an outstanding challenge. Here we develop a general computational approach to designing three-dimensional (3D) protein crystals with pre-specified lattice architectures at atomic accuracy that hierarchically constrains the overall degree of freedoms (DOFs) of the system. We use the approach to design three pairs of oligomers that can be individually purified, and upon mixing, spontaneously self-assemble into large 3D crystals (&gt;100 m). Small-angle X-ray scattering and X-ray crystallography show these crystals are nearly identical to the computational design models, with the design target F4132 and I432 space groups and closely corresponding overall architectures and protein-protein interfaces. The crystal unit cell dimensions can be systematically redesigned while retaining space group symmetry and overall architecture, and the crystals are both extremely porous and highly stable, enabling the robust scaffolding of inorganic nanoparticle arrays. Our approach thus enables the computational design of protein crystals with high accuracy, and since both structure and assembly are encoded in the primary sequence, provides a powerful new platform for biological material engineering.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.19.517184v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Cell-free biosynthesis combined with deep learning accelerates de novo-development of antimicrobial peptides [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.19.517184v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Bioactive peptides are key molecules in health and medicine. Deep learning holds a big promise for the discovery and design of bioactive peptides. Yet, suitable experimental approaches are required to validate candidates in high throughput and at low cost. Here, we established a cell-free protein synthesis (CFPS) pipeline for the rapid and inexpensive production of antimicrobial peptides (AMPs) directly from DNA templates. To validate our platform, we used deep learning to design thousands of AMPs de novo. Using computational methods, we prioritized 500 candidates that we produced and screened with our CFPS pipeline. We identified 30 functional AMPs, which we characterized further through molecular dynamics simulations, antimicrobial activity and toxicity. Notably, six de novo-AMPs feature broad-spectrum activity against multidrug-resistant pathogens and do not develop bacterial resistance. Our work demonstrates the potential of CFPS for production and testing of bioactive peptides within less than 24 hours and &lt;10$ per screen.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.11.18.517127v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Recombinant expression and characterisation of a lipase from the Antarctic zooplankton Salpa thompsoni [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.11.18.517127v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cold marine environments are abundant on earth and represent a rich resource for low temperature enzymes. Here we apply in silico bioprospecting methods followed by in vitro expression and biochemical analyses to characterise a novel low temperature lipase from the Antarctic tunicate Salpa thompsoni. A 586 amino acid pancreatic lipase-like gene was identified from S. thompsoni transcriptomic data, expressed as a hexahistadine fusion protein in Escherichia coli at 10{degrees}C and purified by affinity chromatography. Hydrolysis of the synthetic substrate {rho}-nitrophenyl butyrate (PNPB) showed that this recombinant protein has optimal activity at 20 {degrees}C and pH 7, and a specific activity of 3.16 U/mg under this condition. Over 60% of enzyme activity was maintained between 15 to 25 {degrees}C, with a sharp decrease outside this range. These results are indicative of cold active psychrophilic enzyme activity. A meta-analysis of lipase activities towards PNPB showed that the novel S. thompsoni lipase displays a higher activity at lower temperatures relative to previously characterised enzymes. The work demonstrates a methodology for conversion of transcriptomic to in vitro expression data for the discovery of new cold-active biocatalysts from marine organisms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01531-8"
                        open
                      >
                        <summary class="article-expander__title">High-throughput retrieval of target sequences from complex clone libraries using CRISPRi</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01531-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01531-8/MediaObjects/41587_2022_1531_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A CRISPRi method retrieves sequences of interest from large metagenomic libraries.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-18 datetime="2022-11-18T08:00:00.000Z">2022-11-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-18 datetime="2022-11-18T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M146380X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Trapping of Planar Brownian Motion: Full First Passage Time Distributions by Kinetic Monte Carlo, Asymptotic, and Boundary Integral Methods</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M146380X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 4, Page 1284-1314, December 2022. 
 Abstract. We consider the problem of determining the arrival statistics of unbiased planar random walkers to complex target configurations. In contrast to problems posed in finite domains, simple moments of the first passage time distribution, such as its mean and variance, are not defined. Therefore, it is necessary to obtain the full arrival statistics. We describe several methods to obtain these distributions and other associated quantities such as splitting probabilities. One approach combines a Laplace transform of the underlying parabolic equation with matched asymptotic analysis followed by numerical transform inversion. The second approach is similar but uses a boundary integral equation method to solve for the transformed variable. To validate the results of this theory, and to obtain the arrival time statistics in very general configurations of absorbers, we introduce an efficient kinetic Monte Carlo (KMC) method that describes trajectories as a combination of large but exactly solvable projection steps. The effectiveness of these methodologies is demonstrated on a variety of challenging examples highlighting the applicability of these methods to a variety of practical scenarios, such as source inference. A particularly useful finding arising from these results is that homogenization theories, in which complex configurations are replaced by equivalent simple ones, are remarkably effective at describing arrival time statistics.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1440384?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Modeling Spatial Waves of Wolbachia Invasion for Controlling Mosquito-Borne Diseases</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1440384?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 1903-1929, December 2022. 
 Abstract. Wolbachia is a natural bacterium that can infect mosquitoes and reduce their ability to transmit mosquito-borne diseases, such as dengue fever, Zika, and chikungunya. Field trials and modeling studies have shown that the fraction of infection among the mosquitoes must exceed a threshold level for the infection to persist. To capture this threshold, it is critical to consider the spatial heterogeneity in the distributions of the infected and uninfected mosquitoes created by a local release of the infected mosquitoes. We develop and analyze PDE models to study the invasion dynamics of Wolbachia infection among mosquitoes in the field. Our reaction-diffusion-type models account for the complex vertical transmission and the spatial mosquito dispersion. We characterize the threshold for a successful invasion with a bubble-shaped distribution, called the âcritical bubble.â The critical bubble is optimal in its release size compared to other spatial profiles in a one-dimensional landscape. The fraction of infection near the release center is higher than the threshold level for the corresponding homogeneously mixing ODE models. We show that the proposed spatial models give rise to the traveling waves of Wolbachia infection when above the threshold. We quantify how the threshold condition and traveling-wave velocity depend on the diffusion coefficients and other model parameters. Numerical studies for different scenarios are presented to inform the design of release strategies.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1462830?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Global Dynamics of a Filippov SIQR Model with Delayed Relay Control</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1462830?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 6, Page 1879-1902, December 2022. 
 Abstract. Global dynamics of a Filippov susceptible, infectious, quarantine, and recovered model subject to delayed relay control are investigated in this work. The global stability of equilibria is obtained by means of the generalized Lyapunov method. In addition, combining the PoincarÃ© maps, the displacement function, and a Melnikov-like method, we derive some sufficient conditions to ensure the existence, uniqueness, and global stability of a slowly oscillating periodic solution. Numerical simulations are performed to explore the effectiveness of the quarantine control measure and the effects of the delay.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00311"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Ribosome Stalling of NâLinked Glycoproteins in Cell-Free Extracts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00311">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00311</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100897?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Chaotic synchronization of mutually coupled systemsâarbitrary proportional linear relations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100897?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Considering a system combining two generalized Boolean transformations, we found that depending on the parameters, we can generate generalized synchronization such that the two chaotic orbits have arbitrary proportional linear relations. We rigorously determined its synchronization conditions by the explicit computing conditional Lyapunov exponent using the ergodic property and stable property of the Cauchy distribution. We found that a phenomenon similar to chaotic synchronization occurs even when the synchronization conditions are not strictly satisfied, which exhibits some degree of structural stability of chaotic synchronization. Our model can be further extended to systems with more degrees of freedom and, in the future, can be applied to reservoir computing.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01216-y"
                        open
                      >
                        <summary class="article-expander__title">Popping at the poles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01216-y">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Chemical Biology - Popping at the poles</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01214-0"
                        open
                      >
                        <summary class="article-expander__title">Crowd control</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01214-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01214-0/MediaObjects/41589_2022_1214_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nature Chemical Biology - Crowd control</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01203-3"
                        open
                      >
                        <summary class="article-expander__title">Synthetic protein condensates for cellular and metabolic engineering</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01203-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01203-3/MediaObjects/41589_2022_1203_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein condensates are subcellular structures that enrich and confine molecules in cells. This Review details how condensates can be engineered with responsiveness and on-demand functions, thus pushing cellular and metabolic engineering to a new level.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-17 datetime="2022-11-17T15:22:14.000Z">2022-11-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-17 datetime="2022-11-17T15:22:14.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;120649"
                        open
                      >
                        <summary class="article-expander__title">Why This Universe? A New Calculation Suggests Our Cosmos Is Typical.</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/why-this-universe-new-calculation-suggests-our-cosmos-is-typical-20221117/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/11/CosmicEntropy-cr.KouzouSakai-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Two physicists have calculated that the universe has a higher entropy â and is therefore more likely â than alternative possible universes.            
The post Why This Universe? A New Calculation Suggests Our Cosmos Is Typical. first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34647-1"
                        open
                      >
                        <summary class="article-expander__title">Feedforward growth rate control mitigates gene activation burden</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34647-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34647-1/MediaObjects/41467_2022_34647_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Heterologous gene activation causes non-physiological burden on cellular resources that cells are unable to adjust to. Here the authors present a tunable, modular, and portable feedforward controller that allows dynamic modulation of a genes expression to possibly high-levels without substantially affecting growth rate.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34850-0"
                        open
                      >
                        <summary class="article-expander__title">Identification of purine biosynthesis as an NADH-sensing pathway to mediate energy stress</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34850-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34850-0/MediaObjects/41467_2022_34850_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Reductive stress, reflected by the elevated intracellular NADH/NAD+ ratio, is associated with multiple human diseases. Here, the authors develop a genetic tool to manipulate the ratios of cellular NADH/NAD+ and NADPH/NADP+, and identify purine biosynthesis as an NADH-sensing pathway to mediate reductive stress.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;48
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-16 datetime="2022-11-16T15:40:45.000Z">2022-11-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-16 datetime="2022-11-16T15:40:45.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;120604"
                        open
                      >
                        <summary class="article-expander__title">Computer Proof âBlows Upâ Centuries-Old Fluid Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/computer-helps-prove-long-sought-fluid-equation-singularity-20221116/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/11/3D-Euler-Blowup-cr.DVDP-Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>For more than 250 years, mathematicians have wondered if the Euler equations might sometimes fail to describe a fluidâs flow. A new computer-assisted proof marks a major breakthrough in that quest.            
The post Computer Proof âBlows Upâ Centuries-Old Fluid Equations first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;27
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0378?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Multimodal Genotype and Phenotype Data Integration to Improve Partial Data-Based Longitudinal Prediction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0378?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00520"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A Logically Reversible Double Feynman Gate with Molecular Engineered Bacteria Arranged in an Artificial Neural Network-Type Architecture</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00520">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00520</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0091626?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The impact of inner-coupling and time delay on synchronization: From single-layer network to hypernetwork</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0091626?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Though synchronization of complex dynamical systems has been widely studied in the past few decades, few studies pay attention to the impact of network parameters on synchronization in hypernetworks. In this paper, we focus on a specific hypernetwork model consisting of coupled RÃ¶ssler oscillators and investigate the impact of inner-coupling and time delay on the synchronized region (SR). For the sake of simplicity, the inner-coupling matrix is chosen from three typical forms, which result in classical bounded, unbounded, and empty SR in a single-layer network, respectively. The impact of inner-couplings or time delays on unbounded SR is the most interesting one among the three types of SR. Once the SR of one subnetwork is unbounded, the SR of the whole hypernetwork is also unbounded with a different inner-coupling matrix. In a hypernetwork with unbounded SR, the time delays change not only the size but also the type of SR. In a hypernetwork with bounded or empty SR, the time delays have almost no effect on the type of SR.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0131962?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Mitchell Feigenbaum: His life and legacy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0131962?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34339-w"
                        open
                      >
                        <summary class="article-expander__title">XNAzymes targeting the SARS-CoV-2 genome inhibit viral infection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34339-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34339-w/MediaObjects/41467_2022_34339_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>RNA viruses have been responsible for large-scale epidemics and pandemics throughout the last few centuries. Here, the authors show the design, synthesis and screening of artificial RNA endonuclease XNAzymes capable of cleaving genomic SARS-CoV-2 RNA and self-assembling into enzymatic nanostructures inhibiting cellular viral replication.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01795-3"
                        open
                      >
                        <summary class="article-expander__title">Homeostasis and injectivity: a reaction network perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01795-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Homeostasis represents the idea that a feature may remain invariant despite changes in some external parameters. We establish a connection between homeostasis and injectivity for reaction network models. In particular, we show that a reaction network cannot exhibit homeostasis if a modified version of the network (which we call homeostasis-associated network) is injective. We provide examples of reaction networks which can or cannot exhibit homeostasis by analyzing the injectivity of their homeostasis-associated networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-15 datetime="2022-11-15T05:00:00.000Z">2022-11-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-15 datetime="2022-11-15T05:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00344"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Robotic Construction and Screening of Lanthipeptide Variant Libraries in Escherichia coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00344">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00344</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00498"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A BioBricks Metabolic Engineering Platform for the Biosynthesis of Anthracyclinones in Streptomyces coelicolor</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00498">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00498</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00405"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Expanding the Scope of Bacterial CRISPR Activation with PAM-Flexible dCas9 Variants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00405">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00405</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00380"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Improved Production of Fengycin in Bacillus subtilis by Integrated Strain Engineering Strategy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00380">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00380</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105154?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Derivation and analytical solutions of a non-linear diffusion equation applied to non-constant heat conductivity and ionic diffusion in glasses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105154?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This paper derives a non-linear diffusion equation discussing two possible applications: the ionic diffusion in glasses and temperature-dependent conductivity in semiconductors. The first equation is a logarithmic diffusion derived formally from the continuity of ion concentration, but the latter is a more phenomenological example. A power-law ansatz with time-dependent parameters maximizes a non-standard entropy and gives a set of coupled motion equations we can solve analytically. These results build the general solution to the non-linear diffusion equation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0104907?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Statistical dynamics of driven systems of confined interacting particles in the overdamped-motion regime</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0104907?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Systems consisting of confined, interacting particles doing overdamped motion admit an effective description in terms of nonlinear FokkerâPlanck equations. The behavior of these systems is closely related to the [math] power-law entropies and can be interpreted in terms of the [math]-based thermostatistics. The connection between overdamped systems and the [math] measures provides valuable insights on diverse physical problems, such as the dynamics of interacting vortices in type-II superconductors. The [math]-thermostatistical approach to the study of many-body systems described by nonlinear FokkerâPlanck equations has been intensively explored in recent years, but most of these efforts were restricted to systems affected by time-independent external potentials. Here, we extend this treatment to systems evolving under time-dependent external forces. We establish a lower bound on the work done by these forces when they drive the system during a transformation. The bound is expressed in terms of a free energy based on the [math] entropy and is satisfied even if the driving forces are not derivable from a potential function. It constitutes a generalization, for systems governed by nonlinear FokkerâPlanck equations involving general time-dependent external forces, of the [math]-theorem satisfied by these systems when the external forces arise from a time-independent potential.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41586-022-05548-6"
                        open
                      >
                        <summary class="article-expander__title">Publisher Correction: Potentiating adoptive cell therapy using synthetic IL-9 receptors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41586-022-05548-6">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature - Publisher Correction: Potentiating adoptive cell therapy using synthetic IL-9 receptors</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-14 datetime="2022-11-14T08:00:00.000Z">2022-11-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-14 datetime="2022-11-14T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0067?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Genome-Wide Association with Uncertainty in the Genetic Similarity Matrix</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0067?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0246?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Privacy-Preserving Statistical Analysis of Genomic Data Using Compressive Mechanism with Haar Wavelet Transform</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0246?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105770?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The effect of noise in an HIV infection model with cytotoxic T-lymphocyte impairment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105770?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
The human immunodeficiency virus (HIV) interacts with the immune cells within the human body, where the environment is uncertain and noisy. Stochastic models can successfully encapsulate the effect of such a noisy environment compared to their deterministic counterparts. The human immune system is complex but well-coordinated with various immune cells like [math]T cells, dendritic cells, and cytotoxic T-lymphocyte (CTL) cells, among many others. The CTL can kill the antigenic cells after its recognition. However, the efficacy of CTL in removing the infected [math]T cells is progressively compromised in HIV-infected individuals. This paper considers a noise-induced HIV-immune cell interaction model with immune impairment. A multiplicative white noise is introduced in the infection rate parameter to represent the fluctuations around the average value of the rate parameter as a causative effect of the noise. We analyzed the deterministic and stochastic models and prescribed sufficient conditions for infection eradication and persistence. It is determined under what parametric restrictions the asymptotic solutions of the noise-induced system will be a limiting case of the deterministic solutions. Simulation results revealed that the solutions of the deterministic system either converge to a CTL-dominated interior equilibrium or a CTL-free immunodeficient equilibrium, depending on the initial values of the system. Stochastic analysis divulged that higher noise might be helpful in the infection removal process. The extinction time of infected [math]T cells for some fixed immune impairment gradually decreases with increasing noise intensity and follows the power law.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0112982?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Periodic solutions of coupled Boussinesq equations and Ostrovsky-type models free from zero-mass contradiction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0112982?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Coupled Boussinesq equations are used to describe long weakly nonlinear longitudinal strain waves in a bi-layer with soft bonding between the layers (e.g., a soft adhesive). From a mathematical viewpoint, a particularly difficult case appears when the linear long-wave speeds in the layers are significantly different (high-contrast case). The traditional derivation of the uni-directional models leads to four uncoupled Ostrovsky equations for the right- and left-propagating waves in each layer. However, the models impose a âzero-mass constraintâ; i.e., the initial conditions should necessarily have zero mean, restricting the applicability of that description. Here, we bypass the contradiction in this high-contrast case by constructing the solution for the deviation from the evolving mean value, using asymptotic multiple-scale expansions involving two pairs of fast characteristic variables and two slow time variables. By construction, the Ostrovsky equations emerging within the scope of this derivation are solved for initial conditions with zero mean, while initial conditions for the original system may have non-zero mean values. Asymptotic validity of the solution is carefully examined numerically. We apply the models to the description of counter-propagating waves generated by solitary wave initial conditions, or co-propagating waves generated by cnoidal wave initial conditions, as well as the resulting wave interactions, and contrast with the behavior of the waves in bi-layers when the linear long-wave speeds in the layers are close (low-contrast case). One local (classical) and two non-local (generalized) conservation laws of the coupled Boussinesq equations for strains are derived and used to control the accuracy of the numerical simulations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01830-3"
                        open
                      >
                        <summary class="article-expander__title">Feasibility of sparse large Lotka-Volterra ecosystems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01830-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Consider a large ecosystem (foodweb) with n species, where the abundances follow a LotkaâVolterra system of coupled differential equations. We assume that each species interacts with 
                      
                      \(d&#x3D;d_n\)
                      
                     other species and that their interaction coefficients are independent random variables. This parameter d reflects the connectance of the foodweb and the sparsity of its interactions especially if d is much smaller that n. We address the question of feasibility of the foodweb, that is the existence of an equilibrium solution of the Lotka-Volterra system with no vanishing species. We establish that for a given range of d, namely 
                      
                      \(d\propto n\)
                â¦</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02047-8"
                        open
                      >
                        <summary class="article-expander__title">Total Generalized Variation for Triangulated Surface Data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02047-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
By defining spaces and differential operators on mesh surfaces, we extend the total generalized variation (
                      
                      \(\textrm{TGV}\)
                      
                    ) model on the 2-dimensional space to triangulated surfaces. Based on the new definition of 
                      
                      \(\textrm{TGV}\)
                      
                     model on triangulated surfaces, we introduce total generalized variation restoration optimization problem for data (image pixel/surface normal) restoration over triangulated surfaces. The optimization problem is solved effectively by augmented Lagrangian method (
                      
                      \(\textrm{ALM}\)
                      
                    ). Closed form solutions for subproblems of the 
                      
                      \(\textrm{ALM}\)
                      
                     method are obtained. Convergence analysis of the 
                      
                      \(\textrm{ALM}\)
                      
                     algorithm is presented. Through series of numerical experiments, we show that the 
                      
                      \(\textrm{TGV}\)
                      
                     method can alleviate the staircase effect and recover more structures and details. As a result, the 
                      
                      \(\textrm{TGV}\)
                      
                     model outperforms several existing models visually and quantitatively. The robustness of the 
                      
                      \(\textrm{TGV}\)
                      
                     method is also confirmed numerically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-12 datetime="2022-11-12T05:00:00.000Z">2022-11-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-12 datetime="2022-11-12T05:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00295"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Targeting Transcriptional and Translational Hindrances in a Modular T7RNAP Expression System in Engineered Pseudomonas putida</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00295">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00295</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-11 datetime="2022-11-11T08:00:00.000Z">2022-11-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-11 datetime="2022-11-11T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0262?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Accurate Binning of Metagenomic Contigs Using Composition, Coverage, and Assembly Graphs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0262?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00368"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] The De Novo Synthesis of 2âPhenylethanol from Glucose by the Synthetic Microbial Consortium Composed of Engineered Escherichia coli and Meyerozyma guilliermondii</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00368">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00368</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00423"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A Toolkit for Precise, Multigene Control in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00423">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00423</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00266"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Membrane Fusion Mediated by Non-covalent Binding of Re-engineered Cholera Toxin Assemblies to Glycolipids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00266">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00266</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02050-z"
                        open
                      >
                        <summary class="article-expander__title">Cut-Off Error Splitting Technique for Conservative Nonconforming VEM for N-Coupled Nonlinear SchrÃ¶dingerâBoussinesq Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02050-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this work, the error splitting technique combined with cut-off function method is designed to derive unconditionally optimal error estimates for a fully implicit conservative numerical method of the N-coupled nonlinear SchrÃ¶dingerâBoussinesq equations, which is constructed by an implicit CrankâNicolson-type method in time and new nonconforming virtual element methods in space. The numerical scheme is conservative in the senses of discrete mass and energy, and the cut-off error splitting technique is innovative to remove the standard time-step conditions 
                      
                      \(\tau &#x3D;o(h^{d/4})\)
                      
                     and 
                      
                      \(\tau &#x3D;o(h^{d/2})\)
                      
                    . Finally, several numerical examples are given to confirm our theoretical results. The analytical technique in this work could be used to study other implicit numerical methods of nonlinear physical models, including but not limited to conforming and nonconforming finite element methods/virtual element methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-10 datetime="2022-11-10T05:25:33.000Z">2022-11-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-10 datetime="2022-11-10T05:25:33.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093758?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Hypergraphon mean field games</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093758?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We propose an approach to modeling large-scale multi-agent dynamical systems allowing interactions among more than just pairs of agents using the theory of mean field games and the notion of hypergraphons, which are obtained as limits of large hypergraphs. To the best of our knowledge, ours is the first work on mean field games on hypergraphs. Together with an extension to a multi-layer setup, we obtain limiting descriptions for large systems of non-linear, weakly interacting dynamical agents. On the theoretical side, we prove the well-foundedness of the resulting hypergraphon mean field game, showing both existence and approximate Nash properties. On the applied side, we extend numerical and learning algorithms to compute the hypergraphon mean field equilibria. To verify our approach empirically, we consider a social rumor spreading model, where we give agents intrinsic motivation to spread rumors to unaware agents, and an epidemic control problem.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0111131?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Key role of neuronal diversity in structured reservoir computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0111131?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Chaotic time series have been captured by reservoir computing models composed of a recurrent neural network whose output weights are trained in a supervised manner. These models, however, are typically limited to randomly connected networks of homogeneous units. Here, we propose a new class of structured reservoir models that incorporates a diversity of cell types and their known connections. In a first version of the model, the reservoir was composed of mean-rate units separated into pyramidal, parvalbumin, and somatostatin cells. Stability analysis of this model revealed two distinct dynamical regimes, namely, (i) an inhibition-stabilized network (ISN) where strong recurrent excitation is balanced by strong inhibition and (ii) a non-ISN network with weak excitation. These results were extended to a leaky integrate-and-fire model that captured different cell types along with their network architecture. ISN and non-ISN reservoir networks were trained to relay and generate a chaotic Lorenz attractor. Despite their increased performance, ISN networks operate in a regime of activity near the limits of stability where external perturbations yield a rapid divergence in output. The proposed framework of structured reservoir computing opens avenues for exploring how neural microcircuits can balance performance and stability when representing time series through distinct dynamical regimes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00318"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Negatively Competitive Incoherent Feedforward Loops Mitigate Winner-Take-All Resource Competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00318">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00318</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34644-4"
                        open
                      >
                        <summary class="article-expander__title">Formation of synthetic RNA protein granules using engineered phage-coat-protein -RNA complexes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34644-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34644-4/MediaObjects/41467_2022_34644_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Condensates composed of RNA and proteins are biologically vital but generally poorly understood. Here, the authors engineer synthetic RNA-protein condensates and show that gel-like condensates form as a result of liquid-gel phase separation in a specific and selective fashion.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;52
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34361-y"
                        open
                      >
                        <summary class="article-expander__title">Integrase-mediated differentiation circuits improve evolutionary stability of burdensome and toxic functions in E. coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34361-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34361-y/MediaObjects/41467_2022_34361_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Improving evolution stability of engineered functions is important for bioproduction and synthetic biology. Here, the authors developed an integrase-recombination-based terminal differentiation gene circuit in E. coli to improve the evolutionary stability of engineered function in a general manner.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;52
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Nonlinear dynamics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/nonlinear-dynamics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-23633-8"
                        open
                      >
                        <summary class="article-expander__title">Nonlinear model of infection wavy oscillation of COVID-19 in Japan based on diffusion kinetics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-23633-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-23633-8/MediaObjects/41598_2022_23633_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Nonlinear model of infection wavy oscillation of COVID-19 in Japan based on diffusion kinetics</span>
                                &ensp;<span class="article-reading-time">(&hairsp;23
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01828-x"
                        open
                      >
                        <summary class="article-expander__title">Model selection for assessing the effects of doxorubicin on triple-negative breast cancer cell lines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01828-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Doxorubicin is a chemotherapy widely used to treat several types of cancer, including triple-negative breast cancer. In this work, we use a Bayesian framework to rigorously assess the ability of ten different mathematical models to describe the dynamics of four TNBC cell lines (SUM-149PT, MDA-MB-231, MDA-MB-453, and MDA-MB-468) in response to treatment with doxorubicin at concentrations ranging from 10 to 2500 nM. Each cell line was plated and serially imaged via fluorescence microscopy for 30 days following 6, 12, or 24Â h of in vitro drug exposure. We use the resulting data sets to estimate the parameters of the ten pharmacodynamic models using a Bayesian approach, which accounts for uncertainties in the models, parameters, and observational data. The ten candidate models describe the growth patterns and degree of response to doxorubicin for each cell line by incorporating exponential or logistic tumor growth, and distinct forms of cell death. Cell line and treatment specific model parameters are then estimated from the experimental data for each model. We analyze all competing models using the Bayesian Information Criterion (BIC), and the selection of the best model is made according to the model probabilities (BIC weights). We show that the best model among the candidate set of models depends on the TNBC cell line and the treatment scenario, though, in most cases, there is great uncertainty in choosing the best model. However, we show that the probability of being the best model can be increased by combining treatment data with the same total drug exposure. Our analysis points to the importance of considering multiple models, built on different biological assumptions, to capture the observed variations in tumor growth and treatment response.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02043-y"
                        open
                      >
                        <summary class="article-expander__title">A Div FOSLS Method Suitable for Quadrilateral RT and Hexahedral RTN $$H(\textrm{div})$$ -elements</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02043-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A new div FOSLS mixed finite element method is proposed and analyzed for the first-order system of the general second-order elliptic problems in terms of the scalar variable displacement and the vectorial variable flux. The main feature of the proposed method is to apply a local constant element or (bi,tri)linear element 
                      
                      \(L^2\)
                      
                     projection to the div term, together with a mesh-dependent div term. The method is coercive and symmetric, allowing any combination of conforming approximations for both scalar and vectorial variables. More importantly, the method is suitable for general nonaffine quadrilateral RaviartâThomas (RT) and nonaffine hexahedral RaviartâThomasâNÃ©dÃ©lec (RTN) 
                â¦</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-09 datetime="2022-11-09T02:09:37.000Z">2022-11-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-09 datetime="2022-11-09T02:09:37.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0103436?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The three-dimensional generalized HÃ©non map: Bifurcations and attractors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0103436?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We study dynamics of a generic quadratic diffeomorphism, a 3D generalization of the planar HÃ©non map. Focusing on the dissipative, orientation preserving case, we give a comprehensive parameter study of codimension-one and two bifurcations. Periodic orbits, born at resonant, NeimarkâSacker bifurcations, give rise to Arnold tongues in parameter space. Aperiodic attractors include invariant circles and chaotic orbits; these are distinguished by rotation number and Lyapunov exponents. Chaotic orbits include HÃ©non-like and Lorenz-like attractors, which can arise from period-doubling cascades, and those born from the destruction of invariant circles. The latter lie on paraboloids near the local unstable manifold of a fixed point.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0126848?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Change point detection in multi-agent systems based on higher-order features</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0126848?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Change point detection (CPD) for multi-agent systems helps one to evaluate the state and better control the system. Multivariate CPD methods solve the [math] time series well; however, the multi-agent systems often produce the [math] dimensional data, where [math] is the dimension of multivariate observations, [math] is the total observation time, and [math] is the number of agents. In this paper, we propose two valid approaches based on higher-order features, namely, the Betti number feature extraction and the Persistence feature extraction, to compress the [math]-dimensional features into one dimension so that general CPD methods can be applied to higher-dimensional data. First, a topological structure based on the VietorisâRips complex is constructed on each time-slice snapshot. Then, the Betti number and persistence of the topological structures are obtained to separately constitute two feature matrices for change point estimates. Higher-order features primarily describe the data distribution on each snapshot and are, therefore, independent of the node correspondence cross snapshots, which gives our methods unique advantages in processing missing data. Experiments in multi-agent systems demonstrate the significant performance of our methods. We believe that our methods not only provide a new tool for dimensionality reduction and missing data in multi-agent systems but also have the potential to be applied to a wider range of fields, such as complex networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0106943?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Identifying partial topology of simplicial complexes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0106943?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This paper investigates partial topology identification of simplicial complexes based on adaptive synchronization. For the nodes of interest, the interactions that they participate in can be accurately reconstructed by designing adaptive controllers and parameter estimators. Particularly, not only pairwise interactions but a higher-order structure can be effectively recovered by our method. Moreover, a new linear independence condition with a rigorous definition is established for parameter estimators to converge asymptotically to the true values. Numerical simulations on a general two-dimensional simplicial complex as well as a real-world structure are provided to show the validity of the result and discuss the influence of different parameters on the identification process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02037-w"
                        open
                      >
                        <summary class="article-expander__title">Nonlinear p-Multigrid Preconditioner for Implicit Time Integration of Compressible NavierâStokes Equations with p-Adaptive Flux Reconstruction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02037-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Within the framework of p-adaptive flux reconstruction, we aim to construct efficient polynomial multigrid (pMG) preconditioners for implicit time integration of the NavierâStokes equations using Jacobian-free NewtonâKrylov (JFNK) methods. We hypothesise that in pseudo transient continuation (PTC), as the residual drops, the frequency of error modes that dictates the convergence rate gets higher and higher. We apply nonlinear pMG solvers to stiff steady problems at low Mach number (
                      
                      \(\textrm{Ma}&#x3D;10^{-3}\)
                      
                    ) to verify our hypothesis. It is demonstrated that once the residual drops by a few orders of magnitude, improved smoothing on intermediate p-sublevels will not only maintain the stability oâ¦</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02023-2"
                        open
                      >
                        <summary class="article-expander__title">A Time-Continuous Embedding Method for Scalar Hyperbolic Conservation Laws on Manifolds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02023-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A time-continuous (tc-)embedding method is first proposed for solving nonlinear scalar hyperbolic conservation laws with discontinuous solutions (shocks and rarefaction waves) on codimension 1, connected, smooth, and closed manifolds (surface PDEs or SPDEs in 
                      
                      \({\mathbb {R}}^2\)
                      
                     and 
                      
                      \({\mathbb {R}}^3\)
                      
                    ). The new embedding method improves upon the classical closest point (cp-)embedding method, which requires re-establishments of the constant-along-normal (CAN-)property of the extension function at every time step, in terms of accuracy and efficiency, by incorporating the CAN-property analytically and explicitly in the embedding equation. The tc-embedding SPDEs are solved by the second-order nonlinear central finite volume scheme with a nonlinear minmod slope limiter in space, and the third-order total variation diminished Runge-Kutta scheme in time. An adaptive nonlinear essentially non-oscillatory polynomial interpolation is used to obtain the solution values at the ghost cells. Numerical results in solving the linear wave equation and the Burgersâ equation show that the proposed tc-embedding method has better accuracy, improved resolution, and reduced CPU times than the classical cp-embedding method. The Burgersâ equation, the traffic flow problem, and the Buckley-Leverett equation are solved to demonstrate the robust performance of the tc-embedding method in resolving fine-scale structures efficiently even in the presence of a shock and the essentially non-oscillatory capturing of shocks and rarefaction waves on simple and complex shaped one-dimensional manifolds. Burgersâ equation is also solved on the two-dimensional torus-shaped and spherical-shaped manifolds.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02028-x"
                        open
                      >
                        <summary class="article-expander__title">Exact and Numerical Solutions of the Riemann Problem for a Conservative Model of Compressible Two-Phase Flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02028-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this work we study the solution of the Riemann problem for the barotropic version of the conservative symmetric hyperbolic and thermodynamically compatible (SHTC) two-phase flow model introduced in Romenski et al. (J Sci Comput 42(1):68, 2009, Quart Appl Math 65(2):259â279, 2007). All characteristic fields are carefully studied and explicit expressions are derived for the Riemann invariants and the RankineâHugoniot conditions. Due to the presence of multiple characteristics in the system under consideration, non-standard wave phenomena can occur. Therefore we briefly review admissibility conditions for discontinuities and then discuss possible wave interactions. In particular we will show that overlapping rarefaction waves are possible and moreover we may have shocks that lie inside a rarefaction wave. In contrast to nonconservative two phase flow models, such as the BaerâNunziato system, we can use the advantage of the conservative form of the model under consideration. Furthermore, we show the relation between the considered conservative SHTC system and the corresponding barotropic version of the nonconservative BaerâNunziato model. Additionally, we derive the reduced four equation Kapila system for the case of instantaneous relaxation, which is the common limit system of both, the conservative SHTC model and the non-conservative BaerâNunziato model. Finally, we compare exact solutions of the Riemann problem with numerical results obtained for the conservative two-phase flow model under consideration, for the non-conservative BaerâNunziato system and for the Kapila limit. The examples underline the previous analysis of the different wave phenomena, as well as differences and similarities of the three systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;74
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02042-z"
                        open
                      >
                        <summary class="article-expander__title">On an Eigenvalue Property of Summation-By-Parts Operators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02042-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Summation-By-Parts (SBP) methods provide a systematic way of constructing provably stable numerical schemes. However, many proofs of convergence and accuracy rely on the assumption that the SBP operator possesses a particular eigenvalue property. In this note, three results pertaining to this property are proven. Firstly, the eigenvalue property does not hold for all nullspace consistent SBP operators. Secondly, this issue can be addressed without affecting the accuracy of the method by adding a specially designed, arbitrarily small perturbation term to the SBP operator. Thirdly, all pseudospectral methods satisfy the eigenvalue property.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02015-2"
                        open
                      >
                        <summary class="article-expander__title">Solving Parametric Partial Differential Equations with Deep Rectified Quadratic Unit Neural Networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02015-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Implementing deep neural networks for learning the solution maps of parametric partial differential equations (PDEs) turns out to be more efficient than using many conventional numerical methods. However, limited theoretical analyses have been conducted on this approach. In this study, we investigate the expressive power of deep rectified quadratic unit (ReQU) neural networks for approximating the solution maps of parametric PDEs. The proposed approach is motivated by the recent important work of Kutyniok et al. (Constr Approx 1â53, 2021), which uses deep rectified linear unit (ReLU) neural networks for solving parametric PDEs. In contrast to the previously established complexity-bound 
                      
                      \(\mathcal {O}\left( d^3\log _{2}^{q}(1/ \epsilon ) \right) \)
                      
                     for ReLU neural networks, we derive an upper bound 
                      
                      \(\mathcal {O}\left( d^3\log _{2}^{q}\log _{2}(1/ \epsilon ) \right) \)
                      
                     on the size of the deep ReQU neural network required to achieve accuracy 
                      
                      \(\epsilon &gt;0\)
                      
                    , where d is the dimension of reduced basis representing the solutions. Our method takes full advantage of the inherent low-dimensionality of the solution manifolds and better approximation performance of deep ReQU neural networks. Numerical experiments are performed to verify our theoretical result.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-08 datetime="2022-11-08T08:00:00.000Z">2022-11-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-08 datetime="2022-11-08T08:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.29075.ip?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Special Issue: RECOMB 2022, Part 1</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.29075.ip?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1155-1155, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0212?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Quartet Based Gene Tree Imputation Using Deep Learning Improves Phylogenomic Analyses Despite Missing Data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0212?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1156-1172, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0265?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Inconsistency of Triplet-Based and Quartet-Based Species Tree Estimation under Intralocus Recombination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0265?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1173-1197, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0264?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Multimodal Single-Cell Translation and Alignment with Semi-Supervised Learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0264?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1198-1212, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0270?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">SCOTv2: Single-Cell Multiomic Alignment with Disproportionate Cell-Type Representation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0270?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1213-1228, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0251?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">UNIFAN: A Tool for Unsupervised Single-Cell Clustering and Annotation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0251?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1229-1232, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0244?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">The scINSIGHT Package for Integrating Single-Cell RNA-Seq Data from Different Biological Conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0244?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1233-1236, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0266?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Algorithms for Colinear Chaining with Overlaps and Gap Costs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0266?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1237-1251, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0257?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Efficient Minimum Flow Decomposition via Integer Linear Programming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0257?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 11, Page 1252-1267, November 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102787?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Global Mittag-Leffler synchronization of coupled delayed fractional reaction-diffusion CohenâGrossberg neural networks via sliding mode control</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102787?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This paper studies the sliding mode control method for coupled delayed fractional reaction-diffusion CohenâGrossberg neural networks on a directed non-strongly connected topology. A novel fractional integral sliding mode surface and the corresponding control law are designed to realize global Mittag-Leffler synchronization. The sufficient conditions for synchronization and reachability of the sliding mode surface are derived via the hierarchical method and the Lyapunov method. Finally, simulations are provided to verify our theoretical findings.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0106171?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Exact finite-dimensional reduction for a population of noisy oscillators and its link to OttâAntonsen and WatanabeâStrogatz theories</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0106171?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Populations of globally coupled phase oscillators are described in the thermodynamic limit by kinetic equations for the distribution densities or, equivalently, by infinite hierarchies of equations for the order parameters. Ott and Antonsen [Chaos 18, 037113 (2008)] have found an invariant finite-dimensional subspace on which the dynamics is described by one complex variable per population. For oscillators with Cauchy distributed frequencies or for those driven by Cauchy white noise, this subspace is weakly stable and, thus, describes the asymptotic dynamics. Here, we report on an exact finite-dimensional reduction of the dynamics outside of the OttâAntonsen subspace. We show that the evolution from generic initial states can be reduced to that of three complex variables, plus a constant function. For identical noise-free oscillators, this reduction corresponds to the WatanabeâStrogatz system of equations [Watanabe and Strogatz, Phys. Rev. Lett. 70, 2391 (1993)]. We discuss how the reduced system can be used to explore the transient dynamics of perturbed ensembles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098536?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Spectro-spatial analysis of elastic wave propagation in nonlinear elastic metamaterial systems with damping</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098536?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Different from the linear counterpart, elastic wave propagation in nonlinear elastic metamaterials with damping (EMD) systems has much more novel behaviors. It is little work that reveals the nonlinear wave propagation mechanism in the nonlinear EMD in the literature. In this paper, nonlinear EMD systems with different configurations for the nonlinear elements are constructed to study the nonlinear wave propagation characteristics based on the analytical formulation. Nonlinear wave propagation attenuation characteristics in nonlinear EMD systems are studied in the frequency-domain. Spectro-spatial characteristics of nonlinear traveling wave packet in space- and time-domain are also analyzed to reveal space frequency-domain characteristics. Influences of different configurations for the nonlinear elements on wave propagation in nonlinear EMD systems are discussed. Results shown that locations of band structures can be tuned by changing intensities of nonlinearities in the nonlinear EMD systems, which are not found in the linear system. Moreover, different configurations for the nonlinear elements can lead to localization feature emerged in a nonlinear EMD system, which indicates solitary waves causing from the interaction between dispersion and nonlinearity exist in the nonlinear systems. This interesting wave propagating properties can be used to design new devices for acoustic switches, energy harvesting, and broadband vibration control.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0104610?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Intermittent regimes as a synchronization phenomenon in two sets of nonlinear chemical oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0104610?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Systems of nonlinear chemical oscillators can exhibit a large diversity of non-trivial states depending on the parameters that characterize them. Among these, a synchronization phenomenon is of special interest due to its direct link with chemical and biological processes in nature. We carry out numerical experiments for two different sets of chemical oscillators with different properties and immersed in a BelousovâZhabotinsky solution. We document the emergence of different states of synchronization that depend on the parameters characterizing the solution. We also show that, in the interface regions, this system generates a stable dynamics of intermittency between the different synchronization states where interesting phenomena, such as the âdevil&#x27;s staircase,â emerge. In general, the added complexity introduced with the additional set of oscillators results in more complex non-trivial synchronization states.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0124123?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Transition behavior of the seizure dynamics modulated by the astrocyte inositol triphosphate noise</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0124123?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Epilepsy is a neurological disorder with recurrent seizures, which convey complex dynamical characteristics including chaos and randomness. Until now, the underlying mechanism has not been fully elucidated, especially the bistable property beneath the epileptic random induction phenomena in certain conditions. Inspired by the recent finding that astrocyte GTPase-activating protein (G-protein)-coupled receptors could be involved in stochastic epileptic seizures, we proposed a neuronâastrocyte network model, incorporating the noise of the astrocytic second messenger, inositol triphosphate (IP3) that is modulated by G-protein-coupled receptor activation. Based on this model, we have statistically analyzed the transitions of epileptic seizures by performing repeatable simulation trials. Our simulation results show that the increase in the IP3 noise intensity induces depolarization-block epileptic seizures together with an increase in neuronal firing frequency, consistent with corresponding experiments. Meanwhile, the bistable states of the seizure dynamics were present under certain noise intensities, during which the neuronal firing pattern switches between regular sparse spiking and epileptic seizure states. This random presence of epileptic seizures is absent when the noise intensity continues to increase, accompanying with an increase in the epileptic depolarization block duration. The simulation results also shed light on the fact that calcium signals in astrocytes play significant roles in the pattern formations of the epileptic seizure. Our results provide a potential pathway for understanding the epileptic randomness in certain conditions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0123896?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Inertial motion on the earthâs spheroidal surface</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0123896?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
As seen by an observer in the rotating frame, the earthâs small spheroidal deformations neutralize the centrifugal force, leaving only the smaller Coriolis force to govern the âinertialâ motion of objects that move on its surface, assumed smooth and frictionless. Previous studies of inertial motion employ weakly spheroidal equations of motion that ignore the influence of the centrifugal force and yet treat the earth as a sphere. The latitude dependence of these equations renders them strongly nonlinear. We derive and justify these equations and use them to identify, classify, name, describe, and illustrate all possible classes of inertial motion, including a new class of motion called circumpolar waves, which encircle both poles during each cycle of the motion. We illustrate these classes using CorioVis, our freely available Coriolis visualization software. We identify a rotational/time-reversal symmetry for motion on the earthâs surface and use this symmetry to develop and validate closed-form small-amplitude approximations for the four main classes and one degenerate class of inertial motion. For these five classes, we supply calculations of experimentally relevant frequencies, zonal drifts, and latitude ranges.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01992-8"
                        open
                      >
                        <summary class="article-expander__title">Heterogeneous Multiscale Methods for the LandauâLifshitz Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01992-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we present a finite difference Heterogeneous Multiscale Method for the LandauâLifshitz equation with a highly oscillatory diffusion coefficient. The approach combines a higher order discretization and artificial damping in the so-called micro problem to obtain an efficient implementation. The influence of different parameters on the resulting approximation error is discussed. Further important factors that are taken into account are the choice of time integrator and the initial data for the micro problem which has to be set appropriately to get a consistent scheme. Numerical examples in one and two space dimensions and for both periodic as well as more general coefficients are given to demonstrate the functionality of the approach.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;54
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02040-1"
                        open
                      >
                        <summary class="article-expander__title">Active Learning for Saddle Point Calculation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02040-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The saddle point (SP) calculation is a grand challenge for computationally intensive energy function in computational chemistry area, where the saddle point may represent the transition state. The traditional methods need to evaluate the gradients of the energy function at a very large number of locations. To reduce the number of expensive computations of the true gradients, we propose an active learning framework consisting of a statistical surrogate model, Gaussian process regression (GPR) for the energy function, and a single-walker dynamics method, gentle accent dynamics (GAD), for the saddle-type transition states. SP is detected by the GAD applied to the GPR surrogate for the gradient vector and the Hessian matrix. Our key ingredient for efficiency improvements is an active learning method which sequentially designs the most informative locations and takes evaluations of the original model at these locations to train GPR. We formulate this active learning task as the optimal experimental design problem and propose a very efficient sample-based sub-optimal criterion to construct the optimal locations. We show that the new method significantly decreases the required number of energy or force evaluations of the original model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02029-w"
                        open
                      >
                        <summary class="article-expander__title">Robust Approximation of Generalized Biot-Brinkman Problems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02029-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The generalized Biot-Brinkman equations describe the displacement, pressures and fluxes in an elastic medium permeated by multiple viscous fluid networks and can be used to study complex poromechanical interactions in geophysics, biophysics and other engineering sciences. These equations extend on the Biot and multiple-network poroelasticity equations on the one hand and Brinkman flow models on the other hand, and as such embody a range of singular perturbation problems in realistic parameter regimes. In this paper, we introduce, theoretically analyze and numerically investigate a class of three-field finite element formulations of the generalized Biot-Brinkman equations. By introducing appropriate norms, we demonstrate that the proposed finite element discretization, as well as an associated preconditioning strategy, is robust with respect to the relevant parameter regimes. The theoretical analysis is complemented by numerical examples.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-07 datetime="2022-11-07T00:00:00.000Z">2022-11-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-07 datetime="2022-11-07T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34190-z"
                        open
                      >
                        <summary class="article-expander__title">Circuit-guided population acclimation of a synthetic microbial consortium for improved biochemical production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34190-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34190-z/MediaObjects/41467_2022_34190_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Achieving stable production by co-culturing multiple microorganisms is often challenging due to difficulties in controlling its population. Here, the authors develop a âpopulation guiderâ that can acclimate a population to higher production.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01062-4"
                        open
                      >
                        <summary class="article-expander__title">Photoswitchable gating of non-equilibrium enzymatic feedback in chemically communicating polymersome nanoreactors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01062-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01062-4/MediaObjects/41557_2022_1062_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The circadian rhythm generates out-of-equilibrium metabolite oscillations controlled by feedback loops under light/dark cycles. Now, it has been shown that these life-like properties can emerge from a non-equilibrium nanosystem comprising a binary population of enzyme-containing polymersomes capable of light-gated chemical communication, controllable feedback and coupling to macroscopic oscillations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;49
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-05 datetime="2022-11-05T00:00:00.000Z">2022-11-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-05 datetime="2022-11-05T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34390-7"
                        open
                      >
                        <summary class="article-expander__title">Inducible plasmid copy number control for synthetic biology in commonly used E. coli strains</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34390-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34390-7/MediaObjects/41467_2022_34390_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The ability to externally control gene expression has been important for all areas of biological research, especially for synthetic biology. Here the authors present plasmid TULIP which offers DNA copy number control via chemical induction to accelerate the design, prototyping, and reuse of gene circuits in diverse contexts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-04 datetime="2022-11-04T01:51:02.000Z">2022-11-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-04 datetime="2022-11-04T01:51:02.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0094009?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Graphop mean-field limits and synchronization for the stochastic Kuramoto model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0094009?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Models of coupled oscillator networks play an important role in describing collective synchronization dynamics in biological and technological systems. The Kuramoto model describes oscillatorâs phase evolution and explains the transition from incoherent to coherent oscillations under simplifying assumptions, including all-to-all coupling with uniform strength. Real world networks, however, often display heterogeneous connectivity and coupling weights that influence the critical threshold for this transition. We formulate a general mean-field theory (VlasovâFocker Planck equation) for stochastic Kuramoto-type phase oscillator models, valid for coupling graphs/networks with heterogeneous connectivity and coupling strengths, using graphop theory in the mean-field limit. Considering symmetric odd-valued coupling functions, we mathematically prove an exact formula for the critical threshold for the incoherenceâcoherence transition. We numerically test the predicted threshold using large finite-size representations of the network model. For a large class of graph models, we find that the numerical tests agree very well with the predicted threshold obtained from mean-field theory. However, the prediction is more difficult in practice for graph structures that are sufficiently sparse. Our findings open future research avenues toward a deeper understanding of mean-field theories for heterogeneous systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105654?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Complexity and disequilibrium in the dipole-type Hamiltonian mean-field model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105654?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This research studies information properties, such as complexity and disequilibrium, in the dipole-type Hamiltonian mean-field model. A fundamental analytical assessment is the partition function in the canonical ensemble to derive statistical, thermodynamical, and information measures. They are also analytical, dependent on the number of particles, consistent with the theory for high temperatures, and rising some limitations at shallow temperatures, giving us a notion of the classicality of the system defining an interval of temperatures where the model is well working.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0118706?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Statistical properties of the entropy from ordinal patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0118706?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
The ultimate purpose of the statistical analysis of ordinal patterns is to characterize the distribution of the features they induce. In particular, knowing the joint distribution of the pair entropy-statistical complexity for a large class of time series models would allow statistical tests that are unavailable to date. Working in this direction, we characterize the asymptotic distribution of the empirical Shannonâs entropy for any model under which the true normalized entropy is neither zero nor one. We obtain the asymptotic distribution from the central limit theorem (assuming large time series), the multivariate delta method, and a third-order correction of its mean value. We discuss the applicability of other results (exact, first-, and second-order corrections) regarding their accuracy and numerical stability. Within a general framework for building test statistics about Shannonâs entropy, we present a bilateral test that verifies if there is enough evidence to reject the hypothesis that two signals produce ordinal patterns with the same Shannonâs entropy. We applied this bilateral test to the daily maximum temperature time series from three cities (Dublin, Edinburgh, and Miami) and obtained sensible results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0109451?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Social norms of fairness with reputation-based role assignment in the dictator game</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0109451?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
A vast body of experiments share the view that social norms are major factors for the emergence of fairness in a population of individuals playing the dictator game (DG). Recently, to explore which social norms are conducive to sustaining cooperation has obtained considerable concern. However, thus, far few studies have investigated how social norms influence the evolution of fairness by means of indirect reciprocity. In this study, we propose an indirect reciprocal model of the DG and consider that an individual can be assigned as the dictator due to its good reputation. We investigate the âleading eightâ norms and all second-order social norms by a two-timescale theoretical analysis. We show that when role assignment is based on reputation, four of the âleading eightâ norms, including stern judging and simple standing, lead to a high level of fairness, which increases with the selection intensity. Our work also reveals that not only the correct treatment of making a fair split with good recipients but also distinguishing unjustified unfair split from justified unfair split matters in elevating the level of fairness.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093980?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">A new scenario for Braessâs paradox in power grids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093980?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We consider several topologies of power grids and analyze how the addition of transmission lines affects their dynamics. The main example we are dealing with is a power grid that has a tree-like three-element motif at the periphery. We establish conditions where the addition of a transmission line in the motif enhances its stability or induces Braessâs paradox and reduces stability of the entire grid. By using bifurcation theory and nonlocal stability analysis, we show that two scenarios for Braessâs paradox are realized in the grid. The first scenario is well described and is associated with the disappearance of the synchronous mode. The second scenario has not been previously described and is associated with the reduction of nonlocal stability of the synchronous mode due to the appearance of asynchronous modes. The necessary conditions for stable operation of the grid, under the addition of a line, are derived. It is proved that the new scenario for Braessâs paradox is realized in the grids with more complex topologies even when several lines are added in their bulks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0125873?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Coupled spreading between information and epidemics on multiplex networks with simplicial complexes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0125873?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
The way of information diffusion among individuals can be quite complicated, and it is not only limited to one type of communication, but also impacted by multiple channels. Meanwhile, it is easier for an agent to accept an idea once the proportion of their friends who take it goes beyond a specific threshold. Furthermore, in social networks, some higher-order structures, such as simplicial complexes and hypergraph, can describe more abundant and realistic phenomena. Therefore, based on the classical multiplex network model coupling the infectious disease with its relevant information, we propose a novel epidemic model, in which the lower layer represents the physical contact network depicting the epidemic dissemination, while the upper layer stands for the online social network picturing the diffusion of information. In particular, the upper layer is generated by random simplicial complexes, among which the herd-like threshold model is adopted to characterize the information diffusion, and the unawareâawareâunaware model is also considered simultaneously. Using the microscopic Markov chain approach, we analyze the epidemic threshold of the proposed epidemic model and further check the results with numerous Monte Carlo simulations. It is discovered that the threshold model based on the random simplicial complexes network may still cause abrupt transitions on the epidemic threshold. It is also found that simplicial complexes may greatly influence the epidemic size at a steady state.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0122898?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Linear response theory for renewable fluctuations in power grids with transmission losses</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0122898?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We study the spreading of renewable power fluctuations through grids with Ohmic losses on the lines. By formulating a network-adapted linear response theory, we find that vulnerability patterns are linked to the left Laplacian eigenvectors of the overdamped eigenmodes. We show that for tree-like networks, fluctuations are amplified in the opposite direction of the power flow. This novel mechanism explains vulnerability patterns that were observed in previous numerical simulations of renewable microgrids. While exact mathematical derivations are only possible for tree-like networks with a homogeneous response, we show that the mechanisms discovered also explain vulnerability patterns in realistic heterogeneous meshed grids by studying the IEEE RTS-1996 test system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0118046?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Analytical predictions of stable and unstable firings to chaos in a HindmarshâRose neuron system</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0118046?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
In this paper, analytical predictions of the firing cascades formed by stable and unstable firings in a HindmarshâRose (HR) neuron system are completed via an implicit mapping method. The semi-analytical firing cascades present the route from periodic firings to chaos. For such cascades, the continuous firing flow of the nonlinear neuron system is discretized to form a special mapping structure for nonlinear firing activities. Stability and bifurcation analysis of the nonlinear firings are performed based on resultant eigenvalues of the global mapping structures. Stable and unstable firing solutions in the bifurcation tree exhibit clear period-doubling firing cascades toward chaos. Bifurcations are predicted accurately on the connections. Phase bifurcation trees are observed, which provide deep cognitions of neuronal firings. Harmonic dynamics of the period-doubling firing cascades are obtained and discussed for a better understanding of the contribution of the harmonics in frequency domains. The route into chaos is illustrated by the firing chains from period-1 to period-16 firings and verified by numerical solutions. The applied methods and obtained results provide new perspectives to the complex firing dynamics of the HR neuron system and present a potential strategy to regulate the firings of neurons.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101401?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Exploring bifurcations in BoseâEinstein condensates via phase field crystal models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101401?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
To facilitate the analysis of pattern formation and the related phase transitions in BoseâEinstein condensates, we present an explicit approximate mapping from the nonlocal GrossâPitaevskii equation with cubic nonlinearity to a phase field crystal (PFC) model. This approximation is valid close to the superfluidâsupersolid phase transition boundary. The simplified PFC model permits the exploration of bifurcations and phase transitions via numerical path continuation employing standard software. While revealing the detailed structure of the bifurcations present in the system, we demonstrate the existence of localized states in the PFC approximation. Finally, we discuss how higher-order nonlinearities change the structure of the bifurcation diagram representing the transitions found in the system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0090095?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Pinning control of networks: Dimensionality reduction through simultaneous block-diagonalization of matrices</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0090095?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
In this paper, we study the network pinning control problem in the presence of two different types of coupling: (i) node-to-node coupling among the network nodes and (ii) input-to-node coupling from the source node to the âpinned nodes.â Previous work has mainly focused on the case that (i) and (ii) are of the same type. We decouple the stability analysis of the target synchronous solution into subproblems of the lowest dimension by using the techniques of simultaneous block diagonalization of matrices. Interestingly, we obtain two different types of blocks, driven and undriven. The overall dimension of the driven blocks is equal to the dimension of an appropriately defined controllable subspace, while all the remaining undriven blocks are scalar. Our main result is a decomposition of the stability problem into four independent sets of equations, which we call quotient controllable, quotient uncontrollable, redundant controllable, and redundant uncontrollable. Our analysis shows that the number and location of the pinned nodes affect the number and the dimension of each set of equations. We also observe that in a large variety of complex networks, the stability of the target synchronous solution is de facto only determined by a single quotient controllable block.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01820-5"
                        open
                      >
                        <summary class="article-expander__title">Evolution of a structured cell population endowed with plasticity of traits under constraints on and between the traits</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01820-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Confronted with the biological problem of managing plasticity in cell populations, which is in particular responsible for transient and reversible drug resistance in cancer, we propose a rationale consisting of an integro-differential and a reaction-advection-diffusion equation, the properties of which are studied theoretically and numerically. By using a constructive finite volume method, we show the existence and uniqueness of a weak solution and illustrate by numerical approximations and their simulations the capacity of the model to exhibit divergence of traits. This feature may be theoretically interpreted as describing a physiological step towards multicellularity in animal evolution and, closer to present-day clinical challenges in oncology, as a possible representation of bet hedging in cancer cell populations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-03 datetime="2022-11-03T04:00:00.000Z">2022-11-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-03 datetime="2022-11-03T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00252"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A New Strategy for Increasing Knock-in Efficiency: Multiple Elongase and Desaturase Transgenes Knock-in by Targeting Long Repeated Sequences</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00252">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00252</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34232-6"
                        open
                      >
                        <summary class="article-expander__title">Absolute protein quantification using fluorescence measurements with FPCountR</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34232-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34232-6/MediaObjects/41467_2022_34232_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A challenge in synthetic biology is the empirical characterisation of genetic parts. Here the authors present FPCountR, a validated method and accompanying R package that enables the precise quantification of fluorescent protein reporters per bacterial cell to be enumerated in âproteins per cellâ or nanomolar units without requiring protein purification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01065-1"
                        open
                      >
                        <summary class="article-expander__title">Post-translational backbone-acyl shift yields natural product-like peptides bearing hydroxyhydrocarbon units</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01065-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01065-1/MediaObjects/41557_2022_1065_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Despite recent advances in engineering of in vitro translation systems, direct ribosomal incorporation of hydroxyhydrocarbon moietiesâwhich can endow peptides with unique biochemical/folding propertiesâremains challenging. Now, incorporation of translation-compatible azide/hydroxy acids and their post-translational tandem backbone-acyl shifts have enabled in vitro ribosomal synthesis of peptides containing various hydroxyhydrocarbon units.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-02 datetime="2022-11-02T07:00:00.000Z">2022-11-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-02 datetime="2022-11-02T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0185?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">ApoE Modifier Alleles for Alzheimer&#x27;s Disease Discovered by Information Theory Dependency Measures: MIST Software Package</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0185?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0394?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Excerno: Using Mutational Signatures in Sequencing Data to Filter False Variants Caused by Clinical Archival</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0394?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0117902?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Giant oscillations of diffusion in ac-driven periodic systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0117902?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We revisit the problem of diffusion in a driven system consisting of an inertial Brownian particle moving in a symmetric periodic potential and subjected to a symmetric time-periodic force. We reveal parameter domains in which diffusion is normal in the long time limit and exhibits intriguing giant damped quasiperiodic oscillations as a function of the external driving amplitude. As the mechanism behind this effect, we identify the corresponding oscillations of difference in the number of locked and running trajectories that carry the leading contribution to the diffusion coefficient. Our findings can be verified experimentally in a multitude of physical systems, including colloidal particles, Josephson junction, or cold atoms dwelling in optical lattices, to name only a few.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0115101?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Entropy profiles of Schellingâs segregation model from the WangâLandau algorithm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0115101?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We evaluate Schellingâs segregation outcomes from the square lattice, regular random networks, and clustered regular random networks by situating them in the probability distribution of the entire outcome space of satisfaction and segregation. To do so, we employ the WangâLandau algorithm and calculate the entropy and the number of states as a function of satisfaction and segregation. According to the results, satisfaction tends to increase with segregation, irrespective of the network structure. Moreover, segregation occurs almost surely when satisfaction is maximized, which we also algebraically derive and confirm on infinite-size networks. The average ratios of the neighbors of the same tag are about 67% for the square lattice and regular random networks and about 73% for clustered regular random networks with a clustering coefficient of 0.37(1). Thus, clustering increases segregation on regular random networks. Further, we find that Schellingâs path-dependent process generates sharper segregation than expected by random configurations, an outcome symptomatic of over-optimization from the social welfare perspective. Adopting an alternative rule restricting individualsâ choice set may reduce segregation without compromising on satisfaction.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0089326?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Linear stability analysis and nonlinear simulations of convective dissolution in an inclined porous layer between impermeable surfaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0089326?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Convective dissolution can occur in porous media when a given solute dissolves in a host layer from above and increases the density of the host solution. Buoyancy-driven fingering can then develop, which increases the transfer flux of the solute. We investigate here numerically the properties of this convective dissolution when the porous host layer is inclined by an angle [math] relative to the horizontal direction. We consider an incompressible flow in porous media governed by Darcyâs law, driven by density gradients associated with the concentration of the dissolving solute. The model problem focuses on the case of a very long (infinite) tilted porous layer limited by two parallel impermeable surfaces. A linear stability analysis and nonlinear simulations are performed using the Boussinesq approximation. A vorticity-stream function formulation is adopted to solve the two-dimensional hydrodynamic field through the finite element method. We find that the inclination of the interface decreases the growth rate of the instability and the range of unstable wavenumbers, delaying or even suppressing the onset of the fingering instability. Moreover, it introduces a drift velocity on the perturbations, which is characterized here in both the linear stability analysis and the nonlinear simulations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0090518?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">New 4D and 3D models of chaotic systems developed from the dynamic behavior of nuclear reactors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0090518?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
The complex, highly nonlinear dynamic behavior of nuclear reactors can be captured qualitatively by novel four-dimensional (that is, fourth order) and three-dimensional (that is, third order) models of chaotic systems and analyzed with Lyapunov spectra, bifurcation diagrams, and phase diagrams. The chaotic systems exhibit a rich variety of bifurcation phenomena, including the periodic-doubling route to chaos, reverse bifurcations, anti-monotonicity, and merging chaos. The offset boosting method, which relocates the attractorâs basin of attraction in any direction, is demonstrated in these chaotic systems. Both constant parameters and periodic functions are seen in offset boosting phenomena, yielding chaotic attractors with controlled mean values and coexisting attractors.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0112907?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">An attackâdefense game model in infrastructure networks under link hiding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0112907?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Our increasing dependence on infrastructure networks leads to growing concerns over the protection of these networks. Many methods have been proposed to select protective strategies by combining complex network theory and game theory. However, the misleading effect of hidden links is not considered in previous methods. This work creates an information gap between attackers and defenders by partly hiding network links to mislead the attacker in the game. We first introduce the rule of link hiding that depends on the nodesâ property, where the number of hidden links has a maximum value. Additionally, based on the Stackelberg game model, we establish an attackâdefense game model with link hiding strategies considering node property and cost constraints. Finally, we conduct experiments in a scale-free network and an existing power grid. The experimental results show that the defender tends to combine first-mover advantage and link hiding to get a better payoff under more different costs of the nodes. Hiding half of the links in the existing power grid can effectively reduce network damage by about [math]% on average, with the two sides investing the same resources. The effect of link hiding could be more obvious when the attacker owns more resources than the defender. When an attacker employs the high-degree attacking strategy, the proposed link hiding method can help the defender reduce the damage to the network by 12.2% compared to the link reconnecting method.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0119642?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Invariant tori in dissipative hyperchaos</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0119642?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
One approach to understand the chaotic dynamics of nonlinear dissipative systems is the study of non-chaotic yet dynamically unstable invariant solutions embedded in the systemâs chaotic attractor. The significance of zero-dimensional unstable fixed points and one-dimensional unstable periodic orbits capturing time-periodic dynamics is widely accepted for high-dimensional chaotic systems, including fluid turbulence, while higher-dimensional invariant tori representing quasiperiodic dynamics have rarely been considered. We demonstrate that unstable 2-tori are generically embedded in the hyperchaotic attractor of a dissipative system of ordinary differential equations; tori can be numerically identified via bifurcations of unstable periodic orbits and their parameteric continuation and characterization of stability properties are feasible. As higher-dimensional tori are expected to be structurally unstable, 2-tori together with periodic orbits and equilibria form a complete set of relevant invariant solutions on which to base a dynamical description of chaos.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0116784?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Learning unseen coexisting attractors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0116784?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
Reservoir computing is a machine learning approach that can generate a surrogate model of a dynamical system. It can learn the underlying dynamical system using fewer trainable parameters and, hence, smaller training data sets than competing approaches. Recently, a simpler formulation, known as next-generation reservoir computing, removed many algorithm metaparameters and identified a well-performing traditional reservoir computer, thus simplifying training even further. Here, we study a particularly challenging problem of learning a dynamical system that has both disparate time scales and multiple co-existing dynamical states (attractors). We compare the next-generation and traditional reservoir computer using metrics quantifying the geometry of the ground-truth and forecasted attractors. For the studied four-dimensional system, the next-generation reservoir computing approach uses [math] less training data, requires [math] shorter âwarmupâ time, has fewer metaparameters, and has an [math] higher accuracy in predicting the co-existing attractor characteristics in comparison to a traditional reservoir computer. Furthermore, we demonstrate that it predicts the basin of attraction with high accuracy. This work lends further support to the superior learning ability of this new machine learning algorithm for dynamical systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0113335?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Permutation group entropy: A new route to complexity for real-valued processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0113335?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://aip.scitation.org/action/showOpenGraphArticleImage?doi&#x3D;10.1063/5.0113335&amp;id&#x3D;images/medium/5.0113335.figures.online.highlight_f1.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This is a review of group entropy and its application to permutation complexity. Specifically, we revisit a new approach to the notion of complexity in the time series analysis based on both permutation entropy and group entropy. As a result, the permutation entropy rate can be extended from deterministic dynamics to random processes. More generally, our approach provides a unified framework to discuss chaotic and random behaviors.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0109889?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Revealing recurrent regimes of mid-latitude atmospheric variability using novel machine learning method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0109889?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
The low-frequency variability of the extratropical atmosphere involves hemispheric-scale recurring, often persistent, states known as teleconnection patterns or regimes, which can have a profound impact on predictability on intra-seasonal and longer timescales. However, reliable data-driven identification and dynamical representation of such states are still challenging problems in modeling the dynamics of the atmosphere. We present a new method, which allows us both to detect recurring regimes of atmospheric variability and to obtain dynamical variables serving as an embedding for these regimes. The method combines two approaches from nonlinear data analysis: partitioning a network of recurrent states with studying its properties by the recurrence quantification analysis and the kernel principal component analysis. We apply the method to study teleconnection patterns in a quasi-geostrophical model of atmospheric circulation over the extratropical hemisphere as well as to reanalysis data of geopotential height anomalies in the mid-latitudes of the Northern Hemisphere atmosphere in the winter seasons from 1981 to the present. It is shown that the detected regimes as well as the obtained set of dynamical variables explain large-scale weather patterns, which are associated, in particular, with severe winters over Eurasia and North America. The method presented opens prospects for improving empirical modeling and long-term forecasting of large-scale atmospheric circulation regimes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0095939?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Study of low-dimensional nonlinear fractional difference equations of complex order</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0095939?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
We study the fractional maps of complex order, [math], for [math] and [math] in one and two dimensions. In two dimensions, we study HÃ©non, Duffing, and Lozi maps, and in [math], we study logistic, tent, Gauss, circle, and Bernoulli maps. The generalization in [math] can be done in two different ways, which are not equivalent for fractional order and lead to different bifurcation diagrams. We observed that the smooth maps, such as logistic, Gauss, Duffing, and HÃ©non maps, do not show chaos, while discontinuous maps, such as Bernoulli and circle maps,show chaos. The tent and Lozi map are continuous but not differentiable, and they show chaos as well. In [math], we find that the complex fractional-order maps that show chaos also show multistability. Thus, it can be inferred that the smooth maps of complex fractional order tend to show more regular behavior than the discontinuous or non-differentiable maps.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101667?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Cupolets in a chaotic neuron model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101667?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This paper reports the first finding of cupolets in a chaotic HindmarshâRose neural model. Cupolets (chaotic, unstable, periodic, orbit-lets) are unstable periodic orbits that have been stabilized through a particular control scheme by applying a binary control sequence. We demonstrate different neural dynamics (periodic or chaotic) of the HindmarshâRose model through a bifurcation diagram where the external input current, [math], is the bifurcation parameter. We select a region in the chaotic parameter space and provide the results of numerical simulations. In this chosen parameter space, a control scheme is applied when the trajectory intersects with either of the two control planes. The type of the control is determined by a bit in a binary control sequence. The control is either a small microcontrol (0) or a large macrocontrol (1) that adjusts the future dynamics of the trajectory by a perturbation determined by the coding function [math]. We report the discovery of many cupolets with corresponding control sequences and comment on the differences with previously reported cupolets in the double scroll system. We provide some examples of the generated cupolets and conclude by discussing potential implications for biological neurons.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00315"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Logistic Regression-Guided Identification of Cofactor Specificity-Contributing Residues in Enzyme with Sequence Datasets Partitioned by Catalytic Properties</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00315">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00315</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34141-8"
                        open
                      >
                        <summary class="article-expander__title">speedingCARs: accelerating the engineering of CAR T cells by signaling domain shuffling and single-cell sequencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34141-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34141-8/MediaObjects/41467_2022_34141_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chimeric antigen receptors (CAR) are a promising option for cell-based immunotherapy for cancer and other immune diseases. Here the authors develop speedingCARs, an integrated CAR design and screening platform based on modular signaling domain shuffling and single cell transcriptomic analyses, and test its potential for identifying and validating novel CAR designs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;52
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41586-022-05356-y"
                        open
                      >
                        <summary class="article-expander__title">Real-time bioelectronic sensing of environmental contaminants</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41586-022-05356-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-022-05356-y/MediaObjects/41586_2022_5356_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The combination of synthetic biology and materials engineering enabled the development of biosensors that produce electrical readouts and real-time detection capabilities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-11-01 datetime="2022-11-01T13:00:00.000Z">2022-11-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-11-01 datetime="2022-11-01T13:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/synthetic-gene-expression-control-1101"
                        open
                      >
                        <summary class="article-expander__title">A new control system for synthetic genes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/synthetic-gene-expression-control-1101">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202211/MIT-Programming-Cells-01-PRESS.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Researchers have developed a technique that could help fine-tune the production of monoclonal antibodies and other useful proteins.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1403989?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Statistical Learning of Nonlinear Stochastic Differential Equations from Nonstationary Time Series using Variational Clustering</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1403989?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 4, Page 1251-1283, December 2022. 
 Abstract. Data-driven stochastic parameterization methods use observational data to support and improve existing prediction systems. Specifically in atmospheric sciences, uncertainty in observations is a challenge for modeling because several aspects of the physical processes are yet to be understood in some contexts. A key goal in constructing atmospheric models is to express unresolved processes using the resolved variables of a prediction system. Estimating parameterisations for highly nonlinear systems with non stationary data requires application-specific tools. Previous work on methods for analyzing nonstationary data presents a flexible model-based nonparametric clustering methodology that can be exploited for parameterization development. This paper adapts the existing framework for stochastic parameterization by combining the continuous-time formulation of stochastic differential equations (SDE) with the clustering method. We use a closed-form likelihood function approach based on a suitable Hermite expansion to approximate the parameter values of the SDE with arbitrary nonlinear drift and nonlinear diffusion. The novel parameterization framework provides a smooth classification function that allows us to recover the underlying temporal parameter modulation of a nonstationary one-dimensional SDE. The numerical examples show that the clustering approach recovers a hidden functional relationship between the parameters of the SDE model and an additional auxiliary process. The study builds upon this functional relationship to develop closed-form, nonstationary, data-driven stochastic models for multiscale dynamical systems in real-world applications.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0120822?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dual-domain analysis of gun violence incidents in the United States</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0120822?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 11, November 2022. 
This paper applies new and recently introduced approaches to study trends in gun violence in the United States. We use techniques in both the time and frequency domain to provide a more complete understanding of gun violence dynamics. We analyze gun violence incidents on a state-by-state basis as recorded by the Gun Violence Archive. We have numerous specific phenomena of focus, including periodicity of incidents, locations in time where behavioral changes occur, and shifts in gun violence patterns since April 2020. First, we implement a recently introduced method of spectral density estimation for nonstationary time series to investigate periodicity on a state-by-state basis, including revealing where periodic behaviors change with time. We can also classify different patterns of behavioral changes among the states. We then aim to understand the most significant shifts in gun violence since numerous key events in 2020, including the COVID-19 pandemic, lockdowns, and periods of civil unrest. Our dual-domain analysis provides a more thorough understanding and challenges numerous widely held conceptions regarding the prevalence of gun violence incidents.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-31 datetime="2022-10-31T07:00:00.000Z">2022-10-31</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-31 datetime="2022-10-31T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1466566?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Generic Convergence to Periodic Orbits for Monotone Flows with Respect to 2-Cones and Applications to SEIRS Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1466566?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1829-1850, December 2022. 
 Abstract. We use a general epidemic model, the SEIRS model with a general nonlinear incidence rate, to show how the recently established generic PoincarÃ©âBendixson theory of flows monotone with respect to a cone of rank 2 can be used to establish the generic convergence to periodic orbits. Our approach to establishing generic convergence to periodic solutions applies to high-dimensional epidemic systems and predator-prey systems, very much like the generic convergence to equilibria in cooperative systems using the classical monotone dynamical systems theory.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1469171?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Supervised Optimal Transport</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1469171?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1851-1877, December 2022. 
 Abstract. Optimal transport, a theory for optimal allocation of resources, is widely used in various fields such as astrophysics, machine learning, and imaging science. However, many applications impose elementwise constraints on the transport plan which traditional optimal transport cannot enforce. Here we introduce supervised optimal transport (sOT), which formulates a constrained optimal transport problem where couplings between certain elements are prohibited according to specific applications. sOT is proved to be equivalent to an [math] penalized optimization problem, from which efficient algorithms are designed to solve its entropy regularized formulation. We demonstrate the capability of sOT by comparing it to other variants and extensions of traditional optimal transport in the color transfer problem. We also study the barycenter problem in sOT formulation, where we discover and prove a unique reverse and portion selection (control) mechanism. Supervised optimal transport is broadly applicable to applications in which a constrained transport plan is involved and the original unit should be preserved by avoiding normalization.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M1310321?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Corrigendum: Explicit Construction of a Small Epsilon-Net for Linear Threshold Functions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M1310321?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 5, Page 1692-1702, October 2022. 
 Abstract. The purpose of this note is to correct mistakes and inaccuracies in technical claims in [Y. Rabani and A. Shpilka, SIAM J. Comput., 39 (2010), pp. 3501â3520]. These have no effect on the main results in the paper.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1469753?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">A [math]-Competitive Algorithm for Scheduling Packets with Deadlines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1469753?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 5, Page 1626-1691, October 2022. 
 Abstract. In the online packet scheduling problem with deadlines ([math], for short), the goal is to schedule transmissions of packets that arrive over time in a network switch and need to be sent across a link. Each packet has a deadline, representing its urgency, and a nonnegative weight, which represents its priority. Only one packet can be transmitted in any time slot, so if the system is overloaded, some packets will inevitably miss their deadlines and be dropped. In this scenario, the natural objective is to compute a transmission schedule that maximizes the total weight of packets that are successfully transmitted. The problem is inherently online, with the scheduling decisions made without the knowledge of future packet arrivals. The central problem concerning [math] that has been a subject of intensive study since 2001 is to determine the optimal competitive ratio of online algorithms, namely the worst-case ratio between the optimum total weight of a schedule (computed by an offline algorithm) and the weight of a schedule computed by a (deterministic) online algorithm. We solve this open problem by presenting a [math]-competitive online algorithm for [math] (where [math] is the golden ratio), matching the previously established lower bound.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41477-022-01284-x"
                        open
                      >
                        <summary class="article-expander__title">Plastids as efficient AMP factories</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41477-022-01284-x">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Plants - Plastids as efficient AMP factories</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41419-022-05355-x"
                        open
                      >
                        <summary class="article-expander__title">Thioredoxin-interacting protein deficiency protects against severe acute pancreatitis by suppressing apoptosis signal-regulating kinase 1</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41419-022-05355-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41419-022-05355-x/MediaObjects/41419_2022_5355_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cell Death &amp; Disease - Thioredoxin-interacting protein deficiency protects against severe acute pancreatitis by suppressing apoptosis signal-regulating kinase 1</span>
                                &ensp;<span class="article-reading-time">(&hairsp;41
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01822-3"
                        open
                      >
                        <summary class="article-expander__title">An epidemic model with short-lived mixing groups</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01822-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Almost all epidemic models make the assumption that infection is driven by the interaction between pairs of individuals, one of whom is infectious and the other of whom is susceptible. However, in society individuals mix in groups of varying sizes, at varying times, allowing one or more infectives to be in close contact with one or more susceptible individuals at a given point in time. In this paper we study the effect of mixing groups beyond pairs on the transmission of an infectious disease in an SIR (susceptible 
                      
                      \(\rightarrow \)
                      
                     infective 
                      
                      \(\rightarrow \)
                      
                     recovered) model, both through a branching process approximation for the initial stages of an epidemic with few initial infectives and a functional central limit theorem for the trajectories of the numbers of infectives and susceptibles over time for epidemics with many initial infectives. We also derive central limit theorems for the final size of (i) an epidemic with many initial infectives and (ii) a major outbreak triggered by few initial infectives. We show that, for a given basic reproduction number 
                      
                      \(R_0\)
                      
                    , the distribution of the size of mixing groups has a significant impact on the probability and final size of a major epidemic outbreak. Moreover, the standard pair-based homogeneously mixing epidemic model is shown to represent the worst case scenario, with both the highest probability and the largest final size of a major epidemic.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;54
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-28 datetime="2022-10-28T07:00:00.000Z">2022-10-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-28 datetime="2022-10-28T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1449427?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">NonReciprocal Wave Propagation in Space-Time Modulated Media</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1449427?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 4, Page 1228-1250, December 2022. 
 Abstract. We show the possibility of achieving nonreciprocal wave propagation in space-time modulated media and give an asymptotic analysis of the nonreciprocity property in terms of the amplitude of the time modulation. Such modulation causes a folding of the band structure of the material, which may induce degenerate points. By breaking time-reversal symmetry, we show that these degeneracies may open into nonsymmetric, unidirectional band gaps. Finally, we illustrate our results by several numerical simulations.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M136551X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Diameter, Eccentricities and Distance Oracle Computations on H-Minor Free Graphs and Graphs of Bounded (Distance) VapnikâChervonenkis Dimension</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M136551X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 5, Page 1506-1534, October 2022. 
 Abstract. Under the strong exponential-time hypothesis, the diameter of general unweighted graphs cannot be computed in truly subquadratic time (in the size [math] of the input), as shown by Roditty and Williams. Nevertheless there are several graph classes for which this can be done such as bounded-treewidth graphs, interval graphs, and planar graphs, to name a few. We propose to study unweighted graphs of constant distance VapnikâChervonenkis (VC)-dimension as a broad generalization of many such classesâwhere the distance VC-dimension of a graph [math] is defined as the VC-dimension of its ball hypergraph whose hyperedges are the balls of all possible radii and centers in [math]. In particular for any fixed [mâ¦</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M1375048?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Proximity Search for Maximal Subgraph Enumeration</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M1375048?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 5, Page 1580-1625, October 2022. 
 Abstract. This paper proposes a new general technique for maximal subgraph enumeration which we call proximity search, whose aim is to design efficient enumeration algorithms for problems that could not be solved by existing frameworks. To support this claim and illustrate the technique we include output-polynomial algorithms for several problems for which output-polynomial algorithms were not known, including the enumeration of maximal bipartite subgraphs, maximal [math]-degenerate subgraphs (for bounded [math]), maximal induced chordal subgraphs, and maximal induced trees. Using known techniques, such as reverse search, the space of all maximal solutions induces an implicit directed graph called âsolution graphâ or âsupergraph,â and solutions are enumerated by traversing it; however, nodes in this graph can have exponential out-degree, thus requiring exponential time to be spent on each solution. The novelty of proximity search is a formalization that allows us to define a better solution graph, and a technique, which we call canonical reconstruction, by which we can exploit the properties of given problems to build such graphs. This results in solution graphs whose nodes have significantly smaller (i.e., polynomial) out-degree with respect to existing approaches, but that remain strongly connected, so that all solutions can be enumerated in polynomial delay by a traversal. A drawback of this approach is the space required to keep track of visited solutions, which can be exponential; we further propose a technique to induce a parent-child relationship among solutions and achieve polynomial space when suitable conditions are met.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1440864?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">On the Computability of Continuous Maximum Entropy Distributions with Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1440864?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 5, Page 1451-1505, October 2022. 
 Abstract. We study the following problem: Given a continuous domain [math] along with its convex hull [math], a point [math], and a measure [math] on [math], find the probability density over [math] whose marginal is [math] and that minimizes the KL divergence to the uniform density with respect to [math]. Several distributions in mathematics, physics, statistics, and theoretical computer science arise by different settings of the parameters of this problem. We give a polynomial bound on the norm of the optimizer of the dual problem that holds in a very general setting and relies on a âbalanceâ property of the measure [math] on [math], and exact algorithms for evaluating the dual and its gradient for several interesting settings of [math] and [math]. Together, along with the ellipsoid method, these results imply polynomial-time algorithms to compute such KL divergence minimizing distributions in several cases. Applications of our results include (1) an optimization characterization of the GoemansâWilliamson measure [M. X. Goemans and D. P. Williamson, J ACM, 42 (1995), pp. 1115â1145] that is used to round a positive semidefinite matrix to a vector; (2) the computability of the entropic barrier for convex bodies, given a strong integration oracle, studied by [S. Bubeck and R. Eldan, Proc. Mach. Learn. Res. (PMLR), 40 (2015), p. 279], and (3) a polynomial-time algorithm to compute the barycentric quantum entropy of a density matrix that was proposed as an alternative to von Neumann entropy [W. Band and J. L. Park, Found. Phys., 6 (1976), pp. 249â262; J. L. Park and W. Band, Found. Phys., 7 (1977), pp. 233â244; P. B. Slater, Phys. Lett. A, 159 (1991), pp. 411â414]; this corresponds to the case when [math] is the set of rank-one projection matrices and [math] is derived from the Haar measure on the unit sphere.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1456364?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Tight Revenue Gaps among Multiunit Mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1456364?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjcat&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Computing, Volume 51, Issue 5, Page 1535-1579, October 2022. 
 Abstract. This paper considers Bayesian revenue maximization in the [math]-unit setting, where a monopolist seller has [math] copies of an indivisible item and faces [math] unit-demand buyers (whose value distributions can be nonidentical). Four basic mechanisms among others have been widely employed in practice and widely studied in the literature: Myerson auction, sequential posted-pricing, [math]-th price auction with anonymous reserve, and anonymous pricing. Regarding a pair of mechanisms, we investigate the largest possible ratio between the two revenues (also known as the revenue gap), over all possible value distributions of the buyers. Divide these four mechanisms into two groups: (i) the discriminating mechanism group, Myerson auction and sequential posted-pricing, and (ii) the anonymous mechanism group, anonymous reserve and anonymous pricing. Within one group, the involved two mechanisms have an asymptotically tight revenue gap of [math]. In contrast, any two mechanisms from the different groups have an asymptotically tight revenue gap of [math].</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/angela-koehler-drug-targets-1028"
                        open
                      >
                        <summary class="article-expander__title">Angela Koehler takes on the most challenging drug targets</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/angela-koehler-drug-targets-1028">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202210/MIT-Angela_Koehler-01-press.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using biological, chemical, and engineering tools, she has developed strategies to attack molecules once thought to be âundruggable.â</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01818-z"
                        open
                      >
                        <summary class="article-expander__title">Bifurcation analysis for a single population model with advection</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01818-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, the dynamics of a single population model with a general growth function is investigated in an advective environment. We show the existence of a nonconstant positive steady state, and give sufficient conditions for the occurrence of a Hopf bifurcation at the positive steady state. Moreover, the theoretical results are applied to the diffusive Nicholsonâs blowflies and MackeyâGlassâs models with advection and delay, respectively. We numerically show that the population density decreases as the increase of advection rate or death rate, and a delay-induced Hopf bifurcation is more likely to occur with small advection or low mortality rate.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;32
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01810-7"
                        open
                      >
                        <summary class="article-expander__title">The influence of a transport process on the epidemic threshold</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01810-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
By generating transient encounters between individuals beyond their immediate social environment, transport can have a profound impact on the spreading of an epidemic. In this work, we consider epidemic dynamics in the presence of the transport process that gives rise to a multiplex network model. In addition to a static layer, the (multiplex) epidemic network consists of a second dynamic layer in which any two individuals are connected for the time they occupy the same site during a random walk they perform on a separate transport network. We develop a mean-field description of the stochastic network model and study the influence the transport process has on the epidemic threshold. We show that any transport process generally lowers the epidemic threshold because of the additional connections it generates. In contrast, considering also random walks of fractional order that in some sense are a more realistic model of human mobility, we find that these non-local transport dynamics raise the epidemic threshold in comparison to a classical local random walk. We also test our model on a realistic transport network (the Munich U-Bahn network), and carefully compare mean-field solutions with stochastic trajectories in a range of scenarios.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;73
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/xlxs4/rss-reader/actions/runs/3551477698">
        <time id="build-timestamp" datetime="2022-11-26T00:36:32.186Z">2022-11-26T00:36:32.186Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>