<!DOCTYPE html>
<html lang="en">

  <head>
    <title>osmos::feed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="osmos::feed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <!-- %before-head-end.html% -->
  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-10 datetime="2022-10-10T00:57:57.001Z">2022-10-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-10 datetime="2022-10-10T00:57:57.001Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ScienceDirect Publication: Journal of Differential Equations</button>
                  <a class="source-heading__link" href="https://www.sciencedirect.com/journal/journal-of-differential-equations">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005381"
                        open
                      >
                        <summary class="article-expander__title">Continuity of the integral kernel of Schrödinger propagator</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005381?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Shiliang Zhao, Quan Zheng</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005484"
                        open
                      >
                        <summary class="article-expander__title">Solutions to the Hamilton-Jacobi equation for state constrained Bolza problems with discontinuous time dependence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005484?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Julien Bernis, Piernicola Bettiol, Richard B. Vinter</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005393"
                        open
                      >
                        <summary class="article-expander__title">Supercritical elliptic problems on nonradial domains via a nonsmooth variational approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005393?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Craig Cowan, Abbas Moameni</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005356"
                        open
                      >
                        <summary class="article-expander__title">Photo-acoustic inversion using plasmonic contrast agents: The full Maxwell model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005356?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Ahcene Ghandriche, Mourad Sini</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005411"
                        open
                      >
                        <summary class="article-expander__title">Morse index for the ground state in the energy supercritical Gross–Pitaevskii equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005411?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Dmitry E. Pelinovsky, Szymon Sobieszek</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200540X"
                        open
                      >
                        <summary class="article-expander__title">Orbital stability of periodic standing waves for the cubic fractional nonlinear Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200540X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Gabriel E. Bittencourt Moraes, Handan Borluk, Guilherme de Loreno, Gulcin M. Muslu, Fábio Natali</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005344"
                        open
                      >
                        <summary class="article-expander__title">On the Calderón problem for nonlocal Schrödinger equations with homogeneous, directionally antilocal principal symbols</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005344?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Giovanni Covi, María Ángeles García-Ferrero, Angkana Rüland</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005332"
                        open
                      >
                        <summary class="article-expander__title">A class of supercritical Sobolev type inequalities with logarithm and related elliptic equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005332?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Yinbin Deng, Shuangjie Peng, Xinyue Zhang, Yang Zhou</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005435"
                        open
                      >
                        <summary class="article-expander__title">An alternative for minimal group actions on totally regular curves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005435?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Enhui Shi, Hui Xu, Xiangdong Ye</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005629"
                        open
                      >
                        <summary class="article-expander__title">Optimal decay of compressible Navier-Stokes equations with or without potential force</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005629?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Jincheng Gao, Minling Li, Zheng-an Yao</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005460"
                        open
                      >
                        <summary class="article-expander__title">Existence and multiplicity of solutions to Dirichlet problem for semilinear subelliptic equation with a free perturbation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005460?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Hua Chen, Hong-Ge Chen, Xin-Rui Yuan</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005496"
                        open
                      >
                        <summary class="article-expander__title">A note on the weak regularity theory for degenerate Kolmogorov equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005496?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Francesca Anceschi, Annalaura Rebucci</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005514"
                        open
                      >
                        <summary class="article-expander__title">An inequality of Bessel functions and applications to transcritical bifurcation problems of nonlinear elliptic equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005514?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Hongjing Pan, Shiyi Ye</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005186"
                        open
                      >
                        <summary class="article-expander__title">Gaining two derivatives on a singular force in the 2D Navier-Stokes equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005186?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Alexey Cheskidov, Landon Kavlie</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005630"
                        open
                      >
                        <summary class="article-expander__title">Higher integrability and stability of (p,q)-quasiminimizers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005630?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Antonella Nastasi, Cintia Pacchiano Camacho</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005459"
                        open
                      >
                        <summary class="article-expander__title">Hölder continuity of parabolic quasi-minimizers on metric measure spaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005459?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Andreas Herán</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005472"
                        open
                      >
                        <summary class="article-expander__title">On negative eigenvalues of the spectral problem for water waves of highest amplitude</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005472?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Vladimir Kozlov, Evgeniy Lokharu</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005666"
                        open
                      >
                        <summary class="article-expander__title">On 2D Eulerian limits à la Kuksin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005666?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Benedetta Ferrario</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005617"
                        open
                      >
                        <summary class="article-expander__title">Hydrodynamic limit for the inhomogeneous incompressible Navier-Stokes-Vlasov equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005617?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Yunfei Su, Guochun Wu, Lei Yao, Yinghui Zhang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005368"
                        open
                      >
                        <summary class="article-expander__title">On quantitative uniqueness for parabolic equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005368?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Igor Kukavica, Quinn Le</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005423"
                        open
                      >
                        <summary class="article-expander__title">Nonlinear smoothing for the periodic generalized nonlinear Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005423?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Ryan McConnell</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200537X"
                        open
                      >
                        <summary class="article-expander__title">Decay of entropy solutions of a scalar conservation law beyond a stationary ergodic setting</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200537X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Luís Salvino, Jean Silva</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005502"
                        open
                      >
                        <summary class="article-expander__title">The principle of linearized stability in age-structured diffusive populations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005502?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Christoph Walker, Josef Zehetbauer</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200568X"
                        open
                      >
                        <summary class="article-expander__title">Dynamical system describing cloud of particles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200568X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Dorota Bors, Robert Stańczy</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005605"
                        open
                      >
                        <summary class="article-expander__title">The number of limit cycles for regularized piecewise polynomial systems is unbounded</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005605?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): R. Huzak, K. Uldall Kristiansen</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005642"
                        open
                      >
                        <summary class="article-expander__title">Asymptotic mean value formulas, nonlocal space-time parabolic operators and anomalous tug-of-war games</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005642?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Carmina Fjellström, Kaj Nyström, Yuqiong Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005319"
                        open
                      >
                        <summary class="article-expander__title">Energy conservation law for weak solutions of the full compressible Navier-Stokes equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005319?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Motofumi Aoki, Tsukasa Iwabuchi</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005678"
                        open
                      >
                        <summary class="article-expander__title">A uniqueness criterion for ordinary differential equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005678?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00284-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): A. Constantin</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-09 datetime="2022-10-09T00:00:00.000Z">2022-10-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-09 datetime="2022-10-09T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.08.27.505510v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">CRISPR/Cas-based precision gene replacement in plants via microhomology-mediated end joining [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.08.27.505510v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Highly efficient precision gene editing is ultimately a desirable technology, but implementation remains challenging, especially in plants. Here, we tested a novel CRISPR-Cas-based tool for precision gene replacement in plants employing a microhomology-mediated end joining (MMEJ) mechanism. The MMEJ-mediated precise gene replacement employing 20-bp microhomology MMEJ donors (~100 bp lengths) and a cNHEJ inhibitor, NU7441, produced high targeted editing efficiencies, up to 1.60 {+/-} 0.14, 4.47 {+/-} 1.98, and 8.98 {+/-} 4.73 % in tomato, lettuce, and cabbage, respectively. Our data also revealed the critical impacts of the microhomology length and NU7441 concentration on MMEJ-based precision gene editing in plants.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.04.510900v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Control of spatio-temporal patterning via cell density in a multicellular synthetic gene circuit [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.04.510900v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A major goal in synthetic development is to design and construct gene regulatory circuits that control the patterning and morphogenesis of synthetic multicellular structures. In natural development, an interplay between mechanical and chemical communication shapes the dynamics of gene regulatory circuits that underlie patterning and morphogenesis. However, for synthetic gene circuits, how the non-genetic properties of the growth environment impact circuit behavior remains poorly understood. Here, we describe an occurrence of mechano-chemical coupling in synthetic contact-dependent synNotch patterning circuits demonstrating that cell density modulates the transduction of signal between a sender and receiver cell. By exploiting density-dependent signaling, we construct multicellular signal propagation circuits with synNotch and control the patterning outcome both temporally and spatially via cell density gradients established in vitro via plating or small-molecule mediated modulation of proliferation. Our work demonstrates that synthetic gene circuits can be critically impacted by their context, providing an alternate means for programming multi-cellular circuit patterning outcomes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="285"
                        open
                      >
                        <summary class="article-expander__title">Journal of Mathematical Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/285">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01807-2"
                        open
                      >
                        <summary class="article-expander__title">A group theoretic approach to model comparison with simplicial representations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01807-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The complexity of biological systems, and the increasingly large amount of associated experimental data, necessitates that we develop mathematical models to further our understanding of these systems. Because biological systems are generally not well understood, most mathematical models of these systems are based on experimental data, resulting in a seemingly heterogeneous collection of models that ostensibly represent the same system. To understand the system we therefore need to understand how the different models are related to each other, with a view to obtaining a unified mathematical description. This goal is complicated by the fact that a number of distinct mathematical formalisms may be employed to represent the same system, making direct comparison of the models very diff…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;68
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-08 datetime="2022-10-08T00:00:00.000Z">2022-10-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-08 datetime="2022-10-08T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.04.510903v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Orthogonal tRNA Expression using Endogenous Machinery in Cell-Free Systems [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.04.510903v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A wide variety of non-canonical amino acids (ncAAs) can be incorporated into proteins through the coordinated action of a stop codon suppressing tRNA and aminoacyl-tRNA synthetase. However, methods to discover and characterize suppressor tRNAs are generally lacking. In this work, we show that cell-free systems can express functional suppressor tRNAs using endogenous machinery and characterize their activity. This method is compatible with widely used orthogonal tRNAs, such as the Methanocaldococus jannaschii tyrosyl tRNA, the Methanosarcina barkeri pyrrolysyl tRNA, the Methanomethylophilus Alvus pyrrolysyl tRNA, and an engineered Int pyrroysyl tRNA. Modifying the workflow to evaluate TAA suppression revealed that the M. jannaschii and M. alvus are highly functional TAA suppressors in cell-free systems. Finally, we show that we can express two distinct tRNAs simultaneously, enabling the incorporation of multiple, distinct ncAAs. In total, our work shows that cell-free systems are useful platforms to express and characterize tRNAs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.05.511053v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Characterization of Integrase and Excisionase Activity in Cell-free Protein Expression System Using a Modeling and Analysis Pipeline [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.05.511053v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>We present a full-stack modeling, analysis, and parameter identification pipeline to guide the modeling and design of biological systems starting from specifications to circuit implementations and parameterizations. We demonstrate this pipeline by characterizing the integrase and excisionase activity in cell-free protein expression system. We build on existing Python tools -- BioCRNpyler, AutoReduce, and Bioscrape -- to create this pipeline. For enzyme-mediated DNA recombination in cell-free system, we create detailed chemical reaction network models from simple high-level descriptions of the biological circuits and their context using BioCRNpyler. We use Bioscrape to show that the output of the detailed model is sensitive to many parameters. However, parameter identification is infeasible for this high-dimensional model, hence, we use AutoReduce to automatically obtain reduced models that have fewer parameters. This results in a hierarchy of reduced models under different assumptions to finally arrive at a minimal ODE model for each circuit. Then, we run sensitivity analysis-guided Bayesian inference using Bioscrape for each circuit to identify the model parameters. This process allows us to quantify integrase and excisionase activity in cell extracts enabling complex-circuit designs that depend on accurate control over protein expression levels through DNA recombination. The automated pipeline presented in this paper opens up a new approach to complex circuit design, modeling, reduction, and parameterization.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21366-2"
                        open
                      >
                        <summary class="article-expander__title">Improving protein succinylation sites prediction using embeddings from protein language model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21366-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21366-2/MediaObjects/41598_2022_21366_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Improving protein succinylation sites prediction using embeddings from protein language model</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33718-7"
                        open
                      >
                        <summary class="article-expander__title">Integrative analysis of KRAS wildtype metastatic pancreatic ductal adenocarcinoma reveals mutation and expression-based similarities to cholangiocarcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33718-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33718-7/MediaObjects/41467_2022_33718_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>KRAS wildtype metastatic pancreatic ductal adenocarcinoma (mPDAC) could represent a distinct molecular entity from other PDACs. Here, the authors analyse KRAS wildtype mPDAC tumours using genomics and transcriptomics and find molecular similarities with cholangiocarcinomas.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41597-022-01735-2"
                        open
                      >
                        <summary class="article-expander__title">De novo transcriptome reconstruction in aquacultured early life stages of the cephalopod Octopus vulgaris</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41597-022-01735-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41597-022-01735-2/MediaObjects/41597_2022_1735_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurement(s) Transcriptome sequencing assay Technology Type(s) RNA-seq assay (Illumina) Sample Characteristic - Organism Octopus vulgaris Sample Characteristic - Environment Ocean Sample Characteristic - Location NW Spain (Ría de Vigo, Galicia)</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21291-4"
                        open
                      >
                        <summary class="article-expander__title">The determination of the effect(s) of solute carrier family 22-member 2 (SLC22A2) haplotype variants on drug binding via molecular dynamic simulation systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21291-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21291-4/MediaObjects/41598_2022_21291_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - The determination of the effect(s) of solute carrier family 22-member 2 (SLC22A2) haplotype variants on drug binding via molecular dynamic simulation systems</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10915"
                        open
                      >
                        <summary class="article-expander__title">Journal of Scientific Computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10915">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02019-y"
                        open
                      >
                        <summary class="article-expander__title">Verifiability of the Data-Driven Variational Multiscale Reduced Order Model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02019-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we focus on the mathematical foundations of reduced order model (ROM) closures. First, we extend the verifiability concept from large eddy simulation to the ROM setting. Specifically, we call a ROM closure model verifiable if a small ROM closure model error (i.e., a small difference between the true ROM closure and the modeled ROM closure) implies a small ROM error. Second, we prove that the data-driven ROM closure studied here (i.e., the data-driven variational multiscale ROM) is verifiable. Finally, we investigate the verifiability of the data-driven variational multiscale ROM in the numerical simulation of the one-dimensional Burgers equation and a two-dimensional flow past a circular cylinder at Reynolds numbers 
                      
                      \(Re&#x3D;100\)
                      
                     and 
                      
                      \(Re&#x3D;1000\)
                      
                    .</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-07 datetime="2022-10-07T04:00:00.000Z">2022-10-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-07 datetime="2022-10-07T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00327"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Modular, Synthetic Boolean Logic Gates Enabled in Saccharomyces cerevisiae through T7 Polymerases/CRISPR dCas9 Designs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00327">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00327</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00345"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Efficient Bacterial Genome Engineering throughout the Central Dogma Using the Dual-Selection Marker tetAOPT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00345">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00345</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0117845?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Multistability and evolution of chimera states in a network of type II Morris–Lecar neurons with asymmetrical nonlocal inhibitory connections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0117845?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Formation of synchronous activity patterns is an essential property of neuronal networks that has been of central interest to synchronization theory. Chimera states, where both synchronous and asynchronous activities of neurons co-exist in a single network, are particularly poignant examples of such patterns, whose dynamics and multistability may underlie brain function, such as cognitive tasks. However, dynamical mechanisms of coherent state formation in spiking neuronal networks as well as ways to control these states remain unclear. In this paper, we take a step in this direction by considering the evolution of chimera states in a network of class II excitable Morris–Lecar neurons with asymmetrical nonlocal inhibitory connections. Using the adaptive coherence measure, we are able to partition the network parameter space into regions of various collective behaviors (antiphase synchronous clusters, traveling waves, different types of chimera states as well as a spiking death regime) and have shown multistability between the various regimes. We track the evolution of the chimera states as a function of changed key network parameters and found transitions between various types of chimera states. We further find that the network can demonstrate long transients leading to quasi-persistence of activity patterns in the border regions hinting at near-criticality behaviors.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2021.09.10.459858v4?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Engineered bacteria detect tumor DNA [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2021.09.10.459858v4?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Advances in bacterial engineering have catalysed the development of living cell diagnostics and therapeutics1-3, including microbes that respond to gut inflammation4, intestinal bleeding5, pathogens6 and hypoxic tumors7. Bacteria can access the entire gastrointestinal tract8 to produce outputs measured in stool4 or urine7. Cellular memory, such as bistable switches4,9,10 or genomic rearrangements11, allows bacteria to store information over time. However, living biosensors have not yet been engineered to detect specific DNA sequences or mutations from outside the cell. Here, we engineer naturally competent Acinetobacter baylyi to detect donor DNA from the genomes of colorectal cancer (CRC) cells, organoids and tumors. We characterize the functionality of the biosensors in vitro with co-culture assays and then validate in vivo with sensor bacteria delivered to mice harboring colorectal tumors. We observe horizontal gene transfer from the tumor to the sensor bacteria in our mouse model of CRC. The sensor bacteria achieved 100% discrimination between mice with and without CRC. This Cellular Assay of Targeted, CRISPR-discriminated Horizontal gene transfer (CATCH), establishes a framework for biosensing of mutations or organisms within environments that are difficult to sample, among many other potential applications. Furthermore, the platform could be readily expanded to include production and delivery of antibiotic or antineoplastic therapeutic payloads at the detection site.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.04.509869v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Reconstruct a eukaryotic chromosome arm by de novo design and synthesis [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.04.509869v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The genome of an organism is inherited from its ancestor and keeps evolving over time, however, how much the current version could be altered remains unknown. Here, we use the left arm of chromosome XII (chrXIIL) as an example to probe the genome plasticity in Saccharomyces cerevisiae. A neochromosome was designed to harbor originally dispersed genes. The essentiality of sequences in chrXIIL was dissected by targeted DNA removal, chromosome truncation and random deletion. Notably, 12 genes were sufficient for survival, while 25 genes are required to retain robust fitness. Next, we demonstrated these genes could be reconstructed using synthetic regulatory sequences and recoded open-reading frames with &quot;one-amino-acid-one-codon&quot; strategy. Finally, we built a neochromsome, which could substitute for chrXIIL for cell viability, with these reconstructed genes. Our work not only highlights the high plasticity of yeast genome, but also illustrates the possibility of making functional chromosomes with completely artificial sequences.

HIGHLIGHTSO_LIA neochromosome was designed to facilitate the assembly of exogenous DNA for stable expression in yeast
C_LIO_LIThe left arm of chrXII could be minimized to just 12 genes to maintain viability, but additional genes were required to retain robust fitness
C_LIO_LIComprehensive recoding and transcriptional refactoring using artificial regulatory sequences produced a functional chromosome arm
C_LIO_LIA completely reconstructed neochromosome could replace the chrXIIL to maintain comparable fitness
C_LI</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01642-9"
                        open
                      >
                        <summary class="article-expander__title">BioImageIT: Open-source framework for integration of image data management with analysis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01642-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01642-9/MediaObjects/41592_2022_1642_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nature Methods - BioImageIT: Open-source framework for integration of image data management with analysis</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21417-8"
                        open
                      >
                        <summary class="article-expander__title">Multi-omics disease module detection with an explainable Greedy Decision Forest</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21417-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21417-8/MediaObjects/41598_2022_21417_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Multi-omics disease module detection with an explainable Greedy Decision Forest</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03840-9"
                        open
                      >
                        <summary class="article-expander__title">Delineating cooperative effects of Notch and biomechanical signals on patterned liver differentiation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03840-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03840-9/MediaObjects/42003_2022_3840_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Integrating cellular microarrays, experimental and computational data for liver progenitor cell differentiation patterning, the interplay between Notch signaling and cellular mechanics is assessed.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20598-6"
                        open
                      >
                        <summary class="article-expander__title">Towards a comprehensive approach for characterizing cell activity in bright-field microscopic images</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20598-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20598-6/MediaObjects/41598_2022_20598_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Towards a comprehensive approach for characterizing cell activity in bright-field microscopic images</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33642-w"
                        open
                      >
                        <summary class="article-expander__title">Prevalence and mechanisms of somatic deletions in single human neurons during normal aging and in DNA repair disorders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33642-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33642-w/MediaObjects/41467_2022_33642_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA damage has been implicated in aging and neurodegeneration. Here, the authors develop a bioinformatic method to detect deletions in single neuron genome sequences and reveal an increased burden of somatic deletions during aging and in DNA repair disorders.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33633-x"
                        open
                      >
                        <summary class="article-expander__title">Past and present giant viruses diversity explored through permafrost metagenomics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33633-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33633-x/MediaObjects/41467_2022_33633_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Although giant viruses are abundant in aquatic environments, less is known about giant viruses in soil. Here, the authors use permafrost metagenomics to reveal giant virus diversity and heterogeneity, as well as gene transfers between viruses from different families.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41597-022-01702-x"
                        open
                      >
                        <summary class="article-expander__title">TumorMet: A repository of tumor metabolic networks derived from context-specific Genome-Scale Metabolic Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41597-022-01702-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41597-022-01702-x/MediaObjects/41597_2022_1702_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Measurement(s) gene expression, metabolic relationships Technology Type(s) Genome Scale Metabolic Models; Computational network biology Sample Characteristic - Organism Homo sapiens</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20943-9"
                        open
                      >
                        <summary class="article-expander__title">An assessment tool for computer-assisted semen analysis (CASA) algorithms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20943-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20943-9/MediaObjects/41598_2022_20943_Fig1_HTML.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - An assessment tool for computer-assisted semen analysis (CASA) algorithms</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20750-2"
                        open
                      >
                        <summary class="article-expander__title">Molecular and expression analysis indicate the role of CBL interacting protein kinases (CIPKs) in abiotic stress signaling and development in chickpea</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20750-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20750-2/MediaObjects/41598_2022_20750_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Molecular and expression analysis indicate the role of CBL interacting protein kinases (CIPKs) in abiotic stress signaling and development in chickpea</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20404-3"
                        open
                      >
                        <summary class="article-expander__title">Alzheimer’s disease: insights from a network medicine perspective</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20404-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20404-3/MediaObjects/41598_2022_20404_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Alzheimer’s disease: insights from a network medicine perspective</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21197-1"
                        open
                      >
                        <summary class="article-expander__title">Comparative genomic and biochemical analyses identify a collagen galactosylhydroxylysyl glucosyltransferase from Acanthamoeba polyphaga mimivirus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21197-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21197-1/MediaObjects/41598_2022_21197_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Comparative genomic and biochemical analyses identify a collagen galactosylhydroxylysyl glucosyltransferase from Acanthamoeba polyphaga mimivirus</span>
                                &ensp;<span class="article-reading-time">(&hairsp;34
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01813-4"
                        open
                      >
                        <summary class="article-expander__title">A cancer model with nonlocal free boundary dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01813-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Cancer cells at the tumor boundary move in the direction of the oxygen gradient, while cancer cells far within the tumor are in a necrotic state. This paper introduces a simple mathematical model that accounts for these facts. The model consists of cancer cells, cytotoxic T cells, and oxygen satisfying a system of partial differential equations. Some of the model parameters represent the effect of anti-cancer drugs. The tumor boundary is a free boundary whose dynamics is determined by the movement of cancer cells at the boundary. The model is simulated for radially symmetric and axially symmetric tumors, and it is shown that the tumor may increase or decrease in size, depending on the “strength&quot; of the drugs. Existence theorems are proved, global in-time in the radially symmetric case, and local in-time for any shape of tumor. In the radially symmetric case, it is proved, under different conditions, that the tumor may shrink monotonically, or expand monotonically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01817-0"
                        open
                      >
                        <summary class="article-expander__title">Maximal total population of species in a diffusive logistic model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01817-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we investigate the maximization of the total population of a single species which is governed by a stationary diffusive logistic equation with a fixed amount of resources. For large diffusivity, qualitative properties of the maximizers like symmetry will be addressed. Our results are in line with previous findings which assert that for large diffusion, concentrated resources are favorable for maximizing the total population. Then, an optimality condition for the maximizer is derived based upon rearrangement theory. We develop an efficient numerical algorithm applicable to domains with different geometries in order to compute the maximizer. It is established that the algorithm is convergent. Our numerical simulations give a real insight into the qualitative properties of the maximizer and also lead us to some conjectures about the maximizer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02016-1"
                        open
                      >
                        <summary class="article-expander__title">A Stochastic Nesterov’s Smoothing Accelerated Method for General Nonsmooth Constrained Stochastic Composite Convex Optimization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02016-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We propose a novel stochastic Nesterov’s smoothing accelerated method for general nonsmooth, constrained, stochastic composite convex optimization, the nonsmooth component of which may be not easy to compute its proximal operator. The proposed method combines Nesterov’s smoothing accelerated method (Nesterov in Math Program 103(1):127–152, 2005) for deterministic problems and stochastic approximation for stochastic problems, which allows three variants: single sample and two different mini-batch sizes per iteration, respectively. We prove that all the three variants achieve the best-known complexity bounds in terms of stochastic oracle. Numerical results on a robust linear regression problem, as well as a support vector machine problem show that the proposed method compares favorably with other state-of-the-art first-order methods, and the variants with mini-batch sizes outperform the variant with single sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02008-1"
                        open
                      >
                        <summary class="article-expander__title">Dissipation-Preserving Rational Spectral-Galerkin Method for Strongly Damped Nonlinear Wave System Involving Mixed Fractional Laplacians in Unbounded Domains</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02008-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This paper aims at developing a dissipation-preserving, linearized, and time-stepping-varying spectral method for strongly damped nonlinear wave system in multidimensional unbounded domains 
                      
                      \({\mathbb {R}}^d\)
                      
                     (d&#x3D;1, 2, and 3), where the nonlocal nature is described by the mixed fractional Laplacians. Because the underlying solutions of the problem involving mixed fractional Laplacians decay slowly with certain power law at infinity, we employ the rational spectral-Galerkin method using rational basis (or mapped Gegenbauer functions) for the spatial approximation. To capture the intrinsic dissipative properties of the model equations, we combine the Crank-Nicolson scheme with exponential scalar auxiliary variable approach for the temporal discretization. Based on the rate of nonlocal energy dissipation, we design a novel time-stepping-varying strategy to enhance the efficiency of the scheme. We present the detailed implementation of the scheme, where the main building block of the stiffness matrices is based on the Laguerre-Gauss quadrature rule for the modified Bessel functions of the second kind. The existence, uniqueness, and nonlocal energy dissipation law of the fully discrete scheme are rigourously established. Numerical examples in 3D case are carried out to demonstrate the accuracy and efficiency of the scheme. Finally, we simulate the nonlinear behaviors of 2D/3D dissipative vector solitary waves for damped sine-Gordon system I, for damped sine-Gordon system II, and for damped Klein–Gordon system to provide a deeper understanding of nonlocal physics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-06 datetime="2022-10-06T16:54:27.000Z">2022-10-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-06 datetime="2022-10-06T16:54:27.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119509"
                        open
                      >
                        <summary class="article-expander__title">‘Monumental’ Math Proof Solves Triple Bubble Problem and More</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/monumental-math-proof-solves-triple-bubble-problem-and-more-20221006/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/Multi-Bubble-Conjecture_520x292.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The decades-old Sullivan’s conjecture, about the best way to minimize the surface area of a bubble cluster, was thought to be out of reach for three bubbles and up — until a new breakthrough result.            
The post ‘Monumental’ Math Proof Solves Triple Bubble Problem and More first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00296"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Gatekeeper Residue Replacement in a Phosphite Transporter Enhances Mutational Robustness of the Biocontainment Strategy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00296">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00296</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102483?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The combination of social reward and punishment is conducive to the cooperation and heterogeneity of social relations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102483?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://aip.scitation.org/action/showOpenGraphArticleImage?doi&#x3D;10.1063/5.0102483&amp;id&#x3D;images/medium/5.0102483.figures.online.highlight_f1.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Individual behaviors and social relations influence each other. However, understanding the underlying mechanism remains challenging. From social norms controlling human behavior to individual management of interpersonal relationships, rewards and punishments are some of the most commonly used measures. Through simulating the weak prisoner’s dilemma in finite populations, we find that neither a simple reward measure nor a pure punishment mechanism can extensively promote cooperation. Instead, a combination of appropriate punishment and reward mechanisms can promote cooperation’s prosperity regardless of how large or small the temptation to defect is. In addition, the combination spontaneously produces inhomogeneities in social relations and individual influence, which support the continued existence of cooperative behavior. Finally, we further explain how cooperators establish a sustainable existence under the combination by investigating the social relations at different moments in a small system. These results demonstrate that dispensing rewards and punishments impartially in society is essential to social harmony.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107427?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Generalized weighted permutation entropy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107427?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
A novel heuristic approach is proposed here for time series data analysis, dubbed Generalized weighted permutation entropy, which amalgamates and generalizes beyond their original scope two well established data analysis methods: Permutation entropy and Weighted permutation entropy. The method introduces a scaling parameter to discern the disorder and complexity of ordinal patterns with small and large fluctuations. Using this scaling parameter, the complexity-entropy causality plane is generalized to the complexity-entropy-scale causality box. Simulations conducted on synthetic series generated by stochastic, chaotic, and random processes, as well as real world data, are shown to produce unique signatures in this three dimensional representation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.03.510612v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Engineering Escherichia coli to utilize erythritol as sole carbon source [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.03.510612v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Erythritol, one of the natural sugar alcohols, is widely used as a sugar substitute sweetener in food industries. Humans and their native gut microbiota lack erythritol catabolic microorganisms to utilize erythritol as carbon source, which inspires us to engineer a model chassis that can respond and utilize erythritol for establishing wide applications in synthetic biology fields. Here, we successfully engineered Escherichia coli to respond and utilize erythritol as sole carbon source. First, a heterologous erythritol catabolic gene cluster from Ochrobactrum spp. enables E. coli to catabolize erythritol into erythrose-4-phosphate and flow into pentose-phosphate pathway. Second, we characterized an erythritol binding transcriptional repressor and its DNA binding site. Then, mRNA transcriptional analysis guided us to build up further metabolic engineering strategies for improving erythritol catabolism. Finally, such engineered E. coli strains were applied for in vitro simulated intestinal fluid experiments and performed as an erythritol detector to distinguish soda drinks. We anticipate erythritol catabolic E. coli strains will expand synthetic biology fields and accelerate applications in biomedical engineering and living therapeutics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.24.509315v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">An approach towards identifying the site of interaction between the Streptomyces phage OC31 protein Gp3 and the coiled-coil domain of its serine integrase [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.24.509315v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Streptomyces phage OC31 encodes a serine integrase which recombines the host (attB) and phage (attP) attachment sites to form new attL and attR sites. For attL and attR to be recombined by the integrase, the presence of its cognate recombination directionality factor (RDF) is required. It is hypothesized that the RDF binds the coiled-coil domain of the integrase to accomplish this switch in recombination directionality. Yet, nothing is known about the site of interaction between the RDF and integrase. Here, we tried to identify the region on the coiled-coil domain of OC31 integrase to which its cognate RDF (gp3) might bind. Mutant integrases fused to their cognate RDF (gp3) were created and selected for inactivity in attL x attR recombination. It was not possible to characterize the mutant…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20268-7"
                        open
                      >
                        <summary class="article-expander__title">An interpretable semi-supervised framework for patch-based classification of breast cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20268-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20268-7/MediaObjects/41598_2022_20268_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - An interpretable semi-supervised framework for patch-based classification of breast cancer</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21149-9"
                        open
                      >
                        <summary class="article-expander__title">Research on visual question answering based on dynamic memory network model of multiple attention mechanisms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21149-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21149-9/MediaObjects/41598_2022_21149_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Research on visual question answering based on dynamic memory network model of multiple attention mechanisms</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20651-4"
                        open
                      >
                        <summary class="article-expander__title">Evaluation of two semi-supervised learning methods and their combination for automatic classification of bone marrow cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20651-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20651-4/MediaObjects/41598_2022_20651_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Evaluation of two semi-supervised learning methods and their combination for automatic classification of bone marrow cells</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19994-9"
                        open
                      >
                        <summary class="article-expander__title">Structure-function clustering in weighted brain networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19994-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19994-9/MediaObjects/41598_2022_19994_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Structure-function clustering in weighted brain networks</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04020-5"
                        open
                      >
                        <summary class="article-expander__title">SoloTE for improved analysis of transposable elements in single-cell RNA-Seq data using locus-specific expression</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04020-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04020-5/MediaObjects/42003_2022_4020_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SoloTE includes the genome location of transposable elements (TE) in scRNA-Seq data to determine how TE activity regulates neighboring gene expression.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01625-w"
                        open
                      >
                        <summary class="article-expander__title">neuromaps: structural and functional interpretation of brain maps</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01625-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01625-w/MediaObjects/41592_2022_1625_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>neuromaps is a toolbox for accessing, transforming and comparing human neuroimaging data.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;40
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01623-y"
                        open
                      >
                        <summary class="article-expander__title">Advances and opportunities in RNA structure experimental determination and computational modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01623-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01623-y/MediaObjects/41592_2022_1623_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>This Review summarizes recent methodological advances in experimental and computational tools developed in studying RNA structures, which provides a bridge for communication between both experimentalists and computational experts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s44161-022-00133-6"
                        open
                      >
                        <summary class="article-expander__title">Fat infiltration in the infarcted heart as a paradigm for ventricular arrhythmias</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s44161-022-00133-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs44161-022-00133-6/MediaObjects/44161_2022_133_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In a combined computational and multi-center clinical study to investigate the effects of adipose fat tissue infiltration on ventricular arrhythmias, Sung et al. show that infiltrating adipose tissue, as opposed to scar, is the main cause of infarcted-related ventricular arrhythmias by slowing cardiac conduction in critical sites.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21150-2"
                        open
                      >
                        <summary class="article-expander__title">Prediction of chemical warfare agents based on cholinergic array type meta-predictors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21150-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21150-2/MediaObjects/41598_2022_21150_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Prediction of chemical warfare agents based on cholinergic array type meta-predictors</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21207-2"
                        open
                      >
                        <summary class="article-expander__title">A novel vaccine based on SARS-CoV-2 CD4+ and CD8+ T cell conserved epitopes from variants Alpha to Omicron</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21207-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21207-2/MediaObjects/41598_2022_21207_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A novel vaccine based on SARS-CoV-2 CD4+ and CD8+ T cell conserved epitopes from variants Alpha to Omicron</span>
                                &ensp;<span class="article-reading-time">(&hairsp;60
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01464-2"
                        open
                      >
                        <summary class="article-expander__title">Post-translational modifications reshape the antigenic landscape of the MHC I immunopeptidome in tumors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01464-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01464-2/MediaObjects/41587_2022_1464_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A computational pipeline identifies tumor antigen post-translational modifications guiding immune responses.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33562-9"
                        open
                      >
                        <summary class="article-expander__title">Using domain knowledge for robust and generalizable deep learning-based CT-free PET attenuation and scatter correction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33562-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33562-9/MediaObjects/41467_2022_33562_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Deep learning-based methods have been proposed to substitute CT-based PET attenuation and scatter correction to achieve CT-free PET imaging. Here, the authors present a simple way to integrate domain knowledge in deep learning for CT-free PET imaging.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20224-5"
                        open
                      >
                        <summary class="article-expander__title">Detecting and mitigating simultaneous waves of COVID-19 infections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20224-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20224-5/MediaObjects/41598_2022_20224_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Detecting and mitigating simultaneous waves of COVID-19 infections</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33575-4"
                        open
                      >
                        <summary class="article-expander__title">Predicting the structural basis of targeted protein degradation by integrating molecular dynamics simulations with structural mass spectrometry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33575-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33575-4/MediaObjects/41467_2022_33575_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The formation of ternary degrader-protein complexes is a key step in the targeted degradation of proteins of interest. Here, the authors explore the structure and dynamics of such complexes applying high-performance computer simulations augmented with experimental data.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;79
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33530-3"
                        open
                      >
                        <summary class="article-expander__title">A method for multiplexed full-length single-molecule sequencing of the human mitochondrial genome</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33530-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33530-3/MediaObjects/41467_2022_33530_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Accurate analysis of mitochondrial DNA is important for mitochondrial disease clinical research and diagnostics. Here, authors present a method using Cas9 cleavage, nanopore sequencing and a custom pipeline to identify pathogenic variants, deletions and accurately quantify heteroplasmy to below 1%.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33524-1"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Engineering the stambomycin modular polyketide synthase yields 37-membered mini-stambomycins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33524-1">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Author Correction: Engineering the stambomycin modular polyketide synthase yields 37-membered mini-stambomycins</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01038-4"
                        open
                      >
                        <summary class="article-expander__title">Genetically encoded chemical crosslinking of RNA in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01038-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01038-4/MediaObjects/41557_2022_1038_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein–RNA interactions regulate RNA fate and function, and are generally non-covalent and reversible. Genetically introducing a latent bioreactive amino acid into a protein is now shown to enable the protein to covalently crosslink a bound RNA molecule in vivo. This method offers innovative avenues for developing protein–RNA research and applications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01796-2"
                        open
                      >
                        <summary class="article-expander__title">Cyclic attractors of nonexpanding q-ary networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01796-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Discrete dynamical systems in which model components take on categorical values have been successfully applied to biological networks to study their global dynamic behavior. Boolean models in particular have been used extensively. However, multi-state models have also emerged as effective computational tools for the analysis of complex mechanisms underlying biological networks. Models in which variables assume more than two discrete states provide greater resolution, but this scheme introduces discontinuities. In particular, variables can increase or decrease by more than one unit in one time step. This can be corrected, without changing fixed points of the system, by applying an additional rule to each local activation function. On the other hand, if one is interested in cyclic attractors of their system, then this rule can potentially introduce new cyclic attractors that were not observed previously. This article makes some advancements in understanding the state space dynamics of multi-state network models with synchronous, sequential, or block-sequential update schedules and establishes conditions under which no new cyclic attractors are added to networks when the additional rule is applied. Our analytical results have the potential to be incorporated into modeling software and aid researchers in their analyses of biological multi-state networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-05 datetime="2022-10-05T10:00:42.000Z">2022-10-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-05 datetime="2022-10-05T10:00:42.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119285"
                        open
                      >
                        <summary class="article-expander__title">Molecule-Building Innovators Win Nobel Prize in Chemistry</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/molecule-building-innovators-win-2022-chemistry-nobel-prize-20221005/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/Chemistry_Nobel_1200_Social2.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The chemists Carolyn Bertozzi, Morten Meldal and K. Barry Sharpless were recognized for their development of click chemistry and bioorthogonal chemistry.            
The post Molecule-Building Innovators Win Nobel Prize in Chemistry first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00226"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Metabolic Engineering Mevalonate Pathway Mediated by RNA Scaffolds for Mevalonate and Isoprene Production in Escherichia coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00226">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00226</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00443"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Interaction between the Anchoring Domain of A‑Kinase Anchoring Proteins and the Dimerization and Docking Domain of Protein Kinase A: A Potent Tool for Synthetic Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00443">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00443</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00413"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Control of Enzyme Reaction Initiation inside Giant Unilamellar Vesicles by the Cell-Penetrating Peptide-Mediated Translocation of Cargo Proteins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00413">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00413</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.03.510608v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Design, Construction, and Functional Characterization of a tRNA Neochromosome in Yeast [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.03.510608v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Here we report the design, construction and characterization of a tRNA neochromosome, a designer chromosome that functions as an additional, de novo counterpart to the native complement of Saccharomyces cerevisiae. Intending to address one of the central design principles of the Sc2.0 project, the [~]190 kb tRNA neochromosome houses all 275 relocated nuclear tRNA genes. To maximize stability, the design incorporated orthogonal genetic elements from non-S. cerevisiae yeast species. Furthermore, the presence of 283 rox recombination sites enable an orthogonal SCRaMbLE system capable of adjusting tRNA abundance. Following construction, we obtained evidence of a potent selective force once the neochromosome was introduced into yeast cells, manifesting as a spontaneous doubling in cell ploidy. Furthermore, tRNA sequencing, transcriptomics, proteomics, nucleosome mapping, replication profiling, FISH and Hi-C were undertaken to investigate questions of tRNA neochromosome behavior and function. Its construction demonstrates the remarkable tractability of the yeast model and opens up new opportunities to directly test hypotheses surrounding these essential non-coding RNAs.

HighlightsO_LIDe novo design, construction and functional characterization of a neochromosome containing all 275 nuclear tRNA genes of Saccharomyces cerevisiae.
C_LIO_LIIncreasing the copy number of the 275 highly expressed tRNA genes causes cellular burden, which the host cell likely buffers either by selecting for partial tRNA neochromosome deletions or by increasing its ploidy.
C_LIO_LIThe tRNA neochromosome can be chemically extracted and transformed into new strain backgrounds, enabling its transplantation into multi-synthetic chromosome strains to finalize the Sc2.0 strain.
C_LIO_LIComprehensive functional characterization does not pinpoint a singular cause for the cellular burden caused by the tRNA neochromosome, but does reveal novel insights into its tRNA and structural chromosome biology.
C_LI</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01493-x"
                        open
                      >
                        <summary class="article-expander__title">Modular, programmable RNA sensing using ADAR editing in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01493-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01493-x/MediaObjects/41587_2022_1493_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Specific RNAs are detected in live cells by harnessing endogenous ADAR enzymes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02011-6"
                        open
                      >
                        <summary class="article-expander__title">High-Order ADI-FDTD Schemes for Maxwell’s Equations with Material Interfaces in Two Dimensions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02011-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we apply the immersed interface method (IIM) and the hierarchical derivative matching (HDM) method, respectively, to restore the accuracy of the high-order alternating direction implicit finite-difference time-domain (ADI-FDTD) scheme of the 2D Maxwell’s equations with material interfaces. For the case of discontinuous permittivity 
                      
                      \(\varepsilon \)
                      
                     and continuous permeability 
                      
                      \(\mu \)
                      
                    , we propose four high-order schemes. Two of them are of second order in time and fourth order in space (ADI-IIM-FDTD(2,4) scheme and ADI-HDM-FDTD(2,4) scheme). Others are of fourth order both in time and space (ADI-IIM-FDTD(4,4) scheme and ADI-HDM-FDTD(4,4) scheme). For the case of discontinuous permittivity 
                      
                      \(\varepsilon \)
                      
                     and permeability 
                      
                      \(\mu \)
                      
                    , the (2,4) scheme and the (4,4) scheme are constructed as well (ADI-HDM-FDTD-X(2,4) scheme and ADI-HDM-FDTD-X(4,4) scheme). The proposed six schemes maintain the advantages of ADI-FDTD method such as unconditional stability and computational efficiency. Numerical examples are given to verify the performance of the proposed schemes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-04 datetime="2022-10-04T09:58:09.000Z">2022-10-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-04 datetime="2022-10-04T09:58:09.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119287"
                        open
                      >
                        <summary class="article-expander__title">Pioneering Quantum Physicists Win Nobel Prize in Physics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/pioneering-quantum-physicists-win-nobel-prize-in-physics-20221004/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/1200-social-card-physics-nobel.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Alain Aspect, John Clauser and Anton Zeilinger have won the 2022 Nobel Prize in Physics for groundbreaking experiments with entangled particles.            
The post Pioneering Quantum Physicists Win Nobel Prize in Physics first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105969?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Community formation in wealth-mediated thermodynamic strategy evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105969?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We study a dynamical system defined by a repeated game on a 1D lattice, in which the players keep track of their gross payoffs over time in a bank. Strategy updates are governed by a Boltzmann distribution, which depends on the neighborhood bank values associated with each strategy, relative to a temperature scale, which defines the random fluctuations. Players with higher bank values are, thus, less likely to change strategy than players with a lower bank value. For a parameterized rock–paper–scissors game, we derive a condition under which communities of a given strategy form with either fixed or drifting boundaries. We show the effect of a temperature increase on the underlying system and identify surprising properties of this model through numerical simulations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102390?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Network spreading among areas: A dynamical complex network modeling approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102390?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
With the outbreak of COVID-19, great loss and damage were brought to human society, making the study of epidemic spreading become a significant topic nowadays. To analyze the spread of infectious diseases among different areas, e.g., communities, cities, or countries, we construct a network, based on the epidemic model and the network coupling, whose nodes denote areas, and edges represent population migrations between two areas. Each node follows its dynamic, which describes an epidemic spreading among individuals in an area, and the node also interacts with other nodes, which indicates the spreading among different areas. By giving mathematical proof, we deduce that our model has a stable solution despite the network structure. We propose the peak infected ratio (PIR) as a property of infectious diseases in a certain area, which is not independent of the network structure. We find that increasing the population mobility or the disease infectiousness both cause higher peak infected population all over different by simulation. Furthermore, we apply our model to real-world data on COVID-19 and after properly adjusting the parameters of our model, the distribution of the peak infection ratio in different areas can be well fitted.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101495?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Entropic comparison of Landau–Zener and Demkov interactions in the phase space of a quadrupole billiard</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101495?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We investigate two types of avoided crossings in a chaotic billiard within the framework of information theory. The Shannon entropy in the phase space for the Landau–Zener interaction increases as the center of the avoided crossing is approached, whereas for the Demkov interaction, the Shannon entropy decreases as the center of avoided crossing is passed by with an increase in the deformation parameter. This feature can provide a new indicator for scar formation. In addition, it is found that the Fisher information of the Landau–Zener interaction is significantly larger than that of the Demkov interaction.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00404"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Building an Artificial Plant Cell Wall on a Lipid Bilayer by Assembling Polysaccharides and Engineered Proteins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00404">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00404</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00244"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] SpyPhage: A Cell-Free TXTL Platform for Rapid Engineering of Targeted Phage Therapies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00244">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00244</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00292"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering Endosymbiotic Growth of E. coli in Mammalian Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00292">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00292</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-03 datetime="2022-10-03T20:00:00.000Z">2022-10-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-03 datetime="2022-10-03T20:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/events-illuminate-critical-need-menstruation-science-1003"
                        open
                      >
                        <summary class="article-expander__title">MIT events illuminate critical need for menstruation science</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/events-illuminate-critical-need-menstruation-science-1003">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/MIT-Below-The-Belt-Outside.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>“AI for endometriosis? If only there were data!”</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119283"
                        open
                      >
                        <summary class="article-expander__title">Geneticist Awarded Nobel Prize for Studies of Extinct Human Ancestors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/extinct-human-genomes-studies-win-nobel-prize-for-medicine-2022-20221003/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/Nobel_Medicine_Paabo_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Svante Pääbo has been awarded the 2022 Nobel Prize in Physiology or Medicine for studying our extinct ancestors’ DNA.            
The post Geneticist Awarded Nobel Prize for Studies of Extinct Human Ancestors first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.30.510340v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Hybrid Synthesis of bioplastics polyhydroxybutyrate from carbon dioxide [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.30.510340v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The accelerating environmental crisis has intensified the demand for switching from traditional economy to a renewable one with a reduced carbon footprint. Here we reported a hybrid system, coupling chemical process of CO2 hydrogen reduction and biological process for polyhydroxybutyrate (PHB) synthesis, that utilized CO2 as a raw material to produce PHB in vitro. The synthetic pathway of PHB was optimized by screening more efficient methanol oxidases, high activity mutants of glycolaldehyde synthase and coordinating enzyme dosages in the pathway, which achieved the carbon yield of 93.6% for producing PHB from methanol. Finally, by combining with the chemical process from CO2 to methanol, a scaling-up bio-system was performed to convert CO2 into PHB, yielding 5.8 g/L with the productivity of 1.06 g-1L-1h-1. This approach represents a promising carbon-neutral way to produce biodegradable plastics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19681-9"
                        open
                      >
                        <summary class="article-expander__title">Cell-free protein crystallization for nanocrystal structure determination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19681-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19681-9/MediaObjects/41598_2022_19681_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Cell-free protein crystallization for nanocrystal structure determination</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33606-0"
                        open
                      >
                        <summary class="article-expander__title">Systematic dissection of key factors governing recombination outcomes by GCE-SCRaMbLE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33606-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33606-0/MediaObjects/41467_2022_33606_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Applications of the SCRaMbLE process are hindered due to the lack of facile and tight regulation and limited understanding of key factors that may affect the rearrangement outcomes. Here the authors present an approach to precisely regulate SCRaMbLE recombination in a dose-dependent manner using genetic code expansion technology with low basal activity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01812-5"
                        open
                      >
                        <summary class="article-expander__title">Operating diagrams for a three-tiered microbial food web in the chemostat</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01812-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we consider a three-tiered food web model in a chemostat, including chlorophenol, phenol, and hydrogen substrates and their degraders. The model takes into account the three substrate inflowing concentrations, as well as maintenance, that is, decay terms of the species. The operating diagrams give the asymptotic behavior of the model with respect to the four operating parameters, which are the dilution rate and the three inflowing concentrations of the substrates. These diagrams were obtained only numerically in the existing literature. Using the mathematical analysis of this model obtained in our previous studies, we construct the operating diagrams, by plotting the curves that separate their various regions. Hence, the regions of the operating diagrams are constructed analytically and there is no requirement for time-consuming algorithms to generate the plots, as in the numerical method. Moreover, our method reveals behaviors that have not been detected in the previous numerical studies. The growth functions are of Monod form with the inclusion of a product inhibition term. However, our method applies for a large class of growth functions. We construct operating diagrams with and without maintenance showing the role of maintenance on the stability of the system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;20
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-30 datetime="2022-09-30T19:30:00.000Z">2022-09-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-30 datetime="2022-09-30T19:30:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/smart-researchers-develop-quick-test-determine-immunity-against-covid-19-0930"
                        open
                      >
                        <summary class="article-expander__title">SMART researchers develop quick test to determine immunity against Covid-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/smart-researchers-develop-quick-test-determine-immunity-against-covid-19-0930">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/SMART-and-NTU-scientists-with-the-rapid-test-kit-that-detects-levels-of-COVID19-antibodies-cover.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Test that measures a person’s antibodies requires a drop of blood and takes just 10 minutes to show results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119316"
                        open
                      >
                        <summary class="article-expander__title">How Star Trek’s Lieutenant Uhura Overcame Astronomical Odds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/how-star-treks-lieutenant-uhura-overcame-astronomical-odds-20220930/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/08/Uhura_1200_social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Readers rescued a Star Trek crew from a probabilistic predicament armed only with the power of mathematical reasoning.            
The post How Star Trek’s Lieutenant Uhura Overcame Astronomical Odds first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0280?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Kernel Mixed Model for Transcriptome Association Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0280?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1459782?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Solution of Initial Value Problems by a Dynamical Galerkin Scheme</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1459782?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1147-1166, September 2022. 
 Abstract. Adaptive Galerkin methods for time-dependent partial differential equations are studied and shown to be dissipative. The adaptation implies that the subset of the selected basis function changes over time according to the evolution of the solution. The corresponding projection operator is thus time-dependent and nondifferentiable. We therefore propose to use an integral formulation in time. We analyze the existence and uniqueness of this weak form of the dynamical Galerkin scheme, and we then prove that the nonsmooth projection operator introduces energy dissipation, which is a crucial result for adaptive Galerkin methods, e.g., adaptive wavelet methods. Numerical examples for the inviscid Burgers equation in one dimension and the incompressible Euler equations in two and three spatial dimensions show that the selection of basis functions, for instance by filtering out weak wavelet coefficients from the solution, introduces energy dissipation. Moreover, for the Burgers case we can show that adaptive wavelet regularization yields convergence of the truncated Galerkin solution to the physically relevant entropy solution. These results motivate adaptive wavelet-based Galerkin schemes for nonlinear hyperbolic conservation laws.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1467791?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Geometric Evolution of Bilayers under the Degenerate Functionalized Cahn–Hilliard Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1467791?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1127-1146, September 2022. 
 Abstract. Using a multiscale analysis, we derive a sharp interface limit for the dynamics of bilayer structures of the functionalized Cahn–Hilliard equation with a cutoff diffusion mobility [math] that is degenerate for [math] and a continuously differentiable double-well potential [math]. We show that the bilayer interface does not move in the [math] time scale. The interface motion occurs in the [math] time scale and is determined by porous medium diffusion processes in both phases with no jumps on the interface. In the longer [math] time scale, the interface motion is a complex combination of porous medium diffusion processes in both phases and the property of mass conservation.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1493707?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Erratum: Tunable Eigenvector-Based Centralities for Multiplex and Temporal Networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1493707?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1167-1167, September 2022. 
 Abstract. This erratum corrects several typos in [D. Taylor, M. A. Porter, and P. J. Mucha, Multiscale Model. Simul., 19 (2021), pp. 113–147].</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00367"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Organellar Glue: A Molecular Tool to Artificially Control Chloroplast–Chloroplast Interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00367">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00367</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00232"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Analysis of Slow-Cycling Variants of the Light-Inducible Nuclear Protein Export System LEXY in Mammalian Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00232">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00232</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00382"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] GroovDB: A Database of Ligand-Inducible Transcription Factors</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00382">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00382</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00260"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Synonymous Codons and Hydrophobicity Optimization of Post-translational Signal Peptide PelB Increase Phage Display Efficiency of DARPins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00260">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00260</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02010-7"
                        open
                      >
                        <summary class="article-expander__title">A Posteriori Error Analysis of a Mixed Finite Element Method for the Coupled Brinkman–Forchheimer and Double-Diffusion Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02010-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper we consider a partially augmented fully-mixed variational formulation that has been recently proposed for the coupling of the stationary Brinkman–Forchheimer and double-diffusion equations, and develop an a posteriori error analysis for the 2D and 3D versions of the associated mixed finite element scheme. Indeed, we derive two reliable and efficient residual-based a posteriori error estimators for this problem on arbitrary (convex or non-convex) polygonal and polyhedral regions. The reliability of the proposed estimators draws mainly upon the uniform ellipticity and inf-sup condition of the forms involved, a suitable assumption on the data, stable Helmholtz decompositions in Hilbert and Banach frameworks, and the local approximation properties of the Clément and Raviart–Thomas operators. In turn, inverse inequalities, the localization technique based on bubble functions, and known results from previous works, are the main tools yielding the efficiency estimate. Finally, several numerical examples confirming the theoretical properties of the estimators and illustrating the performance of the associated adaptive algorithms, are reported. In particular, the case of flow through a 3D porous media with channel networks is considered.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-29 datetime="2022-09-29T18:00:00.000Z">2022-09-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-29 datetime="2022-09-29T18:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/processing-waste-biomass-reduce-airborne-emissions-0929"
                        open
                      >
                        <summary class="article-expander__title">Processing waste biomass to reduce airborne emissions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/processing-waste-biomass-reduce-airborne-emissions-0929">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/Takachar-hay-field-MITEI.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MIT spinoff Takachar converts agricultural waste into clean-burning fuel, and wins Earthshot Prize.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00390"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Biofoundry Palette: Planning-Assistant Software for Liquid Handler-Based Experimentation and Operation in the Biofoundry Workflow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00390">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00390</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00360"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Hotspots and Mechanisms of Action of the Thermostable Framework of a Microbial Thermolipase</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00360">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00360</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02009-0"
                        open
                      >
                        <summary class="article-expander__title">Multi-Dimensional Image Recovery via Fully-Connected Tensor Network Decomposition Under the Learnable Transforms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02009-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Multi-dimensional image recovery from incomplete data is a fundamental problem in data processing. Due to its advantage of capturing the correlations between any modes of the multi-dimensional image, i.e., the target tensor, the fully-connected tensor network (FCTN) decomposition has recently shown promising performance on multi-dimensional image recovery. However, FCTN decomposition suffers from computational deficiency, especially for large-scale multi-dimensional images. To address this deficiency, we propose a learnable transform-based FCTN model (termed as T-FCTN), which enjoys the remarkable advantage of FCTN decomposition with cheap computational cost. More concretely, we learn the semi-orthogonal transforms along each mode of the target tensor to project the large-scale te…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02002-7"
                        open
                      >
                        <summary class="article-expander__title">The Spectral Difference Raviart–Thomas Method for Two and Three-Dimensional Elements and Its Connection with the Flux Reconstruction Formulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02002-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The purpose of this work is to describe in detail the development of the spectral difference Raviart–Thomas (SDRT) formulation for two and three-dimensional tensor-product elements and simplexes. Through the process, the authors establish the equivalence between the SDRT method and the flux reconstruction (FR) approach under the assumption of the linearity of the flux and the mesh uniformity. Such a connection allows building a new family of FR schemes for two and three-dimensional simplexes and also to recover the well-known FR-SD method with tensor-product elements. In addition, a thorough analysis of the numerical dissipation and dispersion of both aforementioned schemes and the nodal discontinuous Galerkin FR (FR-DG) method with two and three-dimensional elements is proposed through the use of the combined-mode Fourier approach. SDRT is shown to possess an enhanced temporal linear stability in comparison to FR-DG. On the contrary, SDRT displays larger dissipation and dispersion errors with respect to FR-DG. Finally, the study is concluded with a set of numerical experiments, the linear advection-diffusion problem, the Isentropic Euler Vortex, and the Taylor-Green Vortex (TGV). The latter test case shows that SDRT schemes present a non-linear unstable behavior with simplex elements and certain polynomial degrees. For the sake of completeness, the matrix form of the SDRT method is developed and the computational performance of SDRT with respect to FR schemes is evaluated using GPU architectures.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02006-3"
                        open
                      >
                        <summary class="article-expander__title">Tensor Completion via A Generalized Transformed Tensor T-Product Decomposition Without t-SVD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02006-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Matrix and tensor nuclear norms have been successfully used to promote the low-rankness of tensors in low-rank tensor completion. However, singular value decomposition (SVD), which is computationally expensive for large-scale matrices, frequently appears in solving those nuclear norm minimization models. Based on the tensor-tensor product (T-product), in this paper, we first establish the equivalence between the so-called transformed tubal nuclear norm for a third-order tensor and the minimum of the sum of two factor tensors’ squared Frobenius norms under a general invertible linear transform. Gainfully, we introduce a mode-unfolding (often named as “spatio-temporal” in the internet traffic data recovery literature) regularized tensor completion model that is able to efficiently exploit the hidden structures of tensors. Then, we propose an implementable alternating minimization algorithm to solve the underlying optimization model. It is remarkable that our approach does not require any SVDs and all subproblems of our algorithm enjoy closed-form solutions. A series of numerical experiments on traffic data recovery, color images and videos inpainting demonstrate that our SVD-free approach takes less computing time to achieve satisfactory accuracy than some state-of-the-art tensor nuclear norm minimization approaches.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-28 datetime="2022-09-28T04:00:00.000Z">2022-09-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-28 datetime="2022-09-28T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00391"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineered Protein Nanocages for Concurrent RNA and Protein Packaging In Vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00391">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00391</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00337"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Induced CD45 Proximity Potentiates Natural Killer Cell Receptor Antagonism</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00337">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00337</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01794-4"
                        open
                      >
                        <summary class="article-expander__title">From individual-based epidemic models to McKendrick-von Foerster PDEs: a guide to modeling and inferring COVID-19 dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01794-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We present a unifying, tractable approach for studying the spread of viruses causing complex diseases requiring to be modeled using a large number of types (e.g., infective stage, clinical state, risk factor class). We show that recording each infected individual’s infection age, i.e., the time elapsed since infection, has three benefits. First, regardless of the number of types, the age distribution of the population can be described by means of a first-order, one-dimensional partial differential equation (PDE) known as the McKendrick-von Foerster equation. The frequency of type i is simply obtained by integrating the probability of being in state i at a given age against the age distribution. This representation induces a simple methodology based on the additional assumption of Poisson sampling to infer and forecast the epidemic. We illustrate this technique using French data from the COVID-19 epidemic. Second, our approach generalizes and simplifies standard compartmental models using high-dimensional systems of ordinary differential equations (ODEs) to account for disease complexity. We show that such models can always be rewritten in our framework, thus, providing a low-dimensional yet equivalent representation of these complex models. Third, beyond the simplicity of the approach, we show that our population model naturally appears as a universal scaling limit of a large class of fully stochastic individual-based epidemic models, where the initial condition of the PDE emerges as the limiting age structure of an exponentially growing population starting from a single individual.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;67
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02014-3"
                        open
                      >
                        <summary class="article-expander__title">Optimal Error Estimates of the Local Discontinuous Galerkin Method and High-order Time Discretization Scheme for the Swift–Hohenberg Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02014-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we develop a local discontinuous Galerkin (LDG) method for the Swift–Hohenberg equation. The energy stability and optimal error estimates in 
                      
                      \(L^2\)
                      
                     norm of the semi-discrete LDG scheme are established. To avoid the severe time step restriction of explicit time marching methods, a first-order linear scheme based on the scalar auxiliary variable (SAV) method is employed for temporal discretization. Coupled with the LDG spatial discretization, we achieve a fully-discrete LDG method and prove its energy stability and optimal error estimates. To improve the temporal accuracy, the semi-implicit spectral deferred correction (SDC) method is adapted iteratively. Combining with the SAV method, the SDC method can be linear, high-order accurate and energy stable in our numerical tests. Numerical experiments are presented to verify the theoretical results and to show the efficiency of the proposed methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z">2022-09-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927"
                        open
                      >
                        <summary class="article-expander__title">Neurodegenerative disease can progress in newly identified patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/neuron-cell.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine-learning method finds patterns of health decline in ALS, informing future clinical trial designs and mechanism discovery. The technique also extends to Alzheimer’s and Parkinson’s.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M1377771?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Manifold Learning and Nonlinear Homogenization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M1377771?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1093-1126, September 2022. 
 Abstract. We describe an efficient domain decomposition-based framework for nonlinear multiscale PDE problems. The framework is inspired by manifold learning techniques and exploits the tangent spaces spanned by the nearest neighbors to compress local solution manifolds. Our framework is applied to a semilinear elliptic equation with oscillatory media and a nonlinear radiative transfer equation; in both cases, significant improvements in efficacy are observed. This new method does not rely on a detailed analytical understanding of multiscale PDEs, such as their asymptotic limits, and thus is more versatile for general multiscale problems.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1439808?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Online Multiscale Model Reduction for Nonlinear Stochastic PDEs with Multiplicative Noise</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1439808?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1063-1092, September 2022. 
 Abstract. In this paper, an online multiscale model reduction method is presented for stochastic partial differential equations (SPDEs) with multiplicative noise, where the diffusion coefficient is spatially multiscale and the noise perturbation nonlinearly depends on the diffusion dynamics. It is necessary to efficiently compute all possible trajectories of the stochastic dynamics for quantifying a model’s uncertainty and statistic moments. The multiscale diffusion and nonlinearity may cause the computation to become very intractable. To overcome the multiscale difficulty, a constraint energy minimizing generalized multiscale finite element method (CEM-GMsFEM) is used to localize the computation and obtain an effective coarse model. However, the nonlinear terms are still defined on a fine scale space after the Galerkin projection of CEM-GMsFEM is applied to the nonlinear SPDEs. This significantly impacts the simulation efficiency by CEM-GMsFEM. To this end, a stochastic online discrete empirical interpolation method (DEIM) is proposed to treat the stochastic nonlinearity. The stochastic online DEIM incorporates offline snapshots and online snapshots. The offline snapshots consist of the nonlinear terms at the approximate mean of the stochastic dynamics and are used to construct an offline reduced model. The online snapshots contain some information of the current new trajectory and are used to correct the offline reduced model in an incremental manner. The stochastic online DEIM substantially reduces the dimension of the nonlinear dynamics and enhances the prediction accuracy for the reduced model. Thus, the online multiscale model reduction is constructed by using CEM-GMsFEM and the stochastic online DEIM. A priori error analysis is carried out for the nonlinear SPDEs. We present a few numerical examples with diffusion in heterogeneous porous media and show the effectiveness of the proposed model reduction.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1464592?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Hyperbolic Systems of Moment Equations Describing Sedimentation in Suspensions of Rod-Like Particles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1464592?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1002-1039, September 2022. 
 Abstract. We study mathematical models and approximations for the sedimentation in suspensions of rod-like particles. The starting point of our considerations is a coupled system of partial differential equations, previously derived by Helzel and Tzavaras [Multiscale Model.  Simul., 15 (2017), pp.  500–536], consisting of a kinetic equation for the rod orientation that is coupled to a macroscopic flow equation. It describes the motion of a suspension of rigid rod-like particles under the influence of gravity. Since the coupled system is high dimensional (five dimensions plus time) we are interested in the derivation of models which can adaptively adjust the level of detail. Here we restrict ourselves to simplified lower dimensional flow situations and derive hierarchies of moment equations which can be interpreted as approximations of the coupled kinetic model. We show that the resulting systems of moment equations are hyperbolic.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M146995X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Improved Uniform Error Bounds on Time-Splitting Methods for the Long-Time Dynamics of the Dirac Equation with Small Potentials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M146995X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1040-1062, September 2022. 
 Abstract. We establish improved uniform error bounds on time-splitting methods for the long-time dynamics of the Dirac equation with small electromagnetic potentials characterized by a dimensionless parameter [math] representing the amplitude of the potentials. We begin with a semidiscretization of the Dirac equation in time by a time-splitting method, followed by a full-discretization in space by with the Fourier pseudospectral method in space. By employing the unitary flow property of the second-order time-splitting method for the Dirac equation, we prove uniform error bounds for [math] at [math] and [math] for the semidiscretization and full-discretization, respectively, for any time [math] with [math] and [math]. In the expressions, [math] is the time step, [math] is the mesh size, [math] depends on the regularity of the solution, and [math] grows at most linearly with respect to [math] with [math] and [math] two constants independent of [math], [math], [math], and [math]. Then by adopting the regularity compensation oscillation technique, which controls the high frequency modes by the regularity of the solution and low frequency modes by phase cancellation and energy method, we establish improved uniform error bounds at [math] and [math] for the semidiscretization and full-discretization, respectively, up to the long-time [math]. Numerical results are reported to confirm our error bounds and to demonstrate that they are sharp. Comparisons on the accuracy of different time discretizations for the Dirac equation are also provided.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00389"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Discovery and Heterologous Expression of Microginins from Microcystis aeruginosa LEGE 91341</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00389">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00389</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00287"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Actomyosin-Driven Division of a Synthetic Cell</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00287">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00287</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03999-1"
                        open
                      >
                        <summary class="article-expander__title">Phospholipid synthesis inside phospholipid membrane vesicles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03999-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03999-1/MediaObjects/42003_2022_3999_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A giant vesicle system combines fatty acid synthesis pathways, CoA recycling, and cell-free acyltransferase gene expression to perform phospholipid synthesis inside.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01811-6"
                        open
                      >
                        <summary class="article-expander__title">Noise-driven bifurcations in a neural field system modelling networks of grid cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01811-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The activity generated by an ensemble of neurons is affected by various noise sources. It is a well-recognised challenge to understand the effects of noise on the stability of such networks. We demonstrate that the patterns of activity generated by networks of grid cells emerge from the instability of homogeneous activity for small levels of noise. This is carried out by analysing the robustness of network activity patterns with respect to noise in an upscaled noisy grid cell model in the form of a system of partial differential equations. Inhomogeneous network patterns are numerically understood as branches bifurcating from unstable homogeneous states for small noise levels. We show that there is a phase transition occurring as the level of noise decreases. Our numerical study also indicates the presence of hysteresis phenomena close to the precise critical noise value.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02007-2"
                        open
                      >
                        <summary class="article-expander__title">An Efficient QSC Approximation of Variable-Order Time-Fractional Mobile-Immobile Diffusion Equations with Variably Diffusive Coefficients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02007-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we propose a QSC-L1 method to solve the two-dimensional variable-order time-fractional mobile-immobile diffusion (TF-MID) equations with variably diffusive coefficients, in which the quadratic spline collocation (QSC) method is employed for the spatial discretization, and the classical L1 formula is used for the temporal discretization. We show that the method is unconditionally stable and convergent with first-order in time and second-order in space with respect to some discrete and continuous 
                      
                      \(L^2\)
                      
                     norms. Then, combined with the reduced basis technique, an efficient QSC-L1-RB method is proposed to improve the computational efficiency. Numerical examples are attached to verify the convergence orders, and also the method is applied to identify parameters of the variable-order TF-MID equations. Numerical results confirm the contributions of the reduced basis technique, even when the observation data is contaminated by some levels of random noise.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02003-6"
                        open
                      >
                        <summary class="article-expander__title">A Sixth-Order Quasi-Compact Difference Scheme for Multidimensional Poisson Equations Without Derivatives of Source Term</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02003-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Sixth-order compact difference schemes for Poisson equations have been widely investigated in the literature. Nevertheless, those methods are all constructed based on knowing the exact values of the derivatives of the source term. Therefore, this drawback mostly prevents their actual applications as the analytic form of the source term is rarely available. In this paper, we propose a sixth-order quasi-compact difference method, without having to know the derivatives of the source term, for solving the 2D and 3D Poisson equations. Our strategy is to discretize the equation by the fourth-order compact scheme at the improper interior grid points that adjoin the boundary, while the sixth-order scheme, where it is compact only for the unknowns, is exploited to the proper interior grid points that are not adjoining the boundary. Theoretically, we rigorously prove that the proposed method can achieve the global sixth-order accuracy. Since there are no derivatives of the source term involved in the proposed scheme, our global sixth-order quasi-compact difference method can be developed to solve the time-dependent problems using a time advancing scheme. Numerical experiments are carried out to demonstrate the convergence order and the efficiency of the proposed methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-26 datetime="2022-09-26T04:00:00.000Z">2022-09-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-26 datetime="2022-09-26T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00235"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Reprogramming the Cleavage Specificity of Botulinum Neurotoxin Serotype B1</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00235">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00235</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00156"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Quantification of Genome Editing and Transcriptional Control Capabilities Reveals Hierarchies among Diverse CRISPR/Cas Systems in Human Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00156</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01136-x"
                        open
                      >
                        <summary class="article-expander__title">Chemically inducible split protein regulators for mammalian cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01136-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01136-x/MediaObjects/41589_2022_1136_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>An INSPIRE platform for therapeutic regulators and sensors, based on split human domains binding physiological or clinically approved compounds, has been developed and demonstrated on several human proteins in a relevant ligand concentration range.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33221-z"
                        open
                      >
                        <summary class="article-expander__title">The material properties of a bacterial-derived biomolecular condensate tune biological function in natural and synthetic systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33221-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33221-z/MediaObjects/41467_2022_33221_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>“Intracellular phase separation is emerging as a universal principle for organizing biochemical reactions in time and space. Here the authors show that PopZ condensate dynamics support cell division and using PopZ modular architecture, the tunable PopTag platform was developed to enable designer condensates.”</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01793-5"
                        open
                      >
                        <summary class="article-expander__title">Scaling relations for auxin waves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01793-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We analyze an ‘up-the-gradient’ model for the formation of transport channels of the phytohormone auxin, through auxin-mediated polarization of the PIN1 auxin transporter. We show that this model admits a family of travelling wave solutions that is parameterized by the height of the auxin-pulse. We uncover scaling relations for the speed and width of these waves and verify these rigorous results with numerical computations. In addition, we provide explicit expressions for the leading-order wave profiles, which allows the influence of the biological parameters in the problem to be readily identified. Our proofs are based on a generalization of the scaling principle developed by Friesecke and Pego to construct pulse solutions to the classic Fermi–Pasta–Ulam–Tsingou model, which describes a one-dimensional chain of coupled nonlinear springs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;87
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01806-3"
                        open
                      >
                        <summary class="article-expander__title">Inference in Gaussian state-space models with mixed effects for multiple epidemic dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01806-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The estimation from available data of parameters governing epidemics is a major challenge. In addition to usual issues (data often incomplete and noisy), epidemics of the same nature may be observed in several places or over different periods. The resulting possible inter-epidemic variability is rarely explicitly considered. Here, we propose to tackle multiple epidemics through a unique model incorporating a stochastic representation for each epidemic and to jointly estimate its parameters from noisy and partial observations. By building on a previous work for prevalence data, a Gaussian state-space model is extended to a model with mixed effects on the parameters describing simultaneously several epidemics and their observation process. An appropriate inference method is developed, by coupling the SAEM algorithm with Kalman-type filtering. Moreover, we consider here incidence data, which requires to develop a new version of the filtering algorithm. Its performances are investigated on SIR simulated epidemics for prevalence and incidence data. Our method outperforms an inference method separately processing each dataset. An application to SEIR influenza outbreaks in France over several years using incidence data is also carried out. Parameter estimations highlight a non-negligible variability between influenza seasons, both in transmission and case reporting. The main contribution of our study is to rigorously and explicitly account for the inter-epidemic variability between multiple outbreaks, both from the viewpoint of modeling and inference with a parsimonious statistical model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-24 datetime="2022-09-24T00:00:00.000Z">2022-09-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-24 datetime="2022-09-24T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33288-8"
                        open
                      >
                        <summary class="article-expander__title">Synthetic neuromorphic computing in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33288-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33288-8/MediaObjects/41467_2022_33288_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Computational properties of neuronal networks have been applied to computing systems using simplified models comprising repeated connected nodes. Here the authors create layered assemblies of genetically encoded devices that perform non-binary logic computation and signal processing using combinatorial promoters and feedback regulation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03987-5"
                        open
                      >
                        <summary class="article-expander__title">Targeting non-coding RNA family members with artificial endonuclease XNAzymes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03987-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03987-5/MediaObjects/42003_2022_3987_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Engineered RNA endonuclease XNAzymes can specifically target individual disease-associated non-coding RNAs (ncRNAs) in vitro with potential as platform technology for precision detection or knockdown of individual microRNAs or longer ncRNAs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02012-5"
                        open
                      >
                        <summary class="article-expander__title">A New Adaptation Strategy for Multi-resolution Method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02012-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Adaptive mesh refinement (AMR) and wavelet-based multi-resolution technique, which refine the spatial resolution in regions of interest and coarsen the mesh in other regions, are widely used in scientific computing for higher computational efficiency. The key component of AMR and the multi-resolution method is the adaptation strategy including the regularity estimation. In this paper, a new adaptation strategy for AMR and the multi-resolution method is proposed. Different from AMR, which measures the function regularity based on an empirical gradient operator, and the multi-resolution method with complicated wavelet analysis, the new approach examines the solution smoothness based on the high-order TENO reconstruction (Fu et al. in J Comput Phys 305:333–359, 2016). The core idea is that the TENO scheme not only provides the reconstructed data at the cell interface for flux evaluation but also classifies the local flow scales as smooth or nonsmooth on the discretized mesh. Since the scale-separation procedure is achieved in the spectral and characteristic space, the new adaptation strategy is weakly problem-dependent. Moreover, the extra complexity due to the empirical gradient computing or the wavelet analysis is eliminated. Based on Harten’s finite-volume multi-resolution method (Harten in J Comput Phys 115(2):319–338, 1994), a set of benchmark cases is simulated to validate the performance of the proposed method.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-23 datetime="2022-09-23T04:00:00.000Z">2022-09-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-23 datetime="2022-09-23T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00291"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineered Spore-Forming Bacillus as a Microbial Vessel for Long-Term DNA Data Storage</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00291">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00291</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00216"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Novel Split Intein-Mediated Enzymatic Channeling Accelerates the Multimeric Bioconversion Pathway of Ginsenoside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00216">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00216</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33181-4"
                        open
                      >
                        <summary class="article-expander__title">Activation of stably silenced genes by recruitment of a synthetic de-methylating module</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33181-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33181-4/MediaObjects/41467_2022_33181_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stably silenced genes with methylated CpG at the promoter are refractory to current CRISPR activation systems. Here the authors create a more robust activation system, TETact that recruits DNA-demethylating TET1 with transcriptional activators.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-22 datetime="2022-09-22T15:50:00.000Z">2022-09-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-22 datetime="2022-09-22T15:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/cell-rover-exploring-augmenting-inner-world-cell-0922"
                        open
                      >
                        <summary class="article-expander__title">Cell Rover: Exploring and augmenting the inner world of the cell</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/cell-rover-exploring-augmenting-inner-world-cell-0922">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/cell-antenna.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MIT researchers demonstrate an intracellular antenna that&#x27;s compatible with 3D biological systems and can operate wirelessly inside a living cell.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00111"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A p‑Coumaroyl-CoA Biosensor for Dynamic Regulation of Naringenin Biosynthesis in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00111">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00111</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01802-7"
                        open
                      >
                        <summary class="article-expander__title">Locked fronts in a discrete time discrete space population model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01802-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A model of population growth and dispersal is considered where the spatial habitat is a lattice and reproduction occurs generationally. The resulting discrete dynamical system exhibits velocity locking, where rational speed invasion fronts are observed to persist as parameters are varied. In this article, we construct locked fronts for a particular piecewise linear reproduction function. These fronts are shown to be linear combinations of exponentially decaying solutions to the linear system near the unstable state. Based upon these front solutions, we then derive expressions for the boundary of locking regions in parameter space. We obtain leading order expansions for the locking regions in the limit as the migration parameter tends to zero. Strict spectral stability in exponentially weighted spaces is also established.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-21 datetime="2022-09-21T04:00:00.000Z">2022-09-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-21 datetime="2022-09-21T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00365"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Light-Dependent Control of Bacterial Expression at the mRNA Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00365">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00365</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00638"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Precise Genomic Riboregulator Control of Metabolic Flux in Microbial Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00638">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00638</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33191-2"
                        open
                      >
                        <summary class="article-expander__title">A de novo matrix for macroscopic living materials from bacteria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33191-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33191-2/MediaObjects/41467_2022_33191_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Engineered living materials (ELMs) embed living cells in a biopolymer matrix to create novel materials with tailored functions. In this work, the authors engineered bacteria to grow novel macroscopic materials that can be reshaped, functionalized, and used to filter contaminated water while also showing that the stiffness of these materials can be tuned through genetic changes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01805-4"
                        open
                      >
                        <summary class="article-expander__title">Analysis of the onset of a regime shift and detecting early warning signs of major population changes in a two-trophic three-species predator-prey model with long-term transients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01805-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Identifying early warning signs of sudden population changes and mechanisms leading to regime shifts are highly desirable in population biology. In this paper, a two-trophic ecosystem comprising of two species of predators, competing for their common prey, with explicit interference competition is considered. With proper rescaling, the model is portrayed as a singularly perturbed system with fast prey dynamics and slow dynamics of the predators. In a parameter regime near singular Hopf bifurcation, chaotic mixed-mode oscillations (MMOs), featuring concatenation of small and large amplitude oscillations are observed as long-lasting transients before the system approaches its asymptotic state. To analyze the dynamical cause that initiates a large amplitude oscillation in an MMO orbit, the model is reduced to a suitable normal form near the singular-Hopf point. The normal form possesses a separatrix surface that separates two different types of oscillations. A large amplitude oscillation is initiated if a trajectory moves from the “inner” to the “outer side” of this surface. A set of conditions on the normal form variables are obtained to determine whether a trajectory would exhibit another cycle of MMO dynamics before experiencing a regime shift (i.e. approaching its asymptotic state). These conditions serve as early warning signs for a sudden population shift as well as detect the onset of a regime shift in this ecological model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01978-6"
                        open
                      >
                        <summary class="article-expander__title">A Multipoint Flux Approximation with a Diamond Stencil and a Non-Linear Defect Correction Strategy for the Numerical Solution of Steady State Diffusion Problems in Heterogeneous and Anisotropic Media Satisfying the Discrete Maximum Principle</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01978-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In the present paper, we solve the steady state diffusion equation in 3D domains by means of a cell-centered finite volume method that uses a Multipoint Flux Approximation with a Diamond Stencil and a Non-Linear defect correction strategy (MPFA-DNL) to guarantee the Discrete Maximum Principle (DMP). Our formulation is based in the fact that the flux of MPFA methods can be split into two different parts: a Two Point Flux Approximation (TPFA) component and the Cross-Diffusion Terms (CDT). In the linear MPFA-D method, this split is particularly simple since it lies at the core of the original method construction. In this context, we introduce a non-linear defect correction, aiming to mitigate, whenever necessary, the contributions from the CDT, avoiding, this way, spurious oscillations and DMP violations. Our new MPFA-DNL scheme is locally conservative and capable of dealing with arbitrary anisotropic diffusion tensors and unstructured meshes, without harming the second order convergence rates of the original MPFA-D. To appraise the accuracy and robustness of our formulation, we solve some benchmark problems found in literature. In this paper, we restrict ourselves to tetrahedral meshes, even though, in principle, there is no restriction to extend the method to other polyhedral control volumes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-20 datetime="2022-09-20T07:00:00.000Z">2022-09-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-20 datetime="2022-09-20T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0507?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">virDTL: Viral Recombination Analysis Through Phylogenetic Reconciliation and Its Application to Sarbecoviruses and SARS-CoV-2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0507?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00159"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A Versatile in Vivo DNA Assembly Toolbox for Fungal Strain Engineering</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00159">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00159</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33207-x"
                        open
                      >
                        <summary class="article-expander__title">Screening microbially produced Δ9-tetrahydrocannabinol using a yeast biosensor workflow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33207-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33207-x/MediaObjects/41467_2022_33207_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Microbial production of cannabinoids promises a cheaper and more sustainable route to these important therapeutic molecules, but strain improvement and screening is challenging. Here, the authors develop a yeast-based Δ9-tetrahydrocannabinol (THC) biosensor for screening microbial mutant libraries.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01801-8"
                        open
                      >
                        <summary class="article-expander__title">Randomization for the susceptibility effect of an infectious disease intervention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01801-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Randomized trials of infectious disease interventions, such as vaccines, often focus on groups of connected or potentially interacting individuals. When the pathogen of interest is transmissible between study subjects, interference may occur: individual infection outcomes may depend on treatments received by others. Epidemiologists have defined the primary parameter of interest—called the “susceptibility effect”—as a contrast in infection risk under treatment versus no treatment, while holding exposure to infectiousness constant. A related quantity—the “direct effect”—is defined as an unconditional contrast between the infection risk under treatment versus no treatment. The purpose of this paper is to show that under a widely recommended randomization design, the direct effect may fail to recover the sign of the true susceptibility effect of the intervention in a randomized trial when outcomes are contagious. The analytical approach uses structural features of infectious disease transmission to define the susceptibility effect. A new probabilistic coupling argument reveals stochastic dominance relations between potential infection outcomes under different treatment allocations. The results suggest that estimating the direct effect under randomization may provide misleading conclusions about the effect of an intervention—such as a vaccine—when outcomes are contagious. Investigators who estimate the direct effect may wrongly conclude an intervention that protects treated individuals from infection is harmful, or that a harmful treatment is beneficial.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01804-5"
                        open
                      >
                        <summary class="article-expander__title">Limits of epidemic prediction using SIR models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01804-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The Susceptible-Infectious-Recovered (SIR) equations and their extensions comprise a commonly utilized set of models for understanding and predicting the course of an epidemic. In practice, it is of substantial interest to estimate the model parameters based on noisy observations early in the outbreak, well before the epidemic reaches its peak. This allows prediction of the subsequent course of the epidemic and design of appropriate interventions. However, accurately inferring SIR model parameters in such scenarios is problematic. This article provides novel, theoretical insight on this issue of practical identifiability of the SIR model. Our theory provides new understanding of the inferential limits of routinely used epidemic models and provides a valuable addition to current simulate-and-check methods. We illustrate some practical implications through application to a real-world epidemic data set.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-19 datetime="2022-09-19T07:00:00.000Z">2022-09-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-19 datetime="2022-09-19T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1461277?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">High Order Topological Asymptotics: Reconciling Layer Potentials and Compound Asymptotic Expansions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1461277?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 957-1001, September 2022. 
 Abstract. A systematic two-step procedure is proposed for the derivation of full asymptotic expansions of the solution of elliptic partial differential equations set on a domain perforated with a small hole on which a Dirichlet boundary condition is applied. First, an integral representation of the solution is sought, which enables us to exploit the explicit dependence with respect to the small parameter to predict the correct form of a two-scale ansatz. Second, the terms of the ansatz are characterized by the method of compound asymptotic expansions as the solutions of a cascade of successive interior and exterior domains. This allows us to interpret them as high-order correctors, for which error bounds can be proved using variational estimates. The methodology is illustrated on two different problems: we start by revisiting the perforated Poisson problem with Dirichlet boundary conditions on both the hole and the outer boundary, where we highlight how the method enables us to retrieve very naturally the correct form of the ansatz in the most delicate two-dimensional setting. Then, we provide original and complete asymptotic expansions for a perforated cell problem featuring periodic conditions.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00289"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Transcriptional Reactivation of Lignin Biosynthesis for the Heterologous Production of Etoposide Aglycone in Nicotiana benthamiana</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00289">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00289</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33255-3"
                        open
                      >
                        <summary class="article-expander__title">Biosynthesis of plant hemostatic dencichine in Escherichia coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33255-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33255-3/MediaObjects/41467_2022_33255_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Biosynthetic pathway of dencichine, a plant derived nature product that has found various pharmacological applications, is still elusive. Here, the authors design artificial pathways through retro-biosynthesis approaches and achieve its efficient production in E. coli by systematic metabolic and enzymatic engineering.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01809-0"
                        open
                      >
                        <summary class="article-expander__title">On the qualitative study of a two-trophic plant–herbivore model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01809-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The coexistence of plant-herbivore populations in an ecological system is a fundamental topic of research in mathematical ecology. Plant–herbivore interactions are often described by using discrete-time models in the case of non-overlapping generations: such generations have some specific time interval of life and their old generations are replaced by new generations after some regular interval of time. Keeping in mind the dynamical reliability of continuous-time models we presented two discrete-time plant–herbivore models. Mainly, by applying Euler’s forward method a discrete-time plant–herbivore model is obtained from a continuous-time plant–herbivore model. In addition, a dynamically consistent discrete-time plant–herbivore model is obtained by applying a nonstandard difference scheme. Moreover, local stability is discussed and the existence of bifurcation about positive equilibrium is shown under some mathematical conditions. To control bifurcation and chaos, a modified hybrid technique is developed. Finally, to support our theocratical results and to show the dynamical reliability of the nonstandard difference scheme some numerical examples are provided.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01799-z"
                        open
                      >
                        <summary class="article-expander__title">Vertex results for the robust analysis of uncertain biochemical systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01799-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We consider the problem of assessing the sensitivity of uncertain biochemical systems in the presence of input perturbations (either constant or periodic) around a stable steady state. 

       In particular, we propose approaches for the robust sensitivity analysis of systems with uncertain parameters assumed to take values in a hyper-rectangle. We highlight vertex results, which allow us to check whether a property is satisfied for all parameter choices in the hyper-rectangle by simply checking whether it is satisfied for all parameter choices at the vertices of the hyper-rectangle. We show that, for a vast class of systems, including (bio)chemical reaction networks with mass-action kinetics, the system Jacobian has a totally multiaffine structure (namely, all minors of the Jacobian matrix are multiaffine functions of the uncertain parameters), which can be exploited to obtain several vertex results. We consider different problems: robust non-singularity; robust stability of the steady-state; robust steady-state sensitivity analysis, in the case of constant perturbations; robust frequency-response sensitivity analysis, in the presence of periodic perturbations; and robust adaptation analysis. The developed theory is then applied to gain insight into some examples of uncertain biochemical systems, including the incoherent feed-forward loop, the coherent feed-forward loop, the Brusselator oscillator and the Goldbeter oscillator.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;65
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-17 datetime="2022-09-17T00:00:00.000Z">2022-09-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-17 datetime="2022-09-17T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01788-2"
                        open
                      >
                        <summary class="article-expander__title">A model and a numerical scheme for the description of distribution and abundance of individuals</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01788-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We introduce a model in the context of ecology that can be used to describe the distribution and abundance of individuals when data from field work is extremely limited (for example, in the case of endangered species). Our procedure is based on an intuitive understanding of the physical properties of phenomena. The idea is that individuals have the tendency to be attracted (or repulsed) to certain properties of the environment. At the same time, they are spread in such a way that if there is no reason for them to be in some specific locations, then they are uniformly distributed throughout the region. Our model draws from quantum mechanics, by using quantum Hamiltonians in the context of classical statistical mechanics. The equilibrium between the spreading and the attractive (or repulsive) forces determines the behavior of the species that we model, and this is expressed in terms of a global control problem of an energy operator which is the sum of a kinetic term (spreading) and a potential (attraction or repulsion). We focus on the full probability measure and a global control of the model (instead of looking at conditional measures that generate a global measure). Furthermore, we propose a numerical solution to this global control problem that overcomes the well-known major difficulty of Gibbs sampling (annealing) which is the fact that a global control is hardly reachable when the number of variables is large (the algorithms get stuck in non-optimal states).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01808-1"
                        open
                      >
                        <summary class="article-expander__title">Efficiency functionals for the Lévy flight foraging hypothesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01808-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We consider a forager diffusing via a fractional heat equation and we introduce several efficiency functionals whose optimality is discussed in relation to the Lévy exponent of the evolution equation. Several biological scenarios, such as a target close to the forager, a sparse environment, a target located away from the forager and two targets are specifically taken into account. The optimal strategies of each of these configurations are here analyzed explicitly also with the aid of some special functions of classical flavor and the results are confronted with the existing paradigms of the Lévy foraging hypothesis. Interestingly, one discovers bifurcation phenomena in which a sudden switch occurs between an optimal (but somehow unreliable) Lévy foraging pattern of inverse square law type and a less ideal (but somehow more secure) classical Brownian motion strategy. Additionally, optimal foraging strategies can be detected in the vicinity of the Brownian one even in cases in which the Brownian one is pessimizing an efficiency functional.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01803-6"
                        open
                      >
                        <summary class="article-expander__title">A mathematical model for simulating the spread of a disease through a country divided into geographical regions with different population densities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01803-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The SIR (susceptible-infectious-recovered) model is a well known method for predicting the number of people (or animals) in a population who become infected by and then recover from a disease. Modifications can include categories such people who have been exposed to the disease but are not yet infectious or those who die from the disease. However, the model has nearly always been applied to the entire population of a country or state but there is considerable observational evidence that diseases can spread at different rates in densely populated urban regions and sparsely populated rural areas. This work presents a new approach that applies a SIR type model to a country or state that has been divided into a number of geographical regions, and uses different infection rates in each region which depend on the population density in that region. Further, the model contains a simple matrix based method for simulating the movement of people between different regions. The model is applied to the spread of disease in the United Kingdom and the state of Rio Grande do Sul in Brazil.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-16 datetime="2022-09-16T07:00:00.000Z">2022-09-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-16 datetime="2022-09-16T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0292?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Novel Network Method Major Minor Variation Clustering Enables Identification of Poliovirus Clusters with High-Resolution Linkages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0292?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33111-4"
                        open
                      >
                        <summary class="article-expander__title">Unleashing the potential of noncanonical amino acid biosynthesis to create cells with precision tyrosine sulfation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33111-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33111-4/MediaObjects/41467_2022_33111_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Incorporation of noncanonical amino acids into proteins holds great promise for altering structure and function of these proteins. Here the authors generate metabolically modified prokaryotic and eukaryotic cells that can biosynthesize sTyr and incorporate it into proteins in a site-specific manner.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01792-6"
                        open
                      >
                        <summary class="article-expander__title">The hybrid number of a ploidy profile</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01792-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Polyploidization, whereby an organism inherits multiple copies of the genome of their parents, is an important evolutionary event that has been observed in plants and animals. One way to study such events is in terms of the ploidy number of the species that make up a dataset of interest. It is therefore natural to ask: How much information about the evolutionary past of the set of species that form a dataset can be gleaned from the ploidy numbers of the species? To help answer this question, we introduce and study the novel concept of a ploidy profile which allows us to formalize it in terms of a multiplicity vector indexed by the species the dataset is comprised of. Using the framework of a phylogenetic network, we present a closed formula for computing the hybrid number (i.e. the minimal number of polyploidization events required to explain a ploidy profile) of a large class of ploidy profiles. This formula relies on the construction of a certain phylogenetic network from the simplification sequence of a ploidy profile and the hybrid number of the ploidy profile with which this construction is initialized. Both of them can be computed easily in case the ploidy numbers that make up the ploidy profile are not too large. To help illustrate the applicability of our approach, we apply it to a simplified version of a publicly available Viola dataset.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01991-9"
                        open
                      >
                        <summary class="article-expander__title">A Discontinuous Galerkin Overset Scheme Using WENO Reconstruction and Subcells for Two-Dimensional Problems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01991-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A new scheme for communication between overset grids using subcells and weighted essentially non oscillatory (WENO) reconstruction for two-dimensional problems has been proposed. The effectiveness of this procedure is demonstrated using the discontinuous Galerkin method. This scheme uses a standard WENO reconstruction by dividing the immediate neighbors into subcells to find the degrees of freedom in cells near the overset interface. This also has the added advantage that it also works as a limiter if a discontinuity passes through the overset interface. Accuracy tests to demonstrate the maintenance of higher order are provided. Results containing shocks are also provided to demonstrate the limiter aspect of the data communication procedure.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01999-1"
                        open
                      >
                        <summary class="article-expander__title">A Kernel-Independent Sum-of-Exponentials Method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01999-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We propose an accurate algorithm for a novel sum-of-exponentials (SOE) approximation of kernel functions, and develop a fast algorithm for convolution quadrature based on the SOE, which allows an order N calculation for N time steps of approximating a continuous temporal convolution integral. The SOE method is constructed by a combination of the de la Vallée-Poussin sum for a semi-analytical exponential expansion of a general kernel, and a model reduction technique to minimize the number of exponentials under a given error tolerance. We employ the SOE expansion for the finite part of the splitting convolution kernel so that the convolution integral can be solved as a system of ordinary differential equations. We show that the SOE method works for general kernels with controllable upperbound of positive exponents. Numerical analysis is provided for both the SOE method and the SOE-based convolution quadrature. Numerical results on different kernels demonstrate attractive performance on both accuracy and efficiency of the proposed method.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01990-w"
                        open
                      >
                        <summary class="article-expander__title">Accurate High-Order Tensor-Product Generalized Summation-By-Parts Discretizations of Hyperbolic Conservation Laws: General Curved Domains and Functional Superconvergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01990-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The goal of this paper is to outline the requirements for obtaining accurate solutions and functionals from high-order tensor-product generalized summation-by-parts discretizations of the steady two-dimensional linear convection and Euler equations on general curved domains. Two procedures for constructing high-order grids using either Lagrange or B-spline mappings are outlined. For the linear convection equation, four discretizations are derived and characterized—two based on the mortar-element approach and two based on the global summation-by-parts-operator approach. It is shown numerically that the schemes are dual consistent, and the requirements for achieving functional superconvergence for each set of methods are outlined. For the Euler equations, a dual-consistent mortar-element discretization is proposed and the practical requirements for obtaining accurate solutions and superconvergent functionals for problems of increasing practical relevance are delineated through theory and numerical examples.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01995-5"
                        open
                      >
                        <summary class="article-expander__title">Arbitrarily High Order and Fully Discrete Extrapolated RK–SAV/DG Schemes for Phase-field Gradient Flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01995-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we construct and analyze a fully discrete method for phase-field gradient flows, which uses extrapolated Runge–Kutta with scalar auxiliary variable (RK–SAV) method in time and discontinuous Galerkin (DG) method in space. We propose a novel technique to decouple the system, after which only several elliptic scalar problems with constant coefficients need to be solved independently. Discrete energy decay property of the method is proved for gradient flows. The scheme can be of arbitrarily high order both in time and space, which is demonstrated rigorously for the Allen–Cahn equation and the Cahn–Hilliard equation. More precisely, optimal 
                      
                      \(L^2\)
                      
                    -error bound in space and qth-order convergence rate in time are obtained for q-stage extrapolated RK–SAV/DG method. Several numerical experiments are carried out to verify the theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01998-2"
                        open
                      >
                        <summary class="article-expander__title">A Convergent Iterated Quasi-interpolation for Periodic Domain and Its Applications to Surface PDEs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01998-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The paper first provides an iterated quasi-interpolation scheme for function approximation over periodic domain and then attempts its applications to solve time-dependent surface partial differential equations (PDEs). The key feature of our scheme is that it gives an approximation directly just by taking a weighted average of the available discrete function values evaluated at sampling centers in the periodic domain. As such, it is simple, easy to compute, and the implementation process is stable. Moreover, if the sampling centers distribute uniformly over the periodic domain, it even preserves the same convergence order to all the derivatives. To employ the iterated quasi-interpolation scheme for solving time-dependent surface PDEs, we follow the framework of the method-of-lines. More precisely, we first reformulate the PDEs in terms of the parametric form of the surface. Then we use our quasi-interpolation scheme to approximate both the analytic solution and its spatial derivatives in the reformulated form (of PDEs) to get a semi-discrete ordinary differential equation (ODE) system. Finally, we adopt an appropriate time-integration technique to obtain the full-discrete scheme. As concrete examples, we take the torus for illustration and solve some benchmark reaction-diffusion equations imposed on the torus at the end of the paper. However, the proposed method is general and works on time-dependent PDEs defined on any smooth closed parameterized surfaces without coordinate singularity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01981-x"
                        open
                      >
                        <summary class="article-expander__title">Optimal Convergence of the Scalar Auxiliary Variable Finite Element Method for the Natural Convection Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01981-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This paper concerns an efficient finite element method for the natural convection equations with the scalar auxiliary variable approach. The linearly extrapolated Crank-Nicolson techniques are used to discretize nonlinear terms in the Navier–Stokes equations and the heat equation. The induced scalar auxiliary equation is an univariate ordinary equation. With the benefit of the new defined system, the redefined stability of the proposed method is obtained under the fully explicit scheme for the nonlinear terms without the requirement of skew-symmetric trilinear forms. The optimal convergence rates in space for all the variables are proved. The second order convergence rates in time are also obtained. Finally, numerical experiments are provided to support the theoretical analysis and demonstrate the efficiency of the proposed scheme.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-15 datetime="2022-09-15T04:00:00.000Z">2022-09-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-15 datetime="2022-09-15T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00247"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Functional Comparison between Endogenous and Synthetic Notch Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00247">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00247</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-14 datetime="2022-09-14T00:00:00.000Z">2022-09-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-14 datetime="2022-09-14T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/d41586-022-02231-8"
                        open
                      >
                        <summary class="article-expander__title">Life brought to artificial cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/d41586-022-02231-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.nature.com/lw1024/magazine-assets/d41586-022-02231-8/d41586-022-02231-8_23476970.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A synthetic system that reproduces several attributes of living cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33058-6"
                        open
                      >
                        <summary class="article-expander__title">Layered feedback control overcomes performance trade-off in synthetic biomolecular networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33058-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33058-6/MediaObjects/41467_2022_33058_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Layered feedback is an optimization strategy in feedback control designs widely used in engineering. Here, combining simulation and experimentation, the authors apply layered control - a powerful optimization strategy in engineering - to synthetic biomolecular networks in living bacteria to show layered control overcomes performance trade-offs in biology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41586-022-05223-w"
                        open
                      >
                        <summary class="article-expander__title">Living material assembly of bacteriogenic protocells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41586-022-05223-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-022-05223-w/MediaObjects/41586_2022_5223_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A bacteriogenic strategy for constructing membrane-bounded, molecularly crowded, and compositionally, structurally and morphologically complex synthetic cells provides opportunities for the fabrication of new synthetic cell modules and augmented living/synthetic cell constructs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-13 datetime="2022-09-13T15:20:00.000Z">2022-09-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-13 datetime="2022-09-13T15:20:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/smart-innovation-center-awarded-nrf-grant-0913"
                        open
                      >
                        <summary class="article-expander__title">SMART Innovation Center awarded five-year NRF grant for new deep tech ventures</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/smart-innovation-center-awarded-nrf-grant-0913">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/IC%20stock%20image%202.jpeg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>By providing researchers with financial and strategic support from the early stages, the Innovation Center hopes to bring new and disruptive technologies to market.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0658?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">BDEtools: A MATLAB Package for Boolean Delay Equation Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0658?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">RESISTOR: A New OSPREY Module to Predict Resistance Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00265"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering of the Translesion DNA Synthesis Pathway Enables Controllable C‑to‑G and C‑to‑A Base Editing in Corynebacterium glutamicum</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00265">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00265</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-12 datetime="2022-09-12T07:00:00.000Z">2022-09-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-12 datetime="2022-09-12T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Differentiable Learning of Sequence-Specific Minimizer Schemes with DeepMinimizer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00195"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Steady-State Operation of a Cell-Free Genetic Band-Detection Circuit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00195">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00195</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00314"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Intestinal Engineered Probiotics as Living Therapeutics: Chassis Selection, Colonization Enhancement, Gene Circuit Design, and Biocontainment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00314">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00314</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33046-w"
                        open
                      >
                        <summary class="article-expander__title">Robust data storage in DNA by de Bruijn graph-based de novo strand assembly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33046-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33046-w/MediaObjects/41467_2022_33046_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA data storage is a rapidly developing technology with great potential due to its high density, long-term durability, and low maintenance cost. Here the authors present a strand assembly algorithm (DBGPS) using de Bruijn graph and greedy path search.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33033-1"
                        open
                      >
                        <summary class="article-expander__title">Kinetic compartmentalization by unnatural reaction for itaconate production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33033-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33033-1/MediaObjects/41467_2022_33033_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Unlike eukaryotic system, bacterial hosts lack membranous system, which is one of the limitations for efficient metabolic engineering. Here, the authors report a kinetic compartmentalization strategy to increase substrate availability from competitive reactions for the efficient production of itaconate in E. coli.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/xlxs4/rss-reader/actions/runs/3216211664">
        <time id="build-timestamp" datetime="2022-10-10T00:58:16.916Z">2022-10-10T00:58:16.916Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>