<!DOCTYPE html>
<html lang="en">

  <head>
    <title>osmos::feed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="osmos::feed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <!-- %before-head-end.html% -->
  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-01 datetime="2022-10-01T01:07:11.861Z">2022-10-01</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-01 datetime="2022-10-01T01:07:11.861Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ScienceDirect Publication: Journal of Differential Equations</button>
                  <a class="source-heading__link" href="https://www.sciencedirect.com/journal/journal-of-differential-equations">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005198"
                        open
                      >
                        <summary class="article-expander__title">The Stokes operator in two-dimensional bounded Lipschitz domains</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005198?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Fabian Gabel, Patrick Tolksdorf</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622004922"
                        open
                      >
                        <summary class="article-expander__title">Decay estimates on Besov and Triebel-Lizorkin spaces of the Stokes flows and the incompressible Navier-Stokes flows in half-spaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622004922?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): The Anh Bui, The Quan Bui, Xuan Thinh Duong</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200506X"
                        open
                      >
                        <summary class="article-expander__title">Fractional Sobolev embedding with radial potential</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200506X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Yu Su, Zhaosheng Feng</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005162"
                        open
                      >
                        <summary class="article-expander__title">On the existence and limit behavior of ground states for two coupled Hartree equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005162?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Miao Du, Siyu Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005149"
                        open
                      >
                        <summary class="article-expander__title">Classification of blow-up and global existence of solutions to an initial Neumann problem</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005149?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Bin Guo, Jingjing Zhang, Wenjie Gao, Menglan Liao</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005113"
                        open
                      >
                        <summary class="article-expander__title">Global classical solutions to a higher-dimensional doubly haptotactic cross-diffusion system modeling oncolytic virotherapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005113?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Jiashan Zheng, Jianing Xie</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005137"
                        open
                      >
                        <summary class="article-expander__title">Stationary distribution of stochastic population dynamics with infinite delay</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005137?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Hao Yang, Fuke Wu, Peter E. Kloeden</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005381"
                        open
                      >
                        <summary class="article-expander__title">Continuity of the integral kernel of Schrödinger propagator</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005381?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Shiliang Zhao, Quan Zheng</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005289"
                        open
                      >
                        <summary class="article-expander__title">Well-posedness of the surface wave problem for two dimensional micropolar fluids</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005289?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Xiaoling Hu, Xiaojing Xu, Difan Yuan</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005344"
                        open
                      >
                        <summary class="article-expander__title">On the Calderón problem for nonlocal Schrödinger equations with homogeneous, directionally antilocal principal symbols</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005344?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Giovanni Covi, María Ángeles García-Ferrero, Angkana Rüland</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005332"
                        open
                      >
                        <summary class="article-expander__title">A class of supercritical Sobolev type inequalities with logarithm and related elliptic equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005332?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Yinbin Deng, Shuangjie Peng, Xinyue Zhang, Yang Zhou</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005290"
                        open
                      >
                        <summary class="article-expander__title">Propagation dynamics of forced pulsating waves of a time periodic Lotka-Volterra competition system in a shifting habitat</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005290?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Hongyong Wang, Chaohong Pan, Chunhua Ou</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005277"
                        open
                      >
                        <summary class="article-expander__title">Sobolev space theory and Hölder estimates for the stochastic partial differential equations on conic and polygonal domains</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005277?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Kyeong-Hun Kim, Kijung Lee, Jinsol Seo</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005150"
                        open
                      >
                        <summary class="article-expander__title">Local solution to an energy critical 2-D stochastic wave equation with exponential nonlinearity in a bounded domain</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005150?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Zdzisław Brzeźniak, Nimit Rana</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005496"
                        open
                      >
                        <summary class="article-expander__title">A note on the weak regularity theory for degenerate Kolmogorov equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005496?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Francesca Anceschi, Annalaura Rebucci</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005356"
                        open
                      >
                        <summary class="article-expander__title">Photo-acoustic inversion using plasmonic contrast agents: The full Maxwell model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005356?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Ahcene Ghandriche, Mourad Sini</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005320"
                        open
                      >
                        <summary class="article-expander__title">Estimates for fundamental solutions of parabolic equations in non-divergence form</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005320?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Hongjie Dong, Seick Kim, Sungjin Lee</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005265"
                        open
                      >
                        <summary class="article-expander__title">On Harnack inequality and Hölder continuity to the degenerate parabolic equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005265?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Farman Mamedov</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200540X"
                        open
                      >
                        <summary class="article-expander__title">Orbital stability of periodic standing waves for the cubic fractional nonlinear Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200540X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Gabriel E. Bittencourt Moraes, Handan Borluk, Guilherme de Loreno, Gulcin M. Muslu, Fábio Natali</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005447"
                        open
                      >
                        <summary class="article-expander__title">Straightened characteristics of McKendrick-von Foerster equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005447?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Agnieszka Bartłomiejczyk, Henryk Leszczyński, Milena Matusik</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005368"
                        open
                      >
                        <summary class="article-expander__title">On quantitative uniqueness for parabolic equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005368?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Igor Kukavica, Quinn Le</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005459"
                        open
                      >
                        <summary class="article-expander__title">Hölder continuity of parabolic quasi-minimizers on metric measure spaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005459?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Andreas Herán</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005423"
                        open
                      >
                        <summary class="article-expander__title">Nonlinear smoothing for the periodic generalized nonlinear Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005423?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Ryan McConnell</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005435"
                        open
                      >
                        <summary class="article-expander__title">An alternative for minimal group actions on totally regular curves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005435?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Enhui Shi, Hui Xu, Xiangdong Ye</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200537X"
                        open
                      >
                        <summary class="article-expander__title">Decay of entropy solutions of a scalar conservation law beyond a stationary ergodic setting</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200537X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Luís Salvino, Jean Silva</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005411"
                        open
                      >
                        <summary class="article-expander__title">Morse index for the ground state in the energy supercritical Gross–Pitaevskii equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005411?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Dmitry E. Pelinovsky, Szymon Sobieszek</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005393"
                        open
                      >
                        <summary class="article-expander__title">Supercritical elliptic problems on nonradial domains via a nonsmooth variational approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005393?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Craig Cowan, Abbas Moameni</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005186"
                        open
                      >
                        <summary class="article-expander__title">Gaining two derivatives on a singular force in the 2D Navier-Stokes equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005186?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Alexey Cheskidov, Landon Kavlie</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005460"
                        open
                      >
                        <summary class="article-expander__title">Existence and multiplicity of solutions to Dirichlet problem for semilinear subelliptic equation with a free perturbation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005460?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Hua Chen, Hong-Ge Chen, Xin-Rui Yuan</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005125"
                        open
                      >
                        <summary class="article-expander__title">Geometric rough paths on infinite dimensional spaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005125?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Erlend Grong, Torstein Nilssen, Alexander Schmeding</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005174"
                        open
                      >
                        <summary class="article-expander__title">Heat kernels of non-local Schrödinger operators with Kato potentials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005174?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 December 2022
Source: Journal of Differential Equations, Volume 340
Author(s): Tomasz Grzywny, Kamil Kaleta, Paweł Sztonyk</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005319"
                        open
                      >
                        <summary class="article-expander__title">Energy conservation law for weak solutions of the full compressible Navier-Stokes equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005319?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/S00220396.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 25 December 2022
Source: Journal of Differential Equations, Volume 341
Author(s): Motofumi Aoki, Tsukasa Iwabuchi</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-30 datetime="2022-09-30T13:26:28.000Z">2022-09-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-30 datetime="2022-09-30T13:26:28.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119316"
                        open
                      >
                        <summary class="article-expander__title">How Star Trek’s Lieutenant Uhura Overcame Astronomical Odds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/how-star-treks-lieutenant-uhura-overcame-astronomical-odds-20220930/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/08/Uhura_1200_social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Readers rescued a Star Trek crew from a probabilistic predicament armed only with the power of mathematical reasoning.            
The post How Star Trek’s Lieutenant Uhura Overcame Astronomical Odds first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1459782?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Solution of Initial Value Problems by a Dynamical Galerkin Scheme</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1459782?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1147-1166, September 2022. 
 Abstract. Adaptive Galerkin methods for time-dependent partial differential equations are studied and shown to be dissipative. The adaptation implies that the subset of the selected basis function changes over time according to the evolution of the solution. The corresponding projection operator is thus time-dependent and nondifferentiable. We therefore propose to use an integral formulation in time. We analyze the existence and uniqueness of this weak form of the dynamical Galerkin scheme, and we then prove that the nonsmooth projection operator introduces energy dissipation, which is a crucial result for adaptive Galerkin methods, e.g., adaptive wavelet methods. Numerical examples for the inviscid Burgers equation in one dimension and the incompressible Euler equations in two and three spatial dimensions show that the selection of basis functions, for instance by filtering out weak wavelet coefficients from the solution, introduces energy dissipation. Moreover, for the Burgers case we can show that adaptive wavelet regularization yields convergence of the truncated Galerkin solution to the physically relevant entropy solution. These results motivate adaptive wavelet-based Galerkin schemes for nonlinear hyperbolic conservation laws.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1467791?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Geometric Evolution of Bilayers under the Degenerate Functionalized Cahn–Hilliard Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1467791?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1127-1146, September 2022. 
 Abstract. Using a multiscale analysis, we derive a sharp interface limit for the dynamics of bilayer structures of the functionalized Cahn–Hilliard equation with a cutoff diffusion mobility [math] that is degenerate for [math] and a continuously differentiable double-well potential [math]. We show that the bilayer interface does not move in the [math] time scale. The interface motion occurs in the [math] time scale and is determined by porous medium diffusion processes in both phases with no jumps on the interface. In the longer [math] time scale, the interface motion is a complex combination of porous medium diffusion processes in both phases and the property of mass conservation.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1493707?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Erratum: Tunable Eigenvector-Based Centralities for Multiplex and Temporal Networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1493707?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1167-1167, September 2022. 
 Abstract. This erratum corrects several typos in [D. Taylor, M. A. Porter, and P. J. Mucha, Multiscale Model. Simul., 19 (2021), pp. 113–147].</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.26.509630v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Identification of Rho-dependent termination site in vivo using synthetic sRNA [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.26.509630v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Rho promotes Rho-dependent termination (RDT) at the Rho-dependent terminator, producing a variable-length region at the 3&#x27;-end of mRNA without secondary structure. Determining the exact RDT site in vivo is challenging because the 3&#x27;-end of mRNA is rapidly removed by 3&#x27;- to 5&#x27;-exoribonuclease digestion after RDT. Here, we applied synthetic sRNA (sysRNA) to pinpoint RDT sites in vivo by exploiting its complementary base-pairing ability to target mRNA. Through the combined assays of rapid amplification of cDNA 3&#x27;-ends, primer extension, and capillary electrophoresis, we could precisely locate and quantify mRNA 3&#x27;-ends. We found that complementary double-stranded RNA (dsRNA) formed between sysRNA and mRNA was efficiently cleaved by RNase III in the middle of the dsRNA region. The formation of …</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.29.510219v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">AssemblyTron: Flexible automation of DNA assembly with Opentrons OT-2 lab robots [CONFIRMATORY RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.29.510219v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>As one of the newest fields of engineering, synthetic biology relies upon a trial-and-error Design-Build-Test-Learn approach to simultaneously learn how function is encoded in biology and attempt to engineer it. Many software and hardware platforms have been developed to automate, optimize, and algorithmically perform each step of the Design-Build-Test-Learn cycle. However, there are many fewer options for automating the Build step. Build typically involves DNA assembly, which remains manual, low throughput, and unreliable in most cases, limiting our ability to advance the science and engineering of biology. Here, we present AssemblyTron: an open-source python package to integrate j5 DNA assembly design software outputs with build implementation in Opentrons liquid handling robotics with minimal human intervention. We demonstrate the versatility of AssemblyTron through several scarless, multipart DNA assemblies beginning from fragment amplification. We show that AssemblyTron can perform PCRs across a range of fragment lengths and annealing temperatures by using an optimal annealing temperature gradient calculation algorithm. We then demonstrate that AssemblyTron can perform Golden Gate and homology-dependent in vivo assemblies with comparable fidelity to manual assemblies by simultaneously building four four-fragment assemblies of chromoprotein reporter expression plasmids. Finally, we used AssemblyTron to perform site-directed mutagenesis reactions via homology-dependent in vivo assembly also achieving comparable fidelity to manual assemblies as assessed by sequencing. AssemblyTron can reduce the time, training, costs, and wastes associated with synthetic biology, which along with open-source and affordable automation, will further foster the accessibility of synthetic biology and accelerate biological research and engineering.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.30.509847v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">In situ targeted mutagenesis of gut bacteria [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.30.509847v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Microbiome research is revealing a growing number of bacterial genes that impact our health. While CRISPR-derived tools have shown great success in editing disease-driving genes in human cells, we currently lack the tools to achieve comparable success for bacterial targets. Here we engineer a phage-derived particle to deliver a base editor and modify E. coli colonizing the mouse gut. This was achieved using a non-replicative DNA payload, preventing maintenance and dissemination of the payload, while allowing for an editing efficiency of up to 99.7% of the target bacterial population. The editing of a {beta}-lactamase gene resulted in the stable maintenance of edited bacteria in the mouse gut at least 42 days after treatment. By enabling the in situ modification of bacteria directly in the gut, our approach offers a novel avenue to investigate the function of bacterial genes and provides an opportunity to develop novel microbiome-targeted therapies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.28.509966v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Pathway elucidation of pharmaceutical cucurbitacin IIa in Hemsleya chinensis and high-level production of precursor cucurbitadienol in engineered Saccharomyces cerevisiae and Nicotiana benthamiana [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.28.509966v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cucurbitacin IIa is a triterpene isolated exclusively from Hemsleya plants, which is non-steroidal anti-inflammatory drug that function as the main ingredient of Hemslecin capsules and Supplemental Tablets in China. In this study, the biosynthetic pathway of cucurbitacin IIa was elucidated by characterization of squalene epoxidases (HcSE1, HcSE2), oxidosqualene cyclases HcOSC6, and acyltransferases (HcACT1, HcACT2) in Hemsleya chinensis. Meanwhile, isomultiflorenol synthases (HcOSC1, HcOSC5) and {beta}-amyrin synthase (HcOSC2-4) involved in sterol and triterpenes biosynthesis were functionally illustrated. The high-level production of yeast the key cucurbitacin precursor, cucurbitadienol, was constructed to produce 296.37 mg/L cucurbitadienol and 722.99 mg/L total triterpenoid which is the highest yield cucurbitadienol from known engineered microbes. Moreover, production of cucurbitenol in transient expression of tobacco was employed to achieve 94.8 mg/g dry weight (dw) cucurbitenol from leaves. In this study, the key genes involved in cucurbitacin IIa biosynthesis were identified to facilitate its medical applications via biosynthetic strategy. Meanwhile, the high-level production of cucurbitadienol chassis yeast and tobacco transient expression offered a robust and Supplemental Table substrate for pharmaceutical cucurbitacin production and screening platform for candidate genes involved in cucurbitacin biosynthesis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20143-5"
                        open
                      >
                        <summary class="article-expander__title">StackPR is a new computational approach for large-scale identification of progesterone receptor antagonists using the stacking strategy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20143-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20143-5/MediaObjects/41598_2022_20143_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - StackPR is a new computational approach for large-scale identification of progesterone receptor antagonists using the stacking strategy</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41597-022-01707-6"
                        open
                      >
                        <summary class="article-expander__title">Machine actionable metadata models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41597-022-01707-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41597-022-01707-6/MediaObjects/41597_2022_1707_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Data - Machine actionable metadata models</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21010-z"
                        open
                      >
                        <summary class="article-expander__title">Machine learning to improve the interpretation of intercalating dye-based quantitative PCR results</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21010-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21010-z/MediaObjects/41598_2022_21010_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Machine learning to improve the interpretation of intercalating dye-based quantitative PCR results</span>
                                &ensp;<span class="article-reading-time">(&hairsp;23
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20325-1"
                        open
                      >
                        <summary class="article-expander__title">Identification of hydantoin based Decaprenylphosphoryl-β-d-Ribose Oxidase (DprE1) inhibitors as antimycobacterial agents using computational tools</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20325-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20325-1/MediaObjects/41598_2022_20325_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Identification of hydantoin based Decaprenylphosphoryl-β-d-Ribose Oxidase (DprE1) inhibitors as antimycobacterial agents using computational tools</span>
                                &ensp;<span class="article-reading-time">(&hairsp;34
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20850-z"
                        open
                      >
                        <summary class="article-expander__title">Accurate and fast identification of minimally prepared bacteria phenotypes using Raman spectroscopy assisted by machine learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20850-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20850-z/MediaObjects/41598_2022_20850_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Accurate and fast identification of minimally prepared bacteria phenotypes using Raman spectroscopy assisted by machine learning</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20509-9"
                        open
                      >
                        <summary class="article-expander__title">Mechanistic insight into the role of mevalonate kinase by a natural fatty acid-mediated killing of Leishmania donovani</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20509-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20509-9/MediaObjects/41598_2022_20509_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Mechanistic insight into the role of mevalonate kinase by a natural fatty acid-mediated killing of Leishmania donovani</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20012-1"
                        open
                      >
                        <summary class="article-expander__title">Perception without preconception: comparison between the human and machine learner in recognition of tissues from histological sections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20012-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20012-1/MediaObjects/41598_2022_20012_Fig1_HTML.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Perception without preconception: comparison between the human and machine learner in recognition of tissues from histological sections</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s12276-022-00846-5"
                        open
                      >
                        <summary class="article-expander__title">Machine-learning algorithms for asthma, COPD, and lung cancer risk assessment using circulating microbial extracellular vesicle data and their application to assess dietary effects</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s12276-022-00846-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs12276-022-00846-5/MediaObjects/12276_2022_846_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Artificial intelligence (AI) has enabled researchers to intercept microbial messages bearing clinically useful information about of a variety of respiratory disorders. The organisms that comprise our microbiome communicate via the release of tiny, biomolecule-laden membrane bubbles called ‘extracellular vesicles’ (EVs) into the bloodstream. EVs are also influenced by human disease. South Korean researchers led by Yoon-Keun Kim of MD Healthcare, Seoul, and Young-Koo Jee of Dankook University College of Medicine, Cheonan, have used an AI algorithm to assemble EV-based profiles that can discriminate between healthy people and those with conditions like asthma or lung cancer. Their analysis of 1727 patient serum samples revealed microbial signatures that enabled accurate diagnosis of several respiratory disorders. Preliminary experiments in mice suggest that certain dietary changes could help shift the microbiome of high-risk individuals towards a healthier profile.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20816-1"
                        open
                      >
                        <summary class="article-expander__title">Comparison of image registration methods for combining laparoscopic video and spectral image data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20816-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20816-1/MediaObjects/41598_2022_20816_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Comparison of image registration methods for combining laparoscopic video and spectral image data</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41593-022-01167-6"
                        open
                      >
                        <summary class="article-expander__title">A tool for mapping microglial morphology, morphOMICs, reveals brain-region and sex-dependent phenotypes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41593-022-01167-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-022-01167-6/MediaObjects/41593_2022_1167_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Colombo et al. build a morphological spectrum of over 40,000 microglia across development and disease with a topological data analysis approach that allows mapping of new conditions along these sex-region-specific and brain-region-specific atlases.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;57
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20783-7"
                        open
                      >
                        <summary class="article-expander__title">Panels of mRNAs and miRNAs for decoding molecular mechanisms of Renal Cell Carcinoma (RCC) subtypes utilizing Artificial Intelligence approaches</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20783-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20783-7/MediaObjects/41598_2022_20783_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Panels of mRNAs and miRNAs for decoding molecular mechanisms of Renal Cell Carcinoma (RCC) subtypes utilizing Artificial Intelligence approaches</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41596-022-00739-x"
                        open
                      >
                        <summary class="article-expander__title">Multispectral confocal 3D imaging of intact healthy and tumor tissue using mLSR-3D</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41596-022-00739-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41596-022-00739-x/MediaObjects/41596_2022_739_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multispectral large-scale single-cell resolution 3D imaging allows up to eight fluorophores to be captured in a single acquisition. This protocol enables the visualization and exploration of large intact tissue volumes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20845-w"
                        open
                      >
                        <summary class="article-expander__title">Comparing two machine learning approaches in predicting lupus hospitalization using longitudinal data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20845-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20845-w/MediaObjects/41598_2022_20845_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Comparing two machine learning approaches in predicting lupus hospitalization using longitudinal data</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20758-8"
                        open
                      >
                        <summary class="article-expander__title">Crystal structures of FNIP/FGxxFN motif-containing leucine-rich repeat proteins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20758-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20758-8/MediaObjects/41598_2022_20758_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Crystal structures of FNIP/FGxxFN motif-containing leucine-rich repeat proteins</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10915"
                        open
                      >
                        <summary class="article-expander__title">Journal of Scientific Computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10915">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02010-7"
                        open
                      >
                        <summary class="article-expander__title">A Posteriori Error Analysis of a Mixed Finite Element Method for the Coupled Brinkman–Forchheimer and Double-Diffusion Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02010-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper we consider a partially augmented fully-mixed variational formulation that has been recently proposed for the coupling of the stationary Brinkman–Forchheimer and double-diffusion equations, and develop an a posteriori error analysis for the 2D and 3D versions of the associated mixed finite element scheme. Indeed, we derive two reliable and efficient residual-based a posteriori error estimators for this problem on arbitrary (convex or non-convex) polygonal and polyhedral regions. The reliability of the proposed estimators draws mainly upon the uniform ellipticity and inf-sup condition of the forms involved, a suitable assumption on the data, stable Helmholtz decompositions in Hilbert and Banach frameworks, and the local approximation properties of the Clément and Raviart–Thomas operators. In turn, inverse inequalities, the localization technique based on bubble functions, and known results from previous works, are the main tools yielding the efficiency estimate. Finally, several numerical examples confirming the theoretical properties of the estimators and illustrating the performance of the associated adaptive algorithms, are reported. In particular, the case of flow through a 3D porous media with channel networks is considered.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-29 datetime="2022-09-29T18:00:00.000Z">2022-09-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-29 datetime="2022-09-29T18:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/processing-waste-biomass-reduce-airborne-emissions-0929"
                        open
                      >
                        <summary class="article-expander__title">Processing waste biomass to reduce airborne emissions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/processing-waste-biomass-reduce-airborne-emissions-0929">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/Takachar-hay-field-MITEI.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MIT spinoff Takachar converts agricultural waste into clean-burning fuel, and wins Earthshot Prize.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119250"
                        open
                      >
                        <summary class="article-expander__title">What We Know About Monkeypox</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/epidemiologist-what-we-know-about-monkeypox-20220929/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/MonkeyPox_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Even though we’ve learned a lot about this relative of smallpox, some mysteries remain due to a lack of political will.            
The post What We Know About Monkeypox first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2021.12.08.471839v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Assessing single-component gene drive systems in the mosquito Aedes aegypti via single generation crosses and modeling [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2021.12.08.471839v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The yellow fever mosquito Aedes aegypti is a major vector of arthropod-borne viruses, including dengue, chikungunya, and Zika. A novel approach to mitigate arboviral infections is to generate mosquitoes refractory to infection by overexpressing antiviral effector molecules. Such an approach requires a mechanism to spread these antiviral effectors through a population, for example, by using CRISPR/Cas9-based gene drive (GD) systems. Critical to the design of a single-locus autonomous GD is that the selected genomic locus be amenable to both GD and appropriate expression of the antiviral effector. In our study, we used reverse engineering to target two intergenic genomic loci, which had previously shown to be highly permissive for antiviral effector gene expression, and we further investigat…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32995-6"
                        open
                      >
                        <summary class="article-expander__title">Cost-effective methylome sequencing of cell-free DNA for accurately detecting and locating cancer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32995-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32995-6/MediaObjects/41467_2022_32995_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Early cancer detection by cell-free DNA (cfDNA) is challenged by the low amount of tumour DNA in cfDNA, tumour heterogeneity and the small patient cohorts. Here, the authors develop a method, cfMethyl-Seq, for cost-effective methylome profiling of cfDNA and for detecting and locating cancer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33397-4"
                        open
                      >
                        <summary class="article-expander__title">Deciphering microbial gene function using natural language processing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33397-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33397-4/MediaObjects/41467_2022_33397_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The function of many microbial genes is yet unknown. Here the authors repurposed natural language processing algorithms to explore “gene semantics” and infer function for thousands of genes with defense and secretion systems found to have the most discovery potential.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33266-0"
                        open
                      >
                        <summary class="article-expander__title">Adversarial attacks and adversarial robustness in computational pathology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33266-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33266-0/MediaObjects/41467_2022_33266_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Artificial Intelligence can support diagnostic workflows in oncology, but they are vulnerable to adversarial attacks. Here, the authors show that convolutional neural networks are highly susceptible to white- and black-box adversarial attacks in clinically relevant classification tasks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;32
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20388-0"
                        open
                      >
                        <summary class="article-expander__title">Extraction of sensing data for desired scent impressions using mass spectra of odorant molecules</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20388-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20388-0/MediaObjects/41598_2022_20388_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Extraction of sensing data for desired scent impressions using mass spectra of odorant molecules</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01606-z"
                        open
                      >
                        <summary class="article-expander__title">Analysis of the Human Protein Atlas Weakly Supervised Single-Cell Classification competition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01606-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01606-z/MediaObjects/41592_2022_1606_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine learning competition results in tools for labeling protein patterns of single cells in images with population labels. The winners improve the state of the art and provide strategies to deal with weak classification challenges.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41591-022-02010-y"
                        open
                      >
                        <summary class="article-expander__title">Screening for idiopathic pulmonary fibrosis using comorbidity signatures in electronic health records</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41591-022-02010-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-02010-y/MediaObjects/41591_2022_2010_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine learning algorithm can predict the future risk of idiopathic pulmonary fibrosis, a disease that is difficult to diagnose at its early stages, based on comorbidity signatures in electronic health records.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;48
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20749-9"
                        open
                      >
                        <summary class="article-expander__title">Differentiating a pachychoroid and healthy choroid using an unsupervised machine learning approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20749-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20749-9/MediaObjects/41598_2022_20749_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Differentiating a pachychoroid and healthy choroid using an unsupervised machine learning approach</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20822-3"
                        open
                      >
                        <summary class="article-expander__title">Structural insight into a glucomannan-type extracellular polysaccharide produced by a marine Bacillus altitudinis SORB11 from Southern Ocean</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20822-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20822-3/MediaObjects/41598_2022_20822_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Structural insight into a glucomannan-type extracellular polysaccharide produced by a marine Bacillus altitudinis SORB11 from Southern Ocean</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32874-0"
                        open
                      >
                        <summary class="article-expander__title">Protein structural transitions critically transform the network connectivity and viscoelasticity of RNA-binding protein condensates but RNA can prevent it</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32874-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32874-0/MediaObjects/41467_2022_32874_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this work the authors propose a multiscale computational approach, integrating atomistic and coarse-grained models simulations, to study the thermodynamic and kinetic factors playing a major role in the liquid-to-solid transition of biomolecular condensates. It is revealed how the gradual accumulation of inter-protein β-sheets increases the viscosity of functional liquid-like condensates, transforming them into gel-like pathological aggregates, and it is also shown how high concentrations of RNA can decelerate such transition.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;53
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20858-5"
                        open
                      >
                        <summary class="article-expander__title">Necroptosis-related lncRNA signature predicts prognosis and immune response for cervical squamous cell carcinoma and endocervical adenocarcinomas</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20858-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20858-5/MediaObjects/41598_2022_20858_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Necroptosis-related lncRNA signature predicts prognosis and immune response for cervical squamous cell carcinoma and endocervical adenocarcinomas</span>
                                &ensp;<span class="article-reading-time">(&hairsp;32
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04003-6"
                        open
                      >
                        <summary class="article-expander__title">Changes in and asymmetry of the proteome in the human fetal frontal lobe during early development</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04003-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04003-6/MediaObjects/42003_2022_4003_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Proteomic analysis of human early fetal brain tissue is undertaken to investigate bilateral developmental changes of protein expression and left-right asymmetries of protein expression.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41525-022-00321-0"
                        open
                      >
                        <summary class="article-expander__title">Alternative ANKHD1 transcript promotes proliferation and inhibits migration in uterine corpus endometrial carcinoma</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41525-022-00321-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41525-022-00321-0/MediaObjects/41525_2022_321_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>npj Genomic Medicine - Alternative ANKHD1 transcript promotes proliferation and inhibits migration in uterine corpus endometrial carcinoma</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20682-x"
                        open
                      >
                        <summary class="article-expander__title">DNA methylation landscapes from pig’s limbic structures underline regulatory mechanisms relevant for brain plasticity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20682-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20682-x/MediaObjects/41598_2022_20682_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - DNA methylation landscapes from pig’s limbic structures underline regulatory mechanisms relevant for brain plasticity</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32887-9"
                        open
                      >
                        <summary class="article-expander__title">Systematic identification of intron retention associated variants from massive publicly available transcriptome sequencing data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32887-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32887-9/MediaObjects/41467_2022_32887_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>This paper proposed a novel in-silico framework for automatically screening disease-related variants and applied it to over 200,000 transcriptomes, providing an example to acquire medically relevant knowledge from publicly available sequence data.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02009-0"
                        open
                      >
                        <summary class="article-expander__title">Multi-Dimensional Image Recovery via Fully-Connected Tensor Network Decomposition Under the Learnable Transforms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02009-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Multi-dimensional image recovery from incomplete data is a fundamental problem in data processing. Due to its advantage of capturing the correlations between any modes of the multi-dimensional image, i.e., the target tensor, the fully-connected tensor network (FCTN) decomposition has recently shown promising performance on multi-dimensional image recovery. However, FCTN decomposition suffers from computational deficiency, especially for large-scale multi-dimensional images. To address this deficiency, we propose a learnable transform-based FCTN model (termed as T-FCTN), which enjoys the remarkable advantage of FCTN decomposition with cheap computational cost. More concretely, we learn the semi-orthogonal transforms along each mode of the target tensor to project the large-scale te…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02002-7"
                        open
                      >
                        <summary class="article-expander__title">The Spectral Difference Raviart–Thomas Method for Two and Three-Dimensional Elements and Its Connection with the Flux Reconstruction Formulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02002-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The purpose of this work is to describe in detail the development of the spectral difference Raviart–Thomas (SDRT) formulation for two and three-dimensional tensor-product elements and simplexes. Through the process, the authors establish the equivalence between the SDRT method and the flux reconstruction (FR) approach under the assumption of the linearity of the flux and the mesh uniformity. Such a connection allows building a new family of FR schemes for two and three-dimensional simplexes and also to recover the well-known FR-SD method with tensor-product elements. In addition, a thorough analysis of the numerical dissipation and dispersion of both aforementioned schemes and the nodal discontinuous Galerkin FR (FR-DG) method with two and three-dimensional elements is proposed through the use of the combined-mode Fourier approach. SDRT is shown to possess an enhanced temporal linear stability in comparison to FR-DG. On the contrary, SDRT displays larger dissipation and dispersion errors with respect to FR-DG. Finally, the study is concluded with a set of numerical experiments, the linear advection-diffusion problem, the Isentropic Euler Vortex, and the Taylor-Green Vortex (TGV). The latter test case shows that SDRT schemes present a non-linear unstable behavior with simplex elements and certain polynomial degrees. For the sake of completeness, the matrix form of the SDRT method is developed and the computational performance of SDRT with respect to FR schemes is evaluated using GPU architectures.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02006-3"
                        open
                      >
                        <summary class="article-expander__title">Tensor Completion via A Generalized Transformed Tensor T-Product Decomposition Without t-SVD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02006-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Matrix and tensor nuclear norms have been successfully used to promote the low-rankness of tensors in low-rank tensor completion. However, singular value decomposition (SVD), which is computationally expensive for large-scale matrices, frequently appears in solving those nuclear norm minimization models. Based on the tensor-tensor product (T-product), in this paper, we first establish the equivalence between the so-called transformed tubal nuclear norm for a third-order tensor and the minimum of the sum of two factor tensors’ squared Frobenius norms under a general invertible linear transform. Gainfully, we introduce a mode-unfolding (often named as “spatio-temporal” in the internet traffic data recovery literature) regularized tensor completion model that is able to efficiently exploit the hidden structures of tensors. Then, we propose an implementable alternating minimization algorithm to solve the underlying optimization model. It is remarkable that our approach does not require any SVDs and all subproblems of our algorithm enjoy closed-form solutions. A series of numerical experiments on traffic data recovery, color images and videos inpainting demonstrate that our SVD-free approach takes less computing time to achieve satisfactory accuracy than some state-of-the-art tensor nuclear norm minimization approaches.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-28 datetime="2022-09-28T13:56:44.000Z">2022-09-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-28 datetime="2022-09-28T13:56:44.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119210"
                        open
                      >
                        <summary class="article-expander__title">Biologists Use Genetic Circuits to Program Plant Roots</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/biologists-use-genetic-circuits-to-program-plant-roots-20220928/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Programming-Plant-Roots_1200_Social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using inserted genetic circuitry, synthetic biologists controlled the growth of plant roots for the first time.            
The post Biologists Use Genetic Circuits to Program Plant Roots first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.28.509898v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Synthetic carbon fixation via the autocatalytic serine threonine cycle [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.28.509898v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Atmospheric CO2 poses a major threat to life on Earth by causing global warming and climate change. On the other hand, it is the only carbon source that is scalable enough to establish a circular carbon economy. Accordingly, technologies to capture and convert CO2 to reduced one-carbon (C1) molecules (e.g. formate) using renewable energy are improving fast. Driven by the idea of creating sustainable bioproduction platforms, natural and synthetic C1-utilization pathways are engineered into industrially relevant microbes. The realization of synthetic C1-assimilation cycles in living organisms is a promising but challenging endeavour. Here, we engineer the autocatalytic serine threonine cycle, a synthetic C1-assimilation route in Escherichia coli. Our stepwise engineering approach in tailored selection strains combined with adaptive laboratory evolution experiments enabled the organism to grow on formate. The synthetic strain uses formate as the sole carbon and energy source and is capable of growing at ambient CO2 concentrations, demonstrating the feasibility of establishing synthetic C1-assimilation cycles over laboratory timescales.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.27.509702v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Cell type-independent profiling of interactions between intracellular pathogens and the human phosphoproteome [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.27.509702v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Interactions between proteins from intracellular pathogens and host proteins in an infected cell are often mediated by post-translational modifications encoded in the host proteome. Identifying protein modifications, such as phosphorylation, that dictate these interactions remains a defining challenge in unraveling the molecular mechanisms of pathogenesis. We have developed a platform in engineered bacteria that displays over 110,000 phosphorylated human proteins coupled to a fluorescent reporter system capable of identifying the host-pathogen interactome of phosphoproteins (H-PIP). This resource broadly enables cell-type independent interrogation and discovery of proteins from intracellular pathogens capable of binding phosphorylated human proteins. As an example of the H-PIP platform, we generated a unique, high-resolution SARS-CoV-2 interaction network which expanded our knowledge of viral protein function and identified understudied areas of host pathology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.16.508262v2?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">An integrase toolbox to record gene-expression during plant development [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.16.508262v2?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>There are many open questions about the mechanisms that coordinate the dynamic, multicellular behaviors required for organogenesis. Synthetic circuits that can record in vivo signaling networks have been critical in elucidating animal development. Here, we report on the transfer of this technology to plants using orthogonal serine integrases to mediate site-specific and irreversible DNA recombination visualized by switching between fluorescent reporters. When combined with promoters expressed during lateral root initiation, integrases amplified reporter signal and permanently marked all descendants. In addition, we have developed a suite of methods to tune the threshold for integrase switching, including: RNA/protein degradation tags, a nuclear localization signal, and a split-intein system. These tools improved the robustness of integrase-mediated switching with different promoters and the stability of switching behavior over multiple generations. This integrase toolbox can be used to build history-dependent circuits to decode the order of expression during organogenesis in many contexts.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19565-y"
                        open
                      >
                        <summary class="article-expander__title">De novo metatranscriptomic exploration of gene function in the millipede holobiont</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19565-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19565-y/MediaObjects/41598_2022_19565_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - De novo metatranscriptomic exploration of gene function in the millipede holobiont</span>
                                &ensp;<span class="article-reading-time">(&hairsp;40
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33522-3"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Structure of the PAPP-ABP5 complex reveals mechanism of substrate recognition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33522-3">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Author Correction: Structure of the PAPP-ABP5 complex reveals mechanism of substrate recognition</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="285"
                        open
                      >
                        <summary class="article-expander__title">Journal of Mathematical Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/285">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01794-4"
                        open
                      >
                        <summary class="article-expander__title">From individual-based epidemic models to McKendrick-von Foerster PDEs: a guide to modeling and inferring COVID-19 dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01794-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We present a unifying, tractable approach for studying the spread of viruses causing complex diseases requiring to be modeled using a large number of types (e.g., infective stage, clinical state, risk factor class). We show that recording each infected individual’s infection age, i.e., the time elapsed since infection, has three benefits. First, regardless of the number of types, the age distribution of the population can be described by means of a first-order, one-dimensional partial differential equation (PDE) known as the McKendrick-von Foerster equation. The frequency of type i is simply obtained by integrating the probability of being in state i at a given age against the age distribution. This representation induces a simple methodology based on the additional assumption of Poisson sampling to infer and forecast the epidemic. We illustrate this technique using French data from the COVID-19 epidemic. Second, our approach generalizes and simplifies standard compartmental models using high-dimensional systems of ordinary differential equations (ODEs) to account for disease complexity. We show that such models can always be rewritten in our framework, thus, providing a low-dimensional yet equivalent representation of these complex models. Third, beyond the simplicity of the approach, we show that our population model naturally appears as a universal scaling limit of a large class of fully stochastic individual-based epidemic models, where the initial condition of the PDE emerges as the limiting age structure of an exponentially growing population starting from a single individual.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;67
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02014-3"
                        open
                      >
                        <summary class="article-expander__title">Optimal Error Estimates of the Local Discontinuous Galerkin Method and High-order Time Discretization Scheme for the Swift–Hohenberg Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02014-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we develop a local discontinuous Galerkin (LDG) method for the Swift–Hohenberg equation. The energy stability and optimal error estimates in 
                      
                      \(L^2\)
                      
                     norm of the semi-discrete LDG scheme are established. To avoid the severe time step restriction of explicit time marching methods, a first-order linear scheme based on the scalar auxiliary variable (SAV) method is employed for temporal discretization. Coupled with the LDG spatial discretization, we achieve a fully-discrete LDG method and prove its energy stability and optimal error estimates. To improve the temporal accuracy, the semi-implicit spectral deferred correction (SDC) method is adapted iteratively. Combining with the SAV method, the SDC method can be linear, high-order accurate and energy stable in our numerical tests. Numerical experiments are presented to verify the theoretical results and to show the efficiency of the proposed methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z">2022-09-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927"
                        open
                      >
                        <summary class="article-expander__title">Neurodegenerative disease can progress in newly identified patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/neuron-cell.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine-learning method finds patterns of health decline in ALS, informing future clinical trial designs and mechanism discovery. The technique also extends to Alzheimer’s and Parkinson’s.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119076"
                        open
                      >
                        <summary class="article-expander__title">How Big Is Infinity?</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/how-big-is-infinity-20220927/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Infinity_academy_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Of all the endless questions children and mathematicians have asked about infinity, one of the most fascinating has to do with its size.            
The post How Big Is Infinity? first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M1377771?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Manifold Learning and Nonlinear Homogenization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M1377771?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://epubs.siam.org/na101/home/literatum/publisher/siam/journals/covergifs/mmsubt/cover.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1093-1126, September 2022. 
 Abstract. We describe an efficient domain decomposition-based framework for nonlinear multiscale PDE problems. The framework is inspired by manifold learning techniques and exploits the tangent spaces spanned by the nearest neighbors to compress local solution manifolds. Our framework is applied to a semilinear elliptic equation with oscillatory media and a nonlinear radiative transfer equation; in both cases, significant improvements in efficacy are observed. This new method does not rely on a detailed analytical understanding of multiscale PDEs, such as their asymptotic limits, and thus is more versatile for general multiscale problems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1439808?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Online Multiscale Model Reduction for Nonlinear Stochastic PDEs with Multiplicative Noise</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1439808?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1063-1092, September 2022. 
 Abstract. In this paper, an online multiscale model reduction method is presented for stochastic partial differential equations (SPDEs) with multiplicative noise, where the diffusion coefficient is spatially multiscale and the noise perturbation nonlinearly depends on the diffusion dynamics. It is necessary to efficiently compute all possible trajectories of the stochastic dynamics for quantifying a model’s uncertainty and statistic moments. The multiscale diffusion and nonlinearity may cause the computation to become very intractable. To overcome the multiscale difficulty, a constraint energy minimizing generalized multiscale finite element method (CEM-GMsFEM) is used to localize the computation and obtain an effective coarse model. However, the nonlinear terms are still defined on a fine scale space after the Galerkin projection of CEM-GMsFEM is applied to the nonlinear SPDEs. This significantly impacts the simulation efficiency by CEM-GMsFEM. To this end, a stochastic online discrete empirical interpolation method (DEIM) is proposed to treat the stochastic nonlinearity. The stochastic online DEIM incorporates offline snapshots and online snapshots. The offline snapshots consist of the nonlinear terms at the approximate mean of the stochastic dynamics and are used to construct an offline reduced model. The online snapshots contain some information of the current new trajectory and are used to correct the offline reduced model in an incremental manner. The stochastic online DEIM substantially reduces the dimension of the nonlinear dynamics and enhances the prediction accuracy for the reduced model. Thus, the online multiscale model reduction is constructed by using CEM-GMsFEM and the stochastic online DEIM. A priori error analysis is carried out for the nonlinear SPDEs. We present a few numerical examples with diffusion in heterogeneous porous media and show the effectiveness of the proposed model reduction.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1464592?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Hyperbolic Systems of Moment Equations Describing Sedimentation in Suspensions of Rod-Like Particles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1464592?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1002-1039, September 2022. 
 Abstract. We study mathematical models and approximations for the sedimentation in suspensions of rod-like particles. The starting point of our considerations is a coupled system of partial differential equations, previously derived by Helzel and Tzavaras [Multiscale Model.  Simul., 15 (2017), pp.  500–536], consisting of a kinetic equation for the rod orientation that is coupled to a macroscopic flow equation. It describes the motion of a suspension of rigid rod-like particles under the influence of gravity. Since the coupled system is high dimensional (five dimensions plus time) we are interested in the derivation of models which can adaptively adjust the level of detail. Here we restrict ourselves to simplified lower dimensional flow situations and derive hierarchies of moment equations which can be interpreted as approximations of the coupled kinetic model. We show that the resulting systems of moment equations are hyperbolic.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M146995X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Improved Uniform Error Bounds on Time-Splitting Methods for the Long-Time Dynamics of the Dirac Equation with Small Potentials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M146995X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1040-1062, September 2022. 
 Abstract. We establish improved uniform error bounds on time-splitting methods for the long-time dynamics of the Dirac equation with small electromagnetic potentials characterized by a dimensionless parameter [math] representing the amplitude of the potentials. We begin with a semidiscretization of the Dirac equation in time by a time-splitting method, followed by a full-discretization in space by with the Fourier pseudospectral method in space. By employing the unitary flow property of the second-order time-splitting method for the Dirac equation, we prove uniform error bounds for [math] at [math] and [math] for the semidiscretization and full-discretization, respectively, for any time [math] with [math] and [math]. In the expressions, [math] is the time step, [math] is the mesh size, [math] depends on the regularity of the solution, and [math] grows at most linearly with respect to [math] with [math] and [math] two constants independent of [math], [math], [math], and [math]. Then by adopting the regularity compensation oscillation technique, which controls the high frequency modes by the regularity of the solution and low frequency modes by phase cancellation and energy method, we establish improved uniform error bounds at [math] and [math] for the semidiscretization and full-discretization, respectively, up to the long-time [math]. Numerical results are reported to confirm our error bounds and to demonstrate that they are sharp. Comparisons on the accuracy of different time discretizations for the Dirac equation are also provided.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0095009?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Bifurcation structure and chaos in dynamics of nanomagnet coupled to Josephson junction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0095009?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Irregular easy axis reorientation features are observed in numerical simulations of the nanomagnet coupled to the Josephson junction. We study magnetization bifurcations and chaos that appear in this system due to the interplay of superconductivity and magnetism. The bifurcation structure of magnetization under the variation of Josephson to magnetic energy ratio as a control parameter demonstrates several precessional motions that are related to chaotic behavior and orbits with different periodicities in the ferromagnetic resonance region. The effect of an external periodic signal on the bifurcation structure is also investigated. The results demonstrate high-frequency modes of a periodic motion and a chaotic response near resonance. Far from the ferromagnetic resonance, we observe a quasiperiodic behavior. The obtained results explain the irregular reorientation of the easy axis and the transitions between different types of motion.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.25.509419v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Top-down design of protein nanomaterials with reinforcement learning [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.25.509419v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The multisubunit protein assemblies that play critical roles in biology are the result of evolutionary selection for function of the entire assembly, and hence the subunits in structures such as icosahedral viral capsids often fit together with remarkable shape complementarity1,2. In contrast, the large multisubunit assemblies that have been created by de novo protein design, notably the icosahedral nanocages used in a new generation of potent vaccines3-7, have been built by first designing symmetric oligomers with cyclic symmetry and then assembling these into nanocages while keeping the internal structure fixed8-14, which results in more porous structures with less extensive shape matching between the components. Such hierarchical &quot;bottom-up&quot; design approaches have the advantage that one…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03999-1"
                        open
                      >
                        <summary class="article-expander__title">Phospholipid synthesis inside phospholipid membrane vesicles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03999-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03999-1/MediaObjects/42003_2022_3999_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A giant vesicle system combines fatty acid synthesis pathways, CoA recycling, and cell-free acyltransferase gene expression to perform phospholipid synthesis inside.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01811-6"
                        open
                      >
                        <summary class="article-expander__title">Noise-driven bifurcations in a neural field system modelling networks of grid cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01811-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The activity generated by an ensemble of neurons is affected by various noise sources. It is a well-recognised challenge to understand the effects of noise on the stability of such networks. We demonstrate that the patterns of activity generated by networks of grid cells emerge from the instability of homogeneous activity for small levels of noise. This is carried out by analysing the robustness of network activity patterns with respect to noise in an upscaled noisy grid cell model in the form of a system of partial differential equations. Inhomogeneous network patterns are numerically understood as branches bifurcating from unstable homogeneous states for small noise levels. We show that there is a phase transition occurring as the level of noise decreases. Our numerical study also indicates the presence of hysteresis phenomena close to the precise critical noise value.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02007-2"
                        open
                      >
                        <summary class="article-expander__title">An Efficient QSC Approximation of Variable-Order Time-Fractional Mobile-Immobile Diffusion Equations with Variably Diffusive Coefficients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02007-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we propose a QSC-L1 method to solve the two-dimensional variable-order time-fractional mobile-immobile diffusion (TF-MID) equations with variably diffusive coefficients, in which the quadratic spline collocation (QSC) method is employed for the spatial discretization, and the classical L1 formula is used for the temporal discretization. We show that the method is unconditionally stable and convergent with first-order in time and second-order in space with respect to some discrete and continuous 
                      
                      \(L^2\)
                      
                     norms. Then, combined with the reduced basis technique, an efficient QSC-L1-RB method is proposed to improve the computational efficiency. Numerical examples are attached to verify the convergence orders, and also the method is applied to identify parameters of the variable-order TF-MID equations. Numerical results confirm the contributions of the reduced basis technique, even when the observation data is contaminated by some levels of random noise.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02003-6"
                        open
                      >
                        <summary class="article-expander__title">A Sixth-Order Quasi-Compact Difference Scheme for Multidimensional Poisson Equations Without Derivatives of Source Term</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02003-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Sixth-order compact difference schemes for Poisson equations have been widely investigated in the literature. Nevertheless, those methods are all constructed based on knowing the exact values of the derivatives of the source term. Therefore, this drawback mostly prevents their actual applications as the analytic form of the source term is rarely available. In this paper, we propose a sixth-order quasi-compact difference method, without having to know the derivatives of the source term, for solving the 2D and 3D Poisson equations. Our strategy is to discretize the equation by the fourth-order compact scheme at the improper interior grid points that adjoin the boundary, while the sixth-order scheme, where it is compact only for the unknowns, is exploited to the proper interior grid points that are not adjoining the boundary. Theoretically, we rigorously prove that the proposed method can achieve the global sixth-order accuracy. Since there are no derivatives of the source term involved in the proposed scheme, our global sixth-order quasi-compact difference method can be developed to solve the time-dependent problems using a time advancing scheme. Numerical experiments are carried out to demonstrate the convergence order and the efficiency of the proposed methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-26 datetime="2022-09-26T14:41:16.000Z">2022-09-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-26 datetime="2022-09-26T14:41:16.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119061"
                        open
                      >
                        <summary class="article-expander__title">Physicists Rewrite a Quantum Rule That Clashes With Our Universe</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/physicists-rewrite-a-quantum-rule-that-clashes-with-our-universe-20220926/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/09/Unitarity_1200_Social.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The past and the future are tightly linked in conventional quantum mechanics. Perhaps too tightly. A tweak to the theory could let quantum possibilities increase as space expands.            
The post Physicists Rewrite a Quantum Rule That Clashes With Our Universe first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;29
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00235"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Reprogramming the Cleavage Specificity of Botulinum Neurotoxin Serotype B1</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00235">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00235</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00156"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Quantification of Genome Editing and Transcriptional Control Capabilities Reveals Hierarchies among Diverse CRISPR/Cas Systems in Human Cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00156">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00156</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0087627?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Abrupt transition of the efficient vaccination strategy in a population with heterogeneous fatality rates</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0087627?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
An insufficient supply of an effective SARS-CoV-2 vaccine in most countries demands an effective vaccination strategy to minimize the damage caused by the disease. Currently, many countries vaccinate their population in descending order of age (i.e., descending order of fatality rate) to minimize the deaths caused by the disease; however, the effectiveness of this strategy needs to be quantitatively assessed. We employ the susceptible–infected–recovered–dead model to investigate various vaccination strategies. We constructed a metapopulation model with heterogeneous contact and fatality rates and investigated the effectiveness of vaccination strategies to reduce epidemic mortality. We found that the fatality-based strategy, which is currently employed in many countries, is more effective when the contagion rate is high and vaccine supply is low, but the contact-based method outperforms the fatality-based strategy when there is a sufficiently high supply of the vaccine. We identified a discontinuous transition of the optimal vaccination strategy and path-dependency analogous to hysteresis. This transition and path-dependency imply that combining the fatality-based and contact-based strategies is ineffective in reducing the number of deaths. Furthermore, we demonstrate that such phenomena occur in real-world epidemic diseases, such as tuberculosis and COVID-19. We also show that the conclusions of this research are valid even when the complex epidemic stages, efficacy of the vaccine, and reinfection are considered.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0121186?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Erratum: “Variational inference of the drift function for stochastic differential equations driven by Lévy processes” [Chaos 32, 061103 (2022)]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0121186?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102904?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Crises and chaotic scattering in hydrodynamic pilot-wave experiments</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102904?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Theoretical foundations of chaos have been predominantly laid out for finite-dimensional dynamical systems, such as the three-body problem in classical mechanics and the Lorenz model in dissipative systems. In contrast, many real-world chaotic phenomena, e.g., weather, arise in systems with many (formally infinite) degrees of freedom, which limits direct quantitative analysis of such systems using chaos theory. In the present work, we demonstrate that the hydrodynamic pilot-wave systems offer a bridge between low- and high-dimensional chaotic phenomena by allowing for a systematic study of how the former connects to the latter. Specifically, we present experimental results, which show the formation of low-dimensional chaotic attractors upon destabilization of regular dynamics and a final transition to high-dimensional chaos via the merging of distinct chaotic regions through a crisis bifurcation. Moreover, we show that the post-crisis dynamics of the system can be rationalized as consecutive scatterings from the nonattracting chaotic sets with lifetimes following exponential distributions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107866?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Cluster synchronization induced by manifold deformation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107866?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Pinning control of cluster synchronization in a globally connected network of chaotic oscillators is studied. It is found in simulations that when the pinning strength exceeds a critical value, the oscillators are synchronized into two different clusters, one formed by the pinned oscillators and the other one formed by the unpinned oscillators. The numerical results are analyzed by the generalized method of master stability function (MSF), in which it is shown that whereas the method is able to predict the synchronization behaviors of the pinned oscillators, it fails to predict the synchronization behaviors of the unpinned oscillators. By checking the trajectories of the oscillators in the phase space, it is found that the failure is attributed to the deformed synchronization manifold of the unpinned oscillators, which is clearly deviated from that of isolated oscillator under strong pinnings. A similar phenomenon is also observed in the pinning control of cluster synchronization in a complex network of symmetric structures and in the self-organized cluster synchronization of networked neural oscillators. The findings are important complements to the generalized MSF method and provide an alternative approach to the manipulation of synchronization behaviors in complex network systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100698?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Noise-induced switching in an oscillator with pulse delayed feedback: A discrete stochastic modeling approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100698?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study the dynamics of an oscillatory system with pulse delayed feedback and noise of two types: (i) phase noise acting on the oscillator and (ii) stochastic fluctuations of the feedback delay. Using an event-based approach, we reduce the system dynamics to a stochastic discrete map. For weak noise, we find that the oscillator fluctuates around a deterministic state, and we derive an autoregressive model describing the system dynamics. For stronger noise, the oscillator demonstrates noise-induced switching between various deterministic states; our theory provides a good estimate of the switching statistics in the linear limit. We show that the robustness of the system toward this switching is strikingly different depending on the type of noise. We compare the analytical results for linear coupling to numerical simulations of nonlinear coupling and find that the linear model also provides a qualitative explanation for the differences in robustness to both types of noise. Moreover, phase noise drives the system toward higher frequencies, while stochastic delays do not, and we relate this effect to our theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0110865?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Imaging in reflecting spheres</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0110865?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study the formation of images in a reflective sphere in three configurations using caustics on the field of light rays. The optical wavefront emerging from a source point reaching a subject following passage through the optical system is, in general, a Gaussian surface with partial focus along the two principal directions of the Gaussian surface; i.e., there are two images of the source point, each with partial focus. As the source point moves, the images move on two surfaces, referred to as viewable surfaces. In our systems, one viewable surface consists of points with radial focus and the other consists of points with azimuthal focus. The problems we study are (1) imaging of a parallel beam of light, (2) imaging of the infinite viewed from a location outside the sphere, and (3) imaging of a planar object viewed through the point of its intersection with the radial line normal to the plane. We verify the existence of two images experimentally and show that the distance between them agrees with the computations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100315?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Two competing simplicial irreversible epidemics on simplicial complex</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100315?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Higher-order interactions have significant implications for the dynamics of competing epidemic spreads. In this paper, a competing spread model for two simplicial irreversible epidemics (i.e., susceptible–infected–removed epidemics) on higher-order networks is proposed. The simplicial complexes are based on synthetic (including homogeneous and heterogeneous) and real-world networks. The spread process of two epidemics is theoretically analyzed by extending the microscopic Markov chain approach. When the two epidemics have the same 2-simplex infection rate and the 1-simplex infection rate of epidemic [math] ([math]) is fixed at zero, an increase in the 1-simplex infection rate of epidemic [math] ([math]) causes a transition from continuous growth to sharp growth in the spread of epidemic [math] with [math]. When [math], the growth of epidemic [math] is always continuous. With the increase of [math], the outbreak threshold of epidemic [math] is delayed. When the difference in 1-simplex infection rates between the two epidemics reaches approximately three times, the stronger side obviously dominates. Otherwise, the coexistence of the two epidemics is always observed. When the 1-simplex infection rates are symmetrical, the increase in competition will accelerate the spread process and expand the spread area of both epidemics; when the 1-simplex infection rates are asymmetrical, the spread area of one epidemic increases with an increase in the 1-simplex infection rate from this epidemic while the other decreases. Finally, the influence of 2-simplex infection rates on the competing spread is discussed. An increase in 2-simplex infection rates leads to sharp growth in one of the epidemics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098707?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Learning spatiotemporal chaos using next-generation reservoir computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098707?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Forecasting the behavior of high-dimensional dynamical systems using machine learning requires efficient methods to learn the underlying physical model. We demonstrate spatiotemporal chaos prediction using a machine learning architecture that, when combined with a next-generation reservoir computer, displays state-of-the-art performance with a computational time [math]–[math] times faster for training process and training data set [math] times smaller than other machine learning algorithms. We also take advantage of the translational symmetry of the model to further reduce the computational cost and training data, each by a factor of [math]10.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0095737?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamics study of integrable turbulence with fourth-order nonlinear Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0095737?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In this paper, we focus on the fourth-order nonlinear Schrödinger equation, which can describe the optical system and the Heisenberg spin system. We consider a continuous wave perturbed by the one-dimensional random rough surface as the initial condition. First, we numerically resolve the eigenvalues under different control parameters utilizing the Fourier collocation method. Then, we simulate the evolution of this equation under the above initial conditions via the symmetrical split-step Fourier method. Moreover, we investigate the “steady” chaotic state by evolving a large number of initial conditions for the same control parameters. We find that the control parameters of the initial condition affect the number and intensity of rogue waves (RWs) in integrable turbulence. In particular, we locate the inflection point where the control parameter affects the velocities of solitons and the inconsistency within the parameter of the contribution to the generation of RWs. We further verify that the collision between breathers, solitons, and breathers and solitons can generate RWs. These results will enable us to understand the turbulent state and the formation mechanism of RWs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0088880?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Synchronization in a multilevel network using the Hamilton–Jacobi–Bellman (HJB) technique</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0088880?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
This paper presents the optimal control and synchronization problem of a multilevel network of Rössler chaotic oscillators. Using the Hamilton–Jacobi–Bellman technique, the optimal control law with a three-state variable feedback is designed such that the trajectories of all the Rössler oscillators in the network are optimally synchronized at each level. Furthermore, we provide numerical simulations to demonstrate the effectiveness of the proposed approach for the cases of one and three networks. A perfect correlation between the MATLAB and PSpice results was obtained, thus allowing the experimental validation of our designed controller and shows the effectiveness of the theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.09.23.509237v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Tunable dynamics in a multi-strain transcriptional pulse generator [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.09.23.509237v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A major challenge in synthetic biology is the manipulation of engineered gene circuits toward a specified behavior. This challenge becomes more difficult as synthetic systems become more complex by incorporating additional genes or strains. Here we demonstrate that circuit dynamics can be tuned in synthetic consortia through the manipulation of strain fractions within the community. To do this, we constructed a microbial consortium comprised of three strains of engineered Escherichia coli that, when co-cultured, use homoserine lactone (HSL) mediated intercellular signaling to create a multi-strain incoherent type-1 feedforward loop (I1-FFL). Like naturally occurring I1-FFL motifs in gene networks, this engineered microbial consortium acts as a pulse generator of gene expression. We demonstrated that the amplitude of the pulse can be easily tuned by adjusting the relative population fractions of the strains. We created a mathematical model for the temporal dynamics of the microbial consortium and, using this model, identified population fractions that produced desired pulse characteristics. Our work demonstrates that intercellular gene circuits can be effectively tuned simply by adjusting the starting fractions of each strain type.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01136-x"
                        open
                      >
                        <summary class="article-expander__title">Chemically inducible split protein regulators for mammalian cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01136-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01136-x/MediaObjects/41589_2022_1136_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>An INSPIRE platform for therapeutic regulators and sensors, based on split human domains binding physiological or clinically approved compounds, has been developed and demonstrated on several human proteins in a relevant ligand concentration range.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33221-z"
                        open
                      >
                        <summary class="article-expander__title">The material properties of a bacterial-derived biomolecular condensate tune biological function in natural and synthetic systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33221-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33221-z/MediaObjects/41467_2022_33221_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>“Intracellular phase separation is emerging as a universal principle for organizing biochemical reactions in time and space. Here the authors show that PopZ condensate dynamics support cell division and using PopZ modular architecture, the tunable PopTag platform was developed to enable designer condensates.”</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01793-5"
                        open
                      >
                        <summary class="article-expander__title">Scaling relations for auxin waves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01793-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We analyze an ‘up-the-gradient’ model for the formation of transport channels of the phytohormone auxin, through auxin-mediated polarization of the PIN1 auxin transporter. We show that this model admits a family of travelling wave solutions that is parameterized by the height of the auxin-pulse. We uncover scaling relations for the speed and width of these waves and verify these rigorous results with numerical computations. In addition, we provide explicit expressions for the leading-order wave profiles, which allows the influence of the biological parameters in the problem to be readily identified. Our proofs are based on a generalization of the scaling principle developed by Friesecke and Pego to construct pulse solutions to the classic Fermi–Pasta–Ulam–Tsingou model, which describes a one-dimensional chain of coupled nonlinear springs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;87
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01806-3"
                        open
                      >
                        <summary class="article-expander__title">Inference in Gaussian state-space models with mixed effects for multiple epidemic dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01806-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The estimation from available data of parameters governing epidemics is a major challenge. In addition to usual issues (data often incomplete and noisy), epidemics of the same nature may be observed in several places or over different periods. The resulting possible inter-epidemic variability is rarely explicitly considered. Here, we propose to tackle multiple epidemics through a unique model incorporating a stochastic representation for each epidemic and to jointly estimate its parameters from noisy and partial observations. By building on a previous work for prevalence data, a Gaussian state-space model is extended to a model with mixed effects on the parameters describing simultaneously several epidemics and their observation process. An appropriate inference method is developed, by coupling the SAEM algorithm with Kalman-type filtering. Moreover, we consider here incidence data, which requires to develop a new version of the filtering algorithm. Its performances are investigated on SIR simulated epidemics for prevalence and incidence data. Our method outperforms an inference method separately processing each dataset. An application to SEIR influenza outbreaks in France over several years using incidence data is also carried out. Parameter estimations highlight a non-negligible variability between influenza seasons, both in transmission and case reporting. The main contribution of our study is to rigorously and explicitly account for the inter-epidemic variability between multiple outbreaks, both from the viewpoint of modeling and inference with a parsimonious statistical model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-24 datetime="2022-09-24T00:00:00.000Z">2022-09-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-24 datetime="2022-09-24T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33288-8"
                        open
                      >
                        <summary class="article-expander__title">Synthetic neuromorphic computing in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33288-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33288-8/MediaObjects/41467_2022_33288_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Computational properties of neuronal networks have been applied to computing systems using simplified models comprising repeated connected nodes. Here the authors create layered assemblies of genetically encoded devices that perform non-binary logic computation and signal processing using combinatorial promoters and feedback regulation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03987-5"
                        open
                      >
                        <summary class="article-expander__title">Targeting non-coding RNA family members with artificial endonuclease XNAzymes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03987-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03987-5/MediaObjects/42003_2022_3987_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Engineered RNA endonuclease XNAzymes can specifically target individual disease-associated non-coding RNAs (ncRNAs) in vitro with potential as platform technology for precision detection or knockdown of individual microRNAs or longer ncRNAs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;36
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02012-5"
                        open
                      >
                        <summary class="article-expander__title">A New Adaptation Strategy for Multi-resolution Method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02012-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Adaptive mesh refinement (AMR) and wavelet-based multi-resolution technique, which refine the spatial resolution in regions of interest and coarsen the mesh in other regions, are widely used in scientific computing for higher computational efficiency. The key component of AMR and the multi-resolution method is the adaptation strategy including the regularity estimation. In this paper, a new adaptation strategy for AMR and the multi-resolution method is proposed. Different from AMR, which measures the function regularity based on an empirical gradient operator, and the multi-resolution method with complicated wavelet analysis, the new approach examines the solution smoothness based on the high-order TENO reconstruction (Fu et al. in J Comput Phys 305:333–359, 2016). The core idea is that the TENO scheme not only provides the reconstructed data at the cell interface for flux evaluation but also classifies the local flow scales as smooth or nonsmooth on the discretized mesh. Since the scale-separation procedure is achieved in the spectral and characteristic space, the new adaptation strategy is weakly problem-dependent. Moreover, the extra complexity due to the empirical gradient computing or the wavelet analysis is eliminated. Based on Harten’s finite-volume multi-resolution method (Harten in J Comput Phys 115(2):319–338, 1994), a set of benchmark cases is simulated to validate the performance of the proposed method.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-23 datetime="2022-09-23T04:00:00.000Z">2022-09-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-23 datetime="2022-09-23T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00216"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Novel Split Intein-Mediated Enzymatic Channeling Accelerates the Multimeric Bioconversion Pathway of Ginsenoside</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00216">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00216</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00291"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineered Spore-Forming Bacillus as a Microbial Vessel for Long-Term DNA Data Storage</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00291">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00291</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33181-4"
                        open
                      >
                        <summary class="article-expander__title">Activation of stably silenced genes by recruitment of a synthetic de-methylating module</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33181-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33181-4/MediaObjects/41467_2022_33181_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Stably silenced genes with methylated CpG at the promoter are refractory to current CRISPR activation systems. Here the authors create a more robust activation system, TETact that recruits DNA-demethylating TET1 with transcriptional activators.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-22 datetime="2022-09-22T15:50:00.000Z">2022-09-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-22 datetime="2022-09-22T15:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/cell-rover-exploring-augmenting-inner-world-cell-0922"
                        open
                      >
                        <summary class="article-expander__title">Cell Rover: Exploring and augmenting the inner world of the cell</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/cell-rover-exploring-augmenting-inner-world-cell-0922">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/cell-antenna.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MIT researchers demonstrate an intracellular antenna that&#x27;s compatible with 3D biological systems and can operate wirelessly inside a living cell.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00111"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A p‑Coumaroyl-CoA Biosensor for Dynamic Regulation of Naringenin Biosynthesis in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00111">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00111</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093842?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamical systems defined on simplicial complexes: Symmetries, conjugacies, and invariant subspaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093842?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We consider the general model for dynamical systems defined on a simplicial complex. We describe the conjugacy classes of these systems and show how symmetries in a given simplicial complex manifest in the dynamics defined thereon, especially with regard to invariant subspaces in the dynamics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107583?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Structural position vectors and symmetries in complex networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107583?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Symmetries, due to their fundamental importance to dynamical processes on networks, have attracted a great deal of current research. Finding all symmetric nodes in large complex networks typically relies on automorphism groups from algebraic-group theory, which are solvable in quasipolynomial time. We articulate a conceptually appealing and computationally extremely efficient approach to finding and characterizing all symmetric nodes by introducing a structural position vector (SPV) for each node in networks. We establish the mathematical result that symmetric nodes must have the same SPV value and demonstrate, using six representative complex networks from the real world, that all symmetric nodes in these networks can be found in linear time. Furthermore, the SPVs not only characterize the similarity of nodes but also quantify the nodal influences in propagation dynamics. A caveat is that the proved mathematical result relating the SPV values to nodal symmetries is not sufficient; i.e., nodes having the same SPV values may not be symmetric, which arises in regular networks or networks with a dominant regular component. We point out with an analysis that this caveat is, in fact, shared by the known existing approaches to finding symmetric nodes in the literature. We further argue, with the aid of a mathematical analysis, that our SPV method is generally effective for finding the symmetric nodes in real-world networks that typically do not have a dominant regular component. Our SPV-based framework, therefore, provides a physically intuitive and computationally efficient way to uncover, understand, and exploit symmetric structures in complex networks arising from real-world applications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0108672?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Matrix coupling and generalized frustration in Kuramoto oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0108672?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The Kuramoto model describes the synchronization of coupled oscillators that have different natural frequencies. Among the many generalizations of the original model, Kuramoto and Sakaguchi (KS) proposed a frustrated version that resulted in dynamic behavior of the order parameter, even when the average natural frequency of the oscillators is zero. Here, we consider a generalization of the frustrated KS model that exhibits new transitions to synchronization. The model is identical in form to the original Kuramoto model but written in terms of unit vectors and with the coupling constant replaced by a coupling matrix. The matrix breaks the rotational symmetry and forces the order parameter to point in the direction of the eigenvector with the highest eigenvalue, when the eigenvalues are real. For complex eigenvalues, the module of order parameter oscillates while it rotates around the unit circle, creating active states. We derive the complete phase diagram for the Lorentzian distribution of frequencies using the Ott–Antonsen ansatz. We also show that changing the average value of the natural frequencies leads to further phase transitions where the module of the order parameter goes from oscillatory to static.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107504?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The qualitative and quantitative relationships between pattern formation and average degree in networked reaction-diffusion systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107504?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The Turing pattern is an important dynamic behavior characteristic of activator–inhibitor systems. Differentiating from traditional assumption of activator–inhibitor interactions in a spatially continuous domain, a Turing pattern in networked reaction-diffusion systems has received much attention during the past few decades. In spite of its great progress, it still fails to evaluate the precise influences of network topology on pattern formation. To this end, we try to promote the research on this important and interesting issue from the point of view of average degree—a critical topological feature of networks. We first qualitatively analyze the influence of average degree on pattern formation. Then, a quantitative relationship between pattern formation and average degree, the exponential decay of pattern formation, is proposed via nonlinear regression. The finding holds true for several activator–inhibitor systems including biology model, ecology model, and chemistry model. The significance of this study lies that the exponential decay not only quantitatively depicts the influence of average degree on pattern formation, but also provides the possibility for predicting and controlling pattern formation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01802-7"
                        open
                      >
                        <summary class="article-expander__title">Locked fronts in a discrete time discrete space population model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01802-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A model of population growth and dispersal is considered where the spatial habitat is a lattice and reproduction occurs generationally. The resulting discrete dynamical system exhibits velocity locking, where rational speed invasion fronts are observed to persist as parameters are varied. In this article, we construct locked fronts for a particular piecewise linear reproduction function. These fronts are shown to be linear combinations of exponentially decaying solutions to the linear system near the unstable state. Based upon these front solutions, we then derive expressions for the boundary of locking regions in parameter space. We obtain leading order expansions for the locking regions in the limit as the migration parameter tends to zero. Strict spectral stability in exponentially weighted spaces is also established.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-21 datetime="2022-09-21T04:00:00.000Z">2022-09-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-21 datetime="2022-09-21T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00365"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Light-Dependent Control of Bacterial Expression at the mRNA Level</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00365">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00365</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00638"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Precise Genomic Riboregulator Control of Metabolic Flux in Microbial Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00638">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00638</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33191-2"
                        open
                      >
                        <summary class="article-expander__title">A de novo matrix for macroscopic living materials from bacteria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33191-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33191-2/MediaObjects/41467_2022_33191_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Engineered living materials (ELMs) embed living cells in a biopolymer matrix to create novel materials with tailored functions. In this work, the authors engineered bacteria to grow novel macroscopic materials that can be reshaped, functionalized, and used to filter contaminated water while also showing that the stiffness of these materials can be tuned through genetic changes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01805-4"
                        open
                      >
                        <summary class="article-expander__title">Analysis of the onset of a regime shift and detecting early warning signs of major population changes in a two-trophic three-species predator-prey model with long-term transients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01805-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Identifying early warning signs of sudden population changes and mechanisms leading to regime shifts are highly desirable in population biology. In this paper, a two-trophic ecosystem comprising of two species of predators, competing for their common prey, with explicit interference competition is considered. With proper rescaling, the model is portrayed as a singularly perturbed system with fast prey dynamics and slow dynamics of the predators. In a parameter regime near singular Hopf bifurcation, chaotic mixed-mode oscillations (MMOs), featuring concatenation of small and large amplitude oscillations are observed as long-lasting transients before the system approaches its asymptotic state. To analyze the dynamical cause that initiates a large amplitude oscillation in an MMO orbit, the model is reduced to a suitable normal form near the singular-Hopf point. The normal form possesses a separatrix surface that separates two different types of oscillations. A large amplitude oscillation is initiated if a trajectory moves from the “inner” to the “outer side” of this surface. A set of conditions on the normal form variables are obtained to determine whether a trajectory would exhibit another cycle of MMO dynamics before experiencing a regime shift (i.e. approaching its asymptotic state). These conditions serve as early warning signs for a sudden population shift as well as detect the onset of a regime shift in this ecological model.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01978-6"
                        open
                      >
                        <summary class="article-expander__title">A Multipoint Flux Approximation with a Diamond Stencil and a Non-Linear Defect Correction Strategy for the Numerical Solution of Steady State Diffusion Problems in Heterogeneous and Anisotropic Media Satisfying the Discrete Maximum Principle</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01978-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In the present paper, we solve the steady state diffusion equation in 3D domains by means of a cell-centered finite volume method that uses a Multipoint Flux Approximation with a Diamond Stencil and a Non-Linear defect correction strategy (MPFA-DNL) to guarantee the Discrete Maximum Principle (DMP). Our formulation is based in the fact that the flux of MPFA methods can be split into two different parts: a Two Point Flux Approximation (TPFA) component and the Cross-Diffusion Terms (CDT). In the linear MPFA-D method, this split is particularly simple since it lies at the core of the original method construction. In this context, we introduce a non-linear defect correction, aiming to mitigate, whenever necessary, the contributions from the CDT, avoiding, this way, spurious oscillations and DMP violations. Our new MPFA-DNL scheme is locally conservative and capable of dealing with arbitrary anisotropic diffusion tensors and unstructured meshes, without harming the second order convergence rates of the original MPFA-D. To appraise the accuracy and robustness of our formulation, we solve some benchmark problems found in literature. In this paper, we restrict ourselves to tetrahedral meshes, even though, in principle, there is no restriction to extend the method to other polyhedral control volumes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-20 datetime="2022-09-20T07:00:00.000Z">2022-09-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-20 datetime="2022-09-20T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0507?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">virDTL: Viral Recombination Analysis Through Phylogenetic Reconciliation and Its Application to Sarbecoviruses and SARS-CoV-2</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0507?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00159"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] A Versatile in Vivo DNA Assembly Toolbox for Fungal Strain Engineering</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00159">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00159</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096689?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Synchronization of Belousov–Zhabotinsky oscillators with electrochemical coupling in a spontaneous process</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096689?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
A passive electrochemical coupling approach is proposed to induce spontaneous synchronization between chemical oscillators. The coupling exploits the potential difference between a catalyst redox couple in the Belousov–Zhabotinsky (BZ) reaction, without external feedback, to induce surface reactions that impact the kinetics of the bulk system. The effect of coupling in BZ oscillators under batch condition is characterized using phase synchronization measures. Although the frequency of the oscillators decreases nonlinearly over time, by a factor of 2 or more within 100 cycles, the coupling is strong enough to maintain synchronization. In such a highly drifting system, the Gibbs–Shannon entropy of the cyclic phase difference distribution can be used to quantify the coupling effect. We extend the Oregonator BZ model to account for the drifting natural frequencies in batch condition and for electrochemical coupling, and numerical simulations of the effect of acid concentration on synchronization patterns are in agreement with the experiments. Because of the passive nature of coupling, the proposed coupling scheme can open avenues for designing pattern recognition and neuromorphic computation systems using chemical reactions in a spontaneous process.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101933?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Transport and nonequilibrium phase transitions in polygonal urn models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101933?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study the deterministic dynamics of [math] point particles moving at a constant speed in a 2D table made of two polygonal urns connected by an active rectangular channel, which applies a feedback control on the particles, inverting the horizontal component of their velocities when their number in the channel exceeds a fixed threshold. Such a bounce-back mechanism is non-dissipative: it preserves volumes in phase space. An additional passive channel closes the billiard table forming a circuit in which a stationary current may flow. Under specific constraints on the geometry and on the initial conditions, the large [math] limit allows nonequilibrium phase transitions between homogeneous and inhomogeneous phases. The role of ergodicity in making a probabilistic theory applicable is discussed for both rational and irrational urns. The theoretical predictions are compared with the numerical simulation results. Connections with the dynamics of feedback-controlled biological systems are highlighted.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0091342?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Group percolation in interdependent networks with reinforcement network layer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0091342?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In many real-world interdependent network systems, nodes often work together to form groups, which can enhance robustness to resist risks. However, previous group percolation models are always of a first-order phase transition, regardless of the group size distribution. This motivates us to investigate a generalized model for group percolation in interdependent networks with a reinforcement network layer to eliminate collapse. Some backup devices that are equipped for a density [math] of reinforced nodes constitute the reinforcement network layer. For each group, we assume that at least one node of the group can function in one network and a node in another network depends on the group to function. We find that increasing the density [math] of reinforcement nodes and the size [math] of the dependency group can significantly enhance the robustness of interdependent networks. Importantly, we find the existence of a hybrid phase transition behavior and propose a method for calculating the shift point of percolation types. The most interesting finding is the exact universal solution to the minimal density [math] of reinforced nodes (or the minimum group size [math]) to prevent abrupt collapse for Erdős–Rényi, scale-free, and regular random interdependent networks. Furthermore, we present the validity of the analytic solutions for a triple point [math] (or [math]), the corresponding phase transition point [math], and second-order phase transition points [math] in interdependent networks. These findings might yield a broad perspective for designing more resilient interdependent infrastructure networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0103427?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Application of the Slater criteria to localize invariant tori in Hamiltonian mappings</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0103427?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We investigate the localization of invariant spanning curves for a family of two-dimensional area-preserving mappings described by the dynamical variables [math] and [math] by using Slater’s criterion. The Slater theorem says there are three different return times for an irrational translation over a circle in a given interval. The returning time, which measures the number of iterations a map needs to return to a given periodic or quasi periodic region, has three responses along an invariant spanning curve. They are related to a continued fraction expansion used in the translation and obey the Fibonacci sequence. The rotation numbers for such curves are related to a noble number, leading to a devil’s staircase structure. The behavior of the rotation number as a function of invariant spanning curves located by Slater’s criterion resulted in an expression of a power law in which the absolute value of the exponent is equal to the control parameter [math] that controls the speed of the divergence of [math] in the limit the action [math] is sufficiently small.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33207-x"
                        open
                      >
                        <summary class="article-expander__title">Screening microbially produced Δ9-tetrahydrocannabinol using a yeast biosensor workflow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33207-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33207-x/MediaObjects/41467_2022_33207_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Microbial production of cannabinoids promises a cheaper and more sustainable route to these important therapeutic molecules, but strain improvement and screening is challenging. Here, the authors develop a yeast-based Δ9-tetrahydrocannabinol (THC) biosensor for screening microbial mutant libraries.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01801-8"
                        open
                      >
                        <summary class="article-expander__title">Randomization for the susceptibility effect of an infectious disease intervention</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01801-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Randomized trials of infectious disease interventions, such as vaccines, often focus on groups of connected or potentially interacting individuals. When the pathogen of interest is transmissible between study subjects, interference may occur: individual infection outcomes may depend on treatments received by others. Epidemiologists have defined the primary parameter of interest—called the “susceptibility effect”—as a contrast in infection risk under treatment versus no treatment, while holding exposure to infectiousness constant. A related quantity—the “direct effect”—is defined as an unconditional contrast between the infection risk under treatment versus no treatment. The purpose of this paper is to show that under a widely recommended randomization design, the direct effect may fail to recover the sign of the true susceptibility effect of the intervention in a randomized trial when outcomes are contagious. The analytical approach uses structural features of infectious disease transmission to define the susceptibility effect. A new probabilistic coupling argument reveals stochastic dominance relations between potential infection outcomes under different treatment allocations. The results suggest that estimating the direct effect under randomization may provide misleading conclusions about the effect of an intervention—such as a vaccine—when outcomes are contagious. Investigators who estimate the direct effect may wrongly conclude an intervention that protects treated individuals from infection is harmful, or that a harmful treatment is beneficial.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01804-5"
                        open
                      >
                        <summary class="article-expander__title">Limits of epidemic prediction using SIR models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01804-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The Susceptible-Infectious-Recovered (SIR) equations and their extensions comprise a commonly utilized set of models for understanding and predicting the course of an epidemic. In practice, it is of substantial interest to estimate the model parameters based on noisy observations early in the outbreak, well before the epidemic reaches its peak. This allows prediction of the subsequent course of the epidemic and design of appropriate interventions. However, accurately inferring SIR model parameters in such scenarios is problematic. This article provides novel, theoretical insight on this issue of practical identifiability of the SIR model. Our theory provides new understanding of the inferential limits of routinely used epidemic models and provides a valuable addition to current simulate-and-check methods. We illustrate some practical implications through application to a real-world epidemic data set.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-19 datetime="2022-09-19T12:42:09.000Z">2022-09-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-19 datetime="2022-09-19T12:42:09.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096983?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Resilience of supply-chain systems under perturbations: A network approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096983?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Supply-chain systems (SCSs) are an indispensable part of our daily infrastructures. Note that a small perturbation in a SCS can be amplified, eliciting cascading failures. It is of significant value to ensure a high resilience of SCSs. However, due to the complexity of SCSs, it is quite challenging to study their resilience under conditions of perturbations. In view of this, this paper presents a complex network perspective toward the resilience of SCSs. To achieve this goal, a complex SCS is modeled as a multilayer supply-chain network (SCN) with physical organizations being modeled as nodes and interactions among them as edges. A modeled SCN contains three types of nodes, i.e., suppliers, manufacturers, and retailers. An algorithm is proposed to construct a multilayer SCN. For each layer of a multilayer SCN, two kinds of networks, i.e., networks with Poisson degree distributions and networks with power-law degree distributions, are considered. For a given multilayer SCN, a ripple-effect network model is proposed to analyze its resilience under perturbations. Regarding the perturbations, two scenarios, i.e., random node failures and target node failures, are adopted in this research. In order to validate the effectiveness of the proposed network perspective, simulations on computer-generated SCNs are carried out. Interestingly, it is found that the resilience of SCNs under both random and target perturbations presents a discontinuous phase-change phenomenon, which indicates that SCNs are quite fragile under perturbations. It is further noticed that SCNs with power-law degree distributions are relatively more robust than SCNs with Poisson degree distributions. Although SCNs are found to be fragile, it has been discovered that denser interactions between different system organizations can enhance the network’s resilience.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097417?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dynamic scaling and stochastic fractal in nucleation and growth processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097417?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
A class of nucleation and growth models of a stable phase is investigated for various different growth velocities. It is shown that for growth velocities [math] and [math], where [math] and [math] are the mean domain size of the metastable phase (M-phase) and the mean nucleation time, respectively, the M-phase decays following a power law. Furthermore, snapshots at different time [math] that are taken to collect data for the distribution function [math] of the domain size [math] of the M-phase are found to obey dynamic scaling. Using the idea of data-collapse, we show that each snapshot is a self-similar fractal. However, for [math], such as in the classical Kolmogorov–Johnson–Mehl–Avrami model, and for [math], the decays of the M-phase are exponential and they are not accompanied by dynamic scaling. We find a perfect agreement between numerical simulation and analytical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0121435?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Erratum: “Saddle-center and periodic orbit: Dynamics near symmetric heteroclinic connection” [Chaos 31, 023113 (2021)]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0121435?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1461277?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">High Order Topological Asymptotics: Reconciling Layer Potentials and Compound Asymptotic Expansions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1461277?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 957-1001, September 2022. 
 Abstract. A systematic two-step procedure is proposed for the derivation of full asymptotic expansions of the solution of elliptic partial differential equations set on a domain perforated with a small hole on which a Dirichlet boundary condition is applied. First, an integral representation of the solution is sought, which enables us to exploit the explicit dependence with respect to the small parameter to predict the correct form of a two-scale ansatz. Second, the terms of the ansatz are characterized by the method of compound asymptotic expansions as the solutions of a cascade of successive interior and exterior domains. This allows us to interpret them as high-order correctors, for which error bounds can be proved using variational estimates. The methodology is illustrated on two different problems: we start by revisiting the perforated Poisson problem with Dirichlet boundary conditions on both the hole and the outer boundary, where we highlight how the method enables us to retrieve very naturally the correct form of the ansatz in the most delicate two-dimensional setting. Then, we provide original and complete asymptotic expansions for a perforated cell problem featuring periodic conditions.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00289"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Transcriptional Reactivation of Lignin Biosynthesis for the Heterologous Production of Etoposide Aglycone in Nicotiana benthamiana</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00289">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00289</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33255-3"
                        open
                      >
                        <summary class="article-expander__title">Biosynthesis of plant hemostatic dencichine in Escherichia coli</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33255-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33255-3/MediaObjects/41467_2022_33255_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Biosynthetic pathway of dencichine, a plant derived nature product that has found various pharmacological applications, is still elusive. Here, the authors design artificial pathways through retro-biosynthesis approaches and achieve its efficient production in E. coli by systematic metabolic and enzymatic engineering.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01809-0"
                        open
                      >
                        <summary class="article-expander__title">On the qualitative study of a two-trophic plant–herbivore model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01809-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The coexistence of plant-herbivore populations in an ecological system is a fundamental topic of research in mathematical ecology. Plant–herbivore interactions are often described by using discrete-time models in the case of non-overlapping generations: such generations have some specific time interval of life and their old generations are replaced by new generations after some regular interval of time. Keeping in mind the dynamical reliability of continuous-time models we presented two discrete-time plant–herbivore models. Mainly, by applying Euler’s forward method a discrete-time plant–herbivore model is obtained from a continuous-time plant–herbivore model. In addition, a dynamically consistent discrete-time plant–herbivore model is obtained by applying a nonstandard difference scheme. Moreover, local stability is discussed and the existence of bifurcation about positive equilibrium is shown under some mathematical conditions. To control bifurcation and chaos, a modified hybrid technique is developed. Finally, to support our theocratical results and to show the dynamical reliability of the nonstandard difference scheme some numerical examples are provided.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01799-z"
                        open
                      >
                        <summary class="article-expander__title">Vertex results for the robust analysis of uncertain biochemical systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01799-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We consider the problem of assessing the sensitivity of uncertain biochemical systems in the presence of input perturbations (either constant or periodic) around a stable steady state. 

       In particular, we propose approaches for the robust sensitivity analysis of systems with uncertain parameters assumed to take values in a hyper-rectangle. We highlight vertex results, which allow us to check whether a property is satisfied for all parameter choices in the hyper-rectangle by simply checking whether it is satisfied for all parameter choices at the vertices of the hyper-rectangle. We show that, for a vast class of systems, including (bio)chemical reaction networks with mass-action kinetics, the system Jacobian has a totally multiaffine structure (namely, all minors of the Jacobian matrix are multiaffine functions of the uncertain parameters), which can be exploited to obtain several vertex results. We consider different problems: robust non-singularity; robust stability of the steady-state; robust steady-state sensitivity analysis, in the case of constant perturbations; robust frequency-response sensitivity analysis, in the presence of periodic perturbations; and robust adaptation analysis. The developed theory is then applied to gain insight into some examples of uncertain biochemical systems, including the incoherent feed-forward loop, the coherent feed-forward loop, the Brusselator oscillator and the Goldbeter oscillator.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;65
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-17 datetime="2022-09-17T00:00:00.000Z">2022-09-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-17 datetime="2022-09-17T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01788-2"
                        open
                      >
                        <summary class="article-expander__title">A model and a numerical scheme for the description of distribution and abundance of individuals</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01788-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We introduce a model in the context of ecology that can be used to describe the distribution and abundance of individuals when data from field work is extremely limited (for example, in the case of endangered species). Our procedure is based on an intuitive understanding of the physical properties of phenomena. The idea is that individuals have the tendency to be attracted (or repulsed) to certain properties of the environment. At the same time, they are spread in such a way that if there is no reason for them to be in some specific locations, then they are uniformly distributed throughout the region. Our model draws from quantum mechanics, by using quantum Hamiltonians in the context of classical statistical mechanics. The equilibrium between the spreading and the attractive (or repulsive) forces determines the behavior of the species that we model, and this is expressed in terms of a global control problem of an energy operator which is the sum of a kinetic term (spreading) and a potential (attraction or repulsion). We focus on the full probability measure and a global control of the model (instead of looking at conditional measures that generate a global measure). Furthermore, we propose a numerical solution to this global control problem that overcomes the well-known major difficulty of Gibbs sampling (annealing) which is the fact that a global control is hardly reachable when the number of variables is large (the algorithms get stuck in non-optimal states).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01808-1"
                        open
                      >
                        <summary class="article-expander__title">Efficiency functionals for the Lévy flight foraging hypothesis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01808-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We consider a forager diffusing via a fractional heat equation and we introduce several efficiency functionals whose optimality is discussed in relation to the Lévy exponent of the evolution equation. Several biological scenarios, such as a target close to the forager, a sparse environment, a target located away from the forager and two targets are specifically taken into account. The optimal strategies of each of these configurations are here analyzed explicitly also with the aid of some special functions of classical flavor and the results are confronted with the existing paradigms of the Lévy foraging hypothesis. Interestingly, one discovers bifurcation phenomena in which a sudden switch occurs between an optimal (but somehow unreliable) Lévy foraging pattern of inverse square law type and a less ideal (but somehow more secure) classical Brownian motion strategy. Additionally, optimal foraging strategies can be detected in the vicinity of the Brownian one even in cases in which the Brownian one is pessimizing an efficiency functional.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;28
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01803-6"
                        open
                      >
                        <summary class="article-expander__title">A mathematical model for simulating the spread of a disease through a country divided into geographical regions with different population densities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01803-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The SIR (susceptible-infectious-recovered) model is a well known method for predicting the number of people (or animals) in a population who become infected by and then recover from a disease. Modifications can include categories such people who have been exposed to the disease but are not yet infectious or those who die from the disease. However, the model has nearly always been applied to the entire population of a country or state but there is considerable observational evidence that diseases can spread at different rates in densely populated urban regions and sparsely populated rural areas. This work presents a new approach that applies a SIR type model to a country or state that has been divided into a number of geographical regions, and uses different infection rates in each region which depend on the population density in that region. Further, the model contains a simple matrix based method for simulating the movement of people between different regions. The model is applied to the spread of disease in the United Kingdom and the state of Rio Grande do Sul in Brazil.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-16 datetime="2022-09-16T12:44:04.000Z">2022-09-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-16 datetime="2022-09-16T12:44:04.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0090443?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">A global synchronization theorem for oscillators on a random graph</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0090443?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Consider [math] identical Kuramoto oscillators on a random graph. Specifically, consider Erdős–Rényi random graphs in which any two oscillators are bidirectionally coupled with unit strength, independently and at random, with probability [math]. We say that a network is globally synchronizing if the oscillators converge to the all-in-phase synchronous state for almost all initial conditions. Is there a critical threshold for [math] above which global synchrony is extremely likely but below which it is extremely rare? It is suspected that a critical threshold exists and is close to the so-called connectivity threshold, namely, [math] for [math]. Ling, Xu, and Bandeira made the first progress toward proving a result in this direction: they showed that if [math], then Erdős–Rényi networks of Kuramoto oscillators are globally synchronizing with high probability as [math]. Here, we improve that result by showing that [math] suffices. Our estimates are explicit: for example, we can say that there is more than a [math] chance that a random network with [math] and [math] is globally synchronizing.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0088293?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Riemann–Hilbert approach to two-component modified short-pulse system and its nonlocal reductions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0088293?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In this paper, a Riemann–Hilbert approach to a two-component modified short-pulse (mSP) system on the line with zero boundary conditions is developed. A parametric representation of the solution to the related Cauchy problem is obtained. Four nonlocal integrable reductions, namely, the real reverse space-time nonlocal focusing and defocusing mSP equations and the complex reverse space-time nonlocal focusing and defocusing mSP equations, are studied in detail. For each case, soliton solutions are presented, and, unlike their local counterparts, the nonlocal equations exhibit certain novel properties induced by the impact of nonlocality.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0111507?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Patched patterns and emergence of chaotic interfaces in arrays of nonlocally coupled excitable systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0111507?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We disclose a new class of patterns, called patched patterns, in arrays of non-locally coupled excitable units with attractive and repulsive interactions. The self-organization process involves the formation of two types of patches, majority and minority ones, characterized by uniform average spiking frequencies. Patched patterns may be temporally periodic, quasiperiodic, or chaotic, whereby chaotic patterns may further develop interfaces comprised of units with average frequencies in between those of majority and minority patches. Using chaos and bifurcation theory, we demonstrate that chaos typically emerges via a torus breakup and identify the secondary bifurcation that gives rise to chaotic interfaces. It is shown that the maximal Lyapunov exponent of chaotic patched patterns does not decay, but rather converges to a finite value with system size. Patched patterns with a smaller wavenumber may exhibit diffusive motion of chaotic interfaces, similar to that of the incoherent part of chimeras.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097175?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Target recovery of the economic system based on the target reinforcement path method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097175?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
An effective and stable operation of an economic system leads to a prosperous society and sustainable world development. Unfortunately, the system faces inevitable perturbations of extreme events and is frequently damaged. To maintain the system&#x27;s stability, recovering its damaged functionality is essential and is complementary to strengthening its resilience and forecasting extreme events. This paper proposes a target recovery method based on network and economic equilibrium theories to defend the economic system against perturbations characterized as localized attacks. This novel method stimulates a set of economic sectors that mutually reinforce damaged economic sectors and is intuitively named the target reinforcement path (TRP) method. Developing a nonlinear dynamic model that simulates the economic system&#x27;s operation after being perturbed by a localized attack and recovering based on a target recovery method, we compute the relaxation time for this process to quantify the method&#x27;s efficiency. Furthermore, we adopt a rank aggregation method to comprehensively measure the method&#x27;s efficiency by studying the target recovery of three country-level economic systems (China, India, and Japan) for 73 different regional attack scenarios. Through a comparative analysis of the TRP method and three other classic methods, the TRP method is shown to be more effective and less costly. Applicatively, the proposed method exhibits the potential to recover other vital complex systems with spontaneous recovery ability, such as immune, neurological, and ecological systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105340?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Instability of oscillations in the Rosenzweig–MacArthur model of one consumer and two resources</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105340?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The system of two resources [math] and [math] and one consumer [math] is investigated within the Rosenzweig–MacArthur model with a Holling type II functional response. The rates of consumption of particular resources are normalized as to keep their sum constant. Dynamic switching is introduced as to increase the variable [math] in a process of finite speed. The space of parameters where both resources coexist is explored numerically. The results indicate that oscillations of [math] and mutually synchronized [math], which appear equal for the rates of consumption, are destabilized when these rates are modified. Then, the system is driven to one of fixed points or to a limit cycle with a much smaller amplitude. As a consequence of symmetry between the resources, the consumer cannot change the preferred resource once it is chosen.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0083137?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Identifying the spatiotemporal organization of high-traffic events in a mobile communication system using event synchronization and complex networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0083137?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In mobile communication systems, congestion is related to high-traffic events (HTEs) that occur in the coverage areas of base stations. Understanding, recognizing, and predicting these HTEs and researching their occurrence rules provides theoretical and decision-making support for preventing system congestion. Communication sectors are regarded as nodes, and if HTEs occur synchronously among sectors, then the corresponding nodes are connected. The total number of synchronous HTEs determines the edge weights. The mobile-communication spatiotemporal data are mapped to a weighted network, with the occurrence locations of HTEs as the basic elements. Network analysis provides a structure for representing the interaction of HTEs. By analyzing the topological features of the event synchronization network, the associations among the occurrence times of HTEs can be mined. We find that the event synchronization network is a small-world network, the cumulative strength distribution is exponential, and the edge weight obeys a power law. Moreover, the node clustering coefficient is negatively correlated with the node degree. A congestion coefficient based on several topological parameters is proposed, and the system congestion is visualized. The congestion coefficient contains information about the synchronous occurrence of HTEs between a sector and its neighbors and information about the synchronous occurrence of HTEs among its neighbors. For the mobile communication system considered in this study, the congestion coefficient of a large number of sectors is small and the risk of system congestion is low.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0292?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Novel Network Method Major Minor Variation Clustering Enables Identification of Poliovirus Clusters with High-Resolution Linkages</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0292?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33111-4"
                        open
                      >
                        <summary class="article-expander__title">Unleashing the potential of noncanonical amino acid biosynthesis to create cells with precision tyrosine sulfation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33111-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33111-4/MediaObjects/41467_2022_33111_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Incorporation of noncanonical amino acids into proteins holds great promise for altering structure and function of these proteins. Here the authors generate metabolically modified prokaryotic and eukaryotic cells that can biosynthesize sTyr and incorporate it into proteins in a site-specific manner.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01792-6"
                        open
                      >
                        <summary class="article-expander__title">The hybrid number of a ploidy profile</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01792-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Polyploidization, whereby an organism inherits multiple copies of the genome of their parents, is an important evolutionary event that has been observed in plants and animals. One way to study such events is in terms of the ploidy number of the species that make up a dataset of interest. It is therefore natural to ask: How much information about the evolutionary past of the set of species that form a dataset can be gleaned from the ploidy numbers of the species? To help answer this question, we introduce and study the novel concept of a ploidy profile which allows us to formalize it in terms of a multiplicity vector indexed by the species the dataset is comprised of. Using the framework of a phylogenetic network, we present a closed formula for computing the hybrid number (i.e. the minimal number of polyploidization events required to explain a ploidy profile) of a large class of ploidy profiles. This formula relies on the construction of a certain phylogenetic network from the simplification sequence of a ploidy profile and the hybrid number of the ploidy profile with which this construction is initialized. Both of them can be computed easily in case the ploidy numbers that make up the ploidy profile are not too large. To help illustrate the applicability of our approach, we apply it to a simplified version of a publicly available Viola dataset.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01991-9"
                        open
                      >
                        <summary class="article-expander__title">A Discontinuous Galerkin Overset Scheme Using WENO Reconstruction and Subcells for Two-Dimensional Problems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01991-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A new scheme for communication between overset grids using subcells and weighted essentially non oscillatory (WENO) reconstruction for two-dimensional problems has been proposed. The effectiveness of this procedure is demonstrated using the discontinuous Galerkin method. This scheme uses a standard WENO reconstruction by dividing the immediate neighbors into subcells to find the degrees of freedom in cells near the overset interface. This also has the added advantage that it also works as a limiter if a discontinuity passes through the overset interface. Accuracy tests to demonstrate the maintenance of higher order are provided. Results containing shocks are also provided to demonstrate the limiter aspect of the data communication procedure.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01999-1"
                        open
                      >
                        <summary class="article-expander__title">A Kernel-Independent Sum-of-Exponentials Method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01999-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We propose an accurate algorithm for a novel sum-of-exponentials (SOE) approximation of kernel functions, and develop a fast algorithm for convolution quadrature based on the SOE, which allows an order N calculation for N time steps of approximating a continuous temporal convolution integral. The SOE method is constructed by a combination of the de la Vallée-Poussin sum for a semi-analytical exponential expansion of a general kernel, and a model reduction technique to minimize the number of exponentials under a given error tolerance. We employ the SOE expansion for the finite part of the splitting convolution kernel so that the convolution integral can be solved as a system of ordinary differential equations. We show that the SOE method works for general kernels with controllable upperbound of positive exponents. Numerical analysis is provided for both the SOE method and the SOE-based convolution quadrature. Numerical results on different kernels demonstrate attractive performance on both accuracy and efficiency of the proposed method.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01990-w"
                        open
                      >
                        <summary class="article-expander__title">Accurate High-Order Tensor-Product Generalized Summation-By-Parts Discretizations of Hyperbolic Conservation Laws: General Curved Domains and Functional Superconvergence</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01990-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The goal of this paper is to outline the requirements for obtaining accurate solutions and functionals from high-order tensor-product generalized summation-by-parts discretizations of the steady two-dimensional linear convection and Euler equations on general curved domains. Two procedures for constructing high-order grids using either Lagrange or B-spline mappings are outlined. For the linear convection equation, four discretizations are derived and characterized—two based on the mortar-element approach and two based on the global summation-by-parts-operator approach. It is shown numerically that the schemes are dual consistent, and the requirements for achieving functional superconvergence for each set of methods are outlined. For the Euler equations, a dual-consistent mortar-element discretization is proposed and the practical requirements for obtaining accurate solutions and superconvergent functionals for problems of increasing practical relevance are delineated through theory and numerical examples.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01995-5"
                        open
                      >
                        <summary class="article-expander__title">Arbitrarily High Order and Fully Discrete Extrapolated RK–SAV/DG Schemes for Phase-field Gradient Flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01995-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we construct and analyze a fully discrete method for phase-field gradient flows, which uses extrapolated Runge–Kutta with scalar auxiliary variable (RK–SAV) method in time and discontinuous Galerkin (DG) method in space. We propose a novel technique to decouple the system, after which only several elliptic scalar problems with constant coefficients need to be solved independently. Discrete energy decay property of the method is proved for gradient flows. The scheme can be of arbitrarily high order both in time and space, which is demonstrated rigorously for the Allen–Cahn equation and the Cahn–Hilliard equation. More precisely, optimal 
                      
                      \(L^2\)
                      
                    -error bound in space and qth-order convergence rate in time are obtained for q-stage extrapolated RK–SAV/DG method. Several numerical experiments are carried out to verify the theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01998-2"
                        open
                      >
                        <summary class="article-expander__title">A Convergent Iterated Quasi-interpolation for Periodic Domain and Its Applications to Surface PDEs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01998-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The paper first provides an iterated quasi-interpolation scheme for function approximation over periodic domain and then attempts its applications to solve time-dependent surface partial differential equations (PDEs). The key feature of our scheme is that it gives an approximation directly just by taking a weighted average of the available discrete function values evaluated at sampling centers in the periodic domain. As such, it is simple, easy to compute, and the implementation process is stable. Moreover, if the sampling centers distribute uniformly over the periodic domain, it even preserves the same convergence order to all the derivatives. To employ the iterated quasi-interpolation scheme for solving time-dependent surface PDEs, we follow the framework of the method-of-lines. More precisely, we first reformulate the PDEs in terms of the parametric form of the surface. Then we use our quasi-interpolation scheme to approximate both the analytic solution and its spatial derivatives in the reformulated form (of PDEs) to get a semi-discrete ordinary differential equation (ODE) system. Finally, we adopt an appropriate time-integration technique to obtain the full-discrete scheme. As concrete examples, we take the torus for illustration and solve some benchmark reaction-diffusion equations imposed on the torus at the end of the paper. However, the proposed method is general and works on time-dependent PDEs defined on any smooth closed parameterized surfaces without coordinate singularity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01981-x"
                        open
                      >
                        <summary class="article-expander__title">Optimal Convergence of the Scalar Auxiliary Variable Finite Element Method for the Natural Convection Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01981-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This paper concerns an efficient finite element method for the natural convection equations with the scalar auxiliary variable approach. The linearly extrapolated Crank-Nicolson techniques are used to discretize nonlinear terms in the Navier–Stokes equations and the heat equation. The induced scalar auxiliary equation is an univariate ordinary equation. With the benefit of the new defined system, the redefined stability of the proposed method is obtained under the fully explicit scheme for the nonlinear terms without the requirement of skew-symmetric trilinear forms. The optimal convergence rates in space for all the variables are proved. The second order convergence rates in time are also obtained. Finally, numerical experiments are provided to support the theoretical analysis and demonstrate the efficiency of the proposed scheme.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01993-7"
                        open
                      >
                        <summary class="article-expander__title">Two Harmonic Jacobi–Davidson Methods for Computing a Partial Generalized Singular Value Decomposition of a Large Matrix Pair</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01993-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Two harmonic extraction based Jacobi–Davidson (JD) type algorithms are proposed to compute a partial generalized singular value decomposition (GSVD) of a large regular matrix pair. They are called cross product-free (CPF) and inverse-free (IF) harmonic JDGSVD algorithms, abbreviated as CPF-HJDGSVD and IF-HJDGSVD, respectively. Compared with the standard extraction based JDGSVD algorithm, the harmonic extraction based algorithms converge more regularly and suit better for computing GSVD components corresponding to interior generalized singular values. Thick-restart CPF-HJDGSVD and IF-HJDGSVD algorithms with some deflation and purgation techniques are developed to compute more than one GSVD components. Numerical experiments confirm the superiority of CPF-HJDGSVD and IF-HJDGSVD to the standard extraction based JDGSVD algorithm.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-15 datetime="2022-09-15T04:00:00.000Z">2022-09-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-15 datetime="2022-09-15T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00247"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Functional Comparison between Endogenous and Synthetic Notch Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00247">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00247</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-14 datetime="2022-09-14T02:05:31.000Z">2022-09-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-14 datetime="2022-09-14T02:05:31.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0093841?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Nucleation of stem cell domains in a bistable activator–inhibitor model of the shoot apical meristem</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0093841?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Shoot apical meristems (SAMs) give rise to all above-ground tissues of a plant. Expansion of meristematic tissue is derived from the growth and division of stem cells that reside in a central zone of the SAM. This reservoir of stem cells is maintained by expression of a transcription factor WUSCHEL that is responsible for the development of stem cells in the central zone. WUSCHEL expression is self-activating and downregulated by a signaling pathway initiated by CLAVATA proteins, which are upregulated by WUSCHEL. This classic activator–inhibitor network can generate localized patterns of WUSCHEL activity by a Turing instability provided certain constraints on reaction rates and diffusion constants of WUSCHEL and CLAVATA are satisfied, and most existing mathematical models of nucleation and confinement of stem cells in the SAM rely on Turing&#x27;s mechanism. However, Turing patterns have certain properties that are inconsistent with observed patterns of stem cell differentiation in the SAM. As an alternative mechanism, we propose a model for stem cell confinement based on a bistable-switch in WUSCHEL–CLAVATA interactions. We study the bistable-switch mechanism for pattern formation in a spatially continuous domain and in a discrete cellularized tissue in the presence of a non-uniform field of a rapidly diffusing hormone. By comparing domain formation by Turing and bistable-switch mechanisms in these contexts, we show that bistable switching provides a superior account of nucleation and confinement of the stem cell domain under reasonable assumptions on reaction rates and diffusion constants.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/d41586-022-02231-8"
                        open
                      >
                        <summary class="article-expander__title">Life brought to artificial cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/d41586-022-02231-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.nature.com/lw1024/magazine-assets/d41586-022-02231-8/d41586-022-02231-8_23476970.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A synthetic system that reproduces several attributes of living cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33058-6"
                        open
                      >
                        <summary class="article-expander__title">Layered feedback control overcomes performance trade-off in synthetic biomolecular networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33058-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33058-6/MediaObjects/41467_2022_33058_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Layered feedback is an optimization strategy in feedback control designs widely used in engineering. Here, combining simulation and experimentation, the authors apply layered control - a powerful optimization strategy in engineering - to synthetic biomolecular networks in living bacteria to show layered control overcomes performance trade-offs in biology.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41586-022-05223-w"
                        open
                      >
                        <summary class="article-expander__title">Living material assembly of bacteriogenic protocells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41586-022-05223-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-022-05223-w/MediaObjects/41586_2022_5223_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A bacteriogenic strategy for constructing membrane-bounded, molecularly crowded, and compositionally, structurally and morphologically complex synthetic cells provides opportunities for the fabrication of new synthetic cell modules and augmented living/synthetic cell constructs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01800-9"
                        open
                      >
                        <summary class="article-expander__title">Temperature sensitivity of pest reproductive numbers in age-structured PDE models, with a focus on the invasive spotted lanternfly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01800-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Invasive pest establishment is a pervasive threat to global ecosystems, agriculture, and public health. The recent establishment of the invasive spotted lanternfly in the northeastern United States has proven devastating to farms and vineyards, necessitating urgent development of population dynamical models and effective control practices. In this paper, we propose a stage-age-structured system of PDEs to model insect pest populations, in which underlying dynamics are dictated by ambient temperature through rates of development, fecundity, and mortality. The model incorporates diapause and non-diapause pathways, and is calibrated to experimental and field data on the spotted lanternfly. We develop a novel moving mesh method for capturing age-advection accurately, even for coarse discretization parameters. We define a one-year reproductive number (
                      
                      \(R_0\)
                      
                    ) from the spectrum of a one-year solution operator, and study its sensitivity to variations in the mean and amplitude of the annual temperature profile. We quantify assumptions sufficient to give rise to the low-rank structure of the solution operator characteristic of part of the parameter domain. We discuss establishment potential as it results from the pairing of a favorable 
                      
                      \(R_0\)
                      
                     value and transient population survival, and address implications for pest control strategies.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02000-9"
                        open
                      >
                        <summary class="article-expander__title">A Symmetric Fractional-order Reduction Method for Direct Nonuniform Approximations of Semilinear Diffusion-wave Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02000-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We introduce a symmetric fractional-order reduction (SFOR) method to construct numerical algorithms on general nonuniform temporal meshes for semilinear fractional diffusion-wave equations. By using the novel order reduction method, the governing problem is transformed to an equivalent coupled system, where the explicit orders of time-fractional derivatives involved are all 
                      
                      \(\alpha /2\)
                      
                     
                      
                      \((1&lt;\alpha &lt;2)\)
                      
                    . The linearized L1 scheme and Alikhanov scheme are then proposed on general time meshes. Under some reasonable regularity assumptions and weak restrictions on meshes, the optimal convergence is derived for the two kinds of difference schemes by 
                      
                      \(H^2\)
                      
                     energy method. An adaptive time stepping strategy which based on the (fast linearized) L1 and Alikhanov algorithms is designed for the semilinear diffusion-wave equations. Numerical examples are provided to confirm the accuracy and efficiency of proposed algorithms.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-13 datetime="2022-09-13T15:20:00.000Z">2022-09-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-13 datetime="2022-09-13T15:20:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/smart-innovation-center-awarded-nrf-grant-0913"
                        open
                      >
                        <summary class="article-expander__title">SMART Innovation Center awarded five-year NRF grant for new deep tech ventures</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/smart-innovation-center-awarded-nrf-grant-0913">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/IC%20stock%20image%202.jpeg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>By providing researchers with financial and strategic support from the early stages, the Innovation Center hopes to bring new and disruptive technologies to market.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0658?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">BDEtools: A MATLAB Package for Boolean Delay Equation Modeling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0658?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">RESISTOR: A New OSPREY Module to Predict Resistance Mutations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0254?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00265"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering of the Translesion DNA Synthesis Pathway Enables Controllable C‑to‑G and C‑to‑A Base Editing in Corynebacterium glutamicum</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00265">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00265</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101913?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Fractional Brownian motion with random Hurst exponent: Accelerating diffusion and persistence transitions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101913?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Fractional Brownian motion, a Gaussian non-Markovian self-similar process with stationary long-correlated increments, has been identified to give rise to the anomalous diffusion behavior in a great variety of physical systems. The correlation and diffusion properties of this random motion are fully characterized by its index of self-similarity or the Hurst exponent. However, recent single-particle tracking experiments in biological cells revealed highly complicated anomalous diffusion phenomena that cannot be attributed to a class of self-similar random processes. Inspired by these observations, we here study the process that preserves the properties of the fractional Brownian motion at a single trajectory level; however, the Hurst index randomly changes from trajectory to trajectory. We provide a general mathematical framework for analytical, numerical, and statistical analysis of the fractional Brownian motion with the random Hurst exponent. The explicit formulas for probability density function, mean-squared displacement, and autocovariance function of the increments are presented for three generic distributions of the Hurst exponent, namely, two-point, uniform, and beta distributions. The important features of the process studied here are accelerating diffusion and persistence transition, which we demonstrate analytically and numerically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0087838?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Effect of seasonally forced noisy environment on aquatic prey–predator model with water level fluctuations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0087838?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
In this paper, the impact of environmental noise on prey–predator interactions with seasonal fluctuations in water levels has been studied. Both intensity of noise and water level variations together play a significant role in the dynamics of an aquatic prey–predator system. Analytically, we have shown the existence of a positive solution and its uniqueness, the ultimate bound of the system’s solutions, and the global attractivity of the solution. Moreover, parametric conditions for which model species either persist (strongly or weakly) in the system or go into extinction are derived and their biological significance is also discussed. The sufficient condition of stochastic permanence is also analyzed. To justify the analytical results, we have performed numerical simulations of both deterministic and stochastic systems and observed some significant dynamics in the considered system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0087258?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Linear vs nonlinear transport during chaotic advection in fluid flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0087258?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The goal of this study is explicit demarcation of the region of validity of a linear canonical representation for chaotic advection of Lagrangian fluid parcels in “chaotic seas” in two-dimensional (2D) and three-dimensional (3D) time-periodic fluid flows governed by Hamiltonian mechanics. The concept of lobe dynamics admits exact and unique geometric demarcation of this region and, inherently, distinction of the portions of chaotic seas with essentially linear vs nonlinear Lagrangian transport. This, furthermore, admits explicit establishment of a topological equivalence between the (embedded) Hamiltonian structure of the Lagrangian dynamics in 2D (3D) flows and their canonical form. The linear transport region in physical space encompasses four adjacent subregions that each corresponds to one of the four quadrants in canonical space and may exchange material with their environment in two essentially nonlinear ways. First, exchange between quadrants within the linear transport region and, second, exchange with the exterior of this region. Both forms of exchange can be linked to specific subsets of material elements defined by interacting lobes and combined give rise to circulation through the quadrants of the linear transport region that systematically exchanges the material with the exterior.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0103247?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Effects of species vigilance on coexistence in evolutionary dynamics of spatial rock–paper–scissors game</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0103247?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Recognizing surrounding situations, such as enemy attacks, which can be realized by predator–prey relationships, is one of the common behaviors of the population in ecosystems. In this paper, we explore the relationship between such species’ behavior and biodiversity in the spatial rock–paper–scissors game by employing the ecological concept “vigilance.” In order to describe the vigilance process, we adopt a multiplex structure where two distinct layers describe virtual and physical interactions. By investigating the process of evolution in species, we also found that species with different vigilance go together. In addition, by utilizing the dynamic time warping method, we found that species with the same vigilance have consistent behavior, but species with different vigilance have diverse behavior. Our findings may lead to broader interpretations of mechanisms promoting biodiversity via vigilance in species ecosystems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01797-1"
                        open
                      >
                        <summary class="article-expander__title">An aggregation model of cockroaches with fast-or-slow motion dichotomy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01797-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We propose a mathematical model, namely a reaction–diffusion system, to describe social behaviour of cockroaches. An essential new aspect in our model is that the dispersion behaviour due to overcrowding effect is taken into account as a counterpart to commonly studied aggregation. This consideration leads to an intriguing new phenomenon which has not been observed in the literature. Namely, due to the competition between aggregation towards areas of higher concentration of pheromone and dispersion avoiding overcrowded areas, the cockroaches aggregate more at the transition area of pheromone. Moreover, we also consider the fast reaction limit where the switching rate between active and inactive subpopulations tends to infinity. By utilising improved duality and energy methods, together with the regularisation of heat operator, we prove that the weak solution of the reaction–diffusion system converges to that of a reaction-cross-diffusion system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-12 datetime="2022-09-12T07:00:00.000Z">2022-09-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-12 datetime="2022-09-12T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Differentiable Learning of Sequence-Specific Minimizer Schemes with DeepMinimizer</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0275?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101257?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Transient chaotic behavior of fuzzy controlled polishing processes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101257?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
This paper investigates the dynamics of a fuzzy controlled polishing machine where the effect of temporal sampling is also taken into account. Chaotic and transient chaotic behaviors are experienced for certain control parameter combinations. In the case of transient chaotic motion, closed-form algebraic expressions are determined for the expected value of the kickout number and for the corresponding standard deviation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100166?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Constructing differential equations using only a scalar time-series about continuous time chaotic dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100166?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://aip.scitation.org/action/showOpenGraphArticleImage?doi&#x3D;10.1063/5.0100166&amp;id&#x3D;images/medium/5.0100166.figures.online.highlight_f1.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We propose a simple method of constructing a system of differential equations of chaotic behavior based on the regression only from scalar observable time-series data. The estimated system enables us to reconstruct invariant sets and statistical properties as well as to infer short time-series. Our successful modeling relies on the introduction of a set of Gaussian radial basis functions to capture local structures. The proposed method is used to construct a system of ordinary differential equations whose orbit reconstructs a time-series of a variable of the well-known Lorenz system as a simple but typical example. A system for a macroscopic fluid variable is also constructed.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00314"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Intestinal Engineered Probiotics as Living Therapeutics: Chassis Selection, Colonization Enhancement, Gene Circuit Design, and Biocontainment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00314">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00314</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00195"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Steady-State Operation of a Cell-Free Genetic Band-Detection Circuit</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00195">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00195</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33046-w"
                        open
                      >
                        <summary class="article-expander__title">Robust data storage in DNA by de Bruijn graph-based de novo strand assembly</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33046-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33046-w/MediaObjects/41467_2022_33046_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA data storage is a rapidly developing technology with great potential due to its high density, long-term durability, and low maintenance cost. Here the authors present a strand assembly algorithm (DBGPS) using de Bruijn graph and greedy path search.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33033-1"
                        open
                      >
                        <summary class="article-expander__title">Kinetic compartmentalization by unnatural reaction for itaconate production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33033-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33033-1/MediaObjects/41467_2022_33033_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Unlike eukaryotic system, bacterial hosts lack membranous system, which is one of the limitations for efficient metabolic engineering. Here, the authors report a kinetic compartmentalization strategy to increase substrate availability from competitive reactions for the efficient production of itaconate in E. coli.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01789-1"
                        open
                      >
                        <summary class="article-expander__title">Average abundancy of cooperation in multi-player games with random payoffs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01789-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We consider interactions between players in groups of size 
                      
                      \(d\ge 2\)
                      
                     with payoffs that not only depend on the strategies used in the group but also fluctuate at random over time. An individual can adopt either cooperation or defection as strategy and the population is updated from one time step to the next by a birth-death event according to a Moran model. Assuming recurrent symmetric mutation and payoffs to cooperators and defectors according to the composition of the group whose expected values, variances, and covariances are of the same small order, we derive a first-order approximation for the average abundance of cooperation in the selection-mutation equilibrium. In general, we show that increasing the variance of any payoff for defection or decreasing the variance of any payoff for cooperation increases the average abundance of cooperation. As for the effect of the covariance between any payoff for cooperation and any payoff for defection, we show that it depends on the number of cooperators in the group associated with these payoffs. We study in particular the public goods game, the stag hunt game, and the snowdrift game, all social dilemmas based on random benefit b and random cost c for cooperation, which lead to correlated payoffs to cooperators and defectors within groups. We show that a decrease in the scaled variance of b or c, or an increase in their scaled covariance, makes it easier for weak selection to favor the abundance of cooperation in the stag hunt game and the snowdrift game. The same conclusion holds for the public goods game except that the variance of b has no effect on the average abundance of C. Moreover, while the mutation rate has little effect on which strategy is more abundant at equilibrium, the group size may change it at least in the stag hunt game with a larger group size making it more difficult for cooperation to be more abundant than defection under weak selection.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-10 datetime="2022-09-10T04:00:00.000Z">2022-09-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-10 datetime="2022-09-10T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00363"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Reprogramming T7RNA Polymerase in Escherichia coli Nissle 1917 under Specific Lac Operon for Efficient p‑Coumaric Acid Production</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00363">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00363</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-09 datetime="2022-09-09T04:00:00.000Z">2022-09-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-09 datetime="2022-09-09T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00629"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Pseudotyping Bacteriophage P2 Tail Fibers to Extend the Host Range for Biomedical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00629">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00629</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01997-3"
                        open
                      >
                        <summary class="article-expander__title">Conforming Virtual Element Methods for Sobolev Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01997-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The intention of this paper is to study the 
                      
                      \(H^1\)
                      
                    -conforming virtual element method for a class of Sobolev equations with variable coefficients. The semi-discrete scheme is constructed on the basis of the symmetric Nitsche’s method to impose the homogeneous and inhomogeneous Dirichlet boundary conditions in a unified way. The fully discrete scheme is obtained with the Crank–Nicolson scheme for temporal approximation. Under some assumptions about the penalty parameter in the Nitsche’s method, the existence and uniqueness of the semi-discrete and fully discrete solutions are analyzed. Furthermore, for the semi-discrete and fully discrete schemes, optimal a priori error estimates in both a mesh size dependent norm and 
                      
                      \(L^2\)
                      
                    -norm are proven. Finally, some numerical experiments are carried out to illustrate the theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-01983-9"
                        open
                      >
                        <summary class="article-expander__title">Stability Analysis of Coupled Advection-Diffusion Models with Bulk Interface Condition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-01983-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Numerical stability is of critical importance in general circulation models because it affects the design of algorithms, time to solution, and computational costs associated with the simulations, which are very expensive in practice. In this paper we extend the stability analysis for ocean-atmosphere coupling proposed in [Zhang et al., J. Sci. Comput. 84, 44(2020)] to a more realistic model that includes horizontal advection. We analyze various time-stepping strategies. We find that advection has a stabilizing effect in scenarios common to climate models when bulk interface condition and explicit flux coupling are used. We also show that our method can be used to study the stability impact of advection for other interface conditions such as Dirichlet–Neumann conditions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-08 datetime="2022-09-08T08:05:17.000Z">2022-09-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-08 datetime="2022-09-08T08:05:17.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0094943?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Non-parametric tests for serial dependence in time series based on asymptotic implementations of ordinal-pattern statistics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0094943?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Ordinal patterns can be used to construct non-parametric hypothesis tests that aim to discover (possibly non-linear) serial dependence in a real-valued time series. We derive the asymptotic distribution of the vector of sample frequencies of ordinal patterns and that of various corresponding tests statistics such that the targeted tests for serial dependence are easily implemented based on asymptotic approximations. Simulations are used to check the finite-sample performance of these tests as well as their power properties with respect to various alternative scenarios. The application and interpretation of the tests in practice are illustrated by an environmental data example.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098163?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">On the origin of chaotic attractors with two zero Lyapunov exponents in a system of five biharmonically coupled phase oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098163?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
We study chaotic dynamics in a system of four differential equations describing the interaction of five identical phase oscillators coupled via biharmonic function. We show that this system exhibits strange spiral attractors (Shilnikov attractors) with two zero (indistinguishable from zero in numerics) Lyapunov exponents in a wide region of the parameter space. We explain this phenomenon by means of bifurcation analysis of a three-dimensional Poincaré map for the system under consideration. We show that chaotic dynamics develop here near a codimension three bifurcation, when a periodic orbit (fixed point of the Poincaré map) has the triplet of multipliers [math]. As it is known, the flow normal form for such bifurcation is the well-known three-dimensional Arneodó–Coullet–Spiegel–Tresser (ACST) system, which exhibits spiral attractors. According to this, we conclude that the additional zero Lyapunov exponent for orbits in the observed attractors appears due to the fact that the corresponding three-dimensional Poincaré map is very close to the time-shift map of the ACST-system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102421?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Detecting bifurcations in dynamical systems with CROCKER plots</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102421?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Existing tools for bifurcation detection from signals of dynamical systems typically are either limited to a special class of systems or they require carefully chosen input parameters and a significant expertise to interpret the results. Therefore, we describe an alternative method based on persistent homology—a tool from topological data analysis—that utilizes Betti numbers and CROCKER plots. Betti numbers are topological invariants of topological spaces, while the CROCKER plot is a coarsened but easy to visualize data representation of a one-parameter varying family of persistence barcodes. The specific bifurcations we investigate are transitions from periodic to chaotic behavior or vice versa in a one-parameter collection of differential equations. We validate our methods using numerical experiments on ten dynamical systems and contrast the results with existing tools that use the maximum Lyapunov exponent. We further prove the relationship between the Wasserstein distance to the empty diagram and the norm of the Betti vector, which shows that an even more simplified version of the information has the potential to provide insight into the bifurcation parameter. The results show that our approach reveals more information about the shape of the periodic attractor than standard tools, and it has more favorable computational time in comparison with the Rösenstein algorithm for computing the maximum Lyapunov exponent.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0098454?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Cross-subject emotion recognition using visibility graph and genetic algorithm-based convolution neural network</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0098454?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
An efficient emotion recognition model is an important research branch in electroencephalogram (EEG)-based brain–computer interfaces. However, the input of the emotion recognition model is often a whole set of EEG channels obtained by electrodes placed on subjects. The unnecessary information produced by redundant channels affects the recognition rate and depletes computing resources, thereby hindering the practical applications of emotion recognition. In this work, we aim to optimize the input of EEG channels using a visibility graph (VG) and genetic algorithm-based convolutional neural network (GA-CNN). First, we design an experiment to evoke three types of emotion states using movies and collect the multi-channel EEG signals of each subject under different emotion states. Then, we construct VGs for each EEG channel and derive nonlinear features representing each EEG channel. We employ the genetic algorithm (GA) to find the optimal subset of EEG channels for emotion recognition and use the recognition results of the CNN as fitness values. The experimental results show that the recognition performance of the proposed method using a subset of EEG channels is superior to that of the CNN using all channels for each subject. Last, based on the subset of EEG channels searched by the GA-CNN, we perform cross-subject emotion recognition tasks employing leave-one-subject-out cross-validation. These results demonstrate the effectiveness of the proposed method in recognizing emotion states using fewer EEG channels and further enrich the methods of EEG classification using nonlinear features.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0091738?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Symmetry breaking of solitons in the PT-symmetric nonlinear Schrödinger equation with the cubic–quintic competing saturable nonlinearity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0091738?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The symmetry breaking of solitons in the nonlinear Schrödinger equation with cubic–quintic competing nonlinearity and parity-time symmetric potential is studied. At first, a new asymmetric branch separates from the fundamental symmetric soliton at the first power critical point, and then, the asymmetric branch passes through the branch of the fundamental symmetric soliton and finally merges into the branch of the fundamental symmetric soliton at the second power critical point, while the power of the soliton increases. This leads to the symmetry breaking and double-loop bifurcation of fundamental symmetric solitons. From the power-propagation constant curves of solitons, symmetric fundamental and tripole solitons, asymmetric solitons can also exist. The stability of symmetric fundamental solitons, asymmetric solitons, and symmetric tripole solitons is discussed by the linear stability analysis and direct simulation. Results indicate that symmetric fundamental solitons and symmetric tripole solitons tend to be stable with the increase in the soliton power. Asymmetric solitons are unstable in both high and low power regions. Moreover, with the increase in saturable nonlinearity, the stability region of fundamental symmetric solitons and symmetric tripole solitons becomes wider.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096099?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Dipole and quadrupole nonparaxial solitary waves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096099?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
The cubic nonlinear Helmholtz equation with third and fourth order dispersion and non-Kerr nonlinearity, such as the self steepening and the self frequency shift, is considered. This model describes nonparaxial ultrashort pulse propagation in an optical medium in the presence of spatial dispersion originating from the failure of slowly varying envelope approximation. We show that this system admits periodic (elliptic) solitary waves with a dipole structure within a period and also a transition from a dipole to quadrupole structure within a period depending on the value of the modulus parameter of a Jacobi elliptic function. The parametric conditions to be satisfied for the existence of these solutions are given. The effect of the nonparaxial parameter on physical quantities, such as amplitude, pulse width, and speed of the solitary waves, is examined. It is found that by adjusting the nonparaxial parameter, the speed of solitary waves can be decelerated. The stability and robustness of the solitary waves are discussed numerically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0100772?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">When costly migration helps to improve cooperation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0100772?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Motion is a typical reaction among animals and humans trying to reach better conditions in a changing world. This aspect has been studied intensively in social dilemmas where competing players’ individual and collective interests are in conflict. Starting from the traditional public goods game model, where players are locally fixed and unconditional cooperators or defectors are present, we introduce two additional strategies through which agents can change their positions of dependence on the local cooperation level. More importantly, these so-called sophisticated players should bear an extra cost to maintain their permanent capacity to evaluate their neighborhood and react accordingly. Hence, four strategies compete, and the most successful one can be imitated by its neighbors. Crucially, the introduction of costly movement has a highly biased consequence on the competing main strategies. In the majority of parameter space, it is harmful to defectors and provides a significantly higher cooperation level when the population is rare. At an intermediate population density, which would be otherwise optimal for a system of immobile players, the presence of mobile actors could be detrimental if the interaction pattern changes slightly, thereby blocking the optimal percolation of information flow. In this parameter space, sophisticated cooperators can also show the co-called Moor effect by first avoiding the harmful vicinity of defectors; they subsequently transform into an immobile cooperator state. Hence, paradoxically, the additional cost of movement could be advantageous to reach a higher general income, especially for a rare population when subgroups would be isolated otherwise.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0088465?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Symmetry breaking yields chimeras in two small populations of Kuramoto-type oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0088465?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Despite their simplicity, networks of coupled phase oscillators can give rise to intriguing collective dynamical phenomena. However, the symmetries of globally and identically coupled identical units do not allow solutions where distinct oscillators are frequency-unlocked—a necessary condition for the emergence of chimeras. Thus, forced symmetry breaking is necessary to observe chimera-type solutions. Here, we consider the bifurcations that arise when full permutational symmetry is broken for the network to consist of coupled populations. We consider the smallest possible network composed of four phase oscillators and elucidate the phase space structure, (partial) integrability for some parameter values, and how the bifurcations away from full symmetry lead to frequency-unlocked weak chimera solutions. Since such solutions wind around a torus they must arise in a global bifurcation scenario. Moreover, periodic weak chimeras undergo a period-doubling cascade leading to chaos. The resulting chaotic dynamics with distinct frequencies do not rely on amplitude variation and arise in the smallest networks that support chaos.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01058-0"
                        open
                      >
                        <summary class="article-expander__title">Publisher Correction: Standardized excitable elements for scalable engineering of far-from-equilibrium chemical networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01058-0">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Chemistry - Publisher Correction: Standardized excitable elements for scalable engineering of far-from-equilibrium chemical networks</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-07 datetime="2022-09-07T00:00:00.000Z">2022-09-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-07 datetime="2022-09-07T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01798-0"
                        open
                      >
                        <summary class="article-expander__title">Stoechiometric and dynamical autocatalysis for diluted chemical reaction networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01798-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Autocatalysis underlies the ability of chemical and biochemical systems to replicate. Recently, Blokhuis et al. (PNAS 117(41):25230–25236, 2020) gave a stoechiometric definition of autocatalysis for reaction networks, stating the existence of a combination of reactions such that the balance for all autocatalytic species is strictly positive, and investigated minimal autocatalytic networks, called autocatalytic cores. By contrast, spontaneous autocatalysis—namely, exponential amplification of all species internal to a reaction network, starting from a diluted regime, i.e. low concentrations—is a dynamical property. We introduce here a topological condition (Top) for autocatalysis, namely: restricting the reaction network description to highly diluted species, we assume existence of a strongly connected component possessing at least one reaction with multiple products (including multiple copies of a single species). We find this condition to be necessary and sufficient for stoechiometric autocatalysis. When degradation reactions have small enough rates, the topological condition further ensures dynamical autocatalysis, characterized by a strictly positive Lyapunov exponent giving the instantaneous exponential growth rate of the system. The proof is generally based on the study of auxiliary Markov chains. We provide as examples general autocatalytic cores of Type I and Type III in the typology of Blokhuis et al. (PNAS 117(41):25230–25236, 2020) . In a companion article (Unterberger in Dynamical autocatalysis for autocatalytic cores, 2021), Lyapunov exponents and the behavior in the growth regime are studied quantitatively beyond the present diluted regime .</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-06 datetime="2022-09-06T10:00:00.000Z">2022-09-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-06 datetime="2022-09-06T10:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/alphafold-potential-protein-drug-0906"
                        open
                      >
                        <summary class="article-expander__title">Analyzing the potential of AlphaFold in drug discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/alphafold-potential-protein-drug-0906">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/MIT-Better-Predictions-01-press.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Study finds computer models that predict molecular interactions need improvement before they can help identify drug mechanisms of action.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.1c00561"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Development of Base Editors for Simultaneously Editing Multiple Loci in Lactococcus lactis</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.1c00561">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.1c00561</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01610-3"
                        open
                      >
                        <summary class="article-expander__title">Tracking the order of transcriptional events in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01610-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01610-3/MediaObjects/41592_2022_1610_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Researchers describe Retro-Cascorder, a tool that employs prokaryotic retrons and CRISPR integrases to write transcriptional events into the genome of Escherichia coli.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32744-9"
                        open
                      >
                        <summary class="article-expander__title">Generation of an Escherichia coli strain growing on methanol via the ribulose monophosphate cycle</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32744-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32744-9/MediaObjects/41467_2022_32744_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using one carbon compounds as feedstock is a promising approach in abating climate change. Here, the authors report the conversion of E. coli into a synthetic methylotroph that assimilates methanol via the ribulose monophosphate cycle and a set of distinctive mutations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32496-6"
                        open
                      >
                        <summary class="article-expander__title">MVsim is a toolset for quantifying and designing multivalent interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32496-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32496-6/MediaObjects/41467_2022_32496_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Arising through multiple binding elements, multivalency can specify the avidity, duration, cooperativity, and selectivity of biomolecular interactions, but quantitative prediction and design of these properties has remained challenging. Here the authors enable facile analysis and engineering of multivalent binding by developing MVsim, a simulator that incorporates biochemical and biophysical parameters of interacting molecules and is accessible through a graphical user interface.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32941-6"
                        open
                      >
                        <summary class="article-expander__title">Signal processing and generation of bioactive nitric oxide in a model prototissue</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32941-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32941-6/MediaObjects/41467_2022_32941_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A challenge for synthetic biology is the design and construction of prototissue. Here, the authors spatially segregate layers of enzyme-decorated coacervate protocells as a model prototissue capable of chemical signal processing and modulating outputs of nitric oxide to inhibit blood clot formation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;42
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-05 datetime="2022-09-05T07:00:00.000Z">2022-09-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-05 datetime="2022-09-05T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0447?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Machine-Learning Prospects for Detecting Selection Signatures Using Population Genomics Data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0447?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 943-960, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0399?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">An Integer Linear Programming Approach for Scaffolding Based on Exemplar Breakpoint Distance</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0399?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 961-973, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0417?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Optimal Design of Synthetic DNA Sequences Without Unwanted Binding Sites</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0417?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 974-986, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0269?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">RDscan: A New Method for Improving Germline and Somatic Variant Calling Based on Read Depth Distribution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0269?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 987-1000, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0536?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">kmer2vec: A Novel Method for Comparing DNA Sequences by word2vec Embedding</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0536?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1001-1021, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0549?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Sharing Rewards Undermines Coordinated Hunting</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0549?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1022-1030, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0349?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Cross-Organ Transcriptomic Comparison Reveals Universal Factors During Maturation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0349?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1031-1044, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0464?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Harnessing Fuzzy Rule Based System for Screening Major Histocompatibility Complex Class I Peptide Epitopes from the Whole Proteome: An Implementation on the Proteome of Leishmania donovani</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0464?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 9, Page 1045-1058, September 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01021-z"
                        open
                      >
                        <summary class="article-expander__title">A modular XNAzyme cleaves long, structured RNAs under physiological conditions and enables allele-specific gene silencing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01021-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01021-z/MediaObjects/41557_2022_1021_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Oligonucleotide catalysts such as ribozymes and DNAzymes can cleave RNA efficiently and specifically but are typically dependent on high concentrations of divalent cations, limiting their biological applications. A modular XNAzyme catalyst composed of 2′-deoxy-2′-fluoro-β-d-arabino nucleic acid (FANA) has now been developed that can cleave long (&gt;5 kb), highly structured mRNAs under physiological conditions and enables allele-specific catalytic RNA knockdown inside cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-03 datetime="2022-09-03T00:00:00.000Z">2022-09-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-03 datetime="2022-09-03T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32921-w"
                        open
                      >
                        <summary class="article-expander__title">Expanding the terpene biosynthetic code with non-canonical 16 carbon atom building blocks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32921-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32921-w/MediaObjects/41467_2022_32921_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Establishing methods to access the chemical space that lies beyond canonical terpenoid biosynthesis will increase the applications of isoprenoids. Here, the authors reconstruct the modular structure of terpene biosynthesis on 16-carbon backbones by engineered yeast and synthesize 28 different unique terpenes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;53
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01770-y"
                        open
                      >
                        <summary class="article-expander__title">Upscaling between an agent-based model (smoothed particle approach) and a continuum-based model for skin contractions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01770-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Skin contraction is an important biophysical process that takes place during and after recovery of deep tissue injury. This process is mainly caused by fibroblasts (skin cells) and myofibroblasts (differentiated fibroblasts which exert larger pulling forces and produce larger amounts of collagen) that both exert pulling forces on the surrounding extracellular matrix (ECM). Modelling is done in multiple scales: agent-based modelling on the microscale and continuum-based modelling on the macroscale. In this manuscript we present some results from our study of the connection between these scales. For the one-dimensional case, we managed to rigorously establish the link between the two modelling approaches for both closed-form solutions and finite-element approximations.  For the multi-dimensional case, we computationally evidence the connection between the agent-based and continuum-based modelling approaches.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-02 datetime="2022-09-02T09:53:01.000Z">2022-09-02</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-02 datetime="2022-09-02T09:53:01.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0097075?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Action potential and chaos near the edge of chaos in memristive circuits</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0097075?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Memristor-based neuromorphic systems have a neuro-bionic function, which is critical for possibly overcoming Moore&#x27;s law limitation and the von Neumann bottleneck problem. To explore neural behaviors and complexity mechanisms in memristive circuits, this paper proposes an N-type locally active memristor, based on which a third-order memristive circuit is constructed. Theoretical analysis shows that the memristive circuit can exhibit not only various action potentials but also self-sustained oscillation and chaos. Based on Chua&#x27;s theory of local activity, this paper finds that the neural behaviors and chaos emerge near the edge of chaos through subcritical Hopf bifurcation, in which the small unstable limit cycle is depicted by the dividing line between the attraction basin of the large stable limit cycle and the attraction basin of the stable equilibrium point. Furthermore, an analog circuit is designed to imitate the action potentials and chaos, and the simulation results are in agreement with the theoretical analysis.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096532?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Effects of infection fatality ratio and social contact matrices on vaccine prioritization strategies</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096532?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 9, September 2022. 
Effective strategies of vaccine prioritization are essential to mitigate the impacts of severe infectious diseases. We investigate the role of infection fatality ratio (IFR) and social contact matrices on vaccination prioritization using a compartmental epidemic model fueled by real-world data of different diseases and countries. Our study confirms that massive and early vaccination is extremely effective to reduce the disease fatality if the contagion is mitigated, but the effectiveness is increasingly reduced as vaccination beginning delays in an uncontrolled epidemiological scenario. The optimal and least effective prioritization strategies depend non-linearly on epidemiological variables. Regions of the epidemiological parameter space, in which prioritizing the most vulnerable population is more effective than the most contagious individuals, depend strongly on the IFR age profile being, for example, substantially broader for COVID-19 in comparison with seasonal influenza. Demographics and social contact matrices deform the phase diagrams but do not alter their qualitative shapes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1430546?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Higher Order Phase Averaging for Highly Oscillatory Systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1430546?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 936-956, September 2022. 
 Abstract. We introduce a higher order phase averaging method for nonlinear oscillatory systems. Phase averaging is a technique to filter fast motions from the dynamics while still accounting for their effect on the slow dynamics. Phase averaging is useful for deriving reduced models that can be solved numerically with more efficiency, since larger timesteps can be taken. Recently, Haut and Wingate [SIAM J. Sci. Comput., 36 (2014) pp. A693–A713] introduced the idea of computing finite window numerical phase averages in parallel as the basis for a coarse propagator for a parallel-in-time algorithm. In this contribution, we provide a framework for higher order phase averages that aims to better approximate t…</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03832-9"
                        open
                      >
                        <summary class="article-expander__title">A visualization reporter system for characterizing antibiotic biosynthetic gene clusters expression with high-sensitivity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03832-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03832-9/MediaObjects/42003_2022_3832_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A quorum sensing based visualization reporter system is presented for the characterization of promoters in Gram-positive bacteria, utilizing violacein production, especially for use in the identification of secondary metabolites.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;37
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32829-5"
                        open
                      >
                        <summary class="article-expander__title">Automated model-predictive design of synthetic promoters to control transcriptional profiles in bacteria</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32829-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32829-5/MediaObjects/41467_2022_32829_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Transcription rates are regulated by the interactions between RNA polymerase, sigma factor, and promoter DNA sequences in bacteria. Here the authors combine massively parallel experiments &amp; machine learning to develop a predictive biophysical model of transcription, validated across 22132 bacterial promoters, and apply it to the design and debugging of genetic circuits.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-32889-7"
                        open
                      >
                        <summary class="article-expander__title">Spatiotemporal control of signal-driven enzymatic reaction in artificial cell-like polymersomes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-32889-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-32889-7/MediaObjects/41467_2022_32889_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Researchers have been trying to mimick the cellular spatiotemporal control in normal cells with different approaches. Here, the authors present semi-permeable polymersomes comprising of amphiphilic triblock copolymers to achieve external signal-driven complex coacervation and biophysical reconstitution of cytoskeleton.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;41
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/xlxs4/rss-reader/actions/runs/3162608901">
        <time id="build-timestamp" datetime="2022-10-01T01:07:28.154Z">2022-10-01T01:07:28.154Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>