<!DOCTYPE html>
<html lang="en">

  <head>
    <title>osmos::feed</title>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="robots" content="noindex, nofollow" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
    <link rel="alternate" type="application/rss+xml" title="osmos::feed" href="feed.atom" />
    <link href="index.css?v1.14.4" rel="stylesheet" />
    <!-- %before-head-end.html% -->
  </head>

  <body>
    <!-- %after-body-begin.html% -->
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-24 datetime="2022-10-24T01:03:38.586Z">2022-10-24</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-24 datetime="2022-10-24T01:03:38.586Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ScienceDirect Publication: Journal of Differential Equations</button>
                  <a class="source-heading__link" href="https://www.sciencedirect.com/journal/journal-of-differential-equations">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005617"
                        open
                      >
                        <summary class="article-expander__title">Hydrodynamic limit for the inhomogeneous incompressible Navier-Stokes-Vlasov equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005617?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Yunfei Su, Guochun Wu, Lei Yao, Yinghui Zhang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005952"
                        open
                      >
                        <summary class="article-expander__title">Scaled pressure of dynamical systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005952?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Dandan Cheng, Zhiming Li</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005630"
                        open
                      >
                        <summary class="article-expander__title">Higher integrability and stability of (p,q)-quasiminimizers</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005630?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Antonella Nastasi, Cintia Pacchiano Camacho</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005629"
                        open
                      >
                        <summary class="article-expander__title">Optimal decay of compressible Navier-Stokes equations with or without potential force</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005629?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Jincheng Gao, Minling Li, Zheng-an Yao</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200568X"
                        open
                      >
                        <summary class="article-expander__title">Dynamical system describing cloud of particles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200568X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Dorota Bors, Robert Stańczy</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005915"
                        open
                      >
                        <summary class="article-expander__title">Nonlinear stability of multi-solitons for the Hirota equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005915?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Yamin Xiao, Boling Guo, Zhong Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005745"
                        open
                      >
                        <summary class="article-expander__title">Removable time-dependent singularities of solutions to the Stokes equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005745?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Hideo Kozono, Erika Ushikoshi, Fumitaka Wakabayashi</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S002203962200571X"
                        open
                      >
                        <summary class="article-expander__title">Stackelberg-Nash exact controllability for the Kuramoto-Sivashinsky equation with boundary and distributed controls</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S002203962200571X?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Nicolás Carreño, Maurício C. Santos</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005654"
                        open
                      >
                        <summary class="article-expander__title">An inverse problem for the non-linear fractional magnetic Schrödinger equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005654?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Ru-Yu Lai, Ting Zhou</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005472"
                        open
                      >
                        <summary class="article-expander__title">On negative eigenvalues of the spectral problem for water waves of highest amplitude</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005472?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Vladimir Kozlov, Evgeniy Lokharu</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005666"
                        open
                      >
                        <summary class="article-expander__title">On 2D Eulerian limits à la Kuksin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005666?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Benedetta Ferrario</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005733"
                        open
                      >
                        <summary class="article-expander__title">Inverse spectral problems for radial Schrödinger operators and closed systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005733?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Xin-Jian Xu, Chuan-Fu Yang, Natalia Bondarenko</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005964"
                        open
                      >
                        <summary class="article-expander__title">Monotone reducing mechanism in delayed population model with degenerate diffusion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005964?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Ming Mei, Tianyuan Xu, Jingxue Yin</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005885"
                        open
                      >
                        <summary class="article-expander__title">Aggregation and classification of spatial dynamics of vector-borne disease in advective heterogeneous environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005885?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Kai Wang, Hao Wang, Hongyong Zhao</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005927"
                        open
                      >
                        <summary class="article-expander__title">Interior and boundary regularity criteria for the 6D steady Navier-Stokes equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005927?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Shuai Li, Wendong Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005903"
                        open
                      >
                        <summary class="article-expander__title">Two sequences of solutions for the semilinear elliptic equations with logarithmic nonlinearities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005903?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Wei Shuai</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005836"
                        open
                      >
                        <summary class="article-expander__title">Chern-Simons limit of ground state solutions for the Schrödinger equations coupled with a neutral scalar field</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005836?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Jin-Cai Kang, Xiao-Qi Liu, Chun-Lei Tang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005757"
                        open
                      >
                        <summary class="article-expander__title">Homogenization of a mineral dissolution and precipitation model involving free boundaries at the micro scale</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005757?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): M. Gahn, I.S. Pop</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005708"
                        open
                      >
                        <summary class="article-expander__title">Aspects of control theory on infinite-dimensional Lie groups and G-manifolds</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005708?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Helge Glöckner, Joachim Hilgert</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005976"
                        open
                      >
                        <summary class="article-expander__title">Global solutions to a Keller-Segel-consumption system involving singularly signal-dependent motilities in domains of arbitrary dimension</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005976?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Youshan Tao, Michael Winkler</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005897"
                        open
                      >
                        <summary class="article-expander__title">The Gevrey normalization for quasi-periodic systems under Siegel type small divisor conditions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005897?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Shoujun Xu, Hao Wu</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005642"
                        open
                      >
                        <summary class="article-expander__title">Asymptotic mean value formulas, nonlocal space-time parabolic operators and anomalous tug-of-war games</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005642?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Carmina Fjellström, Kaj Nyström, Yuqiong Wang</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005605"
                        open
                      >
                        <summary class="article-expander__title">The number of limit cycles for regularized piecewise polynomial systems is unbounded</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005605?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): R. Huzak, K. Uldall Kristiansen</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005691"
                        open
                      >
                        <summary class="article-expander__title">Singular perturbations and asymptotic expansions for SPDEs with an application to term structure models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005691?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Sergio Albeverio, Carlo Marinelli, Elisa Mastrogiacomo</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005678"
                        open
                      >
                        <summary class="article-expander__title">A uniqueness criterion for ordinary differential equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005678?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): A. Constantin</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005721"
                        open
                      >
                        <summary class="article-expander__title">Existence and structure of P-area minimizing surfaces in the Heisenberg group</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005721?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 5 January 2023
Source: Journal of Differential Equations, Volume 342
Author(s): Amir Moradifam, Alexander Rowell</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.sciencedirect.com/science/article/pii/S0022039622005769"
                        open
                      >
                        <summary class="article-expander__title">Variation of constants formulae for forward and backward stochastic Volterra integral equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.sciencedirect.com/science/article/pii/S0022039622005769?dgcid&#x3D;rss_sd_all">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022039622X00090-cov150h.gif" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Publication date: 15 January 2023
Source: Journal of Differential Equations, Volume 343
Author(s): Yushi Hamaguchi</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-23 datetime="2022-10-23T00:00:00.000Z">2022-10-23</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-23 datetime="2022-10-23T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33910-9"
                        open
                      >
                        <summary class="article-expander__title">Detection and prevalence of SARS-CoV-2 co-infections during the Omicron variant circulation in France</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33910-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33910-9/MediaObjects/41467_2022_33910_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Monitoring of co-infections of SARS-CoV-2 variants is important to evaluate their clinical impact and the risk of emergence of recombinants. Here, the authors develop and validate a methodological pipeline to detect co-infections and apply it to samples from France in early 2022, when Delta and Omicron were co-circulating.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34130-x"
                        open
                      >
                        <summary class="article-expander__title">Contact tracing reveals community transmission of COVID-19 in New York City</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34130-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34130-x/MediaObjects/41467_2022_34130_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In this study, the authors analyse contact tracing records for ~650,000 suspected or confirmed COVID-19 cases in New York City during the second epidemic wave. They reconstruct transmission networks and find that vaccination and zone-based control policies likely contributed to control of the epidemic.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10915"
                        open
                      >
                        <summary class="article-expander__title">Journal of Scientific Computing</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10915">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02032-1"
                        open
                      >
                        <summary class="article-expander__title">An Efficient Chorin–Temam Projection Proper Orthogonal Decomposition Based Reduced-Order Model for Nonstationary Stokes Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02032-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we propose an efficient proper orthogonal decomposition based reduced-order model (POD-ROM) for nonstationary Stokes equations, which combines the classical projection method with POD technique. This new scheme mainly owns two advantages: the first one is low computational costs since the classical projection method decouples the reduced-order velocity variable and reduced-order pressure variable, and POD technique further improves the computational efficiency; the second advantage consists of circumventing the verification of classical LBB/inf-sup condition for mixed reduced spaces with the help of pressure stabilized Petrov–Galerkin (PSPG)-type projection method, where the pressure stabilization term is inherent which allows the use of non inf-sup stable elements without adding extra stabilization terms. We analyze the proposed projection POD-ROM’s stability and convergence, and numerical experiments validate those theoretical results and also the high-efficiency.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-22 datetime="2022-10-22T00:00:00.000Z">2022-10-22</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-22 datetime="2022-10-22T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.19.512895v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Engineered synthetic one-carbon fixation exceeds yield of the Calvin Cycle [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.19.512895v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>One-carbon (C1) feedstocks derived from CO2 and renewable electricity, such as formate, are promising substrates for sustainable production of chemicals, food and fuels. Energetically more efficient, engineered C1-fixation pathways were proposed to increase biomass yields above their natural counterparts, but have so far not been shown to achieve this. Here, we replace the native  energy-inefficient Calvin-Benson-Bassham (CBB) cycle in Cupriavidus necator by genomic integration of the synthetic reductive glycine pathway for growth on formate. Our final engineered strain reaches a higher biomass yield than the CBB-cycle-utilizing wild type, showing for the first time that efficiencies found in natural metabolism can be exceeded via a synthetic pathway. This yield increase demonstrates the potential of synthetic metabolism and is an important step towards realizing truly sustainable, economically feasible bio-based production.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-34032-y"
                        open
                      >
                        <summary class="article-expander__title">Protein language models trained on multiple sequence alignments learn phylogenetic relationships</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-34032-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-34032-y/MediaObjects/41467_2022_34032_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein language models taking multiple sequence alignments as inputs capture protein structure and mutational effects. Here, the authors show that these models also encode phylogenetic relationships, and can disentangle correlations due to structural constraints from those due to phylogeny.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22585-3"
                        open
                      >
                        <summary class="article-expander__title">A semi-supervised adaptive Markov Gaussian embedding process (SAMGEP) for prediction of phenotype event times using the electronic health record</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22585-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22585-3/MediaObjects/41598_2022_22585_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A semi-supervised adaptive Markov Gaussian embedding process (SAMGEP) for prediction of phenotype event times using the electronic health record</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22157-5"
                        open
                      >
                        <summary class="article-expander__title">Biogeography and eye size evolution of the ogre-faced spiders</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22157-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22157-5/MediaObjects/41598_2022_22157_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Biogeography and eye size evolution of the ogre-faced spiders</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22749-1"
                        open
                      >
                        <summary class="article-expander__title">Identification and functional interpretation of miRNAs affected by rare CNVs in CAKUT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22749-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22749-1/MediaObjects/41598_2022_22749_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Identification and functional interpretation of miRNAs affected by rare CNVs in CAKUT</span>
                                &ensp;<span class="article-reading-time">(&hairsp;34
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22313-x"
                        open
                      >
                        <summary class="article-expander__title">A multicohort geometric deep learning study of age dependent cortical and subcortical morphologic interactions for fluid intelligence prediction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22313-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22313-x/MediaObjects/41598_2022_22313_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A multicohort geometric deep learning study of age dependent cortical and subcortical morphologic interactions for fluid intelligence prediction</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22668-1"
                        open
                      >
                        <summary class="article-expander__title">Characterization of the binding of MRTX1133 as an avenue for the discovery of potential KRASG12D inhibitors for cancer therapy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22668-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22668-1/MediaObjects/41598_2022_22668_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Characterization of the binding of MRTX1133 as an avenue for the discovery of potential KRASG12D inhibitors for cancer therapy</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="285"
                        open
                      >
                        <summary class="article-expander__title">Journal of Mathematical Biology</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/285">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>No content preview</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01814-3"
                        open
                      >
                        <summary class="article-expander__title">Wave speed and critical patch size for integro-difference equations with a strong Allee effect</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01814-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Simplified conditions are given for the existence and positivity of wave speed for an integro-difference equation with a strong Allee effect and an unbounded habitat. The results are used to obtain the existence of a critical patch size for an equation with a bounded habitat. It is shown that if the wave speed is positive there exists a critical patch size such that for a habitat size above the critical patch size solutions can persist in space, and if the wave speed is negative solutions always approach zero. An analytical integral formula is developed to determine the critical patch size when the Laplace dispersal kernel is used, and this formula shows existence of multiple equilibrium solutions. Numerical simulations are provided to demonstrate connections among the wave speed, critical patch size, and Allee threshold.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01827-y"
                        open
                      >
                        <summary class="article-expander__title">Symmetry-breaking longitude bifurcations for a free boundary problem modeling small plaques in three dimensions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01827-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Atherosclerosis, one of the leading causes of death in USA and worldwide, begins with a lesion in the intima of the arterial wall, allowing LDL to penetrate into the intima where they are oxidized. The immune system considers these oxidized LDL as a dangerous substance and tasks the macrophages to attack them; incapacitated macrophages become foam cells and leads to the formation of a plaque. As the plaque continues to grow, it progressively restricts the blood flow, possibly triggering heart attack or stroke. Because the blood vessels tend to be circular, two-space dimensional cross section model is a good approximation, and the two-space dimensional models are studied in Friedman et al. (J Differ Equ 259(4):1227–1255, 2015) and Zhao and Hu (J Differ Equ 288:250–287, 2021). It is interesting to see whether a true three-space dimensional stationary solution can be developed. We shall establish a three-space dimensional stationary solution for the mathematical model of the initiation and development of atherosclerosis which involves LDL and HDL cholesterols, macrophages and foam cells. The model is a highly nonlinear and coupled system of PDEs with a free boundary, the interface between the plaque and the blood flow. We establish infinite branches of symmetry-breaking stationary solutions which bifurcate from the annular stationary solution in the longitude direction.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02025-0"
                        open
                      >
                        <summary class="article-expander__title">Triangularized Orthogonalization-Free Method for Solving Extreme Eigenvalue Problems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02025-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
A novel orthogonalization-free method together with two specific algorithms is proposed to address extreme eigenvalue problems. On top of gradient-based algorithms, the proposed algorithms modify the multicolumn gradient such that earlier columns are decoupled from later ones. Locally, both algorithms converge linearly with convergence rates depending on eigengaps. Momentum acceleration, exact linesearch, and column locking are incorporated to accelerate algorithms and reduce their computational costs. We demonstrate the efficiency of both algorithms on random matrices with different spectrum distributions and matrices from computational chemistry.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-21 datetime="2022-10-21T00:00:00.000Z">2022-10-21</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-21 datetime="2022-10-21T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.21.513145v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">PAM-less Exonuclease-assisted Cas12a for visual detection of Vibrio Species [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.21.513145v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Foodborne pathogens, including Vibrio spp. and norovirus, cause substantial economic and healthcare burdens worldwide. Rapid and sensitive point-of-care testing on-farm or restaurants for batch inspection of pathogenic contamination in raw food products is essential. Here, we present an easy-to-design, cost-effective PAM-less Exonuclease-assisted Cas12A Nucleic-acid Detection (PECAN) assay paired with nucleic acid amplification systems for rapid and sensitive visual detection of 2 pathogenic Vibrio species: Vibrio parahaemolyticus (TDH) and Vibrio Cholerae (ctxA) without protospacer adjacent motif (PAM) site limitation. With T7 exonuclease, PAM-less detection could be achieved with a low concentration of cas12a, costing $0.8 USD per reaction. The system could also be adapted for PAM-less cas12a nucleic acid detection in-field or in-lab for sensitive DNA or RNA detection. We also constructed a low-cost reusable 3D printed heater chassis and reusable sodium acetate heat packs for field use without generating solid waste.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.20.513037v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Comparison of DNA targeting CRISPR editors in human cells [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.20.513037v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Profiling and comparing the performance of current widely used DNA targeting CRISPR systems provide the basic information for the gene-editing toolkit and can be a useful resource for this field. Herein we made a parallel comparison between the recently reported miniature Cas12f1 and the widely used Cas12a and Cas9 nucleases in mammalian cells. The results revealed that as a CRISPRa activator, Un1Cas12f1 could induce gene expression with a comparable level to these of Cas12a and Cas9, while as a cleaving editor, Cas12f1 exhibited similar properties to Cas12a, like high specificity and dominantly induced deletions over insertions, but with less activity. In contrast, wild-type SpCas9 showed the highest activity, lowest specificity, and induced balanced deletions over insertions. Thus, Cas12f1 is recommended for gene-activation-based applications, Cas12a is for therapy applications, and wild-type Cas9 is for in vitro and animal investigations. The comparison provided the editing properties of current widely used DNA-targeting CRISPR systems for the gene-editing field.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.19.512873v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Dual Noncanonical Amino Acid Incorporation Enabling Chemoselective Protein Modification at Two Distinct Sites in Yeast [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.19.512873v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Proteins containing noncanonical amino acids (ncAAs) provide opportunities for dissecting fundamental biological processes and engineering protein therapeutics with diverse chemistries. Incorporation of more than one ncAA into a single construct can endow a protein with multiple useful features, such as unique molecular recognition and covalent crosslinking. Herein, for the first time, we encode two chemically distinct ncAAs into proteins prepared in Saccharomyces cerevisiae. To complement ncAA incorporation in response to the amber (TAG) stop codon in yeast, we evaluated opal (TGA) stop codon suppression using three distinct orthogonal translation systems (OTSs): E. coli tyrosyl-tRNA synthetase/tRNATyr, E. coli leucyl-tRNA synthetase/tRNALeu, and M. alvus pyrrolysyl-tRNA synthetase/tRNAPy…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;5
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.19.512661v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Resource-aware construct design in mammalian cells [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.19.512661v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Resource competition can be the cause of unintended coupling between co-expressed genetic constructs. Here we report quantification of the resource load imposed by different mammalian genetic components and identify construct designs with increased performance and reduced resource footprint. We use these to generate improved synthetic circuits and optimise the co-expression of co-transfected cassettes, shedding light on how this can be useful for bioproduction and biotherapeutic applications. This work provides the scientific community with a framework to take resource demand into consideration when designing mammalian constructs to achieve robust and optimised gene expression.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41592-022-01659-0"
                        open
                      >
                        <summary class="article-expander__title">ShareLoc — an open platform for sharing localization microscopy data</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41592-022-01659-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-022-01659-0/MediaObjects/41592_2022_1659_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Nature Methods - ShareLoc — an open platform for sharing localization microscopy data</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41597-022-01765-w"
                        open
                      >
                        <summary class="article-expander__title">Publisher Correction: TumorMet: A repository of tumor metabolic networks derived from context-specific Genome-Scale Metabolic Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41597-022-01765-w">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Scientific Data - Publisher Correction: TumorMet: A repository of tumor metabolic networks derived from context-specific Genome-Scale Metabolic Models</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04090-5"
                        open
                      >
                        <summary class="article-expander__title">Deep learning-assisted co-registration of full-spectral autofluorescence lifetime microscopic images with H&amp;E-stained histology images</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04090-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04090-5/MediaObjects/42003_2022_4090_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Using unsupervised image-to-image synthesis, homography regression-based co-registration of fluorescence-histology images is improved and can be used on various image formats across full-spectral emission wavelengths.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21748-6"
                        open
                      >
                        <summary class="article-expander__title">Hypothesis-driven dragging of transcriptomic data to analyze proven targeted pathways in Rhinella arenarum larvae exposed to organophosphorus pesticides</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21748-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21748-6/MediaObjects/41598_2022_21748_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Hypothesis-driven dragging of transcriptomic data to analyze proven targeted pathways in Rhinella arenarum larvae exposed to organophosphorus pesticides</span>
                                &ensp;<span class="article-reading-time">(&hairsp;43
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21491-y"
                        open
                      >
                        <summary class="article-expander__title">On the limits of graph neural networks for the early diagnosis of Alzheimer’s disease</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21491-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21491-y/MediaObjects/41598_2022_21491_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - On the limits of graph neural networks for the early diagnosis of Alzheimer’s disease</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20844-x"
                        open
                      >
                        <summary class="article-expander__title">Web and social media searches highlight menstrual irregularities as a global concern in COVID-19 vaccinations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20844-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20844-x/MediaObjects/41598_2022_20844_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Web and social media searches highlight menstrual irregularities as a global concern in COVID-19 vaccinations</span>
                                &ensp;<span class="article-reading-time">(&hairsp;19
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21574-w"
                        open
                      >
                        <summary class="article-expander__title">A stacking-based artificial intelligence framework for an effective detection and localization of colon polyps</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21574-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21574-w/MediaObjects/41598_2022_21574_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - A stacking-based artificial intelligence framework for an effective detection and localization of colon polyps</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41540-022-00251-8"
                        open
                      >
                        <summary class="article-expander__title">Severe testing with high-dimensional omics data for enhancing biomedical scientific discovery</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41540-022-00251-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41540-022-00251-8/MediaObjects/41540_2022_251_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>npj Systems Biology and Applications - Severe testing with high-dimensional omics data for enhancing biomedical scientific discovery</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22578-2"
                        open
                      >
                        <summary class="article-expander__title">Analyzing the GHSI puzzle of whether highly developed countries fared worse in COVID-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22578-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22578-2/MediaObjects/41598_2022_22578_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Analyzing the GHSI puzzle of whether highly developed countries fared worse in COVID-19</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03902-y"
                        open
                      >
                        <summary class="article-expander__title">Adaptation to host cell environment during experimental evolution of Zika virus</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03902-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03902-y/MediaObjects/42003_2022_3902_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In vitro analyses and computational modelling indicate that Zika virus adapts to the cellular environment of its host over time</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-20409-y"
                        open
                      >
                        <summary class="article-expander__title">Comparative transcriptomic analysis of whole blood mycobacterial growth assays and tuberculosis patients’ blood RNA profiles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-20409-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-20409-y/MediaObjects/41598_2022_20409_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Comparative transcriptomic analysis of whole blood mycobacterial growth assays and tuberculosis patients’ blood RNA profiles</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22210-3"
                        open
                      >
                        <summary class="article-expander__title">Biosensors based on novel nonlinear delta-function photonic crystals comprising weak nonlinearities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22210-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22210-3/MediaObjects/41598_2022_22210_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Biosensors based on novel nonlinear delta-function photonic crystals comprising weak nonlinearities</span>
                                &ensp;<span class="article-reading-time">(&hairsp;27
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21546-0"
                        open
                      >
                        <summary class="article-expander__title">In silico investigations identified Butyl Xanalterate to competently target CK2α (CSNK2A1) for therapy of chronic lymphocytic leukemia</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21546-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21546-0/MediaObjects/41598_2022_21546_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - In silico investigations identified Butyl Xanalterate to competently target CK2α (CSNK2A1) for therapy of chronic lymphocytic leukemia</span>
                                &ensp;<span class="article-reading-time">(&hairsp;47
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22253-6"
                        open
                      >
                        <summary class="article-expander__title">The distance distribution of human microRNAs in MirGeneDB database</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22253-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22253-6/MediaObjects/41598_2022_22253_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - The distance distribution of human microRNAs in MirGeneDB database</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-20 datetime="2022-10-20T14:20:53.000Z">2022-10-20</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-20 datetime="2022-10-20T14:20:53.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119930"
                        open
                      >
                        <summary class="article-expander__title">Experiments Spell Doom for Decades-Old Explanation of Quantum Weirdness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/physics-experiments-spell-doom-for-quantum-collapse-theory-20221020/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/Collapse_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Physical-collapse theories have long offered a natural solution to the central mystery of the quantum world. But a series of increasingly precise experiments are making them untenable.              
The post Experiments Spell Doom for Decades-Old Explanation of Quantum Weirdness first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00401"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Heterologous Gene Regulation in Clostridia: Rationally Designed Gene Regulation for Industrial and Medical Applications</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00401">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00401</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00397"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Engineering a Rhodopsin-Based Photo-Electrosynthetic System in Bacteria for CO2 Fixation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00397">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00397</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0095794?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Hopf bifurcation of the model with terms of two time-delays and delay-dependent parameter based on the theory of crossing curves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0095794?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
A three-layer prey–predator model with two time-delays and one delay-dependent parameter is established in this paper. To begin, the paper calculates the conditions for each population in the model to maintain the quantity stable and Hopf bifurcation when [math], [math], [math] is in the stable interval, and [math]. The crossing curves, their type, and the direction of the crossing curves are then obtained using the crossing curve method, which is composed of the threshold values of the dynamic behavior change on the two time-delays plane when [math]. The real data from the forage grass–Ochotona curzoniae–Buteo hemilasius food chain is used to conduct an empirical study of the model. When [math], the feasible region of the crossing curves is open-ended, and the model&#x27;s crossing curves on the [math] plane are truncated. This indicates that the model&#x27;s threshold distribution of dynamic behavior change is a regular curve made of several curves. The simulation using the time-delay value on the crossing curves shows that the model produces different dynamic behaviors such as stability, bifurcation, and chaos depending on the time-delay value on both sides of the curves. The critical values of dynamic behavior change are time-delay values on the crossing curves. The empirical study shows that increasing Ochotona curzoniae’s environmental capacity can easily cause Hopf bifurcation of the system. At this time, the number of each population in the Plateau ecosystem constantly fluctuates, and Ochotona curzoniae is vulnerable to extinction.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0119963?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Model-free prediction of multistability using echo state network</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0119963?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
In the field of complex dynamics, multistable attractors have been gaining significant attention due to their unpredictability in occurrence and extreme sensitivity to initial conditions. Co-existing attractors are abundant in diverse systems ranging from climate to finance and ecological to social systems. In this article, we investigate a data-driven approach to infer different dynamics of a multistable system using an echo state network. We start with a parameter-aware reservoir and predict diverse dynamics for different parameter values. Interestingly, a machine is able to reproduce the dynamics almost perfectly even at distant parameters, which lie considerably far from the parameter values related to the training dynamics. In continuation, we can predict whole bifurcation diagram significant accuracy as well. We extend this study for exploring various dynamics of multistable attractors at an unknown parameter value. While we train the machine with the dynamics of only one attractor at parameter [math], it can capture the dynamics of a co-existing attractor at a new parameter value [math]. Continuing the simulation for a multiple set of initial conditions, we can identify the basins for different attractors. We generalize the results by applying the scheme on two distinct multistable systems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0108050?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">On the averaging principle for stochastic differential equations involving Caputo fractional derivative</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0108050?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
In this paper, we investigate the averaging principle for Caputo-type fractional stochastic differential equations driven by Brownian motion. Different from the approach of integration by parts or decomposing integral interval to deal with the estimation of integral involving singular kernel in the existing literature, we show the desired averaging principle in the sense of mean square by using Hölder inequality via growth conditions on the nonlinear stochastic term. Finally, a simulation example is given to verify the theoretical results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101668?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Physics captured by data-based methods in El Niño prediction</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101668?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
On average once every four years, the Tropical Pacific warms considerably during events called El Niño, leading to weather disruptions over many regions on Earth. Recent machine-learning approaches to El Niño prediction, in particular, Convolutional Neural Networks (CNNs), have shown a surprisingly high skill at relatively long lead times. In an attempt to understand this high skill, we here use data from distorted physics simulations with the intermediate-complexity Zebiak–Cane model to determine what aspects of El Niño physics are represented in a specific CNN-based classification method. We find that the CNN can adequately correct for distortions in the ocean adjustment processes, but that the machine-learning method has far more trouble in dealing with distortions in upwelling feedback strength.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.18.512742v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Engineering ssRNA tile filaments for (dis)assembly and membrane binding [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.18.512742v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Cytoskeletal protein filaments such as actin and microtubules confer mechanical support to cells and facilitate many cellular functions such as motility and division. Recent years have witnessed the development of a variety of molecular scaffolds that mimic such cytoskeletal filaments. Indeed, filaments that are programmable and compatible with biological systems may prove useful in studying or substituting such proteins. Here, we explore the use of ssRNA tiles to build and modify cytoskeletal-like filaments in vitro. We engineer a number of functionalities that are crucial to the function of natural cytoskeletal systems into the ssRNA tiles, including the abilities to assemble or disassemble filaments, to tune the filament stiffness, to induce membrane binding, and to attach binding proteins. The work paves the way for building dynamic cell-like cytoskeletal systems made out of rationally designed ssRNA tiles that can be transcribed in natural or synthetic cells.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Computational biology and bioinformatics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/computational-biology-and-bioinformatics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41591-022-02034-4"
                        open
                      >
                        <summary class="article-expander__title">Artificial intelligence can improve patients’ experience in decentralized clinical trials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41591-022-02034-4">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Medicine - Artificial intelligence can improve patients’ experience in decentralized clinical trials</span>
                                &ensp;<span class="article-reading-time">(&hairsp;10
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-22530-4"
                        open
                      >
                        <summary class="article-expander__title">Trainable joint bilateral filters for enhanced prediction stability in low-dose CT</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-22530-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-22530-4/MediaObjects/41598_2022_22530_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Trainable joint bilateral filters for enhanced prediction stability in low-dose CT</span>
                                &ensp;<span class="article-reading-time">(&hairsp;26
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21645-y"
                        open
                      >
                        <summary class="article-expander__title">Individual-based modeling reveals that the COVID-19 isolation period can be shortened by community vaccination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21645-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21645-y/MediaObjects/41598_2022_21645_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Individual-based modeling reveals that the COVID-19 isolation period can be shortened by community vaccination</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41588-022-01198-6"
                        open
                      >
                        <summary class="article-expander__title">Deciphering single-cell transcriptional programs across species</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41588-022-01198-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41588-022-01198-6/MediaObjects/41588_2022_1198_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The ability to predict gene-expression landscapes at single-cell resolution has long been a challenge in the field of genomics. We mapped whole-body single-cell transcriptomic landscapes of zebrafish, Drosophila, and earthworm using Microwell-seq. We propose the first sequence-based model, Nvwa, that can predict gene expression at single-cell resolution directly from genomic sequences.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33659-1"
                        open
                      >
                        <summary class="article-expander__title">Rapid antibiotic susceptibility testing and species identification for mixed samples</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33659-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33659-1/MediaObjects/41467_2022_33659_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Rapid antibiotic susceptibility testing (AST) is needed. Here the authors report a method for phenotypic AST at the single cell level, using a microfluidic chip that allows for subsequent genotyping with in situ FISH; they apply this to a mixed sample of 7 species and 4 antibiotics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;27
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21732-0"
                        open
                      >
                        <summary class="article-expander__title">Motif and conserved module analysis in DNA (promoters, enhancers) and RNA (lncRNA, mRNA) using AlModules</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21732-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21732-0/MediaObjects/41598_2022_21732_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Motif and conserved module analysis in DNA (promoters, enhancers) and RNA (lncRNA, mRNA) using AlModules</span>
                                &ensp;<span class="article-reading-time">(&hairsp;33
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-21473-0"
                        open
                      >
                        <summary class="article-expander__title">Identification of a novel ceRNA network related to prognosis and immunity in HNSCC based on integrated bioinformatic investigation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-21473-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-21473-0/MediaObjects/41598_2022_21473_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Identification of a novel ceRNA network related to prognosis and immunity in HNSCC based on integrated bioinformatic investigation</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-18839-9"
                        open
                      >
                        <summary class="article-expander__title">In-hospital risk stratification algorithm of Asian elderly patients</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-18839-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-18839-9/MediaObjects/41598_2022_18839_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - In-hospital risk stratification algorithm of Asian elderly patients</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01185-2"
                        open
                      >
                        <summary class="article-expander__title">Food from air</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01185-2">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Chemical Biology - Food from air</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01826-z"
                        open
                      >
                        <summary class="article-expander__title">A game-theoretic model of rabies in domestic dogs with multiple voluntary preventive measures</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01826-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Game theory is now routinely applied to quantitatively model the decision making of individuals presented with various voluntary actions that can prevent a given disease. Most models consider only a single preventive strategy and the case where multiple preventative actions are available is severely understudied. In our paper, we consider a very simple SI compartmental model of rabies in the domestic dog population. We study two choices of the dog owners: to vaccinate their dogs or to restrict the movements of unvaccinated dogs. We analyze the relatively rich patterns of Nash equilibria (NE). We show that there is always at least one NE at which the owners utilize only one form of prevention. However, there can be up to three different NEs at the same time: two NEs at which the owners use exclusively only the vaccination or movement restriction, and the third NE when the owners use both forms of prevention simultaneously. However, we also show that, unlike the first two types of NEs, the third kind of NE is not convergent stable.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;31
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01824-1"
                        open
                      >
                        <summary class="article-expander__title">Detecting minimum energy states and multi-stability in nonlocal advection–diffusion models for interacting species</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01824-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Deriving emergent patterns from models of biological processes is a core concern of mathematical biology. In the context of partial differential equations, these emergent patterns sometimes appear as local minimisers of a corresponding energy functional. Here we give methods for determining the qualitative structure of local minimum energy states of a broad class of multi-species nonlocal advection–diffusion models, recently proposed for modelling the spatial structure of ecosystems. We show that when each pair of species respond to one another in a symmetric fashion (i.e. via mutual avoidance or mutual attraction, with equal strength), the system admits an energy functional that decreases in time and is bounded below. This suggests that the system will eventually reach a local minimum energy steady state, rather than fluctuating in perpetuity. We leverage this energy functional to develop tools, including a novel application of computational algebraic geometry, for making conjectures about the number and qualitative structure of local minimum energy solutions. These conjectures give a guide as to where to look for numerical steady state solutions, which we verify through numerical analysis. Our technique shows that even with two species, multi-stability with up to four classes of local minimum energy states can emerge. The associated dynamics include spatial sorting via aggregation and repulsion both within and between species. The emerging spatial patterns include a mixture of territory-like segregation as well as narrow spike-type solutions. Overall, our study reveals a general picture of rich multi-stability in systems of moving and interacting species.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;68
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02005-4"
                        open
                      >
                        <summary class="article-expander__title">A New Tensor Multi-rank Approximation with Total Variation Regularization for Tensor Completion</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02005-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we present a novel tensor completion model which combines the Laplace function and an anisotropic total variation regularization. The Laplace function is utilized to approximate the tensor multi-rank, and the total variation regularization is added to improve the local piecewise smoothness and preserve the edges of the restored tensor data. An efficient alternating direction method of multipliers is proposed to tackle the tensor completion model, and its convergence theorem is also derived. Extensive experimental results on color images, videos, multispectral images and magnetic resonance imaging data show the efficiency and effectiveness of the proposed method.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02031-2"
                        open
                      >
                        <summary class="article-expander__title">Improved Efficiency of Multilevel Monte Carlo for Stochastic PDE through Strong Pairwise Coupling</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02031-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Multilevel Monte Carlo (MLMC) has become an important methodology in applied mathematics for reducing the computational cost of weak approximations. For many problems, it is well-known that strong pairwise coupling of numerical solutions in the multilevel hierarchy is needed to obtain efficiency gains. In this work, we show that strong pairwise coupling indeed is also important when MLMC is applied to stochastic partial differential equations (SPDE) of reaction-diffusion type, as it can improve the rate of convergence and thus improve tractability. For the MLMC method with strong pairwise coupling that was developed and studied numerically on filtering problems in (Chernov in Num Math 147:71-125, 2021), we prove that the rate of computational efficiency is higher than for existing methods. We also provide numerical comparisons with alternative coupling ideas on linear and nonlinear SPDE to illustrate the importance of this feature.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-19 datetime="2022-10-19T14:21:02.000Z">2022-10-19</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-19 datetime="2022-10-19T14:21:02.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119773"
                        open
                      >
                        <summary class="article-expander__title">Inside the Proton, the ‘Most Complicated Thing You Could Possibly Imagine’</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/inside-the-proton-the-most-complicated-thing-imaginable-20221019/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/PROTON_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The positively charged particle at the heart of the atom is an object of unspeakable complexity, one that changes its appearance depending on how it is probed.             
The post Inside the Proton, the ‘Most Complicated Thing You Could Possibly Imagine’ first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00336"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Crowding-Induced Spatial Organization of Gene Expression in Cell-Sized Vesicles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00336">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00336</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00283"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Bottom-Up Design: A Modular Golden Gate Assembly Platform of Yeast Plasmids for Simultaneous Secretion and Surface Display of Distinct FAP Fusion Proteins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00283">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00283</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-04069-2"
                        open
                      >
                        <summary class="article-expander__title">Deciphering the transcriptional regulation of the catabolism of lignin-derived aromatics in Rhodococcus opacus PD630</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-04069-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-04069-2/MediaObjects/42003_2022_4069_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Gene knockout studies with analysis of aromatic sensors identify transcription factors involved in regulating aromatic degradation pathways in Rhodococcus opacus and substrate hierarchy in aromatic mixtures for applications in lignin valorization.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33961-y"
                        open
                      >
                        <summary class="article-expander__title">Engineered cell differentiation and sexual reproduction in probiotic and mating yeasts</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33961-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33961-y/MediaObjects/41467_2022_33961_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>G protein-coupled receptors (GPCRs) enable cells to sense environmental cues and are indispensable for coordinating vital processes including quorum sensing, proliferation, and sexual reproduction. Here the authors, using heterologous GPCR expression and endogenous ligand production, enable synthetic mating in haploid yeast, and GPCR-mediated biosensing in diploid probiotic yeast.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;49
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33974-7"
                        open
                      >
                        <summary class="article-expander__title">Harnessing bioengineered microbes as a versatile platform for space nutrition</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33974-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33974-7/MediaObjects/41467_2022_33974_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Long-duration human space travel creates challenges for maintaining healthy diets. Here the authors discuss using synthetic biology approaches to modify yeast into an optimal, and enjoyable, food production platform.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01829-w"
                        open
                      >
                        <summary class="article-expander__title">Spatial ecology, optimal control and game theoretical fishing problems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01829-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Of paramount importance in both ecological systems and economic policies are the problems of harvesting of natural resources. A paradigmatic situation where this question is raised is that of fishing strategies. Indeed, overfishing is a well-known problem in the management of live-stocks, as being too greedy may lead to an overall dramatic depletion of the population we are harvesting. A closely related topic is that of Nash equilibria in the context of fishing policies. Namely, two players being in competition for the same pool of resources, is it possible for them to find an equilibrium situation? The goal of this paper is to provide a detailed analysis of these two queries (i.e optimal fishing strategies for single-player models and study of Nash equilibria for multiple players…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02026-z"
                        open
                      >
                        <summary class="article-expander__title">On the Implementation of an Adaptive Multirate Framework for Coupled Transport and Flow</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02026-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This paper presents a multirate in time approach for coupled flow and transport problems combined with goal-oriented error control based on the Dual Weighted Residual (DWR) method. The focus is on the implementation of the multirate concept regarding different time scales for the underlying subproblems. Key ingredients are an arbitrary degree discontinuous Galerkin time discretization, an a posteriori error representation for the transport problem coupled with flow and the concept of space-time slabs based on tensor-product spaces. From the latter, a space-time mesh adaptation with highly economical grids is developed. The performance of the approach is studied carefully by numerical convergence examples as well as an example of physical interest for convection-dominated transport.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;44
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02022-3"
                        open
                      >
                        <summary class="article-expander__title">A RBF-WENO Finite Difference Scheme for Non-linear Degenerate Parabolic Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02022-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this research, a radial basis functions weighted essentially non-oscillatory (RBF-WENO) scheme in the framework of finite difference is developed for solving non-linear degenerate parabolic (NDP) equations which may contain discontinuous solutions. The traditional WENO schemes for solving NDP equations are based on the polynomial interpolation. In this paper, a non-polynomial WENO finite difference scheme in order to enhance the local accuracy is proposed. For a non-polynomial interpolation basis, the infinitely smooth RBFs such as the multi-quadratic are adopted. By optimizing the free parameter in RBFs, the local error generated by the RBFs interpolation can be easily controlled. The formulation of scheme will be described in detail and numerical tests have been prepared to demonstrate the efficiency of the scheme in one and two-dimension. The numerical results demonstrate that the developed non-polynomial WENO schemes enhance the local accuracy and give sharper solution profile than the original WENO based on the polynomial interpolation for solving NDP equations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-18 datetime="2022-10-18T15:20:00.000Z">2022-10-18</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-18 datetime="2022-10-18T15:20:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/interdisciplinary-journey-through-living-machines-sherry-nyeo-1018"
                        open
                      >
                        <summary class="article-expander__title">An interdisciplinary journey through living machines</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/interdisciplinary-journey-through-living-machines-sherry-nyeo-1018">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202210/Sherry-Nyeo.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>With NEET, Sherry Nyeo is discovering MIT’s undergraduate research community at the intersection of computer science and biological engineering.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119841"
                        open
                      >
                        <summary class="article-expander__title">The Computer Scientist Who’s Boosting Privacy on the Internet</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/the-computer-scientist-who-boosts-privacy-with-entropy-20221018/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/Harry-Halpin_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Harry Halpin wants our internet conversations to be more private. He’s helped create a new kind of network that might make it possible.             
The post The Computer Scientist Who’s Boosting Privacy on the Internet first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;24
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0257?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Efficient Minimum Flow Decomposition via Integer Linear Programming</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0257?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1447404?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Nematic Liquid Crystals in a Rectangular Confinement: Solution Landscape, and Bifurcation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1447404?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://epubs.siam.org/na101/home/literatum/publisher/siam/journals/covergifs/smjmap/cover.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1808-1828, December 2022. 
 Abstract. We study the solution landscape and bifurcation diagrams of nematic liquid crystals confined on a rectangle, using a reduced two-dimensional Landau–de Gennes framework in terms of two geometry-dependent variables: half short edge length [math] and aspect ratio [math]. First, we analytically prove that, for any [math] with a small enough [math] or for a large enough [math] with a fixed domain size, there is a unique stable solution that has two line defects on the opposite short edges. Second, we numerically construct solution landscapes by varying [math] and [math], and report a novel X state, which emerges from saddle-node bifurcation and serves as the parent state in such a solution landscape. Various new classes are then found among these solution landscapes, including the X class, the S class, and the L class. By tracking the Morse indices of individual solutions, we present bifurcation diagrams for nematic equilibria, thus illustrating the emergence mechanism of critical points and several effects of geometrical anisotropy on confined defect patterns.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;6
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1454572?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Standing Waves of Reaction-Diffusion Equations on an Unbounded Graph with Two Vertices</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1454572?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1733-1763, December 2022. 
 Abstract. We deal with bistable reaction-diffusion equations in a domain of a metric graph with two vertices, that is, a domain of multiple half-lines with two junctions connected by a line segment. We prove that there exist two types of standing waves: standing front waves and unimodal waves, if the line segment is long enough. We also numerically show the exact number of standing wave solutions for a cubic nonlinearity and for a piecewise linear case. The stability and instability of the standing wave solutions are also investigated. Standing waves play a role in blocking the front propagation.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1472127?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Sparse Optimal Control of Pattern Formations for an SIR Reaction-Diffusion Epidemic Model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1472127?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1764-1790, December 2022. 
 Abstract. Pattern formation arising from the reaction-diffusion epidemic model is a space-time depiction of the distribution and transmission of infectious diseases. Disease control can be achieved by controlling the associated pattern formations. For an SIR reaction-diffusion epidemic model, we review its Turing pattern formations with different transmission rates under the case of a constant recovery rate. To control pattern formations of the SIR epidemic model, we introduce a regulator as a control function of the recovery rate. For a perfect control strategy, it not only leads to a desired pattern formation, but also has a small support in space-time domains. In order to obtain such a control strategy, we propose a sparse optimal control problem governed by the SIR epidemic model. We study the existence of optimal solutions, derive the first order necessary optimality system, obtain the sparsity structure of the control function, and numerically solve the control problem. Numerical results demonstrate the feasibility and effectivity of our method.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1483529?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Robustness of Solutions of Almost Every System of Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1483529?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1791-1807, December 2022. 
 Abstract. In mathematical modeling, it is common to have an equation [math], where the exact form of [math] is not known. This article shows that there are large classes of [math] where almost all [math] share the same properties. The classes we investigate are vector spaces [math] of [math] functions [math] that satisfy the following condition: [math] has “almost constant rank&quot; (ACR) if there is a constant integer [math] such that rank[math] for “almost every&quot; [math] and almost every [math]. If the vector space [math] is finite dimensional, then “almost every&quot; is with respect to the Lebesgue measure on [math], and otherwise, it means almost every in the sense of prevalence, as described herein. Most function spaces commonly used for modeling purposes are ACR. In particular, we show that if all of the functions in [math] are linear or polynomial or real analytic, or if [math] is the set of all functions in a “structured system&quot;, then [math] is ACR. For each [math] and [math], the solution set of [math] is SolSet[math]. A solution set of [math] is called robust if it persists despite small changes in [math] and [math]. The following two global results are proved for almost every [math] in an ACR vector space [math]: (1) Either the solution set SolSet[math] is robust for almost every [math], or none of the solution sets are robust. (2) The solution set SolSet[math] is a [math]-manifold of dimension [math]. In particular, [math] is the same for almost every [math].</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00348"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] EasyGuide Plasmids Support in Vivo Assembly of gRNAs for CRISPR/Cas9 Applications in Saccharomyces cerevisiae</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00348">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00348</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.16.512417v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Directed evolution of aerotolerance in sulfide-dependent thiazole synthases [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.16.512417v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sulfide-dependent THI4 thiazole synthases could potentially be used to replace plant cysteine-dependent suicide THI4s, whose high protein turnover rates make thiamin synthesis exceptionally energy-expensive. However, sulfide-dependent THI4s are anaerobic or microoxic enzymes and hence unadapted to the aerobic conditions in plants; they are also slow enzymes (kcat &lt;1 h-1). To improve aerotolerance and activity, we applied continuous directed evolution under aerobic conditions in the yeast OrthoRep system to two sulfide-dependent bacterial THI4s. Six beneficial single mutations were identified, of which five lie in the active-site cleft predicted by structural modeling and two recapitulate features of naturally aerotolerant THI4s. That single mutations gave substantial improvements suggests that further advance under selection will be possible by stacking mutations. This proof-of-concept study established that the performance of sulfide-dependent THI4s in aerobic conditions is evolvable and, more generally, that yeast OrthoRep provides a plant-like bridge to adapt nonplant enzymes to work better in plants.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33287-9"
                        open
                      >
                        <summary class="article-expander__title">A synthetic transcription platform for programmable gene expression in mammalian cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33287-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33287-9/MediaObjects/41467_2022_33287_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Precise and scalable regulation of gene expression in mammalian cells is challenging. Here, the authors created a highly tunable CRISPR-based synthetic transcription system for programmable control of mammalian gene expression and cellular activity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;53
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01815-2"
                        open
                      >
                        <summary class="article-expander__title">Permanence via invasion graphs: incorporating community assembly into modern coexistence theory</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01815-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
To understand the mechanisms underlying species coexistence, ecologists often study invasion growth rates of theoretical and data-driven models. These growth rates correspond to average per-capita growth rates of one species with respect to an ergodic measure supporting other species. In the ecological literature, coexistence often is equated with the invasion growth rates being positive. Intuitively, positive invasion growth rates ensure that species recover from being rare. To provide a mathematically rigorous framework for this approach, we prove theorems that answer two questions: (i) When do the signs of the invasion growth rates determine coexistence? (ii) When signs are sufficient, which invasion growth rates need to be positive? We focus on deterministic models and equate …</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-17 datetime="2022-10-17T13:44:13.000Z">2022-10-17</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-17 datetime="2022-10-17T13:44:13.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119791"
                        open
                      >
                        <summary class="article-expander__title">Mathematicians Discover the Fibonacci Numbers Hiding in Strange Spaces</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/mathematicians-surprised-by-hidden-fibonacci-numbers-20221017/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/Infinite-Staircases_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Recent explorations of unique geometric worlds reveal perplexing patterns, including the Fibonacci sequence and the golden ratio.            
The post Mathematicians Discover the Fibonacci Numbers Hiding in Strange Spaces first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0106230?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Spread rates of spread models with frozen symbols</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0106230?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
This article aims to compare the long-term behavior of a spread model before and after a type in the model becomes frozen, namely, a type of which an individual only produces individuals of the same type. By means of substitution dynamical systems and matrix analysis, the first result of this work gives the spread rates of a 1-spread model with one frozen symbol. Later, in the work, this is shown to hold under more general settings, which include generalized frozen symbols and frozen symbols in [math]-spread models. Numerical experiments are provided as supporting evidence for the theory.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0104017?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Synchronization and bifurcation in an economic model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0104017?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We study the synchronization of two coupled idealized economies. In the present work, we consider a recently developed economic system that shows a richness of dynamical behavior. By means of the Lyapunov exponents, we analyze that there is overly complex behavior in the transitions in the dynamics of an isolated economy, oscillating between chaotic attractors and limit cycles. Then, for two coupled economies, we analyze the synchronization states for the space of all control parameters as a function of the network coupling parameter. Interestingly, we have evidenced that there is a broad region of fully synchronized states and as we increase the coupling, some phenomena such as a smooth and intermittent loss in synchronization emerge. In the same way, we observe phase synchronization for one of the control parameters. Ultimately, in order to confirm this loss of synchronization, we inspect the stability of synchronized states through the master stability function method for some control parameters. Here, we corroborate what was previously observed, the unexpected vast range of control parameter values of instability corresponding to desynchronization.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00175"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] ACtivE: Assembly and CRISPR-Targeted in Vivo Editing for Yeast Genome Engineering Using Minimum Reagents and Time</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00175">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00175</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.15.512367v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">SLICER: Seamless Loss of Integrated Cassettes Using Endonuclease Cleavage and Recombination in Deinococcus radiodurans [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.15.512367v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Methods for creating seamless genome modifications are an essential part of the microbial genetic toolkit that allows for strain engineering through the recycling of selectable markers. Here, we report the development of a method, termed SLICER, which can be used to create seamless genome modifications in D. radiodurans. We used SLICER to sequentially target four putative restriction-modification (R-M) system genes, recycling the same selective and screening markers for each subsequent deletion. A fifth R-M gene was replaced by a selectable marker to create a final D. radiodurans strain with 5 of the 6 putative R-M systems deleted. While we observed no significant increase in transformation efficiency, SLICER is a promising method to obtain a fully restriction-minus strain and expand the synthetic biology applications of D. radiodurans including as an in vivo DNA assembly platform.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH&#x3D;200 HEIGHT&#x3D;142 SRC&#x3D;&quot;FIGDIR/small/512367v1_ufig1.gif&quot; ALT&#x3D;&quot;Figure 1&quot;&gt;
View larger version (57K):
org.highwire.dtl.DTLVardef@c6d297org.highwire.dtl.DTLVardef@12bc01borg.highwire.dtl.DTLVardef@151649borg.highwire.dtl.DTLVardef@a426d9_HPS_FORMAT_FIGEXP  M_FIG C_FIG</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33890-w"
                        open
                      >
                        <summary class="article-expander__title">A scalable platform to discover antimicrobials of ribosomal origin</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33890-w">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33890-w/MediaObjects/41467_2022_33890_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Ribosomally synthesized and post-translationally modified peptides are a source of antimicrobials. Here, the authors report a platform for the rapid evaluation and characterization of biosynthetic gene clusters that enables the identification of 30 structurally diverse modified peptides, including three showing antimicrobial activities.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;54
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Nonlinear dynamics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/nonlinear-dynamics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s43588-022-00345-7"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Forced and spontaneous symmetry breaking in cell polarization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s43588-022-00345-7">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Computational Science - Author Correction: Forced and spontaneous symmetry breaking in cell polarization</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-16 datetime="2022-10-16T00:00:00.000Z">2022-10-16</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-16 datetime="2022-10-16T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >bioRxiv Subject Collection: Synthetic Biology</button>
                  <a class="source-heading__link" href="http://biorxiv.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.13.512145v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">Chimeric MerR-Family Regulators and Logic Elements for the Design of Metal Sensitive Genetic Circuits in Bacillus subtilis [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.13.512145v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Whole-cell biosensors are emerging as promising tools for monitoring environmental pollutants such as heavy metals. These sensors constitute a genetic circuit comprising a sensing module and an output module, such that a detectable signal is produced in the presence of the desired analyte. The MerR family of metal-responsive regulators offers great potential for the construction of metal sensing circuits, due to their high sensitivity, tight transcription control and large diversity in metal-specificity. However, the sensing diversity is broadest in Gram-negative systems, while chassis organisms are often selected from Gram-positive species, particularly sporulating bacilli. This can be problematic, because Gram-negative biological parts, such as promoters, are frequently observed to be non-functional in Gram-positive hosts. Herein, we combined construction of synthetic genetic circuits and chimeric MerR regulators, supported by structure-guided design, to generate metal-sensitive biosensor modules that are functional in the biotechnological work-horse species Bacillus subtilis. These chimeras consist of a constant Gram-positive derived DNA-binding domain fused to variable metal binding domains of Gram-negative origins. To improve the specificity of the whole-cell biosensor, we developed a modular  AND gate logic system based on the B. subtilis natively split {sigma}-factor, SigO-RsoA, designed to maximise future use for synthetic biology applications in B. subtilis. This work provides insights into the use of modular regulators, such as the MerR family, in the design of synthetic circuits for the detection of heavy metals, with potential wider applicability of the approach to other systems and genetic backgrounds.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://biorxiv.org/cgi/content/short/2022.10.12.511977v1?rss&#x3D;1"
                        open
                      >
                        <summary class="article-expander__title">IS200/IS605 Family-Associated TnpB Increases Transposon Activity and Retention [NEW RESULTS]</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://biorxiv.org/cgi/content/short/2022.10.12.511977v1?rss&#x3D;1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>The IS200/IS605 family are abundant insertion sequences associated with one of the most numerous genes in nature, tnpB. Sequence homologies suggest that transposon-borne TnpB may be an evolutionary precursor to CRISPR-Cas system effector proteins, and TnpB has been shown to function as a Cas-like programmable DNA endonuclease. However, the role of TnpB in transposition and its contributions to genome dynamics remain elusive. Using a suite of fluorescent reporters coupled to transposition, we show that TnpB increases IS608-transposon activity and prevents transposon loss from genomes. Analyzing our results through a mathematical model of transposon dynamics, we discuss the multifaceted roles TnpB may play in transposon regulation. The mutually beneficial transposon-TnpB interaction may explain the prevalence of tnpB, creating conditions for the proliferation and appropriation of TnpBs RNA-guided endonuclease activity for adaptive immunity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-15 datetime="2022-10-15T00:00:00.000Z">2022-10-15</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-15 datetime="2022-10-15T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33679-x"
                        open
                      >
                        <summary class="article-expander__title">In vitro assembly, positioning and contraction of a division ring in minimal cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33679-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33679-x/MediaObjects/41467_2022_33679_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Constructing a minimal protein machinery for self-division of membrane compartments is a major goal of bottom-up synthetic biology. Here, authors achieved the assembly, placement and onset of contraction of a minimal division ring in lipid vesicles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Nonlinear dynamics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/nonlinear-dynamics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33679-x"
                        open
                      >
                        <summary class="article-expander__title">In vitro assembly, positioning and contraction of a division ring in minimal cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33679-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33679-x/MediaObjects/41467_2022_33679_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Constructing a minimal protein machinery for self-division of membrane compartments is a major goal of bottom-up synthetic biology. Here, authors achieved the assembly, placement and onset of contraction of a minimal division ring in lipid vesicles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01823-2"
                        open
                      >
                        <summary class="article-expander__title">Foundations of static and dynamic absolute concentration robustness</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01823-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Absolute Concentration Robustness (ACR) was introduced by Shinar and Feinberg (Science 327:1389-1391, 2010) as robustness of equilibrium species concentration in a mass action dynamical system. Their aim was to devise a mathematical condition that will ensure robustness in the function of the biological system being modeled. The robustness of function rests on what we refer to as empirical robustness—the concentration of a species remains unvarying, when measured in the long run, across arbitrary initial conditions. Even simple examples show that the ACR notion introduced in Shinar and Feinberg (Science 327:1389-1391, 2010) (here referred to as static ACR) is neither necessary nor sufficient for empirical robustness. To make a stronger connection with empirical robustness, we define dynamic ACR, a property related to long-term, global dynamics, rather than only to equilibrium behavior. We discuss general dynamical systems with dynamic ACR properties as well as parametrized families of dynamical systems related to reaction networks. We find necessary and sufficient conditions for dynamic ACR in complex balanced reaction networks, a class of networks that is central to the theory of reaction networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-14 datetime="2022-10-14T07:00:00.000Z">2022-10-14</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-14 datetime="2022-10-14T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0264?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Multimodal Single-Cell Translation and Alignment with Semi-Supervised Learning</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0264?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1429102?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">A Matrix Basis Formulation for the Dyadic Green’s Functions of Maxwell’s Equations in Layered Media</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1429102?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1710-1732, December 2022. 
 Abstract. A matrix basis formulation is introduced to represent the [math] frequency domain dyadic Green’s functions of Maxwell’s equations in 3-dimensional layered media. The formulation can be used to decompose the Maxwell’s Green’s functions into independent TE and TM components, each satisfying a scalar Helmholtz equation. Moreover, the interface transmission conditions for the electromagnetic fields can be reduced to decoupled conditions for the matrix basis coefficients at interfaces. Numerical results for the electric and magnetic dyadic Green’s functions for a 10-layer medium show the capability of the proposed formulation.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1485814?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Spatial Segregation in Reaction-Diffusion Epidemic Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1485814?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1680-1709, December 2022. 
 Abstract. In this paper, we formulate susceptible-infected-susceptible reaction-diffusion epidemic models with cognition and show the impact of movement strategies on disease outbreak and mitigation under a spatially heterogeneous environment. The cognitive diffusion either takes a Fokker–Planck type diffusion obtained by Chapman’s diffusion law (called random diffusion) or follows Fick’s diffusion law (called symmetric diffusion). We derive a variational expression of the basic reproduction number [math] for both models and prove that the disease-free equilibrium is unique and globally asymptotically stable if [math]. Furthermore, if [math], the model following Fick’s diffusion law admits at least one endemic equilibrium and the model following Chapman’s diffusion law has a unique endemic equilibrium. The theoretical results are illustrated by numerical simulations, which additionally show the segregation phenomenon between susceptible and infected populations regulated by different movement strategies. Spatial segregation here is natural, not caused by an isolation policy, and thus is the most important indicator for an infectious disease to spread or wane in the absence of intervention. The first example shows that a heterogeneous random diffusion segregates infected and susceptible populations further than an ODE model and thus reduces the infection size. However, symmetric diffusion never does that. The second example shows that a heterogeneous random diffusion detriments segregation but still reduces the infection severity by moving infected individuals to a disease-free region. In a certain situation, a heterogeneous random diffusion may increase the infection severity as shown in the last example.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00420"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Controllable crRNA Self-Transcription Aided Dual-Amplified CRISPR-Cas12a Strategy for Highly Sensitive Biosensing of FEN1 Activity</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00420">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00420</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00362"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] MGCEP 1.0: A Genetic-Engineered Marine-Derived Chassis Cell for a Scaled Heterologous Expression Platform of Microbial Bioactive Metabolites</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00362">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00362</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102880?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Collective excitability in highly diluted random networks of oscillators</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102880?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We report on collective excitable events in a highly diluted random network of non-excitable nodes. Excitability arises thanks to a self-sustained local adaptation mechanism that drives the system on a slow timescale across a hysteretic phase transition involving states with different degrees of synchronization. These phenomena have been investigated for the Kuramoto model with bimodal distribution of the natural frequencies and for the Kuramoto model with inertia and a unimodal frequency distribution. We consider global and partial stimulation protocols and characterize the system response for different levels of dilution. We compare the results with those obtained in the fully coupled case showing that such collective phenomena are remarkably robust against network diluteness.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107554?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Virial-like thermodynamic uncertainty relation in the tight-binding regime</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107554?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We presented a methodology to approximate the entropy production for Brownian motion in a tilted periodic potential. The approximation stems from the well known thermodynamic uncertainty relation. By applying a virial-like expansion, we provided a tighter lower limit solely in terms of the drift velocity and diffusion. The approach presented is systematically analyzed in the tight-binding regime. We also provide a relative simple rule to validate using the tight-binding approach based on drift and diffusion relations rather than energy barriers and forces. We also discuss the implications of our results outside the tight-binding regime.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0106402?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The structure inference of flocking systems based on the trajectories</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0106402?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
The interaction between the swarm individuals affects the dynamic behavior of the swarm, but it is difficult to obtain directly from outside observation. Therefore, the problem we focus on is inferring the structure of the interactions in the swarm from the individual behavior trajectories. Similar inference problems that existed in network science are named network reconstruction or network inference. It is a fundamental problem pervading research on complex systems. In this paper, a new method, called Motion Trajectory Similarity, is developed for inferring direct interactions from the motion state of individuals in the swarm. It constructs correlations by combining the similarity of the motion trajectories of each cross section of the time series, in which individuals with highly similar motion states are more likely to interact with each other. Experiments on the flocking systems demonstrate that our method can produce a reliable interaction inference and outperform traditional network inference methods. It can withstand a high level of noise and time delay introduced into flocking models, as well as parameter variation in the flocking system, to achieve robust reconstruction. The proposed method provides a new perspective for inferring the interaction structure of a swarm, which helps us to explore the mechanisms of collective movement in swarms and paves the way for developing the flocking models that can be quantified and predicted.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0120404?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Statistical properties of extreme soliton collisions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0120404?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Synchronous collisions between a large number of solitons are considered in the context of a statistical description. It is shown that, during the interaction of solitons of the same signs, the wave field is effectively smoothed out. When the number of solitons increases and the sequence of their amplitudes decay slower, the focused wave becomes even smoother and the statistical moments get frozen for a long time. This quasi-stationary state is characterized by greatly reduced statistical moments and by the density of solitons close to some critical value. This state may be treated as the small-dispersion limit, what makes it possible to analytically estimate all high-order statistical moments. While the focus of the study is made on the Korteweg–de Vries equation and its modified version, a much broader applicability of the results to equations that support soliton-type solutions is discussed.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101935?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Relativistic gas: Lorentz-invariant distribution for the velocities</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101935?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
In 1911, Jüttner proposed the generalization, for a relativistic gas, of the Maxwell–Boltzmann distribution of velocities. Here, we want to discuss, among others, the Jüttner probability density function (PDF). Both the velocity space and, consequently, the momentum space are not flat in special relativity. The velocity space corresponds to the Lobachevsky one, which has a negative curvature. This curvature induces a specific power for the Lorentz factor in the PDF, affecting the Jüttner normalization constant in one, two, and three dimensions. Furthermore, Jüttner distribution, written in terms of a more convenient variable, the rapidity, presents a curvature change at the origin at sufficiently high energy, which does not agree with our computational dynamics simulations of a relativistic gas. However, in one dimension, the rapidity satisfies a simple additivity law. This allows us to obtain, through the central limit theorem, a new, Lorentz-invariant, PDF whose curvature at the origin does not change for any energy value and which agrees with our computational dynamics simulations data. Also, we perform extensive first-principle simulations of a one-dimensional relativistic gas constituted by light and heavy particles.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0089059?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">An empirical study on network conversion of stock time series based on STL method</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0089059?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
A complex network has been widely used to reveal the rule of a complex system. How to convert the stock data into a network is an open issue since the stock data are so large and their random volatility is strong. In this paper, a seasonal trend decomposition procedure based on the loess ([math]) method is applied to convert the stock time series into a directed and weighted symbolic network. Three empirical stock datasets, including the closing price of Shanghai Securities Composite Index, S&amp;P 500 Index, and Nikkei 225 Index, are considered. The properties of these stock time series are revealed from the topological characteristics of corresponding symbolic networks. The results show that: (1) both the weighted indegree and outdegree distributions obey the power-law distribution well; (2) fluctuations of stock closing price are revealed by related network topological properties, such as weighting degree, betweenness, pageranks, and clustering coefficient; and (3) stock closing price, in particular, periods such as financial crises, can be identified by modularity class of the symbolic networks. Moreover, the comparison between the [math] method and the visibility graph further highlights the advantages of the [math] method in terms of the time complexity of the algorithm. Our method offers a new idea to study the network conversion of stock time series.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01821-4"
                        open
                      >
                        <summary class="article-expander__title">Population games with instantaneous behavior and the Rosenzweig–MacArthur model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01821-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
How to determine the spatial distribution and population dynamics of animals are some of the key questions in ecology. These two have been coupled before, but there is no general method for determining spatial distributions based on instantanous behavior coupled with population dynamics. We propose modeling interacting populations with instantaneous habitat choice through mean-field games. By using the framework of variational inequalities, we are able to determine existence and uniqueness for habitat distributions of interacting populations, in both continuous and discrete habitats. With some additional restrictions, we are also able to show existence and uniqueness of fixed-points of the population dynamics along with spatial distributions. We illustrate our theoretical results by studying a Rosenzweig–MacArthur model where predators and consumers inhabit a continuous habitat. This study is conducted both theoretically and numerically. Analyzing the emergent dynamics is possible as viewing the system from the vantage point of variational inequalities allows for applying efficient numerical methods. The generality of our theoretical approach opens up for studying complex ecosystems, e.g. the impact of enrichment on spatial distributions in marine ecosystems.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;49
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-13 datetime="2022-10-13T15:05:40.000Z">2022-10-13</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-13 datetime="2022-10-13T15:05:40.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Quanta Magazine</button>
                  <a class="source-heading__link" href="https://www.quantamagazine.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.quantamagazine.org/?p&#x3D;119742"
                        open
                      >
                        <summary class="article-expander__title">Teenager Solves Stubborn Riddle About Prime Number Look-Alikes</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.quantamagazine.org/teenager-solves-stubborn-riddle-about-prime-number-look-alikes-20221013/">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://d2r55xnwy6nx47.cloudfront.net/uploads/2022/10/WHIZ_Daniel_Larsen_1200_Social.webp" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>In his senior year of high school, Daniel Larsen proved a key theorem about Carmichael numbers — strange entities that mimic the primes.            
The post Teenager Solves Stubborn Riddle About Prime Number Look-Alikes first appeared on Quanta Magazine</span>
                                &ensp;<span class="article-reading-time">(&hairsp;23
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0522?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Inferring Optimal Species Trees in the Presence of Gene Duplication and Loss: Beyond Rooted Gene Trees</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0522?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0241?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Pfeature: A Tool for Computing Wide Range of Protein Features and Building Prediction Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0241?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01069-x"
                        open
                      >
                        <summary class="article-expander__title">Scaling up genelet circuits</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01069-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01069-x/MediaObjects/41557_2022_1069_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>DNA nanotechnology and synthetic biology both aim to expand the range of dynamic behaviours exhibited by rationally programmed biomolecules. Now, the programmability of synthetic transcriptional circuits has been improved to enable synthesis of dynamic biomolecular circuits with unmatched complexity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01050-8"
                        open
                      >
                        <summary class="article-expander__title">A two-residue nascent-strand steric gate controls synthesis of 2′-O-methyl- and 2′-O-(2-methoxyethyl)-RNA</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01050-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01050-8/MediaObjects/41557_2022_1050_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Sterically demanding 2′-modified nucleotides used in antisense therapeutics have thus far been challenging to synthesise enzymatically. Now, it has been shown that mutation of two gatekeeper residues in an archaeal DNA polymerase unlocks efficient synthesis of the modified nucleic acid oligomers 2′-O-methyl-RNA and 2′-O-(2-methoxyethyl)-RNA and enables the evolution of 2′-O-methyl-RNA enzymes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;17
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Nonlinear dynamics : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/nonlinear-dynamics">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41540-022-00248-3"
                        open
                      >
                        <summary class="article-expander__title">Early dynamics of chronic myeloid leukemia on nilotinib predicts deep molecular response</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41540-022-00248-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41540-022-00248-3/MediaObjects/41540_2022_248_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>npj Systems Biology and Applications - Early dynamics of chronic myeloid leukemia on nilotinib predicts deep molecular response</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01819-y"
                        open
                      >
                        <summary class="article-expander__title">Temporal optimization of radiation therapy to heterogeneous tumour populations and cancer stem cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01819-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
External beam radiation therapy is a key part of modern cancer treatments which uses high doses of radiation to destroy tumour cells. Despite its widespread usage and extensive study in theoretical, experimental, and clinical works, many questions still remain about how best to administer it. Many mathematical studies have examined optimal scheduling of radiotherapy, and most come to similar conclusions. Importantly though, these studies generally assume intratumoral homogeneity. But in recent years, it has become clear that tumours are not homogeneous masses of cancerous cells, but wildly heterogeneous masses with various subpopulations which grow and respond to treatment differently. One subpopulation of particular importance is cancer stem cells (CSCs) which are known to exhibi…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01825-0"
                        open
                      >
                        <summary class="article-expander__title">Spatial dynamics of a vegetation model with uptake–diffusion feedback in an arid environment</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01825-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Vegetation patterns with a variety of structures is amazing phenomena in arid or semi-arid areas, which can identify the evolution law of vegetation and are typical signals of ecosystem functions. Many achievements have been made in this respect, yet the mechanisms of uptake–diffusion feedback on the pattern structures of vegetation is not fully understood. To well reveal the influences of parameters perturbation on the pattern formation of vegetation, we give a comprehensive analysis on a vegetation–water model in the forms of reaction–diffusion equation which is posed by Zelnik et al. (Proc Natl Acad Sci 112:12,327–12,331, 2015). We obtain the exact parameters range for stationary patterns and show the dynamical behaviors near the bifurcation point based on nonlinear analysis. It is found that the model has the properties of spot, labyrinth and gap patterns. Moreover, water diffusion rate prohibits the growth of vegetation while shading parameter promotes the increase of vegetation biomass. Our results show that gradual transitions from uniform state to gap pattern can occur for suitable value of parameters which may induce the emergence of desertification.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-12 datetime="2022-10-12T07:00:00.000Z">2022-10-12</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-12 datetime="2022-10-12T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0270?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">SCOTv2: Single-Cell Multiomic Alignment with Disproportionate Cell-Type Representation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0270?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1443650?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Stability and String Stability of Car-Following Models with Reaction-Time Delay</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1443650?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;smjmap&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>SIAM Journal on Applied Mathematics, Volume 82, Issue 5, Page 1661-1679, December 2022. 
 Abstract. We investigate the transfer function emanating from the linearization of a car-following model for human drivers when taking into account their reaction time, which is known to be a cause of the stop-and-go traffic phenomenon. This leads to a nonrational transfer function, implying nontrivial stability conditions which are explicitly given. They are in particular satisfied whenever string stability holds. We also show how this reaction time can introduce a partial string stability, where the transfer function modulus remains smaller than 1 up to some critical frequency. We explore conditions in the parameter space discriminating between 3 different regimes (stability, string stability, and partial string stability).</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01816-1"
                        open
                      >
                        <summary class="article-expander__title">Site-bond percolation model of epidemic spreading with vaccination in complex networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01816-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
To study disease transmission with vaccination based on the network, we map an SIR model to a site-bond percolation model. In the case where the vaccination probability is zero, this model degenerates into a bond percolation model without the immunization. Using the method of generation functions, we obtain exact theoretical results for the epidemic threshold and the average outbreak size. From these exact solutions, we find that the epidemic threshold increases while the average outbreak size decreases with vaccination probability. Numerical simulations show that the size of giant component S increases with transmissibility T but decreases with the probability of vaccination. In addition, we compare the epidemic threshold and the size of the giant component for a Poisson network, an exponential network and a power-law network using numerical simulations. When the probability of vaccination is fixed, the epidemic threshold is the smallest for heterogeneous networks and the size of giant component S in homogeneous networks becomes largest for large transmissibility T(T close to 1).</span>
                                &ensp;<span class="article-reading-time">(&hairsp;22
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02018-z"
                        open
                      >
                        <summary class="article-expander__title">Exponential Integrators for Second-Order in Time Partial Differential Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02018-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Two types of second-order in time partial differential equations, namely semilinear wave equations and semilinear beam equations are considered. To solve these equations with exponential integrators, we present an approach to compute efficiently the action of the matrix exponential as well as those of related matrix functions. Various numerical simulations are presented that illustrate this approach.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;38
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-11 datetime="2022-10-11T00:00:00.000Z">2022-10-11</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-11 datetime="2022-10-11T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02001-8"
                        open
                      >
                        <summary class="article-expander__title">Deep-HyROMnet: A Deep Learning-Based Operator Approximation for Hyper-Reduction of Nonlinear Parametrized PDEs</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02001-8">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
To speed-up the solution of parametrized differential problems, reduced order models (ROMs) have been developed over the years, including projection-based ROMs such as the reduced-basis (RB) method, deep learning-based ROMs, as well as surrogate models obtained through machine learning techniques. Thanks to its physics-based structure, ensured by the use of a Galerkin projection of the full order model (FOM) onto a linear low-dimensional subspace, the Galerkin-RB method yields approximations that fulfill the differential problem at hand. However, to make the assembling of the ROM independent of the FOM dimension, intrusive and expensive hyper-reduction techniques, such as the discrete empirical interpolation method (DEIM), are usually required, thus making this strategy less feasible for problems characterized by (high-order polynomial or nonpolynomial) nonlinearities. To overcome this bottleneck, we propose a novel strategy for learning nonlinear ROM operators using deep neural networks (DNNs). The resulting hyper-reduced order model enhanced by DNNs, to which we refer to as Deep-HyROMnet, is then a physics-based model, still relying on the RB method approach, however employing a DNN architecture to approximate reduced residual vectors and Jacobian matrices once a Galerkin projection has been performed. Numerical results dealing with fast simulations in nonlinear structural mechanics show that Deep-HyROMnets are orders of magnitude faster than POD-Galerkin-DEIM ROMs, still ensuring the same level of accuracy.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;51
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-10 datetime="2022-10-10T07:00:00.000Z">2022-10-10</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-10 datetime="2022-10-10T07:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.29070.zc?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">17th International Symposium on Bioinformatics Research and Applications (ISBRA 2021)</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.29070.zc?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1059-1060, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0108?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Semi-Supervised Cell Detection with Reliable Pseudo-Labels</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0108?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1061-1073, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0118?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Dimensionality Reduction of Single-Cell RNA Sequencing Data by Combining Entropy and Denoising AutoEncoder</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0118?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1074-1084, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0109?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">SuccSPred2.0: A Two-Step Model to Predict Succinylation Sites Based on Multifeature Fusion and Selection Algorithm</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0109?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1085-1094, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0111?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Detecting and Classifying Nuclei Using Multi-Scale Fully Convolutional Network</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0111?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1095-1103, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0113?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">SGFNNs: Signed Graph Filtering-based Neural Networks for Predicting Drug–Drug Interactions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0113?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1104-1116, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0101?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">A Transfer Learning-Based Classification Model for Particle Pruning in Cryo-Electron Microscopy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0101?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1117-1131, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0141?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Determining Significant Correlation Between Pairs of Extant Characters in a Small Parsimony Framework</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0141?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Volume 29, Issue 10, Page 1132-1154, October 2022.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0096959?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Birth of strange nonchaotic attractors in a piecewise linear oscillator</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0096959?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Nonsmooth systems are widely encountered in engineering fields. They have abundant dynamical phenomena, including some results on the complex dynamics in such systems under quasiperiodically forced excitations. In this work, we consider a quasiperiodically forced piecewise linear oscillator and show that strange nonchaotic attractors (SNAs) do exist in such nonsmooth systems. The generation and evolution mechanisms of SNAs are discussed. The torus-doubling, fractal, bubbling, and intermittency routes to SNAs are identified. The strange properties of SNAs are characterized with the aid of the phase sensitivity function, singular continuous spectrum, rational frequency approximation, and the path of the partial Fourier sum of state variables in a complex plane. The nonchaotic properties of SNAs are verified by the methods of maximum Lyapunov exponent and power spectrum.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101321?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">A comprehensive framework for preference aggregation Parrondo’s paradox</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101321?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Individuals can make choices for themselves that are beneficial or detrimental to the entire group. Consider two losing choices that some individuals have to make on behalf of the group. Is it possible that the losing choices combine to give a winning outcome? We show that it is possible through a variant of Parrondo’s paradox—the preference aggregation Parrondo’s paradox (PAPP). This new variant of Parrondo’s paradox makes use of an aggregate rule that combines with a decision-making heuristic that can be applied to individuals or parts of the social group. The aim of this work is to discuss this PAPP framework and exemplify it on a social network. This work enhances existing research by constructing a feedback loop that allows individuals in the social network to adapt its behavior according to the outcome of the Parrondo’s games played.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01059-z"
                        open
                      >
                        <summary class="article-expander__title">Genetically encoded chemical crosslinking of carbohydrate</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01059-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01059-z/MediaObjects/41557_2022_1059_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein–carbohydrate interactions remain challenging to study due to their low binding affinity and non-covalent nature. Now, a genetically encoded bioreactive unnatural amino acid containing sulfonyl fluoride has been shown to crosslink a protein with its bound glycan, offering a solution to probe and exploit protein–carbohydrate interactions.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02013-4"
                        open
                      >
                        <summary class="article-expander__title">A Posteriori Error Control and Adaptivity for the IMEX BDF2 Method for PIDEs with Application to Options Pricing Models</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02013-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we study a posteriori error estimates of the IMEX BDF2 scheme for time discretizations of solving parabolic partial integro-differential equations, which describe the jump-diffusion option pricing model in finance. Because of the initial singularities of the solution which is due to the nonsmoothness of the payoff function, a posteriori error control and adaptivity will be crucial in solving numerically this type of equations. To derive optimal order a posteriori error estimates, quadratic reconstructions for the IMEX BDF2 method are introduced. By using these continuous, piecewise time reconstructions, the upper and lower error bounds depending only on the discretization parameters and the data of the problems are derived. Based on these a posteriori error estimates, we further develop a time adaptive algorithm. The numerical implementations are performed with both nonuniform partitions and adaptivity in time. The adaptive algorithm reduce the computational cost substantially and provides efficient error control for the solution.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;14
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02017-0"
                        open
                      >
                        <summary class="article-expander__title">A New Family of Thermodynamically Compatible Discontinuous Galerkin Methods for Continuum Mechanics and Turbulent Shallow Water Flows</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02017-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this work we propose a new family of high order accurate semi-discrete discontinuous Galerkin (DG) finite element schemes for the thermodynamically compatible discretization of overdetermined first order hyperbolic systems. In particular, we consider a first order hyperbolic model of turbulent shallow water flows, as well as the unified first order hyperbolic Godunov–Peshkov–Romenski (GPR) model of continuum mechanics, which is able to describe at the same time viscous fluids and nonlinear elastic solids at large deformations. Both PDE systems treated in this paper belong to the class of hyperbolic and thermodynamically compatible systems, since both satisfy an entropy inequality and the total energy conservation can be obtained as a direct consequence of all other governing equations via suitable linear combination with the corresponding thermodynamic dual variables. In this paper, we mimic this process for the first time also at the semi-discrete level at the aid of high order discontinuous Galerkin finite element schemes. For the GPR model we directly discretize the entropy inequality and obtain total energy conservation as a consequence of a suitable discretization of all other evolution equations. For turbulent shallow water flows we directly discretize the nonconservative evolution equations related to the Reynolds stress tensor and obtain total energy conservation again as a result of the thermodynamically compatible discretization. As a consequence, for continuum mechanics the new DG schemes satisfy a cell entropy inequality directly by construction and thanks to the discrete thermodynamic compatibility they are provably nonlinearly stable in the energy norm for both systems under consideration.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;66
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-09 datetime="2022-10-09T00:00:00.000Z">2022-10-09</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-09 datetime="2022-10-09T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01807-2"
                        open
                      >
                        <summary class="article-expander__title">A group theoretic approach to model comparison with simplicial representations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01807-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The complexity of biological systems, and the increasingly large amount of associated experimental data, necessitates that we develop mathematical models to further our understanding of these systems. Because biological systems are generally not well understood, most mathematical models of these systems are based on experimental data, resulting in a seemingly heterogeneous collection of models that ostensibly represent the same system. To understand the system we therefore need to understand how the different models are related to each other, with a view to obtaining a unified mathematical description. This goal is complicated by the fact that a number of distinct mathematical formalisms may be employed to represent the same system, making direct comparison of the models very diff…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;68
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-08 datetime="2022-10-08T00:00:00.000Z">2022-10-08</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-08 datetime="2022-10-08T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02019-y"
                        open
                      >
                        <summary class="article-expander__title">Verifiability of the Data-Driven Variational Multiscale Reduced Order Model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02019-y">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we focus on the mathematical foundations of reduced order model (ROM) closures. First, we extend the verifiability concept from large eddy simulation to the ROM setting. Specifically, we call a ROM closure model verifiable if a small ROM closure model error (i.e., a small difference between the true ROM closure and the modeled ROM closure) implies a small ROM error. Second, we prove that the data-driven ROM closure studied here (i.e., the data-driven variational multiscale ROM) is verifiable. Finally, we investigate the verifiability of the data-driven variational multiscale ROM in the numerical simulation of the one-dimensional Burgers equation and a two-dimensional flow past a circular cylinder at Reynolds numbers 
                      
                      \(Re&#x3D;100\)
                      
                     and 
                      
                      \(Re&#x3D;1000\)
                      
                    .</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-07 datetime="2022-10-07T01:54:58.000Z">2022-10-07</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-07 datetime="2022-10-07T01:54:58.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0117845?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Multistability and evolution of chimera states in a network of type II Morris–Lecar neurons with asymmetrical nonlocal inhibitory connections</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0117845?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Formation of synchronous activity patterns is an essential property of neuronal networks that has been of central interest to synchronization theory. Chimera states, where both synchronous and asynchronous activities of neurons co-exist in a single network, are particularly poignant examples of such patterns, whose dynamics and multistability may underlie brain function, such as cognitive tasks. However, dynamical mechanisms of coherent state formation in spiking neuronal networks as well as ways to control these states remain unclear. In this paper, we take a step in this direction by considering the evolution of chimera states in a network of class II excitable Morris–Lecar neurons with asymmetrical nonlocal inhibitory connections. Using the adaptive coherence measure, we are able to partition the network parameter space into regions of various collective behaviors (antiphase synchronous clusters, traveling waves, different types of chimera states as well as a spiking death regime) and have shown multistability between the various regimes. We track the evolution of the chimera states as a function of changed key network parameters and found transitions between various types of chimera states. We further find that the network can demonstrate long transients leading to quasi-persistence of activity patterns in the border regions hinting at near-criticality behaviors.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01813-4"
                        open
                      >
                        <summary class="article-expander__title">A cancer model with nonlocal free boundary dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01813-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Cancer cells at the tumor boundary move in the direction of the oxygen gradient, while cancer cells far within the tumor are in a necrotic state. This paper introduces a simple mathematical model that accounts for these facts. The model consists of cancer cells, cytotoxic T cells, and oxygen satisfying a system of partial differential equations. Some of the model parameters represent the effect of anti-cancer drugs. The tumor boundary is a free boundary whose dynamics is determined by the movement of cancer cells at the boundary. The model is simulated for radially symmetric and axially symmetric tumors, and it is shown that the tumor may increase or decrease in size, depending on the “strength&quot; of the drugs. Existence theorems are proved, global in-time in the radially symmetric case, and local in-time for any shape of tumor. In the radially symmetric case, it is proved, under different conditions, that the tumor may shrink monotonically, or expand monotonically.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01817-0"
                        open
                      >
                        <summary class="article-expander__title">Maximal total population of species in a diffusive logistic model</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01817-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we investigate the maximization of the total population of a single species which is governed by a stationary diffusive logistic equation with a fixed amount of resources. For large diffusivity, qualitative properties of the maximizers like symmetry will be addressed. Our results are in line with previous findings which assert that for large diffusion, concentrated resources are favorable for maximizing the total population. Then, an optimality condition for the maximizer is derived based upon rearrangement theory. We develop an efficient numerical algorithm applicable to domains with different geometries in order to compute the maximizer. It is established that the algorithm is convergent. Our numerical simulations give a real insight into the qualitative properties of the maximizer and also lead us to some conjectures about the maximizer.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02016-1"
                        open
                      >
                        <summary class="article-expander__title">A Stochastic Nesterov’s Smoothing Accelerated Method for General Nonsmooth Constrained Stochastic Composite Convex Optimization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02016-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We propose a novel stochastic Nesterov’s smoothing accelerated method for general nonsmooth, constrained, stochastic composite convex optimization, the nonsmooth component of which may be not easy to compute its proximal operator. The proposed method combines Nesterov’s smoothing accelerated method (Nesterov in Math Program 103(1):127–152, 2005) for deterministic problems and stochastic approximation for stochastic problems, which allows three variants: single sample and two different mini-batch sizes per iteration, respectively. We prove that all the three variants achieve the best-known complexity bounds in terms of stochastic oracle. Numerical results on a robust linear regression problem, as well as a support vector machine problem show that the proposed method compares favorably with other state-of-the-art first-order methods, and the variants with mini-batch sizes outperform the variant with single sample.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02008-1"
                        open
                      >
                        <summary class="article-expander__title">Dissipation-Preserving Rational Spectral-Galerkin Method for Strongly Damped Nonlinear Wave System Involving Mixed Fractional Laplacians in Unbounded Domains</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02008-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
This paper aims at developing a dissipation-preserving, linearized, and time-stepping-varying spectral method for strongly damped nonlinear wave system in multidimensional unbounded domains 
                      
                      \({\mathbb {R}}^d\)
                      
                     (d&#x3D;1, 2, and 3), where the nonlocal nature is described by the mixed fractional Laplacians. Because the underlying solutions of the problem involving mixed fractional Laplacians decay slowly with certain power law at infinity, we employ the rational spectral-Galerkin method using rational basis (or mapped Gegenbauer functions) for the spatial approximation. To capture the intrinsic dissipative properties of the model equations, we combine the Crank-Nicolson scheme with exponential scalar auxiliary variable approach for the temporal discretization. Based on the rate of nonlocal energy dissipation, we design a novel time-stepping-varying strategy to enhance the efficiency of the scheme. We present the detailed implementation of the scheme, where the main building block of the stiffness matrices is based on the Laguerre-Gauss quadrature rule for the modified Bessel functions of the second kind. The existence, uniqueness, and nonlocal energy dissipation law of the fully discrete scheme are rigourously established. Numerical examples in 3D case are carried out to demonstrate the accuracy and efficiency of the scheme. Finally, we simulate the nonlinear behaviors of 2D/3D dissipative vector solitary waves for damped sine-Gordon system I, for damped sine-Gordon system II, and for damped Klein–Gordon system to provide a deeper understanding of nonlocal physics.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-06 datetime="2022-10-06T02:40:52.000Z">2022-10-06</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-06 datetime="2022-10-06T02:40:52.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102483?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">The combination of social reward and punishment is conducive to the cooperation and heterogeneity of social relations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102483?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
Individual behaviors and social relations influence each other. However, understanding the underlying mechanism remains challenging. From social norms controlling human behavior to individual management of interpersonal relationships, rewards and punishments are some of the most commonly used measures. Through simulating the weak prisoner’s dilemma in finite populations, we find that neither a simple reward measure nor a pure punishment mechanism can extensively promote cooperation. Instead, a combination of appropriate punishment and reward mechanisms can promote cooperation’s prosperity regardless of how large or small the temptation to defect is. In addition, the combination spontaneously produces inhomogeneities in social relations and individual influence, which support the continued existence of cooperative behavior. Finally, we further explain how cooperators establish a sustainable existence under the combination by investigating the social relations at different moments in a small system. These results demonstrate that dispensing rewards and punishments impartially in society is essential to social harmony.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0107427?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Generalized weighted permutation entropy</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0107427?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
A novel heuristic approach is proposed here for time series data analysis, dubbed Generalized weighted permutation entropy, which amalgamates and generalizes beyond their original scope two well established data analysis methods: Permutation entropy and Weighted permutation entropy. The method introduces a scaling parameter to discern the disorder and complexity of ordinal patterns with small and large fluctuations. Using this scaling parameter, the complexity-entropy causality plane is generalized to the complexity-entropy-scale causality box. Simulations conducted on synthetic series generated by stochastic, chaotic, and random processes, as well as real world data, are shown to produce unique signatures in this three dimensional representation.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33524-1"
                        open
                      >
                        <summary class="article-expander__title">Author Correction: Engineering the stambomycin modular polyketide synthase yields 37-membered mini-stambomycins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33524-1">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Nature Communications - Author Correction: Engineering the stambomycin modular polyketide synthase yields 37-membered mini-stambomycins</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41557-022-01038-4"
                        open
                      >
                        <summary class="article-expander__title">Genetically encoded chemical crosslinking of RNA in vivo</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41557-022-01038-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41557-022-01038-4/MediaObjects/41557_2022_1038_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Protein–RNA interactions regulate RNA fate and function, and are generally non-covalent and reversible. Genetically introducing a latent bioreactive amino acid into a protein is now shown to enable the protein to covalently crosslink a bound RNA molecule in vivo. This method offers innovative avenues for developing protein–RNA research and applications.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;18
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01796-2"
                        open
                      >
                        <summary class="article-expander__title">Cyclic attractors of nonexpanding q-ary networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01796-2">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Discrete dynamical systems in which model components take on categorical values have been successfully applied to biological networks to study their global dynamic behavior. Boolean models in particular have been used extensively. However, multi-state models have also emerged as effective computational tools for the analysis of complex mechanisms underlying biological networks. Models in which variables assume more than two discrete states provide greater resolution, but this scheme introduces discontinuities. In particular, variables can increase or decrease by more than one unit in one time step. This can be corrected, without changing fixed points of the system, by applying an additional rule to each local activation function. On the other hand, if one is interested in cyclic attractors of their system, then this rule can potentially introduce new cyclic attractors that were not observed previously. This article makes some advancements in understanding the state space dynamics of multi-state network models with synchronous, sequential, or block-sequential update schedules and establishes conditions under which no new cyclic attractors are added to networks when the additional rule is applied. Our analytical results have the potential to be incorporated into modeling software and aid researchers in their analyses of biological multi-state networks.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-05 datetime="2022-10-05T04:00:00.000Z">2022-10-05</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-05 datetime="2022-10-05T04:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >ACS Synthetic Biology: Latest Articles (ACS Publications)</button>
                  <a class="source-heading__link" href="https://pubs.acs.org">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="http://dx.doi.org/10.1021/acssynbio.2c00413"
                        open
                      >
                        <summary class="article-expander__title">[ASAP] Control of Enzyme Reaction Initiation inside Giant Unilamellar Vesicles by the Cell-Penetrating Peptide-Mediated Translocation of Cargo Proteins</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://dx.doi.org/10.1021/acssynbio.2c00413">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>ACS Synthetic Biology
DOI: 10.1021/acssynbio.2c00413</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41587-022-01493-x"
                        open
                      >
                        <summary class="article-expander__title">Modular, programmable RNA sensing using ADAR editing in living cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41587-022-01493-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41587-022-01493-x/MediaObjects/41587_2022_1493_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Specific RNAs are detected in live cells by harnessing endogenous ADAR enzymes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02011-6"
                        open
                      >
                        <summary class="article-expander__title">High-Order ADI-FDTD Schemes for Maxwell’s Equations with Material Interfaces in Two Dimensions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02011-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we apply the immersed interface method (IIM) and the hierarchical derivative matching (HDM) method, respectively, to restore the accuracy of the high-order alternating direction implicit finite-difference time-domain (ADI-FDTD) scheme of the 2D Maxwell’s equations with material interfaces. For the case of discontinuous permittivity 
                      
                      \(\varepsilon \)
                      
                     and continuous permeability 
                      
                      \(\mu \)
                      
                    , we propose four high-order schemes. Two of them are of second order in time and fourth order in space (ADI-IIM-FDTD(2,4) scheme and ADI-HDM-FDTD(2,4) scheme). Others are of fourth order both in time and space (ADI-IIM-FDTD(4,4) scheme and ADI-HDM-FDTD(4,4) scheme). For the case of discontinuous permittivity 
                      
                      \(\varepsilon \)
                      
                     and permeability 
                      
                      \(\mu \)
                      
                    , the (2,4) scheme and the (4,4) scheme are constructed as well (ADI-HDM-FDTD-X(2,4) scheme and ADI-HDM-FDTD-X(4,4) scheme). The proposed six schemes maintain the advantages of ADI-FDTD method such as unconditional stability and computational efficiency. Numerical examples are given to verify the performance of the proposed schemes.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;12
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-04 datetime="2022-10-04T05:26:44.000Z">2022-10-04</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-04 datetime="2022-10-04T05:26:44.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >American Institute of Physics: Chaos: An Interdisciplinary Journal of Nonlinear Science: Table of Contents</button>
                  <a class="source-heading__link" href="https://aip.scitation.org/loi/cha?af&#x3D;R&amp;feed&#x3D;most-recent">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0105969?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Community formation in wealth-mediated thermodynamic strategy evolution</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0105969?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://aip.scitation.org/action/showOpenGraphArticleImage?doi&#x3D;10.1063/5.0105969&amp;id&#x3D;images/medium/5.0105969.figures.online.highlight_f1.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We study a dynamical system defined by a repeated game on a 1D lattice, in which the players keep track of their gross payoffs over time in a bank. Strategy updates are governed by a Boltzmann distribution, which depends on the neighborhood bank values associated with each strategy, relative to a temperature scale, which defines the random fluctuations. Players with higher bank values are, thus, less likely to change strategy than players with a lower bank value. For a parameterized rock–paper–scissors game, we derive a condition under which communities of a given strategy form with either fixed or drifting boundaries. We show the effect of a temperature increase on the underlying system and identify surprising properties of this model through numerical simulations.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;11
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0102390?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Network spreading among areas: A dynamical complex network modeling approach</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0102390?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
With the outbreak of COVID-19, great loss and damage were brought to human society, making the study of epidemic spreading become a significant topic nowadays. To analyze the spread of infectious diseases among different areas, e.g., communities, cities, or countries, we construct a network, based on the epidemic model and the network coupling, whose nodes denote areas, and edges represent population migrations between two areas. Each node follows its dynamic, which describes an epidemic spreading among individuals in an area, and the node also interacts with other nodes, which indicates the spreading among different areas. By giving mathematical proof, we deduce that our model has a stable solution despite the network structure. We propose the peak infected ratio (PIR) as a property of infectious diseases in a certain area, which is not independent of the network structure. We find that increasing the population mobility or the disease infectiousness both cause higher peak infected population all over different by simulation. Furthermore, we apply our model to real-world data on COVID-19 and after properly adjusting the parameters of our model, the distribution of the peak infection ratio in different areas can be well fitted.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://aip.scitation.org/doi/10.1063/5.0101495?af&#x3D;R&amp;feed&#x3D;most-recent"
                        open
                      >
                        <summary class="article-expander__title">Entropic comparison of Landau–Zener and Demkov interactions in the phase space of a quadrupole billiard</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://aip.scitation.org/doi/10.1063/5.0101495?af&#x3D;R&amp;feed&#x3D;most-recent">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Chaos: An Interdisciplinary Journal of Nonlinear Science, Volume 32, Issue 10, October 2022. 
We investigate two types of avoided crossings in a chaotic billiard within the framework of information theory. The Shannon entropy in the phase space for the Landau–Zener interaction increases as the center of the avoided crossing is approached, whereas for the Demkov interaction, the Shannon entropy decreases as the center of avoided crossing is passed by with an increase in the deformation parameter. This feature can provide a new indicator for scar formation. In addition, it is found that the Fisher information of the Landau–Zener interaction is significantly larger than that of the Demkov interaction.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;3
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-10-03 datetime="2022-10-03T20:00:00.000Z">2022-10-03</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-10-03 datetime="2022-10-03T20:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/events-illuminate-critical-need-menstruation-science-1003"
                        open
                      >
                        <summary class="article-expander__title">MIT events illuminate critical need for menstruation science</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/events-illuminate-critical-need-menstruation-science-1003">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/MIT-Below-The-Belt-Outside.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>“AI for endometriosis? If only there were data!”</span>
                                &ensp;<span class="article-reading-time">(&hairsp;7
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41598-022-19681-9"
                        open
                      >
                        <summary class="article-expander__title">Cell-free protein crystallization for nanocrystal structure determination</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41598-022-19681-9">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-022-19681-9/MediaObjects/41598_2022_19681_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Scientific Reports - Cell-free protein crystallization for nanocrystal structure determination</span>
                                &ensp;<span class="article-reading-time">(&hairsp;30
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33606-0"
                        open
                      >
                        <summary class="article-expander__title">Systematic dissection of key factors governing recombination outcomes by GCE-SCRaMbLE</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33606-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33606-0/MediaObjects/41467_2022_33606_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Applications of the SCRaMbLE process are hindered due to the lack of facile and tight regulation and limited understanding of key factors that may affect the rearrangement outcomes. Here the authors present an approach to precisely regulate SCRaMbLE recombination in a dose-dependent manner using genetic code expansion technology with low basal activity.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;39
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01812-5"
                        open
                      >
                        <summary class="article-expander__title">Operating diagrams for a three-tiered microbial food web in the chemostat</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01812-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we consider a three-tiered food web model in a chemostat, including chlorophenol, phenol, and hydrogen substrates and their degraders. The model takes into account the three substrate inflowing concentrations, as well as maintenance, that is, decay terms of the species. The operating diagrams give the asymptotic behavior of the model with respect to the four operating parameters, which are the dilution rate and the three inflowing concentrations of the substrates. These diagrams were obtained only numerically in the existing literature. Using the mathematical analysis of this model obtained in our previous studies, we construct the operating diagrams, by plotting the curves that separate their various regions. Hence, the regions of the operating diagrams are constructed analytically and there is no requirement for time-consuming algorithms to generate the plots, as in the numerical method. Moreover, our method reveals behaviors that have not been detected in the previous numerical studies. The growth functions are of Monod form with the inclusion of a product inhibition term. However, our method applies for a large class of growth functions. We construct operating diagrams with and without maintenance showing the role of maintenance on the stability of the system.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-30 datetime="2022-09-30T19:30:00.000Z">2022-09-30</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-30 datetime="2022-09-30T19:30:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/smart-researchers-develop-quick-test-determine-immunity-against-covid-19-0930"
                        open
                      >
                        <summary class="article-expander__title">SMART researchers develop quick test to determine immunity against Covid-19</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/smart-researchers-develop-quick-test-determine-immunity-against-covid-19-0930">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/SMART-and-NTU-scientists-with-the-rapid-test-kit-that-detects-levels-of-COVID19-antibodies-cover.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Test that measures a person’s antibodies requires a drop of blood and takes just 10 minutes to show results.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;9
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MaryAnnLiebert: Journal of Computational Biology: Table of Contents</button>
                  <a class="source-heading__link" href="https://www.liebertpub.com/loi/cmb?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0280?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R"
                        open
                      >
                        <summary class="article-expander__title">Kernel Mixed Model for Transcriptome Association Study</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2022.0280?ai&#x3D;su&amp;mi&#x3D;cjwv&amp;af&#x3D;R">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://www.liebertpub.com/action/showCoverImage?journalCode&#x3D;" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Journal of Computational Biology, Ahead of Print.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;4
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1459782?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Adaptive Solution of Initial Value Problems by a Dynamical Galerkin Scheme</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1459782?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1147-1166, September 2022. 
 Abstract. Adaptive Galerkin methods for time-dependent partial differential equations are studied and shown to be dissipative. The adaptation implies that the subset of the selected basis function changes over time according to the evolution of the solution. The corresponding projection operator is thus time-dependent and nondifferentiable. We therefore propose to use an integral formulation in time. We analyze the existence and uniqueness of this weak form of the dynamical Galerkin scheme, and we then prove that the nonsmooth projection operator introduces energy dissipation, which is a crucial result for adaptive Galerkin methods, e.g., adaptive wavelet methods. Numerical examples for the inviscid Burgers equation in one dimension and the incompressible Euler equations in two and three spatial dimensions show that the selection of basis functions, for instance by filtering out weak wavelet coefficients from the solution, introduces energy dissipation. Moreover, for the Burgers case we can show that adaptive wavelet regularization yields convergence of the truncated Galerkin solution to the physically relevant entropy solution. These results motivate adaptive wavelet-based Galerkin schemes for nonlinear hyperbolic conservation laws.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1467791?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Geometric Evolution of Bilayers under the Degenerate Functionalized Cahn–Hilliard Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1467791?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1127-1146, September 2022. 
 Abstract. Using a multiscale analysis, we derive a sharp interface limit for the dynamics of bilayer structures of the functionalized Cahn–Hilliard equation with a cutoff diffusion mobility [math] that is degenerate for [math] and a continuously differentiable double-well potential [math]. We show that the bilayer interface does not move in the [math] time scale. The interface motion occurs in the [math] time scale and is determined by porous medium diffusion processes in both phases with no jumps on the interface. In the longer [math] time scale, the interface motion is a complex combination of porous medium diffusion processes in both phases and the property of mass conservation.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M1493707?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Erratum: Tunable Eigenvector-Based Centralities for Multiplex and Temporal Networks</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M1493707?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1167-1167, September 2022. 
 Abstract. This erratum corrects several typos in [D. Taylor, M. A. Porter, and P. J. Mucha, Multiscale Model. Simul., 19 (2021), pp. 113–147].</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02010-7"
                        open
                      >
                        <summary class="article-expander__title">A Posteriori Error Analysis of a Mixed Finite Element Method for the Coupled Brinkman–Forchheimer and Double-Diffusion Equations</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02010-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper we consider a partially augmented fully-mixed variational formulation that has been recently proposed for the coupling of the stationary Brinkman–Forchheimer and double-diffusion equations, and develop an a posteriori error analysis for the 2D and 3D versions of the associated mixed finite element scheme. Indeed, we derive two reliable and efficient residual-based a posteriori error estimators for this problem on arbitrary (convex or non-convex) polygonal and polyhedral regions. The reliability of the proposed estimators draws mainly upon the uniform ellipticity and inf-sup condition of the forms involved, a suitable assumption on the data, stable Helmholtz decompositions in Hilbert and Banach frameworks, and the local approximation properties of the Clément and Raviart–Thomas operators. In turn, inverse inequalities, the localization technique based on bubble functions, and known results from previous works, are the main tools yielding the efficiency estimate. Finally, several numerical examples confirming the theoretical properties of the estimators and illustrating the performance of the associated adaptive algorithms, are reported. In particular, the case of flow through a 3D porous media with channel networks is considered.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;13
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-29 datetime="2022-09-29T18:00:00.000Z">2022-09-29</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-29 datetime="2022-09-29T18:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/processing-waste-biomass-reduce-airborne-emissions-0929"
                        open
                      >
                        <summary class="article-expander__title">Processing waste biomass to reduce airborne emissions</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/processing-waste-biomass-reduce-airborne-emissions-0929">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/Takachar-hay-field-MITEI.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>MIT spinoff Takachar converts agricultural waste into clean-burning fuel, and wins Earthshot Prize.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02009-0"
                        open
                      >
                        <summary class="article-expander__title">Multi-Dimensional Image Recovery via Fully-Connected Tensor Network Decomposition Under the Learnable Transforms</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02009-0">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Multi-dimensional image recovery from incomplete data is a fundamental problem in data processing. Due to its advantage of capturing the correlations between any modes of the multi-dimensional image, i.e., the target tensor, the fully-connected tensor network (FCTN) decomposition has recently shown promising performance on multi-dimensional image recovery. However, FCTN decomposition suffers from computational deficiency, especially for large-scale multi-dimensional images. To address this deficiency, we propose a learnable transform-based FCTN model (termed as T-FCTN), which enjoys the remarkable advantage of FCTN decomposition with cheap computational cost. More concretely, we learn the semi-orthogonal transforms along each mode of the target tensor to project the large-scale te…</span>
                                &ensp;<span class="article-reading-time">(&hairsp;15
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02002-7"
                        open
                      >
                        <summary class="article-expander__title">The Spectral Difference Raviart–Thomas Method for Two and Three-Dimensional Elements and Its Connection with the Flux Reconstruction Formulation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02002-7">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The purpose of this work is to describe in detail the development of the spectral difference Raviart–Thomas (SDRT) formulation for two and three-dimensional tensor-product elements and simplexes. Through the process, the authors establish the equivalence between the SDRT method and the flux reconstruction (FR) approach under the assumption of the linearity of the flux and the mesh uniformity. Such a connection allows building a new family of FR schemes for two and three-dimensional simplexes and also to recover the well-known FR-SD method with tensor-product elements. In addition, a thorough analysis of the numerical dissipation and dispersion of both aforementioned schemes and the nodal discontinuous Galerkin FR (FR-DG) method with two and three-dimensional elements is proposed through the use of the combined-mode Fourier approach. SDRT is shown to possess an enhanced temporal linear stability in comparison to FR-DG. On the contrary, SDRT displays larger dissipation and dispersion errors with respect to FR-DG. Finally, the study is concluded with a set of numerical experiments, the linear advection-diffusion problem, the Isentropic Euler Vortex, and the Taylor-Green Vortex (TGV). The latter test case shows that SDRT schemes present a non-linear unstable behavior with simplex elements and certain polynomial degrees. For the sake of completeness, the matrix form of the SDRT method is developed and the computational performance of SDRT with respect to FR schemes is evaluated using GPU architectures.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;35
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02006-3"
                        open
                      >
                        <summary class="article-expander__title">Tensor Completion via A Generalized Transformed Tensor T-Product Decomposition Without t-SVD</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02006-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
Matrix and tensor nuclear norms have been successfully used to promote the low-rankness of tensors in low-rank tensor completion. However, singular value decomposition (SVD), which is computationally expensive for large-scale matrices, frequently appears in solving those nuclear norm minimization models. Based on the tensor-tensor product (T-product), in this paper, we first establish the equivalence between the so-called transformed tubal nuclear norm for a third-order tensor and the minimum of the sum of two factor tensors’ squared Frobenius norms under a general invertible linear transform. Gainfully, we introduce a mode-unfolding (often named as “spatio-temporal” in the internet traffic data recovery literature) regularized tensor completion model that is able to efficiently exploit the hidden structures of tensors. Then, we propose an implementable alternating minimization algorithm to solve the underlying optimization model. It is remarkable that our approach does not require any SVDs and all subproblems of our algorithm enjoy closed-form solutions. A series of numerical experiments on traffic data recovery, color images and videos inpainting demonstrate that our SVD-free approach takes less computing time to achieve satisfactory accuracy than some state-of-the-art tensor nuclear norm minimization approaches.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;25
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-28 datetime="2022-09-28T00:00:00.000Z">2022-09-28</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-28 datetime="2022-09-28T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01794-4"
                        open
                      >
                        <summary class="article-expander__title">From individual-based epidemic models to McKendrick-von Foerster PDEs: a guide to modeling and inferring COVID-19 dynamics</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01794-4">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We present a unifying, tractable approach for studying the spread of viruses causing complex diseases requiring to be modeled using a large number of types (e.g., infective stage, clinical state, risk factor class). We show that recording each infected individual’s infection age, i.e., the time elapsed since infection, has three benefits. First, regardless of the number of types, the age distribution of the population can be described by means of a first-order, one-dimensional partial differential equation (PDE) known as the McKendrick-von Foerster equation. The frequency of type i is simply obtained by integrating the probability of being in state i at a given age against the age distribution. This representation induces a simple methodology based on the additional assumption of Poisson sampling to infer and forecast the epidemic. We illustrate this technique using French data from the COVID-19 epidemic. Second, our approach generalizes and simplifies standard compartmental models using high-dimensional systems of ordinary differential equations (ODEs) to account for disease complexity. We show that such models can always be rewritten in our framework, thus, providing a low-dimensional yet equivalent representation of these complex models. Third, beyond the simplicity of the approach, we show that our population model naturally appears as a universal scaling limit of a large class of fully stochastic individual-based epidemic models, where the initial condition of the PDE emerges as the limiting age structure of an exponentially growing population starting from a single individual.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;67
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s10915-022-02014-3"
                        open
                      >
                        <summary class="article-expander__title">Optimal Error Estimates of the Local Discontinuous Galerkin Method and High-order Time Discretization Scheme for the Swift–Hohenberg Equation</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s10915-022-02014-3">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/10915.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
In this paper, we develop a local discontinuous Galerkin (LDG) method for the Swift–Hohenberg equation. The energy stability and optimal error estimates in 
                      
                      \(L^2\)
                      
                     norm of the semi-discrete LDG scheme are established. To avoid the severe time step restriction of explicit time marching methods, a first-order linear scheme based on the scalar auxiliary variable (SAV) method is employed for temporal discretization. Coupled with the LDG spatial discretization, we achieve a fully-discrete LDG method and prove its energy stability and optimal error estimates. To improve the temporal accuracy, the semi-implicit spectral deferred correction (SDC) method is adapted iteratively. Combining with the SAV method, the SDC method can be linear, high-order accurate and energy stable in our numerical tests. Numerical experiments are presented to verify the theoretical results and to show the efficiency of the proposed methods.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;16
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z">2022-09-27</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-27 datetime="2022-09-27T19:50:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >MIT News - Bioengineering and biotechnology | Biological engineering | Biotechnology</button>
                  <a class="source-heading__link" href="https://news.mit.edu/rss/topic/biological-engineering-and-biotechnology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927"
                        open
                      >
                        <summary class="article-expander__title">Neurodegenerative disease can progress in newly identified patterns</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://news.mit.edu/2022/neurodegenerative-disease-can-progress-newly-identified-patterns-0927">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://news.mit.edu/sites/default/files/images/202209/neuron-cell.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A machine-learning method finds patterns of health decline in ALS, informing future clinical trial designs and mechanism discovery. The technique also extends to Alzheimer’s and Parkinson’s.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;8
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Society for Industrial and Applied Mathematics: Keyword search for query</button>
                  <a class="source-heading__link" href="https://epubs.siam.org/action/doSearch?type&#x3D;advanced&amp;mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/20M1377771?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Manifold Learning and Nonlinear Homogenization</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/20M1377771?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1093-1126, September 2022. 
 Abstract. We describe an efficient domain decomposition-based framework for nonlinear multiscale PDE problems. The framework is inspired by manifold learning techniques and exploits the tangent spaces spanned by the nearest neighbors to compress local solution manifolds. Our framework is applied to a semilinear elliptic equation with oscillatory media and a nonlinear radiative transfer equation; in both cases, significant improvements in efficacy are observed. This new method does not rely on a detailed analytical understanding of multiscale PDEs, such as their asymptotic limits, and thus is more versatile for general multiscale problems.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1439808?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Online Multiscale Model Reduction for Nonlinear Stochastic PDEs with Multiplicative Noise</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1439808?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1063-1092, September 2022. 
 Abstract. In this paper, an online multiscale model reduction method is presented for stochastic partial differential equations (SPDEs) with multiplicative noise, where the diffusion coefficient is spatially multiscale and the noise perturbation nonlinearly depends on the diffusion dynamics. It is necessary to efficiently compute all possible trajectories of the stochastic dynamics for quantifying a model’s uncertainty and statistic moments. The multiscale diffusion and nonlinearity may cause the computation to become very intractable. To overcome the multiscale difficulty, a constraint energy minimizing generalized multiscale finite element method (CEM-GMsFEM) is used to localize the computation and obtain an effective coarse model. However, the nonlinear terms are still defined on a fine scale space after the Galerkin projection of CEM-GMsFEM is applied to the nonlinear SPDEs. This significantly impacts the simulation efficiency by CEM-GMsFEM. To this end, a stochastic online discrete empirical interpolation method (DEIM) is proposed to treat the stochastic nonlinearity. The stochastic online DEIM incorporates offline snapshots and online snapshots. The offline snapshots consist of the nonlinear terms at the approximate mean of the stochastic dynamics and are used to construct an offline reduced model. The online snapshots contain some information of the current new trajectory and are used to correct the offline reduced model in an incremental manner. The stochastic online DEIM substantially reduces the dimension of the nonlinear dynamics and enhances the prediction accuracy for the reduced model. Thus, the online multiscale model reduction is constructed by using CEM-GMsFEM and the stochastic online DEIM. A priori error analysis is carried out for the nonlinear SPDEs. We present a few numerical examples with diffusion in heterogeneous porous media and show the effectiveness of the proposed model reduction.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/21M1464592?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Hyperbolic Systems of Moment Equations Describing Sedimentation in Suspensions of Rod-Like Particles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/21M1464592?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1002-1039, September 2022. 
 Abstract. We study mathematical models and approximations for the sedimentation in suspensions of rod-like particles. The starting point of our considerations is a coupled system of partial differential equations, previously derived by Helzel and Tzavaras [Multiscale Model.  Simul., 15 (2017), pp.  500–536], consisting of a kinetic equation for the rod orientation that is coupled to a macroscopic flow equation. It describes the motion of a suspension of rigid rod-like particles under the influence of gravity. Since the coupled system is high dimensional (five dimensions plus time) we are interested in the derivation of models which can adaptively adjust the level of detail. Here we restrict ourselves to simplified lower dimensional flow situations and derive hierarchies of moment equations which can be interpreted as approximations of the coupled kinetic model. We show that the resulting systems of moment equations are hyperbolic.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://epubs.siam.org/doi/abs/10.1137/22M146995X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search"
                        open
                      >
                        <summary class="article-expander__title">Improved Uniform Error Bounds on Time-Splitting Methods for the Long-Time Dynamics of the Dirac Equation with Small Potentials</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://epubs.siam.org/doi/abs/10.1137/22M146995X?mi&#x3D;9v4ths&amp;af&#x3D;R&amp;countTerms&#x3D;true&amp;publication&#x3D;mmsubt&amp;searchCollection&#x3D;jb&amp;sortBy&#x3D;Earliest_desc&amp;target&#x3D;multi-search">
                          <div class="article-summary-box-inner media-object">
                            <span class="media-object__text">
                              <span>Multiscale Modeling &amp; Simulation, Volume 20, Issue 3, Page 1040-1062, September 2022. 
 Abstract. We establish improved uniform error bounds on time-splitting methods for the long-time dynamics of the Dirac equation with small electromagnetic potentials characterized by a dimensionless parameter [math] representing the amplitude of the potentials. We begin with a semidiscretization of the Dirac equation in time by a time-splitting method, followed by a full-discretization in space by with the Fourier pseudospectral method in space. By employing the unitary flow property of the second-order time-splitting method for the Dirac equation, we prove uniform error bounds for [math] at [math] and [math] for the semidiscretization and full-discretization, respectively, for any time [math] with [math] and [math]. In the expressions, [math] is the time step, [math] is the mesh size, [math] depends on the regularity of the solution, and [math] grows at most linearly with respect to [math] with [math] and [math] two constants independent of [math], [math], [math], and [math]. Then by adopting the regularity compensation oscillation technique, which controls the high frequency modes by the regularity of the solution and low frequency modes by phase cancellation and energy method, we establish improved uniform error bounds at [math] and [math] for the semidiscretization and full-discretization, respectively, up to the long-time [math]. Numerical results are reported to confirm our error bounds and to demonstrate that they are sharp. Comparisons on the accuracy of different time discretizations for the Dirac equation are also provided.</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s42003-022-03999-1"
                        open
                      >
                        <summary class="article-expander__title">Phospholipid synthesis inside phospholipid membrane vesicles</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s42003-022-03999-1">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs42003-022-03999-1/MediaObjects/42003_2022_3999_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>A giant vesicle system combines fatty acid synthesis pathways, CoA recycling, and cell-free acyltransferase gene expression to perform phospholipid synthesis inside.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;46
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01811-6"
                        open
                      >
                        <summary class="article-expander__title">Noise-driven bifurcations in a neural field system modelling networks of grid cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01811-6">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
The activity generated by an ensemble of neurons is affected by various noise sources. It is a well-recognised challenge to understand the effects of noise on the stability of such networks. We demonstrate that the patterns of activity generated by networks of grid cells emerge from the instability of homogeneous activity for small levels of noise. This is carried out by analysing the robustness of network activity patterns with respect to noise in an upscaled noisy grid cell model in the form of a system of partial differential equations. Inhomogeneous network patterns are numerically understood as branches bifurcating from unstable homogeneous states for small noise levels. We show that there is a phase transition occurring as the level of noise decreases. Our numerical study also indicates the presence of hysteresis phenomena close to the precise critical noise value.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;45
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>
      <section class="daily-content js-toggle-accordions-scope">
        <h2 class="daily-heading">
          <button
            class="daily-heading-toggle"
            data-action="toggle-accordions"
            title="Click to toggle the day, Ctrl + click to toggle all."
          >
            <time class="daily-heading-toggle__weekday js-offset-weekday" data-offset-date=2022-09-26 datetime="2022-09-26T00:00:00.000Z">2022-09-26</time>
            <time class="daily-heading-toggle__date js-offset-date" data-offset-date=2022-09-26 datetime="2022-09-26T00:00:00.000Z"></time>
          </button>
        </h2>
        <ul class="sources card">
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Synthetic biology : nature.com subject feeds</button>
                  <a class="source-heading__link" href="https://www.nature.com/subjects/synthetic-biology">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41589-022-01136-x"
                        open
                      >
                        <summary class="article-expander__title">Chemically inducible split protein regulators for mammalian cells</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41589-022-01136-x">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41589-022-01136-x/MediaObjects/41589_2022_1136_Figa_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>An INSPIRE platform for therapeutic regulators and sensors, based on split human domains binding physiological or clinically approved compounds, has been developed and demonstrated on several human proteins in a relevant ligand concentration range.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;21
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="https://www.nature.com/articles/s41467-022-33221-z"
                        open
                      >
                        <summary class="article-expander__title">The material properties of a bacterial-derived biomolecular condensate tune biological function in natural and synthetic systems</summary>
                        <a class="article-summary-link article-summary-box-outer" href="https://www.nature.com/articles/s41467-022-33221-z">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-022-33221-z/MediaObjects/41467_2022_33221_Fig1_HTML.png" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>“Intracellular phase separation is emerging as a universal principle for organizing biochemical reactions in time and space. Here the authors show that PopZ condensate dynamics support cell division and using PopZ modular architecture, the tunable PopTag platform was developed to enable designer condensates.”</span>
                                &ensp;<span class="article-reading-time">(&hairsp;50
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
            <li class="card__section">
              <section class="js-toggle-accordions-scope">
                <h3 class="source-heading">
                  <button
                    class="source-heading__name"
                    data-action="toggle-accordions"
                    title="Click to toggle the source, Ctrl + click to toggle all."
                  >Latest Results</button>
                  <a class="source-heading__link" href="http://link.springer.com">Open</a>
                </h3>
                <section class="articles-per-source">
                    <article>
                      <details
                        class="article-expander"
                        data-action="toggle-native-accordion"
                        data-accordion-key="10.1007/s00285-022-01793-5"
                        open
                      >
                        <summary class="article-expander__title">Scaling relations for auxin waves</summary>
                        <a class="article-summary-link article-summary-box-outer" href="http://link.springer.com/10.1007/s00285-022-01793-5">
                          <div class="article-summary-box-inner media-object">
                              <img src="https://media.springernature.com/w200/springer-static/cover/journal/285.jpg" loading="lazy" class="media-object__media article-image" />
                            <span class="media-object__text">
                              <span>Abstract
We analyze an ‘up-the-gradient’ model for the formation of transport channels of the phytohormone auxin, through auxin-mediated polarization of the PIN1 auxin transporter. We show that this model admits a family of travelling wave solutions that is parameterized by the height of the auxin-pulse. We uncover scaling relations for the speed and width of these waves and verify these rigorous results with numerical computations. In addition, we provide explicit expressions for the leading-order wave profiles, which allows the influence of the biological parameters in the problem to be readily identified. Our proofs are based on a generalization of the scaling principle developed by Friesecke and Pego to construct pulse solutions to the classic Fermi–Pasta–Ulam–Tsingou model, which describes a one-dimensional chain of coupled nonlinear springs.</span>
                                &ensp;<span class="article-reading-time">(&hairsp;87
                                  min&hairsp;)</span>
                            </span>
                          </div>
                        </a>
                      </details>
                    </article>
                </section>
              </section>
            </li>
        </ul>
      </section>

    <footer>
      <a class="footer-link" href="https://github.com/xlxs4/rss-reader/actions/runs/3309270135">
        <time id="build-timestamp" datetime="2022-10-24T01:03:53.443Z">2022-10-24T01:03:53.443Z</time>
      </a>
      <a class="footer-link" href="https://github.com/osmoscraft/osmosfeed">osmosfeed 1.15.1</a>
    </footer>
    <script src="index.js?v1.14.4"></script>
    <!-- %before-body-end.html% -->
  </body>

</html>